{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "3f0195a7",
   "metadata": {},
   "outputs": [],
   "source": [
    "import py3Dmol\n",
    "import torch\n",
    "import esm\n",
    "import numpy as np\n",
    "import json\n",
    "import requests\n",
    "import os\n",
    "\n",
    "from Bio.Align import PairwiseAligner\n",
    "from sqlalchemy import create_engine\n",
    "from sqlalchemy.orm import sessionmaker\n",
    "\n",
    "from classes import *"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "7369d617",
   "metadata": {},
   "outputs": [],
   "source": [
    "engine = create_engine(\"mysql+pymysql://root:@localhost/pdb?charset=utf8mb4\")\n",
    "Session = sessionmaker(bind=engine)\n",
    "session = Session()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "50a38f9b",
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_pdb(id_):\n",
    "    pdb = session.query(Protein).filter(Protein.id == id_).first()\n",
    "    if pdb is None:\n",
    "        raise Exception(\"There is not file with such id\")\n",
    "    return pdb"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "e26a1b8c",
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_pdb_data(id_or_pdb):\n",
    "    if isinstance(id_or_pdb, Protein):\n",
    "         pdb = id_or_pdb\n",
    "    else:\n",
    "        id_ = id_or_pdb.upper()\n",
    "        pdb = get_pdb(id_)\n",
    "    with open(pdb.structure) as f:\n",
    "        data = f.read()\n",
    "    return data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "621634bb",
   "metadata": {},
   "outputs": [],
   "source": [
    "def view_pdb(id_or_pdb=None, chain_id=None, data=None):\n",
    "    view = py3Dmol.view(width=600, height=400)\n",
    "    if data is None:\n",
    "        data = get_pdb_data(id_or_pdb)\n",
    "    view.addModelsAsFrames(data)\n",
    "    view.setStyle({'model': -1, 'chain': chain_id}, {\"cartoon\": {'color': 'spectrum'}})\n",
    "    view.zoomTo()\n",
    "    view.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "f781dd4a",
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_chains_to_compare(offset, limit, only_chain_id=None):\n",
    "    query = session.query(Chain)\n",
    "    if only_chain_id is not None:\n",
    "        query = query.filter(Chain.chain_id==only_chain_id)\n",
    "    chains = query.limit(limit).offset(offset).all()\n",
    "    return chains"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "51b36cff",
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_most_similars_by_MSA(sequence, only_chain_id=None, top_count=5):\n",
    "    aligner = PairwiseAligner()\n",
    "    offset = 0\n",
    "    limit = 20\n",
    "    tops = []\n",
    "    while True:\n",
    "        chains = get_chains_to_compare(offset, limit, only_chain_id)\n",
    "        if not len(chains):\n",
    "            break\n",
    "        offset += limit\n",
    "        for chain in chains:\n",
    "            alignments = aligner.align(chain.sequence, sequence)\n",
    "            similarity = alignments[0].score\n",
    "            i = len(tops)\n",
    "            while i>0 and similarity>tops[i-1][0]:\n",
    "                i-=1\n",
    "            if i<top_count:\n",
    "                tops.insert(i, (similarity, chain))\n",
    "                tops = tops[:top_count]\n",
    "    return [i for i in tops]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "cecdf401",
   "metadata": {},
   "outputs": [],
   "source": [
    "def evaluate(model, batch_converter, sequence):\n",
    "    data = [(\"seq\", sequence)]\n",
    "    batch_labels, batch_strs, batch_tokens = batch_converter(data)\n",
    "    with torch.no_grad():\n",
    "        results = model(batch_tokens)\n",
    "    avg_tokens = results[\"logits\"][0].mean(axis=0)\n",
    "    return avg_tokens.cpu().numpy()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "5599c92b",
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_most_similars_by_esm(model, batch_converter, sequence, only_chain_id=None, top_count=5):\n",
    "    res1 = evaluate(model, batch_converter, sequence)\n",
    "    offset = 0\n",
    "    limit = 20\n",
    "    tops = []\n",
    "    while True:\n",
    "        chains = get_chains_to_compare(offset, limit, only_chain_id)\n",
    "        if not len(chains):\n",
    "            break\n",
    "        offset += limit\n",
    "        res = np.array([json.loads(chain.esm) for chain in chains])\n",
    "        diffs = np.absolute(res-res1).mean(axis=1)\n",
    "        mins = np.argsort(diffs)[:top_count]\n",
    "        for min_index in mins:\n",
    "            i = len(tops)\n",
    "            while i>0 and diffs[min_index]<tops[i-1][0]:\n",
    "                i-=1\n",
    "            if i<top_count:\n",
    "                tops.insert(i, (diffs[min_index], chains[min_index]))\n",
    "                tops = tops[:top_count]\n",
    "    return [i for i in tops]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "id": "a5e8f5db",
   "metadata": {},
   "outputs": [],
   "source": [
    "def fold_sequence(sequence):\n",
    "    allowed = {'B', 'V', 'X', 'F', 'H', 'K', 'I', 'C', 'S', 'L', 'Y', 'Z', 'T', 'A', 'D', 'P', 'R', 'E', 'N', 'Q', 'J', 'G', 'M', 'W'}\n",
    "    if len(set(sequence).difference(allowed)):\n",
    "        raise Exception(f\"Sequence is only allowed to have {allowed} tokens.\")\n",
    "    res = requests.post(\"https://api.esmatlas.com/foldSequence/v1/pdb/\", data=sequence)\n",
    "    if res.status_code != 200:\n",
    "        raise Exception(res.text)\n",
    "    return res.text"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "33f3f4f3",
   "metadata": {},
   "outputs": [],
   "source": [
    "def to_base_36(s):\n",
    "    BS=\"0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ\"\n",
    "    b = 36\n",
    "    res = \"\"\n",
    "    while s:\n",
    "        res+=BS[s%b]\n",
    "        s//= b\n",
    "    return res[::-1] or \"0\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "9453133a",
   "metadata": {},
   "outputs": [],
   "source": [
    "def save_generated_pdb(sequence, pdb_content):\n",
    "    pdbs_folder = \"/Users/arturmkrtchyan/Desktop/pdb2/generated\"\n",
    "    os.makedirs(pdbs_folder, exist_ok=True)\n",
    "    generated_count = len(os.listdir(pdbs_folder))\n",
    "    pdb_id = \"+\" + to_base_36(generated_count).zfill(3)\n",
    "    pdb_file_name = pdb_id + \".pdb\"\n",
    "    pdb_path = os.path.join(pdbs_folder, pdb_file_name)\n",
    "    with open(pdb_path, \"w\") as f:\n",
    "        f.write(pdb_content)\n",
    "    protein = Protein(pdb_id, \"Generated Protein \" + str(generated_count), pdb_path)\n",
    "    tokens = evaluate(model, sequence)\n",
    "    esm = json.dumps(tokens.tolist())\n",
    "    chain = Chain(pdb_id, \"A\", sequence, esm)\n",
    "    session.add(protein)\n",
    "    session.add(chain)\n",
    "    protein.chains.append(chain)\n",
    "    session.commit()\n",
    "    return pdb_id"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "0fb9c907",
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_similars(sequence, only_chain_id=None, save_to_db=True, algorithm=\"esm\", count=5):\n",
    "    score_title = None\n",
    "    if algorithm==\"esm\":\n",
    "        score_title = \"Difference\"\n",
    "        model, alphabet = esm.pretrained.esm2_t33_650M_UR50D()\n",
    "        batch_converter = alphabet.get_batch_converter()\n",
    "        model.eval();\n",
    "        similars = get_most_similars_by_esm(model, batch_converter, sequence, only_chain_id, top_count=count)\n",
    "    else:\n",
    "        score_title = \"Similarity score\" \n",
    "        similars = get_most_similars_by_MSA(sequence, only_chain_id, top_count=count)\n",
    "    generated_pdb = fold_sequence(sequence)\n",
    "    if save_to_db:\n",
    "        save_generated_pdb(sequence, generated_pdb)\n",
    "    print(f\"Visual representation of this sequence` {sequence}\")\n",
    "    view_pdb(data=generated_pdb, chain_id=\"A\")\n",
    "    print(f\"Below you can see the most {count} similar proteins\")\n",
    "    for similarity, similar in similars:\n",
    "        print(f\"{score_title}: {round(float(similarity), 2)}\\n Protein: {similar.protein.name}({similar.protein.id})\\n Chain Id: {similar.chain_id}\\n Sequence of the chain: {similar.sequence}\")\n",
    "        view_pdb(similar.protein, similar.chain_id)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "id": "6cb681cd",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Visual representation of this sequence` DTDADADCDCDC\n"
     ]
    },
    {
     "data": {
      "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_1684696742110472\"  style=\"position: relative; width: 600px; height: 400px\">\n        <p id=\"3dmolwarning_1684696742110472\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.0.1/3Dmol-min.js');\n}\n\nvar viewer_1684696742110472 = null;\nvar warn = document.getElementById(\"3dmolwarning_1684696742110472\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_1684696742110472 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_1684696742110472\"),{backgroundColor:\"white\"});\nviewer_1684696742110472.zoomTo();\n\tviewer_1684696742110472.addModelsAsFrames(\"HEADER                                            18-OCT-22                     \\nTITLE     ESMFOLD V1 PREDICTION FOR INPUT\\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   ZEMING LIN, HALIL AKIN, ROSHAN RAO, BRIAN HIE, ZHONGKAI ZHU,\\nREMARK   1  AUTH 2 WENTING LU, NIKITA SMETANIN, ROBERT VERKUIL, ORI KABELI,\\nREMARK   1  AUTH 3 YANIV SHMUELI, ALLAN DOS SANTOS COSTA,\\nREMARK   1  AUTH 4 MARYAM FAZEL-ZARANDI, TOM SERCU, SALVATORE CANDIDO,\\nREMARK   1  AUTH 5 ALEXANDER RIVES\\nREMARK   1  TITL   EVOLUTIONARY-SCALE PREDICTION OF ATOMIC LEVEL PROTEIN\\nREMARK   1  TITL 2 STRUCTURE WITH A LANGUAGE MODEL\\nREMARK   1  REF                                                                 \\nREMARK   1  REFN                                                                \\nREMARK   1  PMID                                                                \\nREMARK   1  DOI    10.1101/2022.07.20.500902                                    \\nREMARK   1                                                                      \\nREMARK   1 LICENSE AND DISCLAIMERS                \\nREMARK   1 ESM METAGENOMIC ATLAS DATA IS AVAILABLE UNDER\\nREMARK   1 A CC-BY-4.0 LICENSE FOR ACADEMIC AND COMMERCIAL USE.\\nREMARK   1 COPYRIGHT (C) META PLATFORMS, INC. ALL RIGHTS RESERVED.\\nREMARK   1 USE OF THE ESM METAGENOMIC ATLAS DATA IS SUBJECT\\nREMARK   1 TO THE META OPEN SOURCE TERMS OF USE AND PRIVACY POLICY.\\nATOM      1  N   ASP A   1      -4.527  -8.590 -15.586  1.00  0.70           N  \\nATOM      2  CA  ASP A   1      -3.904  -8.617 -14.265  1.00  0.70           C  \\nATOM      3  C   ASP A   1      -4.096  -7.287 -13.540  1.00  0.71           C  \\nATOM      4  CB  ASP A   1      -2.414  -8.943 -14.381  1.00  0.65           C  \\nATOM      5  O   ASP A   1      -3.796  -6.225 -14.089  1.00  0.67           O  \\nATOM      6  CG  ASP A   1      -2.145 -10.420 -14.608  1.00  0.62           C  \\nATOM      7  OD1 ASP A   1      -2.783 -11.265 -13.943  1.00  0.62           O  \\nATOM      8  OD2 ASP A   1      -1.286 -10.743 -15.456  1.00  0.67           O  \\nATOM      9  N   THR A   2      -5.228  -7.051 -12.825  1.00  0.74           N  \\nATOM     10  CA  THR A   2      -5.584  -5.950 -11.937  1.00  0.74           C  \\nATOM     11  C   THR A   2      -4.548  -5.794 -10.827  1.00  0.73           C  \\nATOM     12  CB  THR A   2      -6.977  -6.160 -11.317  1.00  0.69           C  \\nATOM     13  O   THR A   2      -4.243  -6.753 -10.116  1.00  0.69           O  \\nATOM     14  CG2 THR A   2      -8.057  -6.209 -12.393  1.00  0.62           C  \\nATOM     15  OG1 THR A   2      -6.988  -7.394 -10.588  1.00  0.68           O  \\nATOM     16  N   ASP A   3      -3.542  -5.119 -11.051  1.00  0.67           N  \\nATOM     17  CA  ASP A   3      -2.629  -4.725  -9.983  1.00  0.66           C  \\nATOM     18  C   ASP A   3      -3.324  -3.816  -8.973  1.00  0.66           C  \\nATOM     19  CB  ASP A   3      -1.397  -4.025 -10.561  1.00  0.63           C  \\nATOM     20  O   ASP A   3      -3.678  -2.679  -9.293  1.00  0.65           O  \\nATOM     21  CG  ASP A   3      -0.491  -4.964 -11.338  1.00  0.61           C  \\nATOM     22  OD1 ASP A   3      -0.061  -5.997 -10.781  1.00  0.60           O  \\nATOM     23  OD2 ASP A   3      -0.202  -4.666 -12.518  1.00  0.63           O  \\nATOM     24  N   ALA A   4      -4.473  -4.274  -8.189  1.00  0.63           N  \\nATOM     25  CA  ALA A   4      -5.249  -3.870  -7.020  1.00  0.61           C  \\nATOM     26  C   ALA A   4      -4.339  -3.591  -5.827  1.00  0.62           C  \\nATOM     27  CB  ALA A   4      -6.274  -4.944  -6.663  1.00  0.58           C  \\nATOM     28  O   ALA A   4      -3.440  -4.379  -5.527  1.00  0.59           O  \\nATOM     29  N   ASP A   5      -3.693  -2.519  -5.623  1.00  0.64           N  \\nATOM     30  CA  ASP A   5      -3.469  -1.687  -4.445  1.00  0.62           C  \\nATOM     31  C   ASP A   5      -1.995  -1.682  -4.048  1.00  0.63           C  \\nATOM     32  CB  ASP A   5      -4.327  -2.172  -3.274  1.00  0.59           C  \\nATOM     33  O   ASP A   5      -1.534  -2.578  -3.338  1.00  0.62           O  \\nATOM     34  CG  ASP A   5      -5.815  -1.973  -3.507  1.00  0.58           C  \\nATOM     35  OD1 ASP A   5      -6.249  -0.823  -3.731  1.00  0.58           O  \\nATOM     36  OD2 ASP A   5      -6.560  -2.976  -3.462  1.00  0.60           O  \\nATOM     37  N   ALA A   6      -1.016  -1.423  -4.849  1.00  0.65           N  \\nATOM     38  CA  ALA A   6       0.232  -1.289  -4.103  1.00  0.63           C  \\nATOM     39  C   ALA A   6       0.504   0.170  -3.746  1.00  0.64           C  \\nATOM     40  CB  ALA A   6       1.397  -1.863  -4.906  1.00  0.60           C  \\nATOM     41  O   ALA A   6       1.009   0.934  -4.573  1.00  0.63           O  \\nATOM     42  N   ASP A   7      -0.491   0.945  -3.614  1.00  0.66           N  \\nATOM     43  CA  ASP A   7      -0.255   2.198  -2.903  1.00  0.65           C  \\nATOM     44  C   ASP A   7       0.631   1.977  -1.679  1.00  0.66           C  \\nATOM     45  CB  ASP A   7      -1.581   2.836  -2.484  1.00  0.62           C  \\nATOM     46  O   ASP A   7       0.174   1.450  -0.663  1.00  0.65           O  \\nATOM     47  CG  ASP A   7      -2.389   3.352  -3.662  1.00  0.60           C  \\nATOM     48  OD1 ASP A   7      -1.825   4.055  -4.528  1.00  0.60           O  \\nATOM     49  OD2 ASP A   7      -3.602   3.056  -3.723  1.00  0.62           O  \\nATOM     50  N   CYS A   8       1.901   1.288  -1.745  1.00  0.63           N  \\nATOM     51  CA  CYS A   8       2.951   1.389  -0.737  1.00  0.63           C  \\nATOM     52  C   CYS A   8       3.216   2.844  -0.370  1.00  0.63           C  \\nATOM     53  CB  CYS A   8       4.239   0.737  -1.239  1.00  0.59           C  \\nATOM     54  O   CYS A   8       3.414   3.684  -1.249  1.00  0.62           O  \\nATOM     55  SG  CYS A   8       4.083  -1.034  -1.557  1.00  0.61           S  \\nATOM     56  N   ASP A   9       2.579   3.561   0.543  1.00  0.67           N  \\nATOM     57  CA  ASP A   9       3.059   4.716   1.296  1.00  0.66           C  \\nATOM     58  C   ASP A   9       4.210   4.328   2.221  1.00  0.67           C  \\nATOM     59  CB  ASP A   9       1.920   5.339   2.106  1.00  0.63           C  \\nATOM     60  O   ASP A   9       3.985   3.824   3.323  1.00  0.66           O  \\nATOM     61  CG  ASP A   9       0.917   6.084   1.242  1.00  0.62           C  \\nATOM     62  OD1 ASP A   9       1.210   6.348   0.056  1.00  0.62           O  \\nATOM     63  OD2 ASP A   9      -0.176   6.412   1.754  1.00  0.63           O  \\nATOM     64  N   CYS A  10       5.376   3.679   1.892  1.00  0.65           N  \\nATOM     65  CA  CYS A  10       6.577   3.465   2.692  1.00  0.65           C  \\nATOM     66  C   CYS A  10       7.643   4.503   2.363  1.00  0.65           C  \\nATOM     67  CB  CYS A  10       7.132   2.060   2.461  1.00  0.61           C  \\nATOM     68  O   CYS A  10       7.613   5.113   1.293  1.00  0.63           O  \\nATOM     69  SG  CYS A  10       7.769   1.794   0.792  1.00  0.66           S  \\nATOM     70  N   ASP A  11       8.211   5.354   3.246  1.00  0.71           N  \\nATOM     71  CA  ASP A  11       9.507   6.021   3.320  1.00  0.70           C  \\nATOM     72  C   ASP A  11      10.642   5.051   2.999  1.00  0.70           C  \\nATOM     73  CB  ASP A  11       9.717   6.634   4.706  1.00  0.67           C  \\nATOM     74  O   ASP A  11      10.931   4.143   3.781  1.00  0.69           O  \\nATOM     75  CG  ASP A  11       8.832   7.842   4.960  1.00  0.64           C  \\nATOM     76  OD1 ASP A  11       8.326   8.442   3.987  1.00  0.64           O  \\nATOM     77  OD2 ASP A  11       8.642   8.199   6.143  1.00  0.66           O  \\nATOM     78  N   CYS A  12      10.815   4.162   2.041  1.00  0.66           N  \\nATOM     79  CA  CYS A  12      12.137   3.800   1.543  1.00  0.69           C  \\nATOM     80  C   CYS A  12      12.967   5.042   1.244  1.00  0.66           C  \\nATOM     81  CB  CYS A  12      12.019   2.939   0.286  1.00  0.63           C  \\nATOM     82  O   CYS A  12      12.419   6.095   0.912  1.00  0.65           O  \\nATOM     83  SG  CYS A  12      11.145   3.750  -1.072  1.00  0.66           S  \");\n\tviewer_1684696742110472.setStyle({\"model\": -1, \"chain\": \"A\"},{\"cartoon\": {\"color\": \"spectrum\"}});\n\tviewer_1684696742110472.zoomTo();\nviewer_1684696742110472.render();\n});\n</script>",
      "text/html": [
       "<div id=\"3dmolviewer_1684696742110472\"  style=\"position: relative; width: 600px; height: 400px\">\n",
       "        <p id=\"3dmolwarning_1684696742110472\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
       "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
       "        </div>\n",
       "<script>\n",
       "\n",
       "var loadScriptAsync = function(uri){\n",
       "  return new Promise((resolve, reject) => {\n",
       "    //this is to ignore the existence of requirejs amd\n",
       "    var savedexports, savedmodule;\n",
       "    if (typeof exports !== 'undefined') savedexports = exports;\n",
       "    else exports = {}\n",
       "    if (typeof module !== 'undefined') savedmodule = module;\n",
       "    else module = {}\n",
       "\n",
       "    var tag = document.createElement('script');\n",
       "    tag.src = uri;\n",
       "    tag.async = true;\n",
       "    tag.onload = () => {\n",
       "        exports = savedexports;\n",
       "        module = savedmodule;\n",
       "        resolve();\n",
       "    };\n",
       "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
       "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
       "});\n",
       "};\n",
       "\n",
       "if(typeof $3Dmolpromise === 'undefined') {\n",
       "$3Dmolpromise = null;\n",
       "  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.0.1/3Dmol-min.js');\n",
       "}\n",
       "\n",
       "var viewer_1684696742110472 = null;\n",
       "var warn = document.getElementById(\"3dmolwarning_1684696742110472\");\n",
       "if(warn) {\n",
       "    warn.parentNode.removeChild(warn);\n",
       "}\n",
       "$3Dmolpromise.then(function() {\n",
       "viewer_1684696742110472 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_1684696742110472\"),{backgroundColor:\"white\"});\n",
       "viewer_1684696742110472.zoomTo();\n",
       "\tviewer_1684696742110472.addModelsAsFrames(\"HEADER                                            18-OCT-22                     \\nTITLE     ESMFOLD V1 PREDICTION FOR INPUT\\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   ZEMING LIN, HALIL AKIN, ROSHAN RAO, BRIAN HIE, ZHONGKAI ZHU,\\nREMARK   1  AUTH 2 WENTING LU, NIKITA SMETANIN, ROBERT VERKUIL, ORI KABELI,\\nREMARK   1  AUTH 3 YANIV SHMUELI, ALLAN DOS SANTOS COSTA,\\nREMARK   1  AUTH 4 MARYAM FAZEL-ZARANDI, TOM SERCU, SALVATORE CANDIDO,\\nREMARK   1  AUTH 5 ALEXANDER RIVES\\nREMARK   1  TITL   EVOLUTIONARY-SCALE PREDICTION OF ATOMIC LEVEL PROTEIN\\nREMARK   1  TITL 2 STRUCTURE WITH A LANGUAGE MODEL\\nREMARK   1  REF                                                                 \\nREMARK   1  REFN                                                                \\nREMARK   1  PMID                                                                \\nREMARK   1  DOI    10.1101/2022.07.20.500902                                    \\nREMARK   1                                                                      \\nREMARK   1 LICENSE AND DISCLAIMERS                \\nREMARK   1 ESM METAGENOMIC ATLAS DATA IS AVAILABLE UNDER\\nREMARK   1 A CC-BY-4.0 LICENSE FOR ACADEMIC AND COMMERCIAL USE.\\nREMARK   1 COPYRIGHT (C) META PLATFORMS, INC. ALL RIGHTS RESERVED.\\nREMARK   1 USE OF THE ESM METAGENOMIC ATLAS DATA IS SUBJECT\\nREMARK   1 TO THE META OPEN SOURCE TERMS OF USE AND PRIVACY POLICY.\\nATOM      1  N   ASP A   1      -4.527  -8.590 -15.586  1.00  0.70           N  \\nATOM      2  CA  ASP A   1      -3.904  -8.617 -14.265  1.00  0.70           C  \\nATOM      3  C   ASP A   1      -4.096  -7.287 -13.540  1.00  0.71           C  \\nATOM      4  CB  ASP A   1      -2.414  -8.943 -14.381  1.00  0.65           C  \\nATOM      5  O   ASP A   1      -3.796  -6.225 -14.089  1.00  0.67           O  \\nATOM      6  CG  ASP A   1      -2.145 -10.420 -14.608  1.00  0.62           C  \\nATOM      7  OD1 ASP A   1      -2.783 -11.265 -13.943  1.00  0.62           O  \\nATOM      8  OD2 ASP A   1      -1.286 -10.743 -15.456  1.00  0.67           O  \\nATOM      9  N   THR A   2      -5.228  -7.051 -12.825  1.00  0.74           N  \\nATOM     10  CA  THR A   2      -5.584  -5.950 -11.937  1.00  0.74           C  \\nATOM     11  C   THR A   2      -4.548  -5.794 -10.827  1.00  0.73           C  \\nATOM     12  CB  THR A   2      -6.977  -6.160 -11.317  1.00  0.69           C  \\nATOM     13  O   THR A   2      -4.243  -6.753 -10.116  1.00  0.69           O  \\nATOM     14  CG2 THR A   2      -8.057  -6.209 -12.393  1.00  0.62           C  \\nATOM     15  OG1 THR A   2      -6.988  -7.394 -10.588  1.00  0.68           O  \\nATOM     16  N   ASP A   3      -3.542  -5.119 -11.051  1.00  0.67           N  \\nATOM     17  CA  ASP A   3      -2.629  -4.725  -9.983  1.00  0.66           C  \\nATOM     18  C   ASP A   3      -3.324  -3.816  -8.973  1.00  0.66           C  \\nATOM     19  CB  ASP A   3      -1.397  -4.025 -10.561  1.00  0.63           C  \\nATOM     20  O   ASP A   3      -3.678  -2.679  -9.293  1.00  0.65           O  \\nATOM     21  CG  ASP A   3      -0.491  -4.964 -11.338  1.00  0.61           C  \\nATOM     22  OD1 ASP A   3      -0.061  -5.997 -10.781  1.00  0.60           O  \\nATOM     23  OD2 ASP A   3      -0.202  -4.666 -12.518  1.00  0.63           O  \\nATOM     24  N   ALA A   4      -4.473  -4.274  -8.189  1.00  0.63           N  \\nATOM     25  CA  ALA A   4      -5.249  -3.870  -7.020  1.00  0.61           C  \\nATOM     26  C   ALA A   4      -4.339  -3.591  -5.827  1.00  0.62           C  \\nATOM     27  CB  ALA A   4      -6.274  -4.944  -6.663  1.00  0.58           C  \\nATOM     28  O   ALA A   4      -3.440  -4.379  -5.527  1.00  0.59           O  \\nATOM     29  N   ASP A   5      -3.693  -2.519  -5.623  1.00  0.64           N  \\nATOM     30  CA  ASP A   5      -3.469  -1.687  -4.445  1.00  0.62           C  \\nATOM     31  C   ASP A   5      -1.995  -1.682  -4.048  1.00  0.63           C  \\nATOM     32  CB  ASP A   5      -4.327  -2.172  -3.274  1.00  0.59           C  \\nATOM     33  O   ASP A   5      -1.534  -2.578  -3.338  1.00  0.62           O  \\nATOM     34  CG  ASP A   5      -5.815  -1.973  -3.507  1.00  0.58           C  \\nATOM     35  OD1 ASP A   5      -6.249  -0.823  -3.731  1.00  0.58           O  \\nATOM     36  OD2 ASP A   5      -6.560  -2.976  -3.462  1.00  0.60           O  \\nATOM     37  N   ALA A   6      -1.016  -1.423  -4.849  1.00  0.65           N  \\nATOM     38  CA  ALA A   6       0.232  -1.289  -4.103  1.00  0.63           C  \\nATOM     39  C   ALA A   6       0.504   0.170  -3.746  1.00  0.64           C  \\nATOM     40  CB  ALA A   6       1.397  -1.863  -4.906  1.00  0.60           C  \\nATOM     41  O   ALA A   6       1.009   0.934  -4.573  1.00  0.63           O  \\nATOM     42  N   ASP A   7      -0.491   0.945  -3.614  1.00  0.66           N  \\nATOM     43  CA  ASP A   7      -0.255   2.198  -2.903  1.00  0.65           C  \\nATOM     44  C   ASP A   7       0.631   1.977  -1.679  1.00  0.66           C  \\nATOM     45  CB  ASP A   7      -1.581   2.836  -2.484  1.00  0.62           C  \\nATOM     46  O   ASP A   7       0.174   1.450  -0.663  1.00  0.65           O  \\nATOM     47  CG  ASP A   7      -2.389   3.352  -3.662  1.00  0.60           C  \\nATOM     48  OD1 ASP A   7      -1.825   4.055  -4.528  1.00  0.60           O  \\nATOM     49  OD2 ASP A   7      -3.602   3.056  -3.723  1.00  0.62           O  \\nATOM     50  N   CYS A   8       1.901   1.288  -1.745  1.00  0.63           N  \\nATOM     51  CA  CYS A   8       2.951   1.389  -0.737  1.00  0.63           C  \\nATOM     52  C   CYS A   8       3.216   2.844  -0.370  1.00  0.63           C  \\nATOM     53  CB  CYS A   8       4.239   0.737  -1.239  1.00  0.59           C  \\nATOM     54  O   CYS A   8       3.414   3.684  -1.249  1.00  0.62           O  \\nATOM     55  SG  CYS A   8       4.083  -1.034  -1.557  1.00  0.61           S  \\nATOM     56  N   ASP A   9       2.579   3.561   0.543  1.00  0.67           N  \\nATOM     57  CA  ASP A   9       3.059   4.716   1.296  1.00  0.66           C  \\nATOM     58  C   ASP A   9       4.210   4.328   2.221  1.00  0.67           C  \\nATOM     59  CB  ASP A   9       1.920   5.339   2.106  1.00  0.63           C  \\nATOM     60  O   ASP A   9       3.985   3.824   3.323  1.00  0.66           O  \\nATOM     61  CG  ASP A   9       0.917   6.084   1.242  1.00  0.62           C  \\nATOM     62  OD1 ASP A   9       1.210   6.348   0.056  1.00  0.62           O  \\nATOM     63  OD2 ASP A   9      -0.176   6.412   1.754  1.00  0.63           O  \\nATOM     64  N   CYS A  10       5.376   3.679   1.892  1.00  0.65           N  \\nATOM     65  CA  CYS A  10       6.577   3.465   2.692  1.00  0.65           C  \\nATOM     66  C   CYS A  10       7.643   4.503   2.363  1.00  0.65           C  \\nATOM     67  CB  CYS A  10       7.132   2.060   2.461  1.00  0.61           C  \\nATOM     68  O   CYS A  10       7.613   5.113   1.293  1.00  0.63           O  \\nATOM     69  SG  CYS A  10       7.769   1.794   0.792  1.00  0.66           S  \\nATOM     70  N   ASP A  11       8.211   5.354   3.246  1.00  0.71           N  \\nATOM     71  CA  ASP A  11       9.507   6.021   3.320  1.00  0.70           C  \\nATOM     72  C   ASP A  11      10.642   5.051   2.999  1.00  0.70           C  \\nATOM     73  CB  ASP A  11       9.717   6.634   4.706  1.00  0.67           C  \\nATOM     74  O   ASP A  11      10.931   4.143   3.781  1.00  0.69           O  \\nATOM     75  CG  ASP A  11       8.832   7.842   4.960  1.00  0.64           C  \\nATOM     76  OD1 ASP A  11       8.326   8.442   3.987  1.00  0.64           O  \\nATOM     77  OD2 ASP A  11       8.642   8.199   6.143  1.00  0.66           O  \\nATOM     78  N   CYS A  12      10.815   4.162   2.041  1.00  0.66           N  \\nATOM     79  CA  CYS A  12      12.137   3.800   1.543  1.00  0.69           C  \\nATOM     80  C   CYS A  12      12.967   5.042   1.244  1.00  0.66           C  \\nATOM     81  CB  CYS A  12      12.019   2.939   0.286  1.00  0.63           C  \\nATOM     82  O   CYS A  12      12.419   6.095   0.912  1.00  0.65           O  \\nATOM     83  SG  CYS A  12      11.145   3.750  -1.072  1.00  0.66           S  \");\n",
       "\tviewer_1684696742110472.setStyle({\"model\": -1, \"chain\": \"A\"},{\"cartoon\": {\"color\": \"spectrum\"}});\n",
       "\tviewer_1684696742110472.zoomTo();\n",
       "viewer_1684696742110472.render();\n",
       "});\n",
       "</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Below you can see the most 5 similar proteins\n",
      "Similarity score: 12.0\n",
      " Protein: stable loop in the crystal structure of the intercalated four-stranded cytosine-rich metazoan telomere(200D)\n",
      " Chain Id: A\n",
      " Sequence of the chain: DTDADADCDCDC\n"
     ]
    },
    {
     "data": {
      "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_1684696742114819\"  style=\"position: relative; width: 600px; height: 400px\">\n        <p id=\"3dmolwarning_1684696742114819\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.0.1/3Dmol-min.js');\n}\n\nvar viewer_1684696742114819 = null;\nvar warn = document.getElementById(\"3dmolwarning_1684696742114819\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_1684696742114819 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_1684696742114819\"),{backgroundColor:\"white\"});\nviewer_1684696742114819.zoomTo();\n\tviewer_1684696742114819.addModelsAsFrames(\"HEADER    DNA                                     16-FEB-95   200D              \\nTITLE     STABLE LOOP IN THE CRYSTAL STRUCTURE OF THE INTERCALATED              \\nTITLE    2 FOUR-STRANDED CYTOSINE-RICH METAZOAN TELOMERE                        \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: DNA (5'-D(*TP*AP*AP*CP*CP*C)-3');                          \\nCOMPND   3 CHAIN: A, B;                                                         \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 SYNTHETIC: YES                                                       \\nKEYWDS    U-DNA, QUADRUPLE HELIX, TETRAPLEX, BASE INTERCALATED, LOOP            \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    C.KANG,I.BERGER,C.LOCKSHIN,R.RATLIFF,R.MOYZIS,A.RICH                  \\nREVDAT   2   24-FEB-09 200D    1       VERSN                                    \\nREVDAT   1   31-JUL-95 200D    0                                                \\nJRNL        AUTH   C.KANG,I.BERGER,C.LOCKSHIN,R.RATLIFF,R.MOYZIS,               \\nJRNL        AUTH 2 A.RICH                                                       \\nJRNL        TITL   STABLE LOOP IN THE CRYSTAL STRUCTURE OF THE                  \\nJRNL        TITL 2 INTERCALATED FOUR-STRANDED CYTOSINE-RICH METAZOAN            \\nJRNL        TITL 3 TELOMERE.                                                    \\nJRNL        REF    PROC.NATL.ACAD.SCI.USA        V.  92  3874 1995              \\nJRNL        REFN                   ISSN 0027-8424                               \\nJRNL        PMID   7731999                                                      \\nJRNL        DOI    10.1073/PNAS.92.9.3874                                       \\nREMARK   1                                                                      \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.85 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : X-PLOR                                               \\nREMARK   3   AUTHORS     : BRUNGER                                              \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.85                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 12.00                          \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : 1.000                          \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : 77.0                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 2122                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : 0.190                           \\nREMARK   3   FREE R VALUE                     : 0.223                           \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 0                                       \\nREMARK   3   NUCLEIC ACID ATOMS       : 258                                     \\nREMARK   3   HETEROGEN ATOMS          : 0                                       \\nREMARK   3   SOLVENT ATOMS            : 26                                      \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : 0.030                           \\nREMARK   3   BOND ANGLES            (DEGREES) : 4.00                            \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 200D COMPLIES WITH FORMAT V. 3.15, 01-DEC-08                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : 20-JUL-94                          \\nREMARK 200  TEMPERATURE           (KELVIN) : 277.00                             \\nREMARK 200  PH                             : 6.00                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : N                                  \\nREMARK 200  RADIATION SOURCE               : ROTATING ANODE                     \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : M                                  \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : IMAGE PLATE                        \\nREMARK 200  DETECTOR MANUFACTURER          : RIGAKU RAXIS IIC                   \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : MSC                                \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : 0.04500                            \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): NULL                                      \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): NULL                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: PH 6.00, VAPOR DIFFUSION                 \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: F 2 2 2                          \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -X,-Y,Z                                                 \\nREMARK 290       3555   -X,Y,-Z                                                 \\nREMARK 290       4555   X,-Y,-Z                                                 \\nREMARK 290       5555   X,Y+1/2,Z+1/2                                           \\nREMARK 290       6555   -X,-Y+1/2,Z+1/2                                         \\nREMARK 290       7555   -X,Y+1/2,-Z+1/2                                         \\nREMARK 290       8555   X,-Y+1/2,-Z+1/2                                         \\nREMARK 290       9555   X+1/2,Y,Z+1/2                                           \\nREMARK 290      10555   -X+1/2,-Y,Z+1/2                                         \\nREMARK 290      11555   -X+1/2,Y,-Z+1/2                                         \\nREMARK 290      12555   X+1/2,-Y,-Z+1/2                                         \\nREMARK 290      13555   X+1/2,Y+1/2,Z                                           \\nREMARK 290      14555   -X+1/2,-Y+1/2,Z                                         \\nREMARK 290      15555   -X+1/2,Y+1/2,-Z                                         \\nREMARK 290      16555   X+1/2,-Y+1/2,-Z                                         \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   3 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   3  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   4  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   4  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   5  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   5  0.000000  1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3   5  0.000000  0.000000  1.000000       13.43000            \\nREMARK 290   SMTRY1   6 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   6  0.000000 -1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3   6  0.000000  0.000000  1.000000       13.43000            \\nREMARK 290   SMTRY1   7 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   7  0.000000  1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3   7  0.000000  0.000000 -1.000000       13.43000            \\nREMARK 290   SMTRY1   8  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   8  0.000000 -1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3   8  0.000000  0.000000 -1.000000       13.43000            \\nREMARK 290   SMTRY1   9  1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2   9  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   9  0.000000  0.000000  1.000000       13.43000            \\nREMARK 290   SMTRY1  10 -1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  10  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3  10  0.000000  0.000000  1.000000       13.43000            \\nREMARK 290   SMTRY1  11 -1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  11  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3  11  0.000000  0.000000 -1.000000       13.43000            \\nREMARK 290   SMTRY1  12  1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  12  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3  12  0.000000  0.000000 -1.000000       13.43000            \\nREMARK 290   SMTRY1  13  1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  13  0.000000  1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3  13  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1  14 -1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  14  0.000000 -1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3  14  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1  15 -1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  15  0.000000  1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3  15  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1  16  1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  16  0.000000 -1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3  16  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: TETRAMERIC                        \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A, B                                  \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2 -1.000000  0.000000  0.000000       59.94000            \\nREMARK 350   BIOMT2   2  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000 -1.000000       26.86000            \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND LENGTHS                                      \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,2(A3,1X,A1,I4,A1,1X,A4,3X),1X,F6.3)               \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   RES CSSEQI ATM2   DEVIATION                     \\nREMARK 500     DT A   1   C5     DT A   1   C7      0.038                       \\nREMARK 500     DT A   1   O3'    DA A   2   P      -0.084                       \\nREMARK 500     DA A   3   C4'    DA A   3   C3'    -0.094                       \\nREMARK 500     DA A   3   N9     DA A   3   C4     -0.042                       \\nREMARK 500     DC A   4   N3     DC A   4   C4     -0.047                       \\nREMARK 500     DC A   5   C3'    DC A   5   C2'    -0.055                       \\nREMARK 500     DC A   6   C5'    DC A   6   C4'     0.090                       \\nREMARK 500     DC A   6   C3'    DC A   6   C2'    -0.050                       \\nREMARK 500     DC A   6   N3     DC A   6   C4     -0.052                       \\nREMARK 500     DT B   7   C5'    DT B   7   C4'     0.082                       \\nREMARK 500     DT B   7   C2     DT B   7   N3     -0.049                       \\nREMARK 500     DA B   8   P      DA B   8   O5'     0.063                       \\nREMARK 500     DC B  10   C4'    DC B  10   C3'    -0.083                       \\nREMARK 500     DC B  10   N1     DC B  10   C6     -0.063                       \\nREMARK 500     DC B  10   C4     DC B  10   C5     -0.050                       \\nREMARK 500     DC B  11   N1     DC B  11   C2     -0.094                       \\nREMARK 500     DC B  11   N1     DC B  11   C6     -0.040                       \\nREMARK 500     DC B  11   C2     DC B  11   N3     -0.056                       \\nREMARK 500     DC B  12   O4'    DC B  12   C4'    -0.069                       \\nREMARK 500     DC B  12   N1     DC B  12   C2     -0.062                       \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \\nREMARK 500     DT A   1   O4' -  C1' -  N1  ANGL. DEV. =   2.9 DEGREES          \\nREMARK 500     DT A   1   C6  -  C5  -  C7  ANGL. DEV. =  -4.4 DEGREES          \\nREMARK 500     DA A   2   O4' -  C4' -  C3' ANGL. DEV. =   4.3 DEGREES          \\nREMARK 500     DA A   2   C4' -  C3' -  C2' ANGL. DEV. =  -9.1 DEGREES          \\nREMARK 500     DA A   2   O4' -  C1' -  C2' ANGL. DEV. =  -7.8 DEGREES          \\nREMARK 500     DA A   2   N1  -  C6  -  N6  ANGL. DEV. =   3.7 DEGREES          \\nREMARK 500     DT A   1   C3' -  O3' -  P   ANGL. DEV. =  14.8 DEGREES          \\nREMARK 500     DA A   2   O3' -  P   -  O5' ANGL. DEV. = -11.7 DEGREES          \\nREMARK 500     DA A   3   O4' -  C4' -  C3' ANGL. DEV. =  -7.7 DEGREES          \\nREMARK 500     DA A   3   C1' -  O4' -  C4' ANGL. DEV. =   6.1 DEGREES          \\nREMARK 500     DA A   3   O4' -  C1' -  C2' ANGL. DEV. =  -6.8 DEGREES          \\nREMARK 500     DA A   3   N1  -  C2  -  N3  ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500     DA A   3   C2  -  N3  -  C4  ANGL. DEV. =  -3.9 DEGREES          \\nREMARK 500     DA A   2   C3' -  O3' -  P   ANGL. DEV. =   7.9 DEGREES          \\nREMARK 500     DC A   4   C4' -  C3' -  C2' ANGL. DEV. =  -4.3 DEGREES          \\nREMARK 500     DC A   5   C4' -  C3' -  C2' ANGL. DEV. =  -7.7 DEGREES          \\nREMARK 500     DC A   5   O4' -  C1' -  N1  ANGL. DEV. =  -4.7 DEGREES          \\nREMARK 500     DC A   5   C2  -  N3  -  C4  ANGL. DEV. =   3.5 DEGREES          \\nREMARK 500     DC A   6   P   -  O5' -  C5' ANGL. DEV. =  13.2 DEGREES          \\nREMARK 500     DC A   6   O4' -  C4' -  C3' ANGL. DEV. =  -4.4 DEGREES          \\nREMARK 500     DC A   6   O4' -  C1' -  N1  ANGL. DEV. =   3.9 DEGREES          \\nREMARK 500     DC A   6   N1  -  C2  -  O2  ANGL. DEV. =   3.9 DEGREES          \\nREMARK 500     DC A   6   N3  -  C4  -  N4  ANGL. DEV. =  -5.0 DEGREES          \\nREMARK 500     DC A   6   C5  -  C4  -  N4  ANGL. DEV. =   4.2 DEGREES          \\nREMARK 500     DT B   7   C5' -  C4' -  O4' ANGL. DEV. =   9.2 DEGREES          \\nREMARK 500     DT B   7   N1  -  C2  -  O2  ANGL. DEV. =   5.3 DEGREES          \\nREMARK 500     DT B   7   N3  -  C2  -  O2  ANGL. DEV. =  -4.3 DEGREES          \\nREMARK 500     DA B   8   O4' -  C1' -  C2' ANGL. DEV. =  -5.1 DEGREES          \\nREMARK 500     DA B   8   C2  -  N3  -  C4  ANGL. DEV. =  -3.3 DEGREES          \\nREMARK 500     DA B   8   C5  -  C6  -  N1  ANGL. DEV. =  -4.0 DEGREES          \\nREMARK 500     DT B   7   C3' -  O3' -  P   ANGL. DEV. = -10.9 DEGREES          \\nREMARK 500     DA B   9   O4' -  C1' -  C2' ANGL. DEV. =  -7.3 DEGREES          \\nREMARK 500     DC B  10   C1' -  O4' -  C4' ANGL. DEV. =   6.5 DEGREES          \\nREMARK 500     DC B  10   O4' -  C1' -  C2' ANGL. DEV. = -10.3 DEGREES          \\nREMARK 500     DC B  10   O4' -  C1' -  N1  ANGL. DEV. =   3.6 DEGREES          \\nREMARK 500     DC B  10   N3  -  C2  -  O2  ANGL. DEV. =  -5.1 DEGREES          \\nREMARK 500     DC B  10   C3' -  O3' -  P   ANGL. DEV. =   9.2 DEGREES          \\nREMARK 500     DC B  12   O4' -  C1' -  C2' ANGL. DEV. =  -5.1 DEGREES          \\nREMARK 500     DC B  12   C2  -  N3  -  C4  ANGL. DEV. =   4.6 DEGREES          \\nREMARK 500     DC B  12   N1  -  C2  -  O2  ANGL. DEV. =   4.6 DEGREES          \\nREMARK 500     DC B  11   C3' -  O3' -  P   ANGL. DEV. =   8.3 DEGREES          \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: PLANAR GROUPS                                              \\nREMARK 500                                                                      \\nREMARK 500 PLANAR GROUPS IN THE FOLLOWING RESIDUES HAVE A TOTAL                 \\nREMARK 500 RMS DISTANCE OF ALL ATOMS FROM THE BEST-FIT PLANE                    \\nREMARK 500 BY MORE THAN AN EXPECTED VALUE OF 6*RMSD, WITH AN                    \\nREMARK 500 RMSD 0.02 ANGSTROMS, OR AT LEAST ONE ATOM HAS                        \\nREMARK 500 AN RMSD GREATER THAN THIS VALUE                                      \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        RMS     TYPE                                    \\nREMARK 500     DA A   3         0.06    SIDE_CHAIN                              \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nDBREF  200D A    1     6  PDB    200D     200D             1      6             \\nDBREF  200D B    7    12  PDB    200D     200D             7     12             \\nSEQRES   1 A    6   DT  DA  DA  DC  DC  DC                                      \\nSEQRES   1 B    6   DT  DA  DA  DC  DC  DC                                      \\nFORMUL   3  HOH   *26(H2 O)                                                     \\nCRYST1   59.940   81.330   26.860  90.00  90.00  90.00 F 2 2 2      32          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.016683  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  0.012296  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.037230        0.00000                         \\nATOM      1  O5'  DT A   1      24.565  31.319  24.443  1.00 30.44           O  \\nATOM      2  C5'  DT A   1      23.942  30.467  25.330  1.00 20.89           C  \\nATOM      3  C4'  DT A   1      23.216  29.306  24.650  1.00 27.77           C  \\nATOM      4  O4'  DT A   1      22.175  29.736  23.812  1.00 23.30           O  \\nATOM      5  C3'  DT A   1      24.157  28.454  23.852  1.00 35.38           C  \\nATOM      6  O3'  DT A   1      23.905  27.189  24.366  1.00 33.40           O  \\nATOM      7  C2'  DT A   1      23.531  28.590  22.502  1.00 34.88           C  \\nATOM      8  C1'  DT A   1      22.104  28.825  22.807  1.00 18.33           C  \\nATOM      9  N1   DT A   1      21.415  29.371  21.650  1.00 27.22           N  \\nATOM     10  C2   DT A   1      21.061  28.462  20.735  1.00 17.62           C  \\nATOM     11  O2   DT A   1      21.294  27.274  20.916  1.00 23.45           O  \\nATOM     12  N3   DT A   1      20.409  28.945  19.635  1.00 20.47           N  \\nATOM     13  C4   DT A   1      20.071  30.242  19.367  1.00 19.02           C  \\nATOM     14  O4   DT A   1      19.434  30.520  18.346  1.00 27.67           O  \\nATOM     15  C5   DT A   1      20.491  31.148  20.397  1.00 28.05           C  \\nATOM     16  C7   DT A   1      20.220  32.654  20.284  1.00 24.86           C  \\nATOM     17  C6   DT A   1      21.136  30.694  21.482  1.00 14.22           C  \\nATOM     18  H3   DT A   1      20.149  28.273  18.939  1.00  0.00           H  \\nATOM     19 HO5'  DT A   1      25.263  30.835  24.004  1.00  0.00           H  \\nATOM     20  P    DA A   2      24.766  25.974  24.683  1.00 23.53           P  \\nATOM     21  OP1  DA A   2      24.819  25.775  26.131  1.00 27.65           O  \\nATOM     22  OP2  DA A   2      25.964  25.990  23.827  1.00 38.21           O  \\nATOM     23  O5'  DA A   2      23.749  24.991  23.979  1.00 23.31           O  \\nATOM     24  C5'  DA A   2      22.350  25.133  24.144  1.00 31.45           C  \\nATOM     25  C4'  DA A   2      21.502  24.042  23.575  1.00 10.83           C  \\nATOM     26  O4'  DA A   2      20.221  24.252  24.204  1.00 36.32           O  \\nATOM     27  C3'  DA A   2      21.353  24.038  22.059  1.00 39.61           C  \\nATOM     28  O3'  DA A   2      22.218  23.042  21.474  1.00 19.31           O  \\nATOM     29  C2'  DA A   2      19.926  23.509  22.118  1.00 23.91           C  \\nATOM     30  C1'  DA A   2      19.171  24.220  23.257  1.00 29.47           C  \\nATOM     31  N9   DA A   2      18.839  25.627  23.010  1.00 19.13           N  \\nATOM     32  C8   DA A   2      19.151  26.734  23.759  1.00 21.05           C  \\nATOM     33  N7   DA A   2      18.737  27.848  23.267  1.00 20.27           N  \\nATOM     34  C5   DA A   2      18.093  27.426  22.100  1.00 14.04           C  \\nATOM     35  C6   DA A   2      17.391  28.116  21.124  1.00 18.64           C  \\nATOM     36  N6   DA A   2      17.250  29.453  21.149  1.00 24.64           N  \\nATOM     37  N1   DA A   2      16.861  27.359  20.157  1.00 15.56           N  \\nATOM     38  C2   DA A   2      17.016  26.046  20.152  1.00  9.93           C  \\nATOM     39  N3   DA A   2      17.641  25.306  20.992  1.00 23.82           N  \\nATOM     40  C4   DA A   2      18.160  26.078  21.948  1.00 11.68           C  \\nATOM     41  H61  DA A   2      16.745  29.923  20.410  1.00  0.00           H  \\nATOM     42  H62  DA A   2      17.644  29.994  21.903  1.00  0.00           H  \\nATOM     43  P    DA A   3      23.030  23.149  20.153  1.00 20.46           P  \\nATOM     44  OP1  DA A   3      23.954  22.013  20.095  1.00 24.95           O  \\nATOM     45  OP2  DA A   3      23.531  24.509  20.030  1.00 19.53           O  \\nATOM     46  O5'  DA A   3      21.880  22.885  19.163  1.00 17.18           O  \\nATOM     47  C5'  DA A   3      21.278  21.604  19.070  1.00 13.90           C  \\nATOM     48  C4'  DA A   3      20.159  21.601  18.065  1.00 21.24           C  \\nATOM     49  O4'  DA A   3      19.337  22.719  18.308  1.00 21.96           O  \\nATOM     50  C3'  DA A   3      20.581  21.983  16.756  1.00 22.35           C  \\nATOM     51  O3'  DA A   3      21.091  20.871  16.039  1.00 35.11           O  \\nATOM     52  C2'  DA A   3      19.323  22.493  16.117  1.00 36.04           C  \\nATOM     53  C1'  DA A   3      18.685  23.288  17.172  1.00 20.51           C  \\nATOM     54  N9   DA A   3      18.982  24.750  17.163  1.00 14.73           N  \\nATOM     55  C8   DA A   3      19.499  25.507  18.166  1.00  8.55           C  \\nATOM     56  N7   DA A   3      19.478  26.769  17.943  1.00 16.60           N  \\nATOM     57  C5   DA A   3      18.913  26.871  16.693  1.00 12.55           C  \\nATOM     58  C6   DA A   3      18.601  27.989  15.881  1.00 20.70           C  \\nATOM     59  N6   DA A   3      18.741  29.276  16.217  1.00 14.67           N  \\nATOM     60  N1   DA A   3      18.060  27.707  14.701  1.00 13.89           N  \\nATOM     61  C2   DA A   3      17.850  26.408  14.389  1.00 29.53           C  \\nATOM     62  N3   DA A   3      18.083  25.275  15.052  1.00 18.60           N  \\nATOM     63  C4   DA A   3      18.625  25.618  16.218  1.00  8.22           C  \\nATOM     64  H61  DA A   3      18.524  30.004  15.551  1.00  0.00           H  \\nATOM     65  H62  DA A   3      19.059  29.517  17.142  1.00  0.00           H  \\nATOM     66  P    DC A   4      22.425  21.041  15.096  1.00 24.23           P  \\nATOM     67  OP1  DC A   4      22.465  19.788  14.318  1.00 29.54           O  \\nATOM     68  OP2  DC A   4      23.569  21.448  15.939  1.00 26.74           O  \\nATOM     69  O5'  DC A   4      22.106  22.295  14.117  1.00 24.38           O  \\nATOM     70  C5'  DC A   4      21.326  22.363  12.942  1.00 23.31           C  \\nATOM     71  C4'  DC A   4      21.432  23.763  12.409  1.00 19.02           C  \\nATOM     72  O4'  DC A   4      20.924  24.622  13.375  1.00 13.89           O  \\nATOM     73  C3'  DC A   4      22.882  24.127  12.138  1.00 15.94           C  \\nATOM     74  O3'  DC A   4      23.153  23.807  10.751  1.00 33.78           O  \\nATOM     75  C2'  DC A   4      22.777  25.612  12.409  1.00 26.18           C  \\nATOM     76  C1'  DC A   4      21.638  25.802  13.441  1.00  6.99           C  \\nATOM     77  N1   DC A   4      22.071  25.996  14.858  1.00 15.40           N  \\nATOM     78  C2   DC A   4      22.052  27.275  15.345  1.00 14.75           C  \\nATOM     79  O2   DC A   4      21.597  28.208  14.679  1.00 20.98           O  \\nATOM     80  N3   DC A   4      22.492  27.460  16.606  1.00 19.60           N  \\nATOM     81  C4   DC A   4      22.910  26.491  17.345  1.00  7.46           C  \\nATOM     82  N4   DC A   4      23.363  26.803  18.516  1.00 11.08           N  \\nATOM     83  C5   DC A   4      22.926  25.188  16.895  1.00  7.21           C  \\nATOM     84  C6   DC A   4      22.504  24.981  15.653  1.00  9.76           C  \\nATOM     85  H41  DC A   4      23.402  27.768  18.806  1.00  0.00           H  \\nATOM     86  H42  DC A   4      23.675  26.076  19.149  1.00  0.00           H  \\nATOM     87  P    DC A   5      24.446  22.970  10.188  1.00 28.52           P  \\nATOM     88  OP1  DC A   5      24.144  22.120   9.020  1.00 24.29           O  \\nATOM     89  OP2  DC A   5      25.053  22.349  11.400  1.00 34.89           O  \\nATOM     90  O5'  DC A   5      25.373  24.138   9.643  1.00 30.47           O  \\nATOM     91  C5'  DC A   5      24.837  25.048   8.687  1.00 27.30           C  \\nATOM     92  C4'  DC A   5      25.648  26.351   8.642  1.00 20.98           C  \\nATOM     93  O4'  DC A   5      25.627  26.952   9.905  1.00 21.19           O  \\nATOM     94  C3'  DC A   5      27.100  26.119   8.375  1.00 20.71           C  \\nATOM     95  O3'  DC A   5      27.341  25.805   7.002  1.00 23.11           O  \\nATOM     96  C2'  DC A   5      27.496  27.486   8.703  1.00 13.44           C  \\nATOM     97  C1'  DC A   5      26.696  27.830   9.919  1.00 13.82           C  \\nATOM     98  N1   DC A   5      27.308  27.595  11.235  1.00 22.24           N  \\nATOM     99  C2   DC A   5      27.581  28.683  11.989  1.00  9.58           C  \\nATOM    100  O2   DC A   5      27.383  29.828  11.573  1.00 25.04           O  \\nATOM    101  N3   DC A   5      28.050  28.457  13.220  1.00 23.52           N  \\nATOM    102  C4   DC A   5      28.242  27.259  13.710  1.00  9.77           C  \\nATOM    103  N4   DC A   5      28.680  27.143  14.947  1.00 20.94           N  \\nATOM    104  C5   DC A   5      27.978  26.108  12.961  1.00 15.31           C  \\nATOM    105  C6   DC A   5      27.515  26.334  11.726  1.00 20.31           C  \\nATOM    106  H41  DC A   5      28.833  26.230  15.355  1.00  0.00           H  \\nATOM    107  H42  DC A   5      28.860  27.971  15.496  1.00  0.00           H  \\nATOM    108  P    DC A   6      28.699  25.257   6.381  1.00 26.27           P  \\nATOM    109  OP1  DC A   6      28.416  24.653   5.094  1.00 29.77           O  \\nATOM    110  OP2  DC A   6      29.457  24.544   7.410  1.00 23.78           O  \\nATOM    111  O5'  DC A   6      29.520  26.539   6.046  1.00 32.49           O  \\nATOM    112  C5'  DC A   6      29.282  27.692   5.226  1.00 30.24           C  \\nATOM    113  C4'  DC A   6      30.531  28.674   5.429  1.00 22.63           C  \\nATOM    114  O4'  DC A   6      30.781  28.988   6.818  1.00 18.93           O  \\nATOM    115  C3'  DC A   6      31.823  27.970   5.079  1.00 28.36           C  \\nATOM    116  O3'  DC A   6      32.102  28.065   3.709  1.00 27.07           O  \\nATOM    117  C2'  DC A   6      32.812  28.770   5.808  1.00 13.00           C  \\nATOM    118  C1'  DC A   6      32.139  29.157   7.068  1.00  9.92           C  \\nATOM    119  N1   DC A   6      32.607  28.338   8.228  1.00 22.07           N  \\nATOM    120  C2   DC A   6      33.155  29.092   9.255  1.00 26.49           C  \\nATOM    121  O2   DC A   6      33.154  30.333   9.252  1.00 26.06           O  \\nATOM    122  N3   DC A   6      33.673  28.433  10.285  1.00 17.53           N  \\nATOM    123  C4   DC A   6      33.677  27.152  10.362  1.00  7.84           C  \\nATOM    124  N4   DC A   6      34.255  26.711  11.432  1.00 17.65           N  \\nATOM    125  C5   DC A   6      33.130  26.339   9.369  1.00  9.69           C  \\nATOM    126  C6   DC A   6      32.596  26.974   8.310  1.00 25.15           C  \\nATOM    127 HO3'  DC A   6      31.406  27.590   3.246  1.00  0.00           H  \\nATOM    128  H41  DC A   6      34.358  25.716  11.590  1.00  0.00           H  \\nATOM    129  H42  DC A   6      34.605  27.355  12.121  1.00  0.00           H  \\nTER     130       DC A   6                                                      \\nATOM    131  O5'  DT B   7      11.293  29.851  15.858  1.00 28.85           O  \\nATOM    132  C5'  DT B   7      11.549  31.182  16.395  1.00 25.35           C  \\nATOM    133  C4'  DT B   7      12.980  31.877  16.290  1.00 39.66           C  \\nATOM    134  O4'  DT B   7      14.126  31.292  17.044  1.00 37.54           O  \\nATOM    135  C3'  DT B   7      13.456  31.813  14.854  1.00 39.01           C  \\nATOM    136  O3'  DT B   7      14.165  32.997  14.544  1.00 38.98           O  \\nATOM    137  C2'  DT B   7      14.424  30.692  14.753  1.00 20.58           C  \\nATOM    138  C1'  DT B   7      14.994  30.617  16.125  1.00 20.74           C  \\nATOM    139  N1   DT B   7      15.053  29.255  16.540  1.00 18.62           N  \\nATOM    140  C2   DT B   7      15.671  29.004  17.731  1.00 20.33           C  \\nATOM    141  O2   DT B   7      16.113  29.817  18.554  1.00 26.72           O  \\nATOM    142  N3   DT B   7      15.786  27.715  18.009  1.00 22.63           N  \\nATOM    143  C4   DT B   7      15.361  26.621  17.272  1.00 33.64           C  \\nATOM    144  O4   DT B   7      15.606  25.447  17.626  1.00 34.71           O  \\nATOM    145  C5   DT B   7      14.708  26.988  16.073  1.00 14.16           C  \\nATOM    146  C7   DT B   7      14.102  25.888  15.216  1.00 39.75           C  \\nATOM    147  C6   DT B   7      14.575  28.259  15.732  1.00 24.49           C  \\nATOM    148  H3   DT B   7      16.244  27.571  18.880  1.00  0.00           H  \\nATOM    149 HO5'  DT B   7      11.404  29.913  14.905  1.00  0.00           H  \\nATOM    150  P    DA B   8      13.931  33.345  13.024  1.00 39.22           P  \\nATOM    151  OP1  DA B   8      12.612  34.036  13.039  1.00 34.77           O  \\nATOM    152  OP2  DA B   8      14.206  32.074  12.263  1.00 39.31           O  \\nATOM    153  O5'  DA B   8      15.000  34.463  12.432  1.00 39.80           O  \\nATOM    154  C5'  DA B   8      16.303  34.113  11.867  1.00 39.02           C  \\nATOM    155  C4'  DA B   8      17.406  35.058  12.322  1.00 17.13           C  \\nATOM    156  O4'  DA B   8      16.819  36.368  12.502  1.00 22.28           O  \\nATOM    157  C3'  DA B   8      18.055  34.660  13.684  1.00 35.33           C  \\nATOM    158  O3'  DA B   8      19.437  35.008  13.398  1.00 39.28           O  \\nATOM    159  C2'  DA B   8      17.310  35.602  14.642  1.00 35.21           C  \\nATOM    160  C1'  DA B   8      17.143  36.915  13.808  1.00 14.63           C  \\nATOM    161  N9   DA B   8      16.003  37.727  14.228  1.00 20.81           N  \\nATOM    162  C8   DA B   8      14.676  37.344  14.239  1.00 37.43           C  \\nATOM    163  N7   DA B   8      13.839  38.247  14.635  1.00 23.05           N  \\nATOM    164  C5   DA B   8      14.647  39.318  14.912  1.00 19.42           C  \\nATOM    165  C6   DA B   8      14.326  40.578  15.398  1.00 14.47           C  \\nATOM    166  N6   DA B   8      13.065  41.064  15.553  1.00 20.96           N  \\nATOM    167  N1   DA B   8      15.373  41.351  15.597  1.00 33.53           N  \\nATOM    168  C2   DA B   8      16.622  40.929  15.326  1.00 27.49           C  \\nATOM    169  N3   DA B   8      17.042  39.778  14.845  1.00 27.07           N  \\nATOM    170  C4   DA B   8      15.968  39.007  14.666  1.00 10.79           C  \\nATOM    171  H61  DA B   8      12.269  40.498  15.302  1.00  0.00           H  \\nATOM    172  H62  DA B   8      12.942  41.998  15.918  1.00  0.00           H  \\nATOM    173  P    DA B   9      20.719  34.445  14.194  1.00 35.42           P  \\nATOM    174  OP1  DA B   9      21.959  35.022  13.557  1.00 39.82           O  \\nATOM    175  OP2  DA B   9      20.528  32.939  14.321  1.00 28.79           O  \\nATOM    176  O5'  DA B   9      20.556  35.250  15.553  1.00 39.94           O  \\nATOM    177  C5'  DA B   9      21.196  36.518  15.727  1.00 27.77           C  \\nATOM    178  C4'  DA B   9      20.828  37.045  17.056  1.00 16.24           C  \\nATOM    179  O4'  DA B   9      19.420  37.260  17.115  1.00 24.83           O  \\nATOM    180  C3'  DA B   9      21.131  36.015  18.071  1.00 22.96           C  \\nATOM    181  O3'  DA B   9      21.775  36.603  19.212  1.00 29.07           O  \\nATOM    182  C2'  DA B   9      19.740  35.550  18.407  1.00 26.00           C  \\nATOM    183  C1'  DA B   9      18.975  36.797  18.352  1.00 18.88           C  \\nATOM    184  N9   DA B   9      17.533  36.628  18.310  1.00 17.60           N  \\nATOM    185  C8   DA B   9      16.740  35.511  18.232  1.00 23.85           C  \\nATOM    186  N7   DA B   9      15.467  35.795  18.264  1.00 25.36           N  \\nATOM    187  C5   DA B   9      15.416  37.185  18.373  1.00 14.91           C  \\nATOM    188  C6   DA B   9      14.358  38.117  18.472  1.00 24.27           C  \\nATOM    189  N6   DA B   9      13.051  37.812  18.565  1.00 30.71           N  \\nATOM    190  N1   DA B   9      14.730  39.429  18.573  1.00 34.89           N  \\nATOM    191  C2   DA B   9      16.025  39.778  18.587  1.00 15.13           C  \\nATOM    192  N3   DA B   9      17.077  38.975  18.506  1.00 18.79           N  \\nATOM    193  C4   DA B   9      16.688  37.686  18.401  1.00 21.13           C  \\nATOM    194  H61  DA B   9      12.726  36.865  18.446  1.00  0.00           H  \\nATOM    195  H62  DA B   9      12.408  38.552  18.781  1.00  0.00           H  \\nATOM    196  P    DC B  10      23.405  36.738  19.343  1.00 28.67           P  \\nATOM    197  OP1  DC B  10      23.679  37.914  20.242  1.00 34.27           O  \\nATOM    198  OP2  DC B  10      23.910  36.731  17.961  1.00 38.84           O  \\nATOM    199  O5'  DC B  10      24.087  35.454  20.011  1.00 18.77           O  \\nATOM    200  C5'  DC B  10      24.087  35.100  21.368  1.00 30.20           C  \\nATOM    201  C4'  DC B  10      24.575  33.695  21.529  1.00  9.80           C  \\nATOM    202  O4'  DC B  10      23.800  32.700  20.879  1.00 18.83           O  \\nATOM    203  C3'  DC B  10      25.878  33.620  20.926  1.00 14.61           C  \\nATOM    204  O3'  DC B  10      26.615  32.959  21.958  1.00 36.36           O  \\nATOM    205  C2'  DC B  10      25.606  32.814  19.678  1.00 19.97           C  \\nATOM    206  C1'  DC B  10      24.481  31.842  19.992  1.00 15.38           C  \\nATOM    207  N1   DC B  10      23.686  31.565  18.773  1.00 11.72           N  \\nATOM    208  C2   DC B  10      23.456  30.308  18.325  1.00 11.51           C  \\nATOM    209  O2   DC B  10      23.748  29.333  18.974  1.00 25.69           O  \\nATOM    210  N3   DC B  10      22.811  30.099  17.201  1.00 14.49           N  \\nATOM    211  C4   DC B  10      22.370  31.109  16.513  1.00 12.31           C  \\nATOM    212  N4   DC B  10      21.662  30.932  15.398  1.00 25.78           N  \\nATOM    213  C5   DC B  10      22.573  32.400  16.939  1.00 14.33           C  \\nATOM    214  C6   DC B  10      23.237  32.570  18.074  1.00 13.01           C  \\nATOM    215  H41  DC B  10      21.523  29.994  15.042  1.00  0.00           H  \\nATOM    216  H42  DC B  10      21.261  31.720  14.909  1.00  0.00           H  \\nATOM    217  P    DC B  11      28.130  33.237  22.467  1.00 34.54           P  \\nATOM    218  OP1  DC B  11      28.322  32.261  23.594  1.00 29.17           O  \\nATOM    219  OP2  DC B  11      28.366  34.724  22.567  1.00 26.05           O  \\nATOM    220  O5'  DC B  11      28.851  32.747  21.146  1.00 29.65           O  \\nATOM    221  C5'  DC B  11      30.240  32.966  20.860  1.00 31.52           C  \\nATOM    222  C4'  DC B  11      30.646  31.847  19.984  1.00 13.84           C  \\nATOM    223  O4'  DC B  11      29.762  31.772  18.892  1.00 25.81           O  \\nATOM    224  C3'  DC B  11      32.016  32.042  19.492  1.00 15.90           C  \\nATOM    225  O3'  DC B  11      32.776  31.224  20.405  1.00 35.61           O  \\nATOM    226  C2'  DC B  11      31.886  31.528  18.055  1.00 19.50           C  \\nATOM    227  C1'  DC B  11      30.453  31.059  17.897  1.00 14.84           C  \\nATOM    228  N1   DC B  11      29.801  31.387  16.621  1.00  8.89           N  \\nATOM    229  C2   DC B  11      29.420  30.373  15.896  1.00 10.51           C  \\nATOM    230  O2   DC B  11      29.691  29.265  16.247  1.00 16.37           O  \\nATOM    231  N3   DC B  11      28.767  30.545  14.789  1.00 14.37           N  \\nATOM    232  C4   DC B  11      28.472  31.744  14.372  1.00  8.47           C  \\nATOM    233  N4   DC B  11      27.746  31.854  13.292  1.00 24.16           N  \\nATOM    234  C5   DC B  11      28.866  32.886  15.096  1.00 19.21           C  \\nATOM    235  C6   DC B  11      29.541  32.630  16.236  1.00 19.62           C  \\nATOM    236  H41  DC B  11      27.427  32.762  12.970  1.00  0.00           H  \\nATOM    237  H42  DC B  11      27.498  31.025  12.770  1.00  0.00           H  \\nATOM    238  P    DC B  12      34.436  31.147  20.578  1.00 35.23           P  \\nATOM    239  OP1  DC B  12      34.764  29.859  21.271  1.00 39.08           O  \\nATOM    240  OP2  DC B  12      34.907  32.477  21.122  1.00 36.25           O  \\nATOM    241  O5'  DC B  12      35.015  30.931  19.062  1.00 30.38           O  \\nATOM    242  C5'  DC B  12      36.391  30.665  18.739  1.00 30.68           C  \\nATOM    243  C4'  DC B  12      36.518  29.663  17.628  1.00 28.20           C  \\nATOM    244  O4'  DC B  12      35.889  30.201  16.528  1.00 29.99           O  \\nATOM    245  C3'  DC B  12      37.900  29.433  17.125  1.00 23.57           C  \\nATOM    246  O3'  DC B  12      38.418  28.232  17.727  1.00 39.77           O  \\nATOM    247  C2'  DC B  12      37.691  29.249  15.673  1.00 21.13           C  \\nATOM    248  C1'  DC B  12      36.254  29.370  15.451  1.00 17.03           C  \\nATOM    249  N1   DC B  12      35.819  30.088  14.254  1.00  9.75           N  \\nATOM    250  C2   DC B  12      35.372  29.340  13.243  1.00 20.35           C  \\nATOM    251  O2   DC B  12      35.274  28.103  13.297  1.00 21.58           O  \\nATOM    252  N3   DC B  12      34.928  30.008  12.187  1.00 14.37           N  \\nATOM    253  C4   DC B  12      34.893  31.316  12.097  1.00 11.85           C  \\nATOM    254  N4   DC B  12      34.397  31.868  10.987  1.00 17.73           N  \\nATOM    255  C5   DC B  12      35.351  32.092  13.152  1.00 12.20           C  \\nATOM    256  C6   DC B  12      35.811  31.425  14.209  1.00 15.76           C  \\nATOM    257 HO3'  DC B  12      38.451  28.385  18.677  1.00  0.00           H  \\nATOM    258  H41  DC B  12      34.400  32.875  10.890  1.00  0.00           H  \\nATOM    259  H42  DC B  12      34.013  31.291  10.256  1.00  0.00           H  \\nTER     260       DC B  12                                                      \\nHETATM  261  O   HOH A  13      28.213  22.100  27.530  1.00 42.22           O  \\nHETATM  262  O   HOH A  14      26.586  21.386   7.132  1.00 44.58           O  \\nHETATM  263  O   HOH A  15      21.619  32.763  24.469  1.00 36.66           O  \\nHETATM  264  O   HOH A  16      28.321  21.893  10.903  1.00 34.10           O  \\nHETATM  265  O   HOH A  20      22.484  26.473  28.343  1.00 38.32           O  \\nHETATM  266  O   HOH A  21      19.359  34.276  23.539  1.00 31.79           O  \\nHETATM  267  O   HOH A  22      20.296  23.598   9.052  1.00 39.71           O  \\nHETATM  268  O   HOH A  24      28.013  24.110  25.005  1.00 29.37           O  \\nHETATM  269  O   HOH A  30      29.595  24.355  16.471  1.00 35.57           O  \\nHETATM  270  O   HOH A  31      34.129  23.669  11.033  1.00 30.44           O  \\nHETATM  271  O   HOH A  32      26.296  25.396  21.262  1.00 22.48           O  \\nHETATM  272  O   HOH A  33      29.926  21.907   4.334  1.00 43.21           O  \\nHETATM  273  O   HOH A  34      25.342  21.361  22.578  1.00 27.71           O  \\nHETATM  274  O   HOH B  17      21.384  39.724  15.164  1.00 37.35           O  \\nHETATM  275  O   HOH B  18      15.873  22.366  19.329  1.00 48.06           O  \\nHETATM  276  O   HOH B  19      18.356  32.435  15.970  1.00 25.93           O  \\nHETATM  277  O   HOH B  23      41.100  27.441  15.831  1.00 27.63           O  \\nHETATM  278  O   HOH B  25      22.484  34.287  11.145  1.00 40.90           O  \\nHETATM  279  O   HOH B  26      12.592  34.741  18.612  1.00 23.80           O  \\nHETATM  280  O   HOH B  27      11.090  39.558  17.897  1.00 41.31           O  \\nHETATM  281  O   HOH B  28      13.559  32.507  19.884  1.00 29.78           O  \\nHETATM  282  O   HOH B  29      34.412  35.088  10.781  1.00 21.68           O  \\nHETATM  283  O   HOH B  35      19.923  40.404  18.291  1.00 31.79           O  \\nHETATM  284  O   HOH B  36      18.633  37.421  10.594  1.00 28.62           O  \\nHETATM  285  O   HOH B  37      41.530  30.514  14.417  1.00 36.59           O  \\nHETATM  286  O   HOH B  38      26.648  34.690  11.974  1.00 32.48           O  \\nMASTER      362    0    0    0    0    0    0    6  284    2    0    2          \\nEND                                                                             \\n\");\n\tviewer_1684696742114819.setStyle({\"model\": -1, \"chain\": \"A\"},{\"cartoon\": {\"color\": \"spectrum\"}});\n\tviewer_1684696742114819.zoomTo();\nviewer_1684696742114819.render();\n});\n</script>",
      "text/html": [
       "<div id=\"3dmolviewer_1684696742114819\"  style=\"position: relative; width: 600px; height: 400px\">\n",
       "        <p id=\"3dmolwarning_1684696742114819\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
       "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
       "        </div>\n",
       "<script>\n",
       "\n",
       "var loadScriptAsync = function(uri){\n",
       "  return new Promise((resolve, reject) => {\n",
       "    //this is to ignore the existence of requirejs amd\n",
       "    var savedexports, savedmodule;\n",
       "    if (typeof exports !== 'undefined') savedexports = exports;\n",
       "    else exports = {}\n",
       "    if (typeof module !== 'undefined') savedmodule = module;\n",
       "    else module = {}\n",
       "\n",
       "    var tag = document.createElement('script');\n",
       "    tag.src = uri;\n",
       "    tag.async = true;\n",
       "    tag.onload = () => {\n",
       "        exports = savedexports;\n",
       "        module = savedmodule;\n",
       "        resolve();\n",
       "    };\n",
       "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
       "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
       "});\n",
       "};\n",
       "\n",
       "if(typeof $3Dmolpromise === 'undefined') {\n",
       "$3Dmolpromise = null;\n",
       "  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.0.1/3Dmol-min.js');\n",
       "}\n",
       "\n",
       "var viewer_1684696742114819 = null;\n",
       "var warn = document.getElementById(\"3dmolwarning_1684696742114819\");\n",
       "if(warn) {\n",
       "    warn.parentNode.removeChild(warn);\n",
       "}\n",
       "$3Dmolpromise.then(function() {\n",
       "viewer_1684696742114819 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_1684696742114819\"),{backgroundColor:\"white\"});\n",
       "viewer_1684696742114819.zoomTo();\n",
       "\tviewer_1684696742114819.addModelsAsFrames(\"HEADER    DNA                                     16-FEB-95   200D              \\nTITLE     STABLE LOOP IN THE CRYSTAL STRUCTURE OF THE INTERCALATED              \\nTITLE    2 FOUR-STRANDED CYTOSINE-RICH METAZOAN TELOMERE                        \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: DNA (5'-D(*TP*AP*AP*CP*CP*C)-3');                          \\nCOMPND   3 CHAIN: A, B;                                                         \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 SYNTHETIC: YES                                                       \\nKEYWDS    U-DNA, QUADRUPLE HELIX, TETRAPLEX, BASE INTERCALATED, LOOP            \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    C.KANG,I.BERGER,C.LOCKSHIN,R.RATLIFF,R.MOYZIS,A.RICH                  \\nREVDAT   2   24-FEB-09 200D    1       VERSN                                    \\nREVDAT   1   31-JUL-95 200D    0                                                \\nJRNL        AUTH   C.KANG,I.BERGER,C.LOCKSHIN,R.RATLIFF,R.MOYZIS,               \\nJRNL        AUTH 2 A.RICH                                                       \\nJRNL        TITL   STABLE LOOP IN THE CRYSTAL STRUCTURE OF THE                  \\nJRNL        TITL 2 INTERCALATED FOUR-STRANDED CYTOSINE-RICH METAZOAN            \\nJRNL        TITL 3 TELOMERE.                                                    \\nJRNL        REF    PROC.NATL.ACAD.SCI.USA        V.  92  3874 1995              \\nJRNL        REFN                   ISSN 0027-8424                               \\nJRNL        PMID   7731999                                                      \\nJRNL        DOI    10.1073/PNAS.92.9.3874                                       \\nREMARK   1                                                                      \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.85 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : X-PLOR                                               \\nREMARK   3   AUTHORS     : BRUNGER                                              \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.85                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 12.00                          \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : 1.000                          \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : 77.0                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 2122                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : 0.190                           \\nREMARK   3   FREE R VALUE                     : 0.223                           \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 0                                       \\nREMARK   3   NUCLEIC ACID ATOMS       : 258                                     \\nREMARK   3   HETEROGEN ATOMS          : 0                                       \\nREMARK   3   SOLVENT ATOMS            : 26                                      \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : 0.030                           \\nREMARK   3   BOND ANGLES            (DEGREES) : 4.00                            \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 200D COMPLIES WITH FORMAT V. 3.15, 01-DEC-08                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : 20-JUL-94                          \\nREMARK 200  TEMPERATURE           (KELVIN) : 277.00                             \\nREMARK 200  PH                             : 6.00                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : N                                  \\nREMARK 200  RADIATION SOURCE               : ROTATING ANODE                     \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : M                                  \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : IMAGE PLATE                        \\nREMARK 200  DETECTOR MANUFACTURER          : RIGAKU RAXIS IIC                   \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : MSC                                \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : 0.04500                            \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): NULL                                      \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): NULL                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: PH 6.00, VAPOR DIFFUSION                 \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: F 2 2 2                          \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -X,-Y,Z                                                 \\nREMARK 290       3555   -X,Y,-Z                                                 \\nREMARK 290       4555   X,-Y,-Z                                                 \\nREMARK 290       5555   X,Y+1/2,Z+1/2                                           \\nREMARK 290       6555   -X,-Y+1/2,Z+1/2                                         \\nREMARK 290       7555   -X,Y+1/2,-Z+1/2                                         \\nREMARK 290       8555   X,-Y+1/2,-Z+1/2                                         \\nREMARK 290       9555   X+1/2,Y,Z+1/2                                           \\nREMARK 290      10555   -X+1/2,-Y,Z+1/2                                         \\nREMARK 290      11555   -X+1/2,Y,-Z+1/2                                         \\nREMARK 290      12555   X+1/2,-Y,-Z+1/2                                         \\nREMARK 290      13555   X+1/2,Y+1/2,Z                                           \\nREMARK 290      14555   -X+1/2,-Y+1/2,Z                                         \\nREMARK 290      15555   -X+1/2,Y+1/2,-Z                                         \\nREMARK 290      16555   X+1/2,-Y+1/2,-Z                                         \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   3 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   3  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   4  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   4  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   5  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   5  0.000000  1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3   5  0.000000  0.000000  1.000000       13.43000            \\nREMARK 290   SMTRY1   6 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   6  0.000000 -1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3   6  0.000000  0.000000  1.000000       13.43000            \\nREMARK 290   SMTRY1   7 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   7  0.000000  1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3   7  0.000000  0.000000 -1.000000       13.43000            \\nREMARK 290   SMTRY1   8  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   8  0.000000 -1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3   8  0.000000  0.000000 -1.000000       13.43000            \\nREMARK 290   SMTRY1   9  1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2   9  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   9  0.000000  0.000000  1.000000       13.43000            \\nREMARK 290   SMTRY1  10 -1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  10  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3  10  0.000000  0.000000  1.000000       13.43000            \\nREMARK 290   SMTRY1  11 -1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  11  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3  11  0.000000  0.000000 -1.000000       13.43000            \\nREMARK 290   SMTRY1  12  1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  12  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3  12  0.000000  0.000000 -1.000000       13.43000            \\nREMARK 290   SMTRY1  13  1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  13  0.000000  1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3  13  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1  14 -1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  14  0.000000 -1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3  14  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1  15 -1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  15  0.000000  1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3  15  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1  16  1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  16  0.000000 -1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3  16  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: TETRAMERIC                        \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A, B                                  \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2 -1.000000  0.000000  0.000000       59.94000            \\nREMARK 350   BIOMT2   2  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000 -1.000000       26.86000            \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND LENGTHS                                      \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,2(A3,1X,A1,I4,A1,1X,A4,3X),1X,F6.3)               \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   RES CSSEQI ATM2   DEVIATION                     \\nREMARK 500     DT A   1   C5     DT A   1   C7      0.038                       \\nREMARK 500     DT A   1   O3'    DA A   2   P      -0.084                       \\nREMARK 500     DA A   3   C4'    DA A   3   C3'    -0.094                       \\nREMARK 500     DA A   3   N9     DA A   3   C4     -0.042                       \\nREMARK 500     DC A   4   N3     DC A   4   C4     -0.047                       \\nREMARK 500     DC A   5   C3'    DC A   5   C2'    -0.055                       \\nREMARK 500     DC A   6   C5'    DC A   6   C4'     0.090                       \\nREMARK 500     DC A   6   C3'    DC A   6   C2'    -0.050                       \\nREMARK 500     DC A   6   N3     DC A   6   C4     -0.052                       \\nREMARK 500     DT B   7   C5'    DT B   7   C4'     0.082                       \\nREMARK 500     DT B   7   C2     DT B   7   N3     -0.049                       \\nREMARK 500     DA B   8   P      DA B   8   O5'     0.063                       \\nREMARK 500     DC B  10   C4'    DC B  10   C3'    -0.083                       \\nREMARK 500     DC B  10   N1     DC B  10   C6     -0.063                       \\nREMARK 500     DC B  10   C4     DC B  10   C5     -0.050                       \\nREMARK 500     DC B  11   N1     DC B  11   C2     -0.094                       \\nREMARK 500     DC B  11   N1     DC B  11   C6     -0.040                       \\nREMARK 500     DC B  11   C2     DC B  11   N3     -0.056                       \\nREMARK 500     DC B  12   O4'    DC B  12   C4'    -0.069                       \\nREMARK 500     DC B  12   N1     DC B  12   C2     -0.062                       \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \\nREMARK 500     DT A   1   O4' -  C1' -  N1  ANGL. DEV. =   2.9 DEGREES          \\nREMARK 500     DT A   1   C6  -  C5  -  C7  ANGL. DEV. =  -4.4 DEGREES          \\nREMARK 500     DA A   2   O4' -  C4' -  C3' ANGL. DEV. =   4.3 DEGREES          \\nREMARK 500     DA A   2   C4' -  C3' -  C2' ANGL. DEV. =  -9.1 DEGREES          \\nREMARK 500     DA A   2   O4' -  C1' -  C2' ANGL. DEV. =  -7.8 DEGREES          \\nREMARK 500     DA A   2   N1  -  C6  -  N6  ANGL. DEV. =   3.7 DEGREES          \\nREMARK 500     DT A   1   C3' -  O3' -  P   ANGL. DEV. =  14.8 DEGREES          \\nREMARK 500     DA A   2   O3' -  P   -  O5' ANGL. DEV. = -11.7 DEGREES          \\nREMARK 500     DA A   3   O4' -  C4' -  C3' ANGL. DEV. =  -7.7 DEGREES          \\nREMARK 500     DA A   3   C1' -  O4' -  C4' ANGL. DEV. =   6.1 DEGREES          \\nREMARK 500     DA A   3   O4' -  C1' -  C2' ANGL. DEV. =  -6.8 DEGREES          \\nREMARK 500     DA A   3   N1  -  C2  -  N3  ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500     DA A   3   C2  -  N3  -  C4  ANGL. DEV. =  -3.9 DEGREES          \\nREMARK 500     DA A   2   C3' -  O3' -  P   ANGL. DEV. =   7.9 DEGREES          \\nREMARK 500     DC A   4   C4' -  C3' -  C2' ANGL. DEV. =  -4.3 DEGREES          \\nREMARK 500     DC A   5   C4' -  C3' -  C2' ANGL. DEV. =  -7.7 DEGREES          \\nREMARK 500     DC A   5   O4' -  C1' -  N1  ANGL. DEV. =  -4.7 DEGREES          \\nREMARK 500     DC A   5   C2  -  N3  -  C4  ANGL. DEV. =   3.5 DEGREES          \\nREMARK 500     DC A   6   P   -  O5' -  C5' ANGL. DEV. =  13.2 DEGREES          \\nREMARK 500     DC A   6   O4' -  C4' -  C3' ANGL. DEV. =  -4.4 DEGREES          \\nREMARK 500     DC A   6   O4' -  C1' -  N1  ANGL. DEV. =   3.9 DEGREES          \\nREMARK 500     DC A   6   N1  -  C2  -  O2  ANGL. DEV. =   3.9 DEGREES          \\nREMARK 500     DC A   6   N3  -  C4  -  N4  ANGL. DEV. =  -5.0 DEGREES          \\nREMARK 500     DC A   6   C5  -  C4  -  N4  ANGL. DEV. =   4.2 DEGREES          \\nREMARK 500     DT B   7   C5' -  C4' -  O4' ANGL. DEV. =   9.2 DEGREES          \\nREMARK 500     DT B   7   N1  -  C2  -  O2  ANGL. DEV. =   5.3 DEGREES          \\nREMARK 500     DT B   7   N3  -  C2  -  O2  ANGL. DEV. =  -4.3 DEGREES          \\nREMARK 500     DA B   8   O4' -  C1' -  C2' ANGL. DEV. =  -5.1 DEGREES          \\nREMARK 500     DA B   8   C2  -  N3  -  C4  ANGL. DEV. =  -3.3 DEGREES          \\nREMARK 500     DA B   8   C5  -  C6  -  N1  ANGL. DEV. =  -4.0 DEGREES          \\nREMARK 500     DT B   7   C3' -  O3' -  P   ANGL. DEV. = -10.9 DEGREES          \\nREMARK 500     DA B   9   O4' -  C1' -  C2' ANGL. DEV. =  -7.3 DEGREES          \\nREMARK 500     DC B  10   C1' -  O4' -  C4' ANGL. DEV. =   6.5 DEGREES          \\nREMARK 500     DC B  10   O4' -  C1' -  C2' ANGL. DEV. = -10.3 DEGREES          \\nREMARK 500     DC B  10   O4' -  C1' -  N1  ANGL. DEV. =   3.6 DEGREES          \\nREMARK 500     DC B  10   N3  -  C2  -  O2  ANGL. DEV. =  -5.1 DEGREES          \\nREMARK 500     DC B  10   C3' -  O3' -  P   ANGL. DEV. =   9.2 DEGREES          \\nREMARK 500     DC B  12   O4' -  C1' -  C2' ANGL. DEV. =  -5.1 DEGREES          \\nREMARK 500     DC B  12   C2  -  N3  -  C4  ANGL. DEV. =   4.6 DEGREES          \\nREMARK 500     DC B  12   N1  -  C2  -  O2  ANGL. DEV. =   4.6 DEGREES          \\nREMARK 500     DC B  11   C3' -  O3' -  P   ANGL. DEV. =   8.3 DEGREES          \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: PLANAR GROUPS                                              \\nREMARK 500                                                                      \\nREMARK 500 PLANAR GROUPS IN THE FOLLOWING RESIDUES HAVE A TOTAL                 \\nREMARK 500 RMS DISTANCE OF ALL ATOMS FROM THE BEST-FIT PLANE                    \\nREMARK 500 BY MORE THAN AN EXPECTED VALUE OF 6*RMSD, WITH AN                    \\nREMARK 500 RMSD 0.02 ANGSTROMS, OR AT LEAST ONE ATOM HAS                        \\nREMARK 500 AN RMSD GREATER THAN THIS VALUE                                      \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        RMS     TYPE                                    \\nREMARK 500     DA A   3         0.06    SIDE_CHAIN                              \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nDBREF  200D A    1     6  PDB    200D     200D             1      6             \\nDBREF  200D B    7    12  PDB    200D     200D             7     12             \\nSEQRES   1 A    6   DT  DA  DA  DC  DC  DC                                      \\nSEQRES   1 B    6   DT  DA  DA  DC  DC  DC                                      \\nFORMUL   3  HOH   *26(H2 O)                                                     \\nCRYST1   59.940   81.330   26.860  90.00  90.00  90.00 F 2 2 2      32          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.016683  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  0.012296  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.037230        0.00000                         \\nATOM      1  O5'  DT A   1      24.565  31.319  24.443  1.00 30.44           O  \\nATOM      2  C5'  DT A   1      23.942  30.467  25.330  1.00 20.89           C  \\nATOM      3  C4'  DT A   1      23.216  29.306  24.650  1.00 27.77           C  \\nATOM      4  O4'  DT A   1      22.175  29.736  23.812  1.00 23.30           O  \\nATOM      5  C3'  DT A   1      24.157  28.454  23.852  1.00 35.38           C  \\nATOM      6  O3'  DT A   1      23.905  27.189  24.366  1.00 33.40           O  \\nATOM      7  C2'  DT A   1      23.531  28.590  22.502  1.00 34.88           C  \\nATOM      8  C1'  DT A   1      22.104  28.825  22.807  1.00 18.33           C  \\nATOM      9  N1   DT A   1      21.415  29.371  21.650  1.00 27.22           N  \\nATOM     10  C2   DT A   1      21.061  28.462  20.735  1.00 17.62           C  \\nATOM     11  O2   DT A   1      21.294  27.274  20.916  1.00 23.45           O  \\nATOM     12  N3   DT A   1      20.409  28.945  19.635  1.00 20.47           N  \\nATOM     13  C4   DT A   1      20.071  30.242  19.367  1.00 19.02           C  \\nATOM     14  O4   DT A   1      19.434  30.520  18.346  1.00 27.67           O  \\nATOM     15  C5   DT A   1      20.491  31.148  20.397  1.00 28.05           C  \\nATOM     16  C7   DT A   1      20.220  32.654  20.284  1.00 24.86           C  \\nATOM     17  C6   DT A   1      21.136  30.694  21.482  1.00 14.22           C  \\nATOM     18  H3   DT A   1      20.149  28.273  18.939  1.00  0.00           H  \\nATOM     19 HO5'  DT A   1      25.263  30.835  24.004  1.00  0.00           H  \\nATOM     20  P    DA A   2      24.766  25.974  24.683  1.00 23.53           P  \\nATOM     21  OP1  DA A   2      24.819  25.775  26.131  1.00 27.65           O  \\nATOM     22  OP2  DA A   2      25.964  25.990  23.827  1.00 38.21           O  \\nATOM     23  O5'  DA A   2      23.749  24.991  23.979  1.00 23.31           O  \\nATOM     24  C5'  DA A   2      22.350  25.133  24.144  1.00 31.45           C  \\nATOM     25  C4'  DA A   2      21.502  24.042  23.575  1.00 10.83           C  \\nATOM     26  O4'  DA A   2      20.221  24.252  24.204  1.00 36.32           O  \\nATOM     27  C3'  DA A   2      21.353  24.038  22.059  1.00 39.61           C  \\nATOM     28  O3'  DA A   2      22.218  23.042  21.474  1.00 19.31           O  \\nATOM     29  C2'  DA A   2      19.926  23.509  22.118  1.00 23.91           C  \\nATOM     30  C1'  DA A   2      19.171  24.220  23.257  1.00 29.47           C  \\nATOM     31  N9   DA A   2      18.839  25.627  23.010  1.00 19.13           N  \\nATOM     32  C8   DA A   2      19.151  26.734  23.759  1.00 21.05           C  \\nATOM     33  N7   DA A   2      18.737  27.848  23.267  1.00 20.27           N  \\nATOM     34  C5   DA A   2      18.093  27.426  22.100  1.00 14.04           C  \\nATOM     35  C6   DA A   2      17.391  28.116  21.124  1.00 18.64           C  \\nATOM     36  N6   DA A   2      17.250  29.453  21.149  1.00 24.64           N  \\nATOM     37  N1   DA A   2      16.861  27.359  20.157  1.00 15.56           N  \\nATOM     38  C2   DA A   2      17.016  26.046  20.152  1.00  9.93           C  \\nATOM     39  N3   DA A   2      17.641  25.306  20.992  1.00 23.82           N  \\nATOM     40  C4   DA A   2      18.160  26.078  21.948  1.00 11.68           C  \\nATOM     41  H61  DA A   2      16.745  29.923  20.410  1.00  0.00           H  \\nATOM     42  H62  DA A   2      17.644  29.994  21.903  1.00  0.00           H  \\nATOM     43  P    DA A   3      23.030  23.149  20.153  1.00 20.46           P  \\nATOM     44  OP1  DA A   3      23.954  22.013  20.095  1.00 24.95           O  \\nATOM     45  OP2  DA A   3      23.531  24.509  20.030  1.00 19.53           O  \\nATOM     46  O5'  DA A   3      21.880  22.885  19.163  1.00 17.18           O  \\nATOM     47  C5'  DA A   3      21.278  21.604  19.070  1.00 13.90           C  \\nATOM     48  C4'  DA A   3      20.159  21.601  18.065  1.00 21.24           C  \\nATOM     49  O4'  DA A   3      19.337  22.719  18.308  1.00 21.96           O  \\nATOM     50  C3'  DA A   3      20.581  21.983  16.756  1.00 22.35           C  \\nATOM     51  O3'  DA A   3      21.091  20.871  16.039  1.00 35.11           O  \\nATOM     52  C2'  DA A   3      19.323  22.493  16.117  1.00 36.04           C  \\nATOM     53  C1'  DA A   3      18.685  23.288  17.172  1.00 20.51           C  \\nATOM     54  N9   DA A   3      18.982  24.750  17.163  1.00 14.73           N  \\nATOM     55  C8   DA A   3      19.499  25.507  18.166  1.00  8.55           C  \\nATOM     56  N7   DA A   3      19.478  26.769  17.943  1.00 16.60           N  \\nATOM     57  C5   DA A   3      18.913  26.871  16.693  1.00 12.55           C  \\nATOM     58  C6   DA A   3      18.601  27.989  15.881  1.00 20.70           C  \\nATOM     59  N6   DA A   3      18.741  29.276  16.217  1.00 14.67           N  \\nATOM     60  N1   DA A   3      18.060  27.707  14.701  1.00 13.89           N  \\nATOM     61  C2   DA A   3      17.850  26.408  14.389  1.00 29.53           C  \\nATOM     62  N3   DA A   3      18.083  25.275  15.052  1.00 18.60           N  \\nATOM     63  C4   DA A   3      18.625  25.618  16.218  1.00  8.22           C  \\nATOM     64  H61  DA A   3      18.524  30.004  15.551  1.00  0.00           H  \\nATOM     65  H62  DA A   3      19.059  29.517  17.142  1.00  0.00           H  \\nATOM     66  P    DC A   4      22.425  21.041  15.096  1.00 24.23           P  \\nATOM     67  OP1  DC A   4      22.465  19.788  14.318  1.00 29.54           O  \\nATOM     68  OP2  DC A   4      23.569  21.448  15.939  1.00 26.74           O  \\nATOM     69  O5'  DC A   4      22.106  22.295  14.117  1.00 24.38           O  \\nATOM     70  C5'  DC A   4      21.326  22.363  12.942  1.00 23.31           C  \\nATOM     71  C4'  DC A   4      21.432  23.763  12.409  1.00 19.02           C  \\nATOM     72  O4'  DC A   4      20.924  24.622  13.375  1.00 13.89           O  \\nATOM     73  C3'  DC A   4      22.882  24.127  12.138  1.00 15.94           C  \\nATOM     74  O3'  DC A   4      23.153  23.807  10.751  1.00 33.78           O  \\nATOM     75  C2'  DC A   4      22.777  25.612  12.409  1.00 26.18           C  \\nATOM     76  C1'  DC A   4      21.638  25.802  13.441  1.00  6.99           C  \\nATOM     77  N1   DC A   4      22.071  25.996  14.858  1.00 15.40           N  \\nATOM     78  C2   DC A   4      22.052  27.275  15.345  1.00 14.75           C  \\nATOM     79  O2   DC A   4      21.597  28.208  14.679  1.00 20.98           O  \\nATOM     80  N3   DC A   4      22.492  27.460  16.606  1.00 19.60           N  \\nATOM     81  C4   DC A   4      22.910  26.491  17.345  1.00  7.46           C  \\nATOM     82  N4   DC A   4      23.363  26.803  18.516  1.00 11.08           N  \\nATOM     83  C5   DC A   4      22.926  25.188  16.895  1.00  7.21           C  \\nATOM     84  C6   DC A   4      22.504  24.981  15.653  1.00  9.76           C  \\nATOM     85  H41  DC A   4      23.402  27.768  18.806  1.00  0.00           H  \\nATOM     86  H42  DC A   4      23.675  26.076  19.149  1.00  0.00           H  \\nATOM     87  P    DC A   5      24.446  22.970  10.188  1.00 28.52           P  \\nATOM     88  OP1  DC A   5      24.144  22.120   9.020  1.00 24.29           O  \\nATOM     89  OP2  DC A   5      25.053  22.349  11.400  1.00 34.89           O  \\nATOM     90  O5'  DC A   5      25.373  24.138   9.643  1.00 30.47           O  \\nATOM     91  C5'  DC A   5      24.837  25.048   8.687  1.00 27.30           C  \\nATOM     92  C4'  DC A   5      25.648  26.351   8.642  1.00 20.98           C  \\nATOM     93  O4'  DC A   5      25.627  26.952   9.905  1.00 21.19           O  \\nATOM     94  C3'  DC A   5      27.100  26.119   8.375  1.00 20.71           C  \\nATOM     95  O3'  DC A   5      27.341  25.805   7.002  1.00 23.11           O  \\nATOM     96  C2'  DC A   5      27.496  27.486   8.703  1.00 13.44           C  \\nATOM     97  C1'  DC A   5      26.696  27.830   9.919  1.00 13.82           C  \\nATOM     98  N1   DC A   5      27.308  27.595  11.235  1.00 22.24           N  \\nATOM     99  C2   DC A   5      27.581  28.683  11.989  1.00  9.58           C  \\nATOM    100  O2   DC A   5      27.383  29.828  11.573  1.00 25.04           O  \\nATOM    101  N3   DC A   5      28.050  28.457  13.220  1.00 23.52           N  \\nATOM    102  C4   DC A   5      28.242  27.259  13.710  1.00  9.77           C  \\nATOM    103  N4   DC A   5      28.680  27.143  14.947  1.00 20.94           N  \\nATOM    104  C5   DC A   5      27.978  26.108  12.961  1.00 15.31           C  \\nATOM    105  C6   DC A   5      27.515  26.334  11.726  1.00 20.31           C  \\nATOM    106  H41  DC A   5      28.833  26.230  15.355  1.00  0.00           H  \\nATOM    107  H42  DC A   5      28.860  27.971  15.496  1.00  0.00           H  \\nATOM    108  P    DC A   6      28.699  25.257   6.381  1.00 26.27           P  \\nATOM    109  OP1  DC A   6      28.416  24.653   5.094  1.00 29.77           O  \\nATOM    110  OP2  DC A   6      29.457  24.544   7.410  1.00 23.78           O  \\nATOM    111  O5'  DC A   6      29.520  26.539   6.046  1.00 32.49           O  \\nATOM    112  C5'  DC A   6      29.282  27.692   5.226  1.00 30.24           C  \\nATOM    113  C4'  DC A   6      30.531  28.674   5.429  1.00 22.63           C  \\nATOM    114  O4'  DC A   6      30.781  28.988   6.818  1.00 18.93           O  \\nATOM    115  C3'  DC A   6      31.823  27.970   5.079  1.00 28.36           C  \\nATOM    116  O3'  DC A   6      32.102  28.065   3.709  1.00 27.07           O  \\nATOM    117  C2'  DC A   6      32.812  28.770   5.808  1.00 13.00           C  \\nATOM    118  C1'  DC A   6      32.139  29.157   7.068  1.00  9.92           C  \\nATOM    119  N1   DC A   6      32.607  28.338   8.228  1.00 22.07           N  \\nATOM    120  C2   DC A   6      33.155  29.092   9.255  1.00 26.49           C  \\nATOM    121  O2   DC A   6      33.154  30.333   9.252  1.00 26.06           O  \\nATOM    122  N3   DC A   6      33.673  28.433  10.285  1.00 17.53           N  \\nATOM    123  C4   DC A   6      33.677  27.152  10.362  1.00  7.84           C  \\nATOM    124  N4   DC A   6      34.255  26.711  11.432  1.00 17.65           N  \\nATOM    125  C5   DC A   6      33.130  26.339   9.369  1.00  9.69           C  \\nATOM    126  C6   DC A   6      32.596  26.974   8.310  1.00 25.15           C  \\nATOM    127 HO3'  DC A   6      31.406  27.590   3.246  1.00  0.00           H  \\nATOM    128  H41  DC A   6      34.358  25.716  11.590  1.00  0.00           H  \\nATOM    129  H42  DC A   6      34.605  27.355  12.121  1.00  0.00           H  \\nTER     130       DC A   6                                                      \\nATOM    131  O5'  DT B   7      11.293  29.851  15.858  1.00 28.85           O  \\nATOM    132  C5'  DT B   7      11.549  31.182  16.395  1.00 25.35           C  \\nATOM    133  C4'  DT B   7      12.980  31.877  16.290  1.00 39.66           C  \\nATOM    134  O4'  DT B   7      14.126  31.292  17.044  1.00 37.54           O  \\nATOM    135  C3'  DT B   7      13.456  31.813  14.854  1.00 39.01           C  \\nATOM    136  O3'  DT B   7      14.165  32.997  14.544  1.00 38.98           O  \\nATOM    137  C2'  DT B   7      14.424  30.692  14.753  1.00 20.58           C  \\nATOM    138  C1'  DT B   7      14.994  30.617  16.125  1.00 20.74           C  \\nATOM    139  N1   DT B   7      15.053  29.255  16.540  1.00 18.62           N  \\nATOM    140  C2   DT B   7      15.671  29.004  17.731  1.00 20.33           C  \\nATOM    141  O2   DT B   7      16.113  29.817  18.554  1.00 26.72           O  \\nATOM    142  N3   DT B   7      15.786  27.715  18.009  1.00 22.63           N  \\nATOM    143  C4   DT B   7      15.361  26.621  17.272  1.00 33.64           C  \\nATOM    144  O4   DT B   7      15.606  25.447  17.626  1.00 34.71           O  \\nATOM    145  C5   DT B   7      14.708  26.988  16.073  1.00 14.16           C  \\nATOM    146  C7   DT B   7      14.102  25.888  15.216  1.00 39.75           C  \\nATOM    147  C6   DT B   7      14.575  28.259  15.732  1.00 24.49           C  \\nATOM    148  H3   DT B   7      16.244  27.571  18.880  1.00  0.00           H  \\nATOM    149 HO5'  DT B   7      11.404  29.913  14.905  1.00  0.00           H  \\nATOM    150  P    DA B   8      13.931  33.345  13.024  1.00 39.22           P  \\nATOM    151  OP1  DA B   8      12.612  34.036  13.039  1.00 34.77           O  \\nATOM    152  OP2  DA B   8      14.206  32.074  12.263  1.00 39.31           O  \\nATOM    153  O5'  DA B   8      15.000  34.463  12.432  1.00 39.80           O  \\nATOM    154  C5'  DA B   8      16.303  34.113  11.867  1.00 39.02           C  \\nATOM    155  C4'  DA B   8      17.406  35.058  12.322  1.00 17.13           C  \\nATOM    156  O4'  DA B   8      16.819  36.368  12.502  1.00 22.28           O  \\nATOM    157  C3'  DA B   8      18.055  34.660  13.684  1.00 35.33           C  \\nATOM    158  O3'  DA B   8      19.437  35.008  13.398  1.00 39.28           O  \\nATOM    159  C2'  DA B   8      17.310  35.602  14.642  1.00 35.21           C  \\nATOM    160  C1'  DA B   8      17.143  36.915  13.808  1.00 14.63           C  \\nATOM    161  N9   DA B   8      16.003  37.727  14.228  1.00 20.81           N  \\nATOM    162  C8   DA B   8      14.676  37.344  14.239  1.00 37.43           C  \\nATOM    163  N7   DA B   8      13.839  38.247  14.635  1.00 23.05           N  \\nATOM    164  C5   DA B   8      14.647  39.318  14.912  1.00 19.42           C  \\nATOM    165  C6   DA B   8      14.326  40.578  15.398  1.00 14.47           C  \\nATOM    166  N6   DA B   8      13.065  41.064  15.553  1.00 20.96           N  \\nATOM    167  N1   DA B   8      15.373  41.351  15.597  1.00 33.53           N  \\nATOM    168  C2   DA B   8      16.622  40.929  15.326  1.00 27.49           C  \\nATOM    169  N3   DA B   8      17.042  39.778  14.845  1.00 27.07           N  \\nATOM    170  C4   DA B   8      15.968  39.007  14.666  1.00 10.79           C  \\nATOM    171  H61  DA B   8      12.269  40.498  15.302  1.00  0.00           H  \\nATOM    172  H62  DA B   8      12.942  41.998  15.918  1.00  0.00           H  \\nATOM    173  P    DA B   9      20.719  34.445  14.194  1.00 35.42           P  \\nATOM    174  OP1  DA B   9      21.959  35.022  13.557  1.00 39.82           O  \\nATOM    175  OP2  DA B   9      20.528  32.939  14.321  1.00 28.79           O  \\nATOM    176  O5'  DA B   9      20.556  35.250  15.553  1.00 39.94           O  \\nATOM    177  C5'  DA B   9      21.196  36.518  15.727  1.00 27.77           C  \\nATOM    178  C4'  DA B   9      20.828  37.045  17.056  1.00 16.24           C  \\nATOM    179  O4'  DA B   9      19.420  37.260  17.115  1.00 24.83           O  \\nATOM    180  C3'  DA B   9      21.131  36.015  18.071  1.00 22.96           C  \\nATOM    181  O3'  DA B   9      21.775  36.603  19.212  1.00 29.07           O  \\nATOM    182  C2'  DA B   9      19.740  35.550  18.407  1.00 26.00           C  \\nATOM    183  C1'  DA B   9      18.975  36.797  18.352  1.00 18.88           C  \\nATOM    184  N9   DA B   9      17.533  36.628  18.310  1.00 17.60           N  \\nATOM    185  C8   DA B   9      16.740  35.511  18.232  1.00 23.85           C  \\nATOM    186  N7   DA B   9      15.467  35.795  18.264  1.00 25.36           N  \\nATOM    187  C5   DA B   9      15.416  37.185  18.373  1.00 14.91           C  \\nATOM    188  C6   DA B   9      14.358  38.117  18.472  1.00 24.27           C  \\nATOM    189  N6   DA B   9      13.051  37.812  18.565  1.00 30.71           N  \\nATOM    190  N1   DA B   9      14.730  39.429  18.573  1.00 34.89           N  \\nATOM    191  C2   DA B   9      16.025  39.778  18.587  1.00 15.13           C  \\nATOM    192  N3   DA B   9      17.077  38.975  18.506  1.00 18.79           N  \\nATOM    193  C4   DA B   9      16.688  37.686  18.401  1.00 21.13           C  \\nATOM    194  H61  DA B   9      12.726  36.865  18.446  1.00  0.00           H  \\nATOM    195  H62  DA B   9      12.408  38.552  18.781  1.00  0.00           H  \\nATOM    196  P    DC B  10      23.405  36.738  19.343  1.00 28.67           P  \\nATOM    197  OP1  DC B  10      23.679  37.914  20.242  1.00 34.27           O  \\nATOM    198  OP2  DC B  10      23.910  36.731  17.961  1.00 38.84           O  \\nATOM    199  O5'  DC B  10      24.087  35.454  20.011  1.00 18.77           O  \\nATOM    200  C5'  DC B  10      24.087  35.100  21.368  1.00 30.20           C  \\nATOM    201  C4'  DC B  10      24.575  33.695  21.529  1.00  9.80           C  \\nATOM    202  O4'  DC B  10      23.800  32.700  20.879  1.00 18.83           O  \\nATOM    203  C3'  DC B  10      25.878  33.620  20.926  1.00 14.61           C  \\nATOM    204  O3'  DC B  10      26.615  32.959  21.958  1.00 36.36           O  \\nATOM    205  C2'  DC B  10      25.606  32.814  19.678  1.00 19.97           C  \\nATOM    206  C1'  DC B  10      24.481  31.842  19.992  1.00 15.38           C  \\nATOM    207  N1   DC B  10      23.686  31.565  18.773  1.00 11.72           N  \\nATOM    208  C2   DC B  10      23.456  30.308  18.325  1.00 11.51           C  \\nATOM    209  O2   DC B  10      23.748  29.333  18.974  1.00 25.69           O  \\nATOM    210  N3   DC B  10      22.811  30.099  17.201  1.00 14.49           N  \\nATOM    211  C4   DC B  10      22.370  31.109  16.513  1.00 12.31           C  \\nATOM    212  N4   DC B  10      21.662  30.932  15.398  1.00 25.78           N  \\nATOM    213  C5   DC B  10      22.573  32.400  16.939  1.00 14.33           C  \\nATOM    214  C6   DC B  10      23.237  32.570  18.074  1.00 13.01           C  \\nATOM    215  H41  DC B  10      21.523  29.994  15.042  1.00  0.00           H  \\nATOM    216  H42  DC B  10      21.261  31.720  14.909  1.00  0.00           H  \\nATOM    217  P    DC B  11      28.130  33.237  22.467  1.00 34.54           P  \\nATOM    218  OP1  DC B  11      28.322  32.261  23.594  1.00 29.17           O  \\nATOM    219  OP2  DC B  11      28.366  34.724  22.567  1.00 26.05           O  \\nATOM    220  O5'  DC B  11      28.851  32.747  21.146  1.00 29.65           O  \\nATOM    221  C5'  DC B  11      30.240  32.966  20.860  1.00 31.52           C  \\nATOM    222  C4'  DC B  11      30.646  31.847  19.984  1.00 13.84           C  \\nATOM    223  O4'  DC B  11      29.762  31.772  18.892  1.00 25.81           O  \\nATOM    224  C3'  DC B  11      32.016  32.042  19.492  1.00 15.90           C  \\nATOM    225  O3'  DC B  11      32.776  31.224  20.405  1.00 35.61           O  \\nATOM    226  C2'  DC B  11      31.886  31.528  18.055  1.00 19.50           C  \\nATOM    227  C1'  DC B  11      30.453  31.059  17.897  1.00 14.84           C  \\nATOM    228  N1   DC B  11      29.801  31.387  16.621  1.00  8.89           N  \\nATOM    229  C2   DC B  11      29.420  30.373  15.896  1.00 10.51           C  \\nATOM    230  O2   DC B  11      29.691  29.265  16.247  1.00 16.37           O  \\nATOM    231  N3   DC B  11      28.767  30.545  14.789  1.00 14.37           N  \\nATOM    232  C4   DC B  11      28.472  31.744  14.372  1.00  8.47           C  \\nATOM    233  N4   DC B  11      27.746  31.854  13.292  1.00 24.16           N  \\nATOM    234  C5   DC B  11      28.866  32.886  15.096  1.00 19.21           C  \\nATOM    235  C6   DC B  11      29.541  32.630  16.236  1.00 19.62           C  \\nATOM    236  H41  DC B  11      27.427  32.762  12.970  1.00  0.00           H  \\nATOM    237  H42  DC B  11      27.498  31.025  12.770  1.00  0.00           H  \\nATOM    238  P    DC B  12      34.436  31.147  20.578  1.00 35.23           P  \\nATOM    239  OP1  DC B  12      34.764  29.859  21.271  1.00 39.08           O  \\nATOM    240  OP2  DC B  12      34.907  32.477  21.122  1.00 36.25           O  \\nATOM    241  O5'  DC B  12      35.015  30.931  19.062  1.00 30.38           O  \\nATOM    242  C5'  DC B  12      36.391  30.665  18.739  1.00 30.68           C  \\nATOM    243  C4'  DC B  12      36.518  29.663  17.628  1.00 28.20           C  \\nATOM    244  O4'  DC B  12      35.889  30.201  16.528  1.00 29.99           O  \\nATOM    245  C3'  DC B  12      37.900  29.433  17.125  1.00 23.57           C  \\nATOM    246  O3'  DC B  12      38.418  28.232  17.727  1.00 39.77           O  \\nATOM    247  C2'  DC B  12      37.691  29.249  15.673  1.00 21.13           C  \\nATOM    248  C1'  DC B  12      36.254  29.370  15.451  1.00 17.03           C  \\nATOM    249  N1   DC B  12      35.819  30.088  14.254  1.00  9.75           N  \\nATOM    250  C2   DC B  12      35.372  29.340  13.243  1.00 20.35           C  \\nATOM    251  O2   DC B  12      35.274  28.103  13.297  1.00 21.58           O  \\nATOM    252  N3   DC B  12      34.928  30.008  12.187  1.00 14.37           N  \\nATOM    253  C4   DC B  12      34.893  31.316  12.097  1.00 11.85           C  \\nATOM    254  N4   DC B  12      34.397  31.868  10.987  1.00 17.73           N  \\nATOM    255  C5   DC B  12      35.351  32.092  13.152  1.00 12.20           C  \\nATOM    256  C6   DC B  12      35.811  31.425  14.209  1.00 15.76           C  \\nATOM    257 HO3'  DC B  12      38.451  28.385  18.677  1.00  0.00           H  \\nATOM    258  H41  DC B  12      34.400  32.875  10.890  1.00  0.00           H  \\nATOM    259  H42  DC B  12      34.013  31.291  10.256  1.00  0.00           H  \\nTER     260       DC B  12                                                      \\nHETATM  261  O   HOH A  13      28.213  22.100  27.530  1.00 42.22           O  \\nHETATM  262  O   HOH A  14      26.586  21.386   7.132  1.00 44.58           O  \\nHETATM  263  O   HOH A  15      21.619  32.763  24.469  1.00 36.66           O  \\nHETATM  264  O   HOH A  16      28.321  21.893  10.903  1.00 34.10           O  \\nHETATM  265  O   HOH A  20      22.484  26.473  28.343  1.00 38.32           O  \\nHETATM  266  O   HOH A  21      19.359  34.276  23.539  1.00 31.79           O  \\nHETATM  267  O   HOH A  22      20.296  23.598   9.052  1.00 39.71           O  \\nHETATM  268  O   HOH A  24      28.013  24.110  25.005  1.00 29.37           O  \\nHETATM  269  O   HOH A  30      29.595  24.355  16.471  1.00 35.57           O  \\nHETATM  270  O   HOH A  31      34.129  23.669  11.033  1.00 30.44           O  \\nHETATM  271  O   HOH A  32      26.296  25.396  21.262  1.00 22.48           O  \\nHETATM  272  O   HOH A  33      29.926  21.907   4.334  1.00 43.21           O  \\nHETATM  273  O   HOH A  34      25.342  21.361  22.578  1.00 27.71           O  \\nHETATM  274  O   HOH B  17      21.384  39.724  15.164  1.00 37.35           O  \\nHETATM  275  O   HOH B  18      15.873  22.366  19.329  1.00 48.06           O  \\nHETATM  276  O   HOH B  19      18.356  32.435  15.970  1.00 25.93           O  \\nHETATM  277  O   HOH B  23      41.100  27.441  15.831  1.00 27.63           O  \\nHETATM  278  O   HOH B  25      22.484  34.287  11.145  1.00 40.90           O  \\nHETATM  279  O   HOH B  26      12.592  34.741  18.612  1.00 23.80           O  \\nHETATM  280  O   HOH B  27      11.090  39.558  17.897  1.00 41.31           O  \\nHETATM  281  O   HOH B  28      13.559  32.507  19.884  1.00 29.78           O  \\nHETATM  282  O   HOH B  29      34.412  35.088  10.781  1.00 21.68           O  \\nHETATM  283  O   HOH B  35      19.923  40.404  18.291  1.00 31.79           O  \\nHETATM  284  O   HOH B  36      18.633  37.421  10.594  1.00 28.62           O  \\nHETATM  285  O   HOH B  37      41.530  30.514  14.417  1.00 36.59           O  \\nHETATM  286  O   HOH B  38      26.648  34.690  11.974  1.00 32.48           O  \\nMASTER      362    0    0    0    0    0    0    6  284    2    0    2          \\nEND                                                                             \\n\");\n",
       "\tviewer_1684696742114819.setStyle({\"model\": -1, \"chain\": \"A\"},{\"cartoon\": {\"color\": \"spectrum\"}});\n",
       "\tviewer_1684696742114819.zoomTo();\n",
       "viewer_1684696742114819.render();\n",
       "});\n",
       "</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Similarity score: 12.0\n",
      " Protein: stable loop in the crystal structure of the intercalated four-stranded cytosine-rich metazoan telomere(200D)\n",
      " Chain Id: B\n",
      " Sequence of the chain: DTDADADCDCDC\n"
     ]
    },
    {
     "data": {
      "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_1684696742117703\"  style=\"position: relative; width: 600px; height: 400px\">\n        <p id=\"3dmolwarning_1684696742117703\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.0.1/3Dmol-min.js');\n}\n\nvar viewer_1684696742117703 = null;\nvar warn = document.getElementById(\"3dmolwarning_1684696742117703\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_1684696742117703 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_1684696742117703\"),{backgroundColor:\"white\"});\nviewer_1684696742117703.zoomTo();\n\tviewer_1684696742117703.addModelsAsFrames(\"HEADER    DNA                                     16-FEB-95   200D              \\nTITLE     STABLE LOOP IN THE CRYSTAL STRUCTURE OF THE INTERCALATED              \\nTITLE    2 FOUR-STRANDED CYTOSINE-RICH METAZOAN TELOMERE                        \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: DNA (5'-D(*TP*AP*AP*CP*CP*C)-3');                          \\nCOMPND   3 CHAIN: A, B;                                                         \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 SYNTHETIC: YES                                                       \\nKEYWDS    U-DNA, QUADRUPLE HELIX, TETRAPLEX, BASE INTERCALATED, LOOP            \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    C.KANG,I.BERGER,C.LOCKSHIN,R.RATLIFF,R.MOYZIS,A.RICH                  \\nREVDAT   2   24-FEB-09 200D    1       VERSN                                    \\nREVDAT   1   31-JUL-95 200D    0                                                \\nJRNL        AUTH   C.KANG,I.BERGER,C.LOCKSHIN,R.RATLIFF,R.MOYZIS,               \\nJRNL        AUTH 2 A.RICH                                                       \\nJRNL        TITL   STABLE LOOP IN THE CRYSTAL STRUCTURE OF THE                  \\nJRNL        TITL 2 INTERCALATED FOUR-STRANDED CYTOSINE-RICH METAZOAN            \\nJRNL        TITL 3 TELOMERE.                                                    \\nJRNL        REF    PROC.NATL.ACAD.SCI.USA        V.  92  3874 1995              \\nJRNL        REFN                   ISSN 0027-8424                               \\nJRNL        PMID   7731999                                                      \\nJRNL        DOI    10.1073/PNAS.92.9.3874                                       \\nREMARK   1                                                                      \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.85 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : X-PLOR                                               \\nREMARK   3   AUTHORS     : BRUNGER                                              \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.85                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 12.00                          \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : 1.000                          \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : 77.0                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 2122                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : 0.190                           \\nREMARK   3   FREE R VALUE                     : 0.223                           \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 0                                       \\nREMARK   3   NUCLEIC ACID ATOMS       : 258                                     \\nREMARK   3   HETEROGEN ATOMS          : 0                                       \\nREMARK   3   SOLVENT ATOMS            : 26                                      \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : 0.030                           \\nREMARK   3   BOND ANGLES            (DEGREES) : 4.00                            \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 200D COMPLIES WITH FORMAT V. 3.15, 01-DEC-08                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : 20-JUL-94                          \\nREMARK 200  TEMPERATURE           (KELVIN) : 277.00                             \\nREMARK 200  PH                             : 6.00                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : N                                  \\nREMARK 200  RADIATION SOURCE               : ROTATING ANODE                     \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : M                                  \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : IMAGE PLATE                        \\nREMARK 200  DETECTOR MANUFACTURER          : RIGAKU RAXIS IIC                   \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : MSC                                \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : 0.04500                            \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): NULL                                      \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): NULL                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: PH 6.00, VAPOR DIFFUSION                 \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: F 2 2 2                          \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -X,-Y,Z                                                 \\nREMARK 290       3555   -X,Y,-Z                                                 \\nREMARK 290       4555   X,-Y,-Z                                                 \\nREMARK 290       5555   X,Y+1/2,Z+1/2                                           \\nREMARK 290       6555   -X,-Y+1/2,Z+1/2                                         \\nREMARK 290       7555   -X,Y+1/2,-Z+1/2                                         \\nREMARK 290       8555   X,-Y+1/2,-Z+1/2                                         \\nREMARK 290       9555   X+1/2,Y,Z+1/2                                           \\nREMARK 290      10555   -X+1/2,-Y,Z+1/2                                         \\nREMARK 290      11555   -X+1/2,Y,-Z+1/2                                         \\nREMARK 290      12555   X+1/2,-Y,-Z+1/2                                         \\nREMARK 290      13555   X+1/2,Y+1/2,Z                                           \\nREMARK 290      14555   -X+1/2,-Y+1/2,Z                                         \\nREMARK 290      15555   -X+1/2,Y+1/2,-Z                                         \\nREMARK 290      16555   X+1/2,-Y+1/2,-Z                                         \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   3 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   3  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   4  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   4  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   5  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   5  0.000000  1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3   5  0.000000  0.000000  1.000000       13.43000            \\nREMARK 290   SMTRY1   6 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   6  0.000000 -1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3   6  0.000000  0.000000  1.000000       13.43000            \\nREMARK 290   SMTRY1   7 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   7  0.000000  1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3   7  0.000000  0.000000 -1.000000       13.43000            \\nREMARK 290   SMTRY1   8  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   8  0.000000 -1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3   8  0.000000  0.000000 -1.000000       13.43000            \\nREMARK 290   SMTRY1   9  1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2   9  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   9  0.000000  0.000000  1.000000       13.43000            \\nREMARK 290   SMTRY1  10 -1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  10  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3  10  0.000000  0.000000  1.000000       13.43000            \\nREMARK 290   SMTRY1  11 -1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  11  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3  11  0.000000  0.000000 -1.000000       13.43000            \\nREMARK 290   SMTRY1  12  1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  12  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3  12  0.000000  0.000000 -1.000000       13.43000            \\nREMARK 290   SMTRY1  13  1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  13  0.000000  1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3  13  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1  14 -1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  14  0.000000 -1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3  14  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1  15 -1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  15  0.000000  1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3  15  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1  16  1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  16  0.000000 -1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3  16  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: TETRAMERIC                        \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A, B                                  \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2 -1.000000  0.000000  0.000000       59.94000            \\nREMARK 350   BIOMT2   2  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000 -1.000000       26.86000            \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND LENGTHS                                      \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,2(A3,1X,A1,I4,A1,1X,A4,3X),1X,F6.3)               \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   RES CSSEQI ATM2   DEVIATION                     \\nREMARK 500     DT A   1   C5     DT A   1   C7      0.038                       \\nREMARK 500     DT A   1   O3'    DA A   2   P      -0.084                       \\nREMARK 500     DA A   3   C4'    DA A   3   C3'    -0.094                       \\nREMARK 500     DA A   3   N9     DA A   3   C4     -0.042                       \\nREMARK 500     DC A   4   N3     DC A   4   C4     -0.047                       \\nREMARK 500     DC A   5   C3'    DC A   5   C2'    -0.055                       \\nREMARK 500     DC A   6   C5'    DC A   6   C4'     0.090                       \\nREMARK 500     DC A   6   C3'    DC A   6   C2'    -0.050                       \\nREMARK 500     DC A   6   N3     DC A   6   C4     -0.052                       \\nREMARK 500     DT B   7   C5'    DT B   7   C4'     0.082                       \\nREMARK 500     DT B   7   C2     DT B   7   N3     -0.049                       \\nREMARK 500     DA B   8   P      DA B   8   O5'     0.063                       \\nREMARK 500     DC B  10   C4'    DC B  10   C3'    -0.083                       \\nREMARK 500     DC B  10   N1     DC B  10   C6     -0.063                       \\nREMARK 500     DC B  10   C4     DC B  10   C5     -0.050                       \\nREMARK 500     DC B  11   N1     DC B  11   C2     -0.094                       \\nREMARK 500     DC B  11   N1     DC B  11   C6     -0.040                       \\nREMARK 500     DC B  11   C2     DC B  11   N3     -0.056                       \\nREMARK 500     DC B  12   O4'    DC B  12   C4'    -0.069                       \\nREMARK 500     DC B  12   N1     DC B  12   C2     -0.062                       \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \\nREMARK 500     DT A   1   O4' -  C1' -  N1  ANGL. DEV. =   2.9 DEGREES          \\nREMARK 500     DT A   1   C6  -  C5  -  C7  ANGL. DEV. =  -4.4 DEGREES          \\nREMARK 500     DA A   2   O4' -  C4' -  C3' ANGL. DEV. =   4.3 DEGREES          \\nREMARK 500     DA A   2   C4' -  C3' -  C2' ANGL. DEV. =  -9.1 DEGREES          \\nREMARK 500     DA A   2   O4' -  C1' -  C2' ANGL. DEV. =  -7.8 DEGREES          \\nREMARK 500     DA A   2   N1  -  C6  -  N6  ANGL. DEV. =   3.7 DEGREES          \\nREMARK 500     DT A   1   C3' -  O3' -  P   ANGL. DEV. =  14.8 DEGREES          \\nREMARK 500     DA A   2   O3' -  P   -  O5' ANGL. DEV. = -11.7 DEGREES          \\nREMARK 500     DA A   3   O4' -  C4' -  C3' ANGL. DEV. =  -7.7 DEGREES          \\nREMARK 500     DA A   3   C1' -  O4' -  C4' ANGL. DEV. =   6.1 DEGREES          \\nREMARK 500     DA A   3   O4' -  C1' -  C2' ANGL. DEV. =  -6.8 DEGREES          \\nREMARK 500     DA A   3   N1  -  C2  -  N3  ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500     DA A   3   C2  -  N3  -  C4  ANGL. DEV. =  -3.9 DEGREES          \\nREMARK 500     DA A   2   C3' -  O3' -  P   ANGL. DEV. =   7.9 DEGREES          \\nREMARK 500     DC A   4   C4' -  C3' -  C2' ANGL. DEV. =  -4.3 DEGREES          \\nREMARK 500     DC A   5   C4' -  C3' -  C2' ANGL. DEV. =  -7.7 DEGREES          \\nREMARK 500     DC A   5   O4' -  C1' -  N1  ANGL. DEV. =  -4.7 DEGREES          \\nREMARK 500     DC A   5   C2  -  N3  -  C4  ANGL. DEV. =   3.5 DEGREES          \\nREMARK 500     DC A   6   P   -  O5' -  C5' ANGL. DEV. =  13.2 DEGREES          \\nREMARK 500     DC A   6   O4' -  C4' -  C3' ANGL. DEV. =  -4.4 DEGREES          \\nREMARK 500     DC A   6   O4' -  C1' -  N1  ANGL. DEV. =   3.9 DEGREES          \\nREMARK 500     DC A   6   N1  -  C2  -  O2  ANGL. DEV. =   3.9 DEGREES          \\nREMARK 500     DC A   6   N3  -  C4  -  N4  ANGL. DEV. =  -5.0 DEGREES          \\nREMARK 500     DC A   6   C5  -  C4  -  N4  ANGL. DEV. =   4.2 DEGREES          \\nREMARK 500     DT B   7   C5' -  C4' -  O4' ANGL. DEV. =   9.2 DEGREES          \\nREMARK 500     DT B   7   N1  -  C2  -  O2  ANGL. DEV. =   5.3 DEGREES          \\nREMARK 500     DT B   7   N3  -  C2  -  O2  ANGL. DEV. =  -4.3 DEGREES          \\nREMARK 500     DA B   8   O4' -  C1' -  C2' ANGL. DEV. =  -5.1 DEGREES          \\nREMARK 500     DA B   8   C2  -  N3  -  C4  ANGL. DEV. =  -3.3 DEGREES          \\nREMARK 500     DA B   8   C5  -  C6  -  N1  ANGL. DEV. =  -4.0 DEGREES          \\nREMARK 500     DT B   7   C3' -  O3' -  P   ANGL. DEV. = -10.9 DEGREES          \\nREMARK 500     DA B   9   O4' -  C1' -  C2' ANGL. DEV. =  -7.3 DEGREES          \\nREMARK 500     DC B  10   C1' -  O4' -  C4' ANGL. DEV. =   6.5 DEGREES          \\nREMARK 500     DC B  10   O4' -  C1' -  C2' ANGL. DEV. = -10.3 DEGREES          \\nREMARK 500     DC B  10   O4' -  C1' -  N1  ANGL. DEV. =   3.6 DEGREES          \\nREMARK 500     DC B  10   N3  -  C2  -  O2  ANGL. DEV. =  -5.1 DEGREES          \\nREMARK 500     DC B  10   C3' -  O3' -  P   ANGL. DEV. =   9.2 DEGREES          \\nREMARK 500     DC B  12   O4' -  C1' -  C2' ANGL. DEV. =  -5.1 DEGREES          \\nREMARK 500     DC B  12   C2  -  N3  -  C4  ANGL. DEV. =   4.6 DEGREES          \\nREMARK 500     DC B  12   N1  -  C2  -  O2  ANGL. DEV. =   4.6 DEGREES          \\nREMARK 500     DC B  11   C3' -  O3' -  P   ANGL. DEV. =   8.3 DEGREES          \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: PLANAR GROUPS                                              \\nREMARK 500                                                                      \\nREMARK 500 PLANAR GROUPS IN THE FOLLOWING RESIDUES HAVE A TOTAL                 \\nREMARK 500 RMS DISTANCE OF ALL ATOMS FROM THE BEST-FIT PLANE                    \\nREMARK 500 BY MORE THAN AN EXPECTED VALUE OF 6*RMSD, WITH AN                    \\nREMARK 500 RMSD 0.02 ANGSTROMS, OR AT LEAST ONE ATOM HAS                        \\nREMARK 500 AN RMSD GREATER THAN THIS VALUE                                      \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        RMS     TYPE                                    \\nREMARK 500     DA A   3         0.06    SIDE_CHAIN                              \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nDBREF  200D A    1     6  PDB    200D     200D             1      6             \\nDBREF  200D B    7    12  PDB    200D     200D             7     12             \\nSEQRES   1 A    6   DT  DA  DA  DC  DC  DC                                      \\nSEQRES   1 B    6   DT  DA  DA  DC  DC  DC                                      \\nFORMUL   3  HOH   *26(H2 O)                                                     \\nCRYST1   59.940   81.330   26.860  90.00  90.00  90.00 F 2 2 2      32          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.016683  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  0.012296  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.037230        0.00000                         \\nATOM      1  O5'  DT A   1      24.565  31.319  24.443  1.00 30.44           O  \\nATOM      2  C5'  DT A   1      23.942  30.467  25.330  1.00 20.89           C  \\nATOM      3  C4'  DT A   1      23.216  29.306  24.650  1.00 27.77           C  \\nATOM      4  O4'  DT A   1      22.175  29.736  23.812  1.00 23.30           O  \\nATOM      5  C3'  DT A   1      24.157  28.454  23.852  1.00 35.38           C  \\nATOM      6  O3'  DT A   1      23.905  27.189  24.366  1.00 33.40           O  \\nATOM      7  C2'  DT A   1      23.531  28.590  22.502  1.00 34.88           C  \\nATOM      8  C1'  DT A   1      22.104  28.825  22.807  1.00 18.33           C  \\nATOM      9  N1   DT A   1      21.415  29.371  21.650  1.00 27.22           N  \\nATOM     10  C2   DT A   1      21.061  28.462  20.735  1.00 17.62           C  \\nATOM     11  O2   DT A   1      21.294  27.274  20.916  1.00 23.45           O  \\nATOM     12  N3   DT A   1      20.409  28.945  19.635  1.00 20.47           N  \\nATOM     13  C4   DT A   1      20.071  30.242  19.367  1.00 19.02           C  \\nATOM     14  O4   DT A   1      19.434  30.520  18.346  1.00 27.67           O  \\nATOM     15  C5   DT A   1      20.491  31.148  20.397  1.00 28.05           C  \\nATOM     16  C7   DT A   1      20.220  32.654  20.284  1.00 24.86           C  \\nATOM     17  C6   DT A   1      21.136  30.694  21.482  1.00 14.22           C  \\nATOM     18  H3   DT A   1      20.149  28.273  18.939  1.00  0.00           H  \\nATOM     19 HO5'  DT A   1      25.263  30.835  24.004  1.00  0.00           H  \\nATOM     20  P    DA A   2      24.766  25.974  24.683  1.00 23.53           P  \\nATOM     21  OP1  DA A   2      24.819  25.775  26.131  1.00 27.65           O  \\nATOM     22  OP2  DA A   2      25.964  25.990  23.827  1.00 38.21           O  \\nATOM     23  O5'  DA A   2      23.749  24.991  23.979  1.00 23.31           O  \\nATOM     24  C5'  DA A   2      22.350  25.133  24.144  1.00 31.45           C  \\nATOM     25  C4'  DA A   2      21.502  24.042  23.575  1.00 10.83           C  \\nATOM     26  O4'  DA A   2      20.221  24.252  24.204  1.00 36.32           O  \\nATOM     27  C3'  DA A   2      21.353  24.038  22.059  1.00 39.61           C  \\nATOM     28  O3'  DA A   2      22.218  23.042  21.474  1.00 19.31           O  \\nATOM     29  C2'  DA A   2      19.926  23.509  22.118  1.00 23.91           C  \\nATOM     30  C1'  DA A   2      19.171  24.220  23.257  1.00 29.47           C  \\nATOM     31  N9   DA A   2      18.839  25.627  23.010  1.00 19.13           N  \\nATOM     32  C8   DA A   2      19.151  26.734  23.759  1.00 21.05           C  \\nATOM     33  N7   DA A   2      18.737  27.848  23.267  1.00 20.27           N  \\nATOM     34  C5   DA A   2      18.093  27.426  22.100  1.00 14.04           C  \\nATOM     35  C6   DA A   2      17.391  28.116  21.124  1.00 18.64           C  \\nATOM     36  N6   DA A   2      17.250  29.453  21.149  1.00 24.64           N  \\nATOM     37  N1   DA A   2      16.861  27.359  20.157  1.00 15.56           N  \\nATOM     38  C2   DA A   2      17.016  26.046  20.152  1.00  9.93           C  \\nATOM     39  N3   DA A   2      17.641  25.306  20.992  1.00 23.82           N  \\nATOM     40  C4   DA A   2      18.160  26.078  21.948  1.00 11.68           C  \\nATOM     41  H61  DA A   2      16.745  29.923  20.410  1.00  0.00           H  \\nATOM     42  H62  DA A   2      17.644  29.994  21.903  1.00  0.00           H  \\nATOM     43  P    DA A   3      23.030  23.149  20.153  1.00 20.46           P  \\nATOM     44  OP1  DA A   3      23.954  22.013  20.095  1.00 24.95           O  \\nATOM     45  OP2  DA A   3      23.531  24.509  20.030  1.00 19.53           O  \\nATOM     46  O5'  DA A   3      21.880  22.885  19.163  1.00 17.18           O  \\nATOM     47  C5'  DA A   3      21.278  21.604  19.070  1.00 13.90           C  \\nATOM     48  C4'  DA A   3      20.159  21.601  18.065  1.00 21.24           C  \\nATOM     49  O4'  DA A   3      19.337  22.719  18.308  1.00 21.96           O  \\nATOM     50  C3'  DA A   3      20.581  21.983  16.756  1.00 22.35           C  \\nATOM     51  O3'  DA A   3      21.091  20.871  16.039  1.00 35.11           O  \\nATOM     52  C2'  DA A   3      19.323  22.493  16.117  1.00 36.04           C  \\nATOM     53  C1'  DA A   3      18.685  23.288  17.172  1.00 20.51           C  \\nATOM     54  N9   DA A   3      18.982  24.750  17.163  1.00 14.73           N  \\nATOM     55  C8   DA A   3      19.499  25.507  18.166  1.00  8.55           C  \\nATOM     56  N7   DA A   3      19.478  26.769  17.943  1.00 16.60           N  \\nATOM     57  C5   DA A   3      18.913  26.871  16.693  1.00 12.55           C  \\nATOM     58  C6   DA A   3      18.601  27.989  15.881  1.00 20.70           C  \\nATOM     59  N6   DA A   3      18.741  29.276  16.217  1.00 14.67           N  \\nATOM     60  N1   DA A   3      18.060  27.707  14.701  1.00 13.89           N  \\nATOM     61  C2   DA A   3      17.850  26.408  14.389  1.00 29.53           C  \\nATOM     62  N3   DA A   3      18.083  25.275  15.052  1.00 18.60           N  \\nATOM     63  C4   DA A   3      18.625  25.618  16.218  1.00  8.22           C  \\nATOM     64  H61  DA A   3      18.524  30.004  15.551  1.00  0.00           H  \\nATOM     65  H62  DA A   3      19.059  29.517  17.142  1.00  0.00           H  \\nATOM     66  P    DC A   4      22.425  21.041  15.096  1.00 24.23           P  \\nATOM     67  OP1  DC A   4      22.465  19.788  14.318  1.00 29.54           O  \\nATOM     68  OP2  DC A   4      23.569  21.448  15.939  1.00 26.74           O  \\nATOM     69  O5'  DC A   4      22.106  22.295  14.117  1.00 24.38           O  \\nATOM     70  C5'  DC A   4      21.326  22.363  12.942  1.00 23.31           C  \\nATOM     71  C4'  DC A   4      21.432  23.763  12.409  1.00 19.02           C  \\nATOM     72  O4'  DC A   4      20.924  24.622  13.375  1.00 13.89           O  \\nATOM     73  C3'  DC A   4      22.882  24.127  12.138  1.00 15.94           C  \\nATOM     74  O3'  DC A   4      23.153  23.807  10.751  1.00 33.78           O  \\nATOM     75  C2'  DC A   4      22.777  25.612  12.409  1.00 26.18           C  \\nATOM     76  C1'  DC A   4      21.638  25.802  13.441  1.00  6.99           C  \\nATOM     77  N1   DC A   4      22.071  25.996  14.858  1.00 15.40           N  \\nATOM     78  C2   DC A   4      22.052  27.275  15.345  1.00 14.75           C  \\nATOM     79  O2   DC A   4      21.597  28.208  14.679  1.00 20.98           O  \\nATOM     80  N3   DC A   4      22.492  27.460  16.606  1.00 19.60           N  \\nATOM     81  C4   DC A   4      22.910  26.491  17.345  1.00  7.46           C  \\nATOM     82  N4   DC A   4      23.363  26.803  18.516  1.00 11.08           N  \\nATOM     83  C5   DC A   4      22.926  25.188  16.895  1.00  7.21           C  \\nATOM     84  C6   DC A   4      22.504  24.981  15.653  1.00  9.76           C  \\nATOM     85  H41  DC A   4      23.402  27.768  18.806  1.00  0.00           H  \\nATOM     86  H42  DC A   4      23.675  26.076  19.149  1.00  0.00           H  \\nATOM     87  P    DC A   5      24.446  22.970  10.188  1.00 28.52           P  \\nATOM     88  OP1  DC A   5      24.144  22.120   9.020  1.00 24.29           O  \\nATOM     89  OP2  DC A   5      25.053  22.349  11.400  1.00 34.89           O  \\nATOM     90  O5'  DC A   5      25.373  24.138   9.643  1.00 30.47           O  \\nATOM     91  C5'  DC A   5      24.837  25.048   8.687  1.00 27.30           C  \\nATOM     92  C4'  DC A   5      25.648  26.351   8.642  1.00 20.98           C  \\nATOM     93  O4'  DC A   5      25.627  26.952   9.905  1.00 21.19           O  \\nATOM     94  C3'  DC A   5      27.100  26.119   8.375  1.00 20.71           C  \\nATOM     95  O3'  DC A   5      27.341  25.805   7.002  1.00 23.11           O  \\nATOM     96  C2'  DC A   5      27.496  27.486   8.703  1.00 13.44           C  \\nATOM     97  C1'  DC A   5      26.696  27.830   9.919  1.00 13.82           C  \\nATOM     98  N1   DC A   5      27.308  27.595  11.235  1.00 22.24           N  \\nATOM     99  C2   DC A   5      27.581  28.683  11.989  1.00  9.58           C  \\nATOM    100  O2   DC A   5      27.383  29.828  11.573  1.00 25.04           O  \\nATOM    101  N3   DC A   5      28.050  28.457  13.220  1.00 23.52           N  \\nATOM    102  C4   DC A   5      28.242  27.259  13.710  1.00  9.77           C  \\nATOM    103  N4   DC A   5      28.680  27.143  14.947  1.00 20.94           N  \\nATOM    104  C5   DC A   5      27.978  26.108  12.961  1.00 15.31           C  \\nATOM    105  C6   DC A   5      27.515  26.334  11.726  1.00 20.31           C  \\nATOM    106  H41  DC A   5      28.833  26.230  15.355  1.00  0.00           H  \\nATOM    107  H42  DC A   5      28.860  27.971  15.496  1.00  0.00           H  \\nATOM    108  P    DC A   6      28.699  25.257   6.381  1.00 26.27           P  \\nATOM    109  OP1  DC A   6      28.416  24.653   5.094  1.00 29.77           O  \\nATOM    110  OP2  DC A   6      29.457  24.544   7.410  1.00 23.78           O  \\nATOM    111  O5'  DC A   6      29.520  26.539   6.046  1.00 32.49           O  \\nATOM    112  C5'  DC A   6      29.282  27.692   5.226  1.00 30.24           C  \\nATOM    113  C4'  DC A   6      30.531  28.674   5.429  1.00 22.63           C  \\nATOM    114  O4'  DC A   6      30.781  28.988   6.818  1.00 18.93           O  \\nATOM    115  C3'  DC A   6      31.823  27.970   5.079  1.00 28.36           C  \\nATOM    116  O3'  DC A   6      32.102  28.065   3.709  1.00 27.07           O  \\nATOM    117  C2'  DC A   6      32.812  28.770   5.808  1.00 13.00           C  \\nATOM    118  C1'  DC A   6      32.139  29.157   7.068  1.00  9.92           C  \\nATOM    119  N1   DC A   6      32.607  28.338   8.228  1.00 22.07           N  \\nATOM    120  C2   DC A   6      33.155  29.092   9.255  1.00 26.49           C  \\nATOM    121  O2   DC A   6      33.154  30.333   9.252  1.00 26.06           O  \\nATOM    122  N3   DC A   6      33.673  28.433  10.285  1.00 17.53           N  \\nATOM    123  C4   DC A   6      33.677  27.152  10.362  1.00  7.84           C  \\nATOM    124  N4   DC A   6      34.255  26.711  11.432  1.00 17.65           N  \\nATOM    125  C5   DC A   6      33.130  26.339   9.369  1.00  9.69           C  \\nATOM    126  C6   DC A   6      32.596  26.974   8.310  1.00 25.15           C  \\nATOM    127 HO3'  DC A   6      31.406  27.590   3.246  1.00  0.00           H  \\nATOM    128  H41  DC A   6      34.358  25.716  11.590  1.00  0.00           H  \\nATOM    129  H42  DC A   6      34.605  27.355  12.121  1.00  0.00           H  \\nTER     130       DC A   6                                                      \\nATOM    131  O5'  DT B   7      11.293  29.851  15.858  1.00 28.85           O  \\nATOM    132  C5'  DT B   7      11.549  31.182  16.395  1.00 25.35           C  \\nATOM    133  C4'  DT B   7      12.980  31.877  16.290  1.00 39.66           C  \\nATOM    134  O4'  DT B   7      14.126  31.292  17.044  1.00 37.54           O  \\nATOM    135  C3'  DT B   7      13.456  31.813  14.854  1.00 39.01           C  \\nATOM    136  O3'  DT B   7      14.165  32.997  14.544  1.00 38.98           O  \\nATOM    137  C2'  DT B   7      14.424  30.692  14.753  1.00 20.58           C  \\nATOM    138  C1'  DT B   7      14.994  30.617  16.125  1.00 20.74           C  \\nATOM    139  N1   DT B   7      15.053  29.255  16.540  1.00 18.62           N  \\nATOM    140  C2   DT B   7      15.671  29.004  17.731  1.00 20.33           C  \\nATOM    141  O2   DT B   7      16.113  29.817  18.554  1.00 26.72           O  \\nATOM    142  N3   DT B   7      15.786  27.715  18.009  1.00 22.63           N  \\nATOM    143  C4   DT B   7      15.361  26.621  17.272  1.00 33.64           C  \\nATOM    144  O4   DT B   7      15.606  25.447  17.626  1.00 34.71           O  \\nATOM    145  C5   DT B   7      14.708  26.988  16.073  1.00 14.16           C  \\nATOM    146  C7   DT B   7      14.102  25.888  15.216  1.00 39.75           C  \\nATOM    147  C6   DT B   7      14.575  28.259  15.732  1.00 24.49           C  \\nATOM    148  H3   DT B   7      16.244  27.571  18.880  1.00  0.00           H  \\nATOM    149 HO5'  DT B   7      11.404  29.913  14.905  1.00  0.00           H  \\nATOM    150  P    DA B   8      13.931  33.345  13.024  1.00 39.22           P  \\nATOM    151  OP1  DA B   8      12.612  34.036  13.039  1.00 34.77           O  \\nATOM    152  OP2  DA B   8      14.206  32.074  12.263  1.00 39.31           O  \\nATOM    153  O5'  DA B   8      15.000  34.463  12.432  1.00 39.80           O  \\nATOM    154  C5'  DA B   8      16.303  34.113  11.867  1.00 39.02           C  \\nATOM    155  C4'  DA B   8      17.406  35.058  12.322  1.00 17.13           C  \\nATOM    156  O4'  DA B   8      16.819  36.368  12.502  1.00 22.28           O  \\nATOM    157  C3'  DA B   8      18.055  34.660  13.684  1.00 35.33           C  \\nATOM    158  O3'  DA B   8      19.437  35.008  13.398  1.00 39.28           O  \\nATOM    159  C2'  DA B   8      17.310  35.602  14.642  1.00 35.21           C  \\nATOM    160  C1'  DA B   8      17.143  36.915  13.808  1.00 14.63           C  \\nATOM    161  N9   DA B   8      16.003  37.727  14.228  1.00 20.81           N  \\nATOM    162  C8   DA B   8      14.676  37.344  14.239  1.00 37.43           C  \\nATOM    163  N7   DA B   8      13.839  38.247  14.635  1.00 23.05           N  \\nATOM    164  C5   DA B   8      14.647  39.318  14.912  1.00 19.42           C  \\nATOM    165  C6   DA B   8      14.326  40.578  15.398  1.00 14.47           C  \\nATOM    166  N6   DA B   8      13.065  41.064  15.553  1.00 20.96           N  \\nATOM    167  N1   DA B   8      15.373  41.351  15.597  1.00 33.53           N  \\nATOM    168  C2   DA B   8      16.622  40.929  15.326  1.00 27.49           C  \\nATOM    169  N3   DA B   8      17.042  39.778  14.845  1.00 27.07           N  \\nATOM    170  C4   DA B   8      15.968  39.007  14.666  1.00 10.79           C  \\nATOM    171  H61  DA B   8      12.269  40.498  15.302  1.00  0.00           H  \\nATOM    172  H62  DA B   8      12.942  41.998  15.918  1.00  0.00           H  \\nATOM    173  P    DA B   9      20.719  34.445  14.194  1.00 35.42           P  \\nATOM    174  OP1  DA B   9      21.959  35.022  13.557  1.00 39.82           O  \\nATOM    175  OP2  DA B   9      20.528  32.939  14.321  1.00 28.79           O  \\nATOM    176  O5'  DA B   9      20.556  35.250  15.553  1.00 39.94           O  \\nATOM    177  C5'  DA B   9      21.196  36.518  15.727  1.00 27.77           C  \\nATOM    178  C4'  DA B   9      20.828  37.045  17.056  1.00 16.24           C  \\nATOM    179  O4'  DA B   9      19.420  37.260  17.115  1.00 24.83           O  \\nATOM    180  C3'  DA B   9      21.131  36.015  18.071  1.00 22.96           C  \\nATOM    181  O3'  DA B   9      21.775  36.603  19.212  1.00 29.07           O  \\nATOM    182  C2'  DA B   9      19.740  35.550  18.407  1.00 26.00           C  \\nATOM    183  C1'  DA B   9      18.975  36.797  18.352  1.00 18.88           C  \\nATOM    184  N9   DA B   9      17.533  36.628  18.310  1.00 17.60           N  \\nATOM    185  C8   DA B   9      16.740  35.511  18.232  1.00 23.85           C  \\nATOM    186  N7   DA B   9      15.467  35.795  18.264  1.00 25.36           N  \\nATOM    187  C5   DA B   9      15.416  37.185  18.373  1.00 14.91           C  \\nATOM    188  C6   DA B   9      14.358  38.117  18.472  1.00 24.27           C  \\nATOM    189  N6   DA B   9      13.051  37.812  18.565  1.00 30.71           N  \\nATOM    190  N1   DA B   9      14.730  39.429  18.573  1.00 34.89           N  \\nATOM    191  C2   DA B   9      16.025  39.778  18.587  1.00 15.13           C  \\nATOM    192  N3   DA B   9      17.077  38.975  18.506  1.00 18.79           N  \\nATOM    193  C4   DA B   9      16.688  37.686  18.401  1.00 21.13           C  \\nATOM    194  H61  DA B   9      12.726  36.865  18.446  1.00  0.00           H  \\nATOM    195  H62  DA B   9      12.408  38.552  18.781  1.00  0.00           H  \\nATOM    196  P    DC B  10      23.405  36.738  19.343  1.00 28.67           P  \\nATOM    197  OP1  DC B  10      23.679  37.914  20.242  1.00 34.27           O  \\nATOM    198  OP2  DC B  10      23.910  36.731  17.961  1.00 38.84           O  \\nATOM    199  O5'  DC B  10      24.087  35.454  20.011  1.00 18.77           O  \\nATOM    200  C5'  DC B  10      24.087  35.100  21.368  1.00 30.20           C  \\nATOM    201  C4'  DC B  10      24.575  33.695  21.529  1.00  9.80           C  \\nATOM    202  O4'  DC B  10      23.800  32.700  20.879  1.00 18.83           O  \\nATOM    203  C3'  DC B  10      25.878  33.620  20.926  1.00 14.61           C  \\nATOM    204  O3'  DC B  10      26.615  32.959  21.958  1.00 36.36           O  \\nATOM    205  C2'  DC B  10      25.606  32.814  19.678  1.00 19.97           C  \\nATOM    206  C1'  DC B  10      24.481  31.842  19.992  1.00 15.38           C  \\nATOM    207  N1   DC B  10      23.686  31.565  18.773  1.00 11.72           N  \\nATOM    208  C2   DC B  10      23.456  30.308  18.325  1.00 11.51           C  \\nATOM    209  O2   DC B  10      23.748  29.333  18.974  1.00 25.69           O  \\nATOM    210  N3   DC B  10      22.811  30.099  17.201  1.00 14.49           N  \\nATOM    211  C4   DC B  10      22.370  31.109  16.513  1.00 12.31           C  \\nATOM    212  N4   DC B  10      21.662  30.932  15.398  1.00 25.78           N  \\nATOM    213  C5   DC B  10      22.573  32.400  16.939  1.00 14.33           C  \\nATOM    214  C6   DC B  10      23.237  32.570  18.074  1.00 13.01           C  \\nATOM    215  H41  DC B  10      21.523  29.994  15.042  1.00  0.00           H  \\nATOM    216  H42  DC B  10      21.261  31.720  14.909  1.00  0.00           H  \\nATOM    217  P    DC B  11      28.130  33.237  22.467  1.00 34.54           P  \\nATOM    218  OP1  DC B  11      28.322  32.261  23.594  1.00 29.17           O  \\nATOM    219  OP2  DC B  11      28.366  34.724  22.567  1.00 26.05           O  \\nATOM    220  O5'  DC B  11      28.851  32.747  21.146  1.00 29.65           O  \\nATOM    221  C5'  DC B  11      30.240  32.966  20.860  1.00 31.52           C  \\nATOM    222  C4'  DC B  11      30.646  31.847  19.984  1.00 13.84           C  \\nATOM    223  O4'  DC B  11      29.762  31.772  18.892  1.00 25.81           O  \\nATOM    224  C3'  DC B  11      32.016  32.042  19.492  1.00 15.90           C  \\nATOM    225  O3'  DC B  11      32.776  31.224  20.405  1.00 35.61           O  \\nATOM    226  C2'  DC B  11      31.886  31.528  18.055  1.00 19.50           C  \\nATOM    227  C1'  DC B  11      30.453  31.059  17.897  1.00 14.84           C  \\nATOM    228  N1   DC B  11      29.801  31.387  16.621  1.00  8.89           N  \\nATOM    229  C2   DC B  11      29.420  30.373  15.896  1.00 10.51           C  \\nATOM    230  O2   DC B  11      29.691  29.265  16.247  1.00 16.37           O  \\nATOM    231  N3   DC B  11      28.767  30.545  14.789  1.00 14.37           N  \\nATOM    232  C4   DC B  11      28.472  31.744  14.372  1.00  8.47           C  \\nATOM    233  N4   DC B  11      27.746  31.854  13.292  1.00 24.16           N  \\nATOM    234  C5   DC B  11      28.866  32.886  15.096  1.00 19.21           C  \\nATOM    235  C6   DC B  11      29.541  32.630  16.236  1.00 19.62           C  \\nATOM    236  H41  DC B  11      27.427  32.762  12.970  1.00  0.00           H  \\nATOM    237  H42  DC B  11      27.498  31.025  12.770  1.00  0.00           H  \\nATOM    238  P    DC B  12      34.436  31.147  20.578  1.00 35.23           P  \\nATOM    239  OP1  DC B  12      34.764  29.859  21.271  1.00 39.08           O  \\nATOM    240  OP2  DC B  12      34.907  32.477  21.122  1.00 36.25           O  \\nATOM    241  O5'  DC B  12      35.015  30.931  19.062  1.00 30.38           O  \\nATOM    242  C5'  DC B  12      36.391  30.665  18.739  1.00 30.68           C  \\nATOM    243  C4'  DC B  12      36.518  29.663  17.628  1.00 28.20           C  \\nATOM    244  O4'  DC B  12      35.889  30.201  16.528  1.00 29.99           O  \\nATOM    245  C3'  DC B  12      37.900  29.433  17.125  1.00 23.57           C  \\nATOM    246  O3'  DC B  12      38.418  28.232  17.727  1.00 39.77           O  \\nATOM    247  C2'  DC B  12      37.691  29.249  15.673  1.00 21.13           C  \\nATOM    248  C1'  DC B  12      36.254  29.370  15.451  1.00 17.03           C  \\nATOM    249  N1   DC B  12      35.819  30.088  14.254  1.00  9.75           N  \\nATOM    250  C2   DC B  12      35.372  29.340  13.243  1.00 20.35           C  \\nATOM    251  O2   DC B  12      35.274  28.103  13.297  1.00 21.58           O  \\nATOM    252  N3   DC B  12      34.928  30.008  12.187  1.00 14.37           N  \\nATOM    253  C4   DC B  12      34.893  31.316  12.097  1.00 11.85           C  \\nATOM    254  N4   DC B  12      34.397  31.868  10.987  1.00 17.73           N  \\nATOM    255  C5   DC B  12      35.351  32.092  13.152  1.00 12.20           C  \\nATOM    256  C6   DC B  12      35.811  31.425  14.209  1.00 15.76           C  \\nATOM    257 HO3'  DC B  12      38.451  28.385  18.677  1.00  0.00           H  \\nATOM    258  H41  DC B  12      34.400  32.875  10.890  1.00  0.00           H  \\nATOM    259  H42  DC B  12      34.013  31.291  10.256  1.00  0.00           H  \\nTER     260       DC B  12                                                      \\nHETATM  261  O   HOH A  13      28.213  22.100  27.530  1.00 42.22           O  \\nHETATM  262  O   HOH A  14      26.586  21.386   7.132  1.00 44.58           O  \\nHETATM  263  O   HOH A  15      21.619  32.763  24.469  1.00 36.66           O  \\nHETATM  264  O   HOH A  16      28.321  21.893  10.903  1.00 34.10           O  \\nHETATM  265  O   HOH A  20      22.484  26.473  28.343  1.00 38.32           O  \\nHETATM  266  O   HOH A  21      19.359  34.276  23.539  1.00 31.79           O  \\nHETATM  267  O   HOH A  22      20.296  23.598   9.052  1.00 39.71           O  \\nHETATM  268  O   HOH A  24      28.013  24.110  25.005  1.00 29.37           O  \\nHETATM  269  O   HOH A  30      29.595  24.355  16.471  1.00 35.57           O  \\nHETATM  270  O   HOH A  31      34.129  23.669  11.033  1.00 30.44           O  \\nHETATM  271  O   HOH A  32      26.296  25.396  21.262  1.00 22.48           O  \\nHETATM  272  O   HOH A  33      29.926  21.907   4.334  1.00 43.21           O  \\nHETATM  273  O   HOH A  34      25.342  21.361  22.578  1.00 27.71           O  \\nHETATM  274  O   HOH B  17      21.384  39.724  15.164  1.00 37.35           O  \\nHETATM  275  O   HOH B  18      15.873  22.366  19.329  1.00 48.06           O  \\nHETATM  276  O   HOH B  19      18.356  32.435  15.970  1.00 25.93           O  \\nHETATM  277  O   HOH B  23      41.100  27.441  15.831  1.00 27.63           O  \\nHETATM  278  O   HOH B  25      22.484  34.287  11.145  1.00 40.90           O  \\nHETATM  279  O   HOH B  26      12.592  34.741  18.612  1.00 23.80           O  \\nHETATM  280  O   HOH B  27      11.090  39.558  17.897  1.00 41.31           O  \\nHETATM  281  O   HOH B  28      13.559  32.507  19.884  1.00 29.78           O  \\nHETATM  282  O   HOH B  29      34.412  35.088  10.781  1.00 21.68           O  \\nHETATM  283  O   HOH B  35      19.923  40.404  18.291  1.00 31.79           O  \\nHETATM  284  O   HOH B  36      18.633  37.421  10.594  1.00 28.62           O  \\nHETATM  285  O   HOH B  37      41.530  30.514  14.417  1.00 36.59           O  \\nHETATM  286  O   HOH B  38      26.648  34.690  11.974  1.00 32.48           O  \\nMASTER      362    0    0    0    0    0    0    6  284    2    0    2          \\nEND                                                                             \\n\");\n\tviewer_1684696742117703.setStyle({\"model\": -1, \"chain\": \"B\"},{\"cartoon\": {\"color\": \"spectrum\"}});\n\tviewer_1684696742117703.zoomTo();\nviewer_1684696742117703.render();\n});\n</script>",
      "text/html": [
       "<div id=\"3dmolviewer_1684696742117703\"  style=\"position: relative; width: 600px; height: 400px\">\n",
       "        <p id=\"3dmolwarning_1684696742117703\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
       "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
       "        </div>\n",
       "<script>\n",
       "\n",
       "var loadScriptAsync = function(uri){\n",
       "  return new Promise((resolve, reject) => {\n",
       "    //this is to ignore the existence of requirejs amd\n",
       "    var savedexports, savedmodule;\n",
       "    if (typeof exports !== 'undefined') savedexports = exports;\n",
       "    else exports = {}\n",
       "    if (typeof module !== 'undefined') savedmodule = module;\n",
       "    else module = {}\n",
       "\n",
       "    var tag = document.createElement('script');\n",
       "    tag.src = uri;\n",
       "    tag.async = true;\n",
       "    tag.onload = () => {\n",
       "        exports = savedexports;\n",
       "        module = savedmodule;\n",
       "        resolve();\n",
       "    };\n",
       "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
       "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
       "});\n",
       "};\n",
       "\n",
       "if(typeof $3Dmolpromise === 'undefined') {\n",
       "$3Dmolpromise = null;\n",
       "  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.0.1/3Dmol-min.js');\n",
       "}\n",
       "\n",
       "var viewer_1684696742117703 = null;\n",
       "var warn = document.getElementById(\"3dmolwarning_1684696742117703\");\n",
       "if(warn) {\n",
       "    warn.parentNode.removeChild(warn);\n",
       "}\n",
       "$3Dmolpromise.then(function() {\n",
       "viewer_1684696742117703 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_1684696742117703\"),{backgroundColor:\"white\"});\n",
       "viewer_1684696742117703.zoomTo();\n",
       "\tviewer_1684696742117703.addModelsAsFrames(\"HEADER    DNA                                     16-FEB-95   200D              \\nTITLE     STABLE LOOP IN THE CRYSTAL STRUCTURE OF THE INTERCALATED              \\nTITLE    2 FOUR-STRANDED CYTOSINE-RICH METAZOAN TELOMERE                        \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: DNA (5'-D(*TP*AP*AP*CP*CP*C)-3');                          \\nCOMPND   3 CHAIN: A, B;                                                         \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 SYNTHETIC: YES                                                       \\nKEYWDS    U-DNA, QUADRUPLE HELIX, TETRAPLEX, BASE INTERCALATED, LOOP            \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    C.KANG,I.BERGER,C.LOCKSHIN,R.RATLIFF,R.MOYZIS,A.RICH                  \\nREVDAT   2   24-FEB-09 200D    1       VERSN                                    \\nREVDAT   1   31-JUL-95 200D    0                                                \\nJRNL        AUTH   C.KANG,I.BERGER,C.LOCKSHIN,R.RATLIFF,R.MOYZIS,               \\nJRNL        AUTH 2 A.RICH                                                       \\nJRNL        TITL   STABLE LOOP IN THE CRYSTAL STRUCTURE OF THE                  \\nJRNL        TITL 2 INTERCALATED FOUR-STRANDED CYTOSINE-RICH METAZOAN            \\nJRNL        TITL 3 TELOMERE.                                                    \\nJRNL        REF    PROC.NATL.ACAD.SCI.USA        V.  92  3874 1995              \\nJRNL        REFN                   ISSN 0027-8424                               \\nJRNL        PMID   7731999                                                      \\nJRNL        DOI    10.1073/PNAS.92.9.3874                                       \\nREMARK   1                                                                      \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    1.85 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : X-PLOR                                               \\nREMARK   3   AUTHORS     : BRUNGER                                              \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 1.85                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 12.00                          \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : 1.000                          \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : 77.0                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 2122                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : 0.190                           \\nREMARK   3   FREE R VALUE                     : 0.223                           \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 0                                       \\nREMARK   3   NUCLEIC ACID ATOMS       : 258                                     \\nREMARK   3   HETEROGEN ATOMS          : 0                                       \\nREMARK   3   SOLVENT ATOMS            : 26                                      \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : 0.030                           \\nREMARK   3   BOND ANGLES            (DEGREES) : 4.00                            \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 200D COMPLIES WITH FORMAT V. 3.15, 01-DEC-08                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : 20-JUL-94                          \\nREMARK 200  TEMPERATURE           (KELVIN) : 277.00                             \\nREMARK 200  PH                             : 6.00                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : N                                  \\nREMARK 200  RADIATION SOURCE               : ROTATING ANODE                     \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : M                                  \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : IMAGE PLATE                        \\nREMARK 200  DETECTOR MANUFACTURER          : RIGAKU RAXIS IIC                   \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : MSC                                \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : NULL                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : NULL                               \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY                : NULL                               \\nREMARK 200  R MERGE                    (I) : 0.04500                            \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): NULL                                      \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): NULL                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: PH 6.00, VAPOR DIFFUSION                 \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: F 2 2 2                          \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -X,-Y,Z                                                 \\nREMARK 290       3555   -X,Y,-Z                                                 \\nREMARK 290       4555   X,-Y,-Z                                                 \\nREMARK 290       5555   X,Y+1/2,Z+1/2                                           \\nREMARK 290       6555   -X,-Y+1/2,Z+1/2                                         \\nREMARK 290       7555   -X,Y+1/2,-Z+1/2                                         \\nREMARK 290       8555   X,-Y+1/2,-Z+1/2                                         \\nREMARK 290       9555   X+1/2,Y,Z+1/2                                           \\nREMARK 290      10555   -X+1/2,-Y,Z+1/2                                         \\nREMARK 290      11555   -X+1/2,Y,-Z+1/2                                         \\nREMARK 290      12555   X+1/2,-Y,-Z+1/2                                         \\nREMARK 290      13555   X+1/2,Y+1/2,Z                                           \\nREMARK 290      14555   -X+1/2,-Y+1/2,Z                                         \\nREMARK 290      15555   -X+1/2,Y+1/2,-Z                                         \\nREMARK 290      16555   X+1/2,-Y+1/2,-Z                                         \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   3 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   3  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   4  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   4  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   5  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   5  0.000000  1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3   5  0.000000  0.000000  1.000000       13.43000            \\nREMARK 290   SMTRY1   6 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   6  0.000000 -1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3   6  0.000000  0.000000  1.000000       13.43000            \\nREMARK 290   SMTRY1   7 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   7  0.000000  1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3   7  0.000000  0.000000 -1.000000       13.43000            \\nREMARK 290   SMTRY1   8  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   8  0.000000 -1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3   8  0.000000  0.000000 -1.000000       13.43000            \\nREMARK 290   SMTRY1   9  1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2   9  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   9  0.000000  0.000000  1.000000       13.43000            \\nREMARK 290   SMTRY1  10 -1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  10  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3  10  0.000000  0.000000  1.000000       13.43000            \\nREMARK 290   SMTRY1  11 -1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  11  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3  11  0.000000  0.000000 -1.000000       13.43000            \\nREMARK 290   SMTRY1  12  1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  12  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3  12  0.000000  0.000000 -1.000000       13.43000            \\nREMARK 290   SMTRY1  13  1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  13  0.000000  1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3  13  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1  14 -1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  14  0.000000 -1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3  14  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1  15 -1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  15  0.000000  1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3  15  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1  16  1.000000  0.000000  0.000000       29.97000            \\nREMARK 290   SMTRY2  16  0.000000 -1.000000  0.000000       40.66500            \\nREMARK 290   SMTRY3  16  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: TETRAMERIC                        \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A, B                                  \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2 -1.000000  0.000000  0.000000       59.94000            \\nREMARK 350   BIOMT2   2  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000 -1.000000       26.86000            \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND LENGTHS                                      \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,2(A3,1X,A1,I4,A1,1X,A4,3X),1X,F6.3)               \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   RES CSSEQI ATM2   DEVIATION                     \\nREMARK 500     DT A   1   C5     DT A   1   C7      0.038                       \\nREMARK 500     DT A   1   O3'    DA A   2   P      -0.084                       \\nREMARK 500     DA A   3   C4'    DA A   3   C3'    -0.094                       \\nREMARK 500     DA A   3   N9     DA A   3   C4     -0.042                       \\nREMARK 500     DC A   4   N3     DC A   4   C4     -0.047                       \\nREMARK 500     DC A   5   C3'    DC A   5   C2'    -0.055                       \\nREMARK 500     DC A   6   C5'    DC A   6   C4'     0.090                       \\nREMARK 500     DC A   6   C3'    DC A   6   C2'    -0.050                       \\nREMARK 500     DC A   6   N3     DC A   6   C4     -0.052                       \\nREMARK 500     DT B   7   C5'    DT B   7   C4'     0.082                       \\nREMARK 500     DT B   7   C2     DT B   7   N3     -0.049                       \\nREMARK 500     DA B   8   P      DA B   8   O5'     0.063                       \\nREMARK 500     DC B  10   C4'    DC B  10   C3'    -0.083                       \\nREMARK 500     DC B  10   N1     DC B  10   C6     -0.063                       \\nREMARK 500     DC B  10   C4     DC B  10   C5     -0.050                       \\nREMARK 500     DC B  11   N1     DC B  11   C2     -0.094                       \\nREMARK 500     DC B  11   N1     DC B  11   C6     -0.040                       \\nREMARK 500     DC B  11   C2     DC B  11   N3     -0.056                       \\nREMARK 500     DC B  12   O4'    DC B  12   C4'    -0.069                       \\nREMARK 500     DC B  12   N1     DC B  12   C2     -0.062                       \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \\nREMARK 500     DT A   1   O4' -  C1' -  N1  ANGL. DEV. =   2.9 DEGREES          \\nREMARK 500     DT A   1   C6  -  C5  -  C7  ANGL. DEV. =  -4.4 DEGREES          \\nREMARK 500     DA A   2   O4' -  C4' -  C3' ANGL. DEV. =   4.3 DEGREES          \\nREMARK 500     DA A   2   C4' -  C3' -  C2' ANGL. DEV. =  -9.1 DEGREES          \\nREMARK 500     DA A   2   O4' -  C1' -  C2' ANGL. DEV. =  -7.8 DEGREES          \\nREMARK 500     DA A   2   N1  -  C6  -  N6  ANGL. DEV. =   3.7 DEGREES          \\nREMARK 500     DT A   1   C3' -  O3' -  P   ANGL. DEV. =  14.8 DEGREES          \\nREMARK 500     DA A   2   O3' -  P   -  O5' ANGL. DEV. = -11.7 DEGREES          \\nREMARK 500     DA A   3   O4' -  C4' -  C3' ANGL. DEV. =  -7.7 DEGREES          \\nREMARK 500     DA A   3   C1' -  O4' -  C4' ANGL. DEV. =   6.1 DEGREES          \\nREMARK 500     DA A   3   O4' -  C1' -  C2' ANGL. DEV. =  -6.8 DEGREES          \\nREMARK 500     DA A   3   N1  -  C2  -  N3  ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500     DA A   3   C2  -  N3  -  C4  ANGL. DEV. =  -3.9 DEGREES          \\nREMARK 500     DA A   2   C3' -  O3' -  P   ANGL. DEV. =   7.9 DEGREES          \\nREMARK 500     DC A   4   C4' -  C3' -  C2' ANGL. DEV. =  -4.3 DEGREES          \\nREMARK 500     DC A   5   C4' -  C3' -  C2' ANGL. DEV. =  -7.7 DEGREES          \\nREMARK 500     DC A   5   O4' -  C1' -  N1  ANGL. DEV. =  -4.7 DEGREES          \\nREMARK 500     DC A   5   C2  -  N3  -  C4  ANGL. DEV. =   3.5 DEGREES          \\nREMARK 500     DC A   6   P   -  O5' -  C5' ANGL. DEV. =  13.2 DEGREES          \\nREMARK 500     DC A   6   O4' -  C4' -  C3' ANGL. DEV. =  -4.4 DEGREES          \\nREMARK 500     DC A   6   O4' -  C1' -  N1  ANGL. DEV. =   3.9 DEGREES          \\nREMARK 500     DC A   6   N1  -  C2  -  O2  ANGL. DEV. =   3.9 DEGREES          \\nREMARK 500     DC A   6   N3  -  C4  -  N4  ANGL. DEV. =  -5.0 DEGREES          \\nREMARK 500     DC A   6   C5  -  C4  -  N4  ANGL. DEV. =   4.2 DEGREES          \\nREMARK 500     DT B   7   C5' -  C4' -  O4' ANGL. DEV. =   9.2 DEGREES          \\nREMARK 500     DT B   7   N1  -  C2  -  O2  ANGL. DEV. =   5.3 DEGREES          \\nREMARK 500     DT B   7   N3  -  C2  -  O2  ANGL. DEV. =  -4.3 DEGREES          \\nREMARK 500     DA B   8   O4' -  C1' -  C2' ANGL. DEV. =  -5.1 DEGREES          \\nREMARK 500     DA B   8   C2  -  N3  -  C4  ANGL. DEV. =  -3.3 DEGREES          \\nREMARK 500     DA B   8   C5  -  C6  -  N1  ANGL. DEV. =  -4.0 DEGREES          \\nREMARK 500     DT B   7   C3' -  O3' -  P   ANGL. DEV. = -10.9 DEGREES          \\nREMARK 500     DA B   9   O4' -  C1' -  C2' ANGL. DEV. =  -7.3 DEGREES          \\nREMARK 500     DC B  10   C1' -  O4' -  C4' ANGL. DEV. =   6.5 DEGREES          \\nREMARK 500     DC B  10   O4' -  C1' -  C2' ANGL. DEV. = -10.3 DEGREES          \\nREMARK 500     DC B  10   O4' -  C1' -  N1  ANGL. DEV. =   3.6 DEGREES          \\nREMARK 500     DC B  10   N3  -  C2  -  O2  ANGL. DEV. =  -5.1 DEGREES          \\nREMARK 500     DC B  10   C3' -  O3' -  P   ANGL. DEV. =   9.2 DEGREES          \\nREMARK 500     DC B  12   O4' -  C1' -  C2' ANGL. DEV. =  -5.1 DEGREES          \\nREMARK 500     DC B  12   C2  -  N3  -  C4  ANGL. DEV. =   4.6 DEGREES          \\nREMARK 500     DC B  12   N1  -  C2  -  O2  ANGL. DEV. =   4.6 DEGREES          \\nREMARK 500     DC B  11   C3' -  O3' -  P   ANGL. DEV. =   8.3 DEGREES          \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: PLANAR GROUPS                                              \\nREMARK 500                                                                      \\nREMARK 500 PLANAR GROUPS IN THE FOLLOWING RESIDUES HAVE A TOTAL                 \\nREMARK 500 RMS DISTANCE OF ALL ATOMS FROM THE BEST-FIT PLANE                    \\nREMARK 500 BY MORE THAN AN EXPECTED VALUE OF 6*RMSD, WITH AN                    \\nREMARK 500 RMSD 0.02 ANGSTROMS, OR AT LEAST ONE ATOM HAS                        \\nREMARK 500 AN RMSD GREATER THAN THIS VALUE                                      \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        RMS     TYPE                                    \\nREMARK 500     DA A   3         0.06    SIDE_CHAIN                              \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nDBREF  200D A    1     6  PDB    200D     200D             1      6             \\nDBREF  200D B    7    12  PDB    200D     200D             7     12             \\nSEQRES   1 A    6   DT  DA  DA  DC  DC  DC                                      \\nSEQRES   1 B    6   DT  DA  DA  DC  DC  DC                                      \\nFORMUL   3  HOH   *26(H2 O)                                                     \\nCRYST1   59.940   81.330   26.860  90.00  90.00  90.00 F 2 2 2      32          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.016683  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  0.012296  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.037230        0.00000                         \\nATOM      1  O5'  DT A   1      24.565  31.319  24.443  1.00 30.44           O  \\nATOM      2  C5'  DT A   1      23.942  30.467  25.330  1.00 20.89           C  \\nATOM      3  C4'  DT A   1      23.216  29.306  24.650  1.00 27.77           C  \\nATOM      4  O4'  DT A   1      22.175  29.736  23.812  1.00 23.30           O  \\nATOM      5  C3'  DT A   1      24.157  28.454  23.852  1.00 35.38           C  \\nATOM      6  O3'  DT A   1      23.905  27.189  24.366  1.00 33.40           O  \\nATOM      7  C2'  DT A   1      23.531  28.590  22.502  1.00 34.88           C  \\nATOM      8  C1'  DT A   1      22.104  28.825  22.807  1.00 18.33           C  \\nATOM      9  N1   DT A   1      21.415  29.371  21.650  1.00 27.22           N  \\nATOM     10  C2   DT A   1      21.061  28.462  20.735  1.00 17.62           C  \\nATOM     11  O2   DT A   1      21.294  27.274  20.916  1.00 23.45           O  \\nATOM     12  N3   DT A   1      20.409  28.945  19.635  1.00 20.47           N  \\nATOM     13  C4   DT A   1      20.071  30.242  19.367  1.00 19.02           C  \\nATOM     14  O4   DT A   1      19.434  30.520  18.346  1.00 27.67           O  \\nATOM     15  C5   DT A   1      20.491  31.148  20.397  1.00 28.05           C  \\nATOM     16  C7   DT A   1      20.220  32.654  20.284  1.00 24.86           C  \\nATOM     17  C6   DT A   1      21.136  30.694  21.482  1.00 14.22           C  \\nATOM     18  H3   DT A   1      20.149  28.273  18.939  1.00  0.00           H  \\nATOM     19 HO5'  DT A   1      25.263  30.835  24.004  1.00  0.00           H  \\nATOM     20  P    DA A   2      24.766  25.974  24.683  1.00 23.53           P  \\nATOM     21  OP1  DA A   2      24.819  25.775  26.131  1.00 27.65           O  \\nATOM     22  OP2  DA A   2      25.964  25.990  23.827  1.00 38.21           O  \\nATOM     23  O5'  DA A   2      23.749  24.991  23.979  1.00 23.31           O  \\nATOM     24  C5'  DA A   2      22.350  25.133  24.144  1.00 31.45           C  \\nATOM     25  C4'  DA A   2      21.502  24.042  23.575  1.00 10.83           C  \\nATOM     26  O4'  DA A   2      20.221  24.252  24.204  1.00 36.32           O  \\nATOM     27  C3'  DA A   2      21.353  24.038  22.059  1.00 39.61           C  \\nATOM     28  O3'  DA A   2      22.218  23.042  21.474  1.00 19.31           O  \\nATOM     29  C2'  DA A   2      19.926  23.509  22.118  1.00 23.91           C  \\nATOM     30  C1'  DA A   2      19.171  24.220  23.257  1.00 29.47           C  \\nATOM     31  N9   DA A   2      18.839  25.627  23.010  1.00 19.13           N  \\nATOM     32  C8   DA A   2      19.151  26.734  23.759  1.00 21.05           C  \\nATOM     33  N7   DA A   2      18.737  27.848  23.267  1.00 20.27           N  \\nATOM     34  C5   DA A   2      18.093  27.426  22.100  1.00 14.04           C  \\nATOM     35  C6   DA A   2      17.391  28.116  21.124  1.00 18.64           C  \\nATOM     36  N6   DA A   2      17.250  29.453  21.149  1.00 24.64           N  \\nATOM     37  N1   DA A   2      16.861  27.359  20.157  1.00 15.56           N  \\nATOM     38  C2   DA A   2      17.016  26.046  20.152  1.00  9.93           C  \\nATOM     39  N3   DA A   2      17.641  25.306  20.992  1.00 23.82           N  \\nATOM     40  C4   DA A   2      18.160  26.078  21.948  1.00 11.68           C  \\nATOM     41  H61  DA A   2      16.745  29.923  20.410  1.00  0.00           H  \\nATOM     42  H62  DA A   2      17.644  29.994  21.903  1.00  0.00           H  \\nATOM     43  P    DA A   3      23.030  23.149  20.153  1.00 20.46           P  \\nATOM     44  OP1  DA A   3      23.954  22.013  20.095  1.00 24.95           O  \\nATOM     45  OP2  DA A   3      23.531  24.509  20.030  1.00 19.53           O  \\nATOM     46  O5'  DA A   3      21.880  22.885  19.163  1.00 17.18           O  \\nATOM     47  C5'  DA A   3      21.278  21.604  19.070  1.00 13.90           C  \\nATOM     48  C4'  DA A   3      20.159  21.601  18.065  1.00 21.24           C  \\nATOM     49  O4'  DA A   3      19.337  22.719  18.308  1.00 21.96           O  \\nATOM     50  C3'  DA A   3      20.581  21.983  16.756  1.00 22.35           C  \\nATOM     51  O3'  DA A   3      21.091  20.871  16.039  1.00 35.11           O  \\nATOM     52  C2'  DA A   3      19.323  22.493  16.117  1.00 36.04           C  \\nATOM     53  C1'  DA A   3      18.685  23.288  17.172  1.00 20.51           C  \\nATOM     54  N9   DA A   3      18.982  24.750  17.163  1.00 14.73           N  \\nATOM     55  C8   DA A   3      19.499  25.507  18.166  1.00  8.55           C  \\nATOM     56  N7   DA A   3      19.478  26.769  17.943  1.00 16.60           N  \\nATOM     57  C5   DA A   3      18.913  26.871  16.693  1.00 12.55           C  \\nATOM     58  C6   DA A   3      18.601  27.989  15.881  1.00 20.70           C  \\nATOM     59  N6   DA A   3      18.741  29.276  16.217  1.00 14.67           N  \\nATOM     60  N1   DA A   3      18.060  27.707  14.701  1.00 13.89           N  \\nATOM     61  C2   DA A   3      17.850  26.408  14.389  1.00 29.53           C  \\nATOM     62  N3   DA A   3      18.083  25.275  15.052  1.00 18.60           N  \\nATOM     63  C4   DA A   3      18.625  25.618  16.218  1.00  8.22           C  \\nATOM     64  H61  DA A   3      18.524  30.004  15.551  1.00  0.00           H  \\nATOM     65  H62  DA A   3      19.059  29.517  17.142  1.00  0.00           H  \\nATOM     66  P    DC A   4      22.425  21.041  15.096  1.00 24.23           P  \\nATOM     67  OP1  DC A   4      22.465  19.788  14.318  1.00 29.54           O  \\nATOM     68  OP2  DC A   4      23.569  21.448  15.939  1.00 26.74           O  \\nATOM     69  O5'  DC A   4      22.106  22.295  14.117  1.00 24.38           O  \\nATOM     70  C5'  DC A   4      21.326  22.363  12.942  1.00 23.31           C  \\nATOM     71  C4'  DC A   4      21.432  23.763  12.409  1.00 19.02           C  \\nATOM     72  O4'  DC A   4      20.924  24.622  13.375  1.00 13.89           O  \\nATOM     73  C3'  DC A   4      22.882  24.127  12.138  1.00 15.94           C  \\nATOM     74  O3'  DC A   4      23.153  23.807  10.751  1.00 33.78           O  \\nATOM     75  C2'  DC A   4      22.777  25.612  12.409  1.00 26.18           C  \\nATOM     76  C1'  DC A   4      21.638  25.802  13.441  1.00  6.99           C  \\nATOM     77  N1   DC A   4      22.071  25.996  14.858  1.00 15.40           N  \\nATOM     78  C2   DC A   4      22.052  27.275  15.345  1.00 14.75           C  \\nATOM     79  O2   DC A   4      21.597  28.208  14.679  1.00 20.98           O  \\nATOM     80  N3   DC A   4      22.492  27.460  16.606  1.00 19.60           N  \\nATOM     81  C4   DC A   4      22.910  26.491  17.345  1.00  7.46           C  \\nATOM     82  N4   DC A   4      23.363  26.803  18.516  1.00 11.08           N  \\nATOM     83  C5   DC A   4      22.926  25.188  16.895  1.00  7.21           C  \\nATOM     84  C6   DC A   4      22.504  24.981  15.653  1.00  9.76           C  \\nATOM     85  H41  DC A   4      23.402  27.768  18.806  1.00  0.00           H  \\nATOM     86  H42  DC A   4      23.675  26.076  19.149  1.00  0.00           H  \\nATOM     87  P    DC A   5      24.446  22.970  10.188  1.00 28.52           P  \\nATOM     88  OP1  DC A   5      24.144  22.120   9.020  1.00 24.29           O  \\nATOM     89  OP2  DC A   5      25.053  22.349  11.400  1.00 34.89           O  \\nATOM     90  O5'  DC A   5      25.373  24.138   9.643  1.00 30.47           O  \\nATOM     91  C5'  DC A   5      24.837  25.048   8.687  1.00 27.30           C  \\nATOM     92  C4'  DC A   5      25.648  26.351   8.642  1.00 20.98           C  \\nATOM     93  O4'  DC A   5      25.627  26.952   9.905  1.00 21.19           O  \\nATOM     94  C3'  DC A   5      27.100  26.119   8.375  1.00 20.71           C  \\nATOM     95  O3'  DC A   5      27.341  25.805   7.002  1.00 23.11           O  \\nATOM     96  C2'  DC A   5      27.496  27.486   8.703  1.00 13.44           C  \\nATOM     97  C1'  DC A   5      26.696  27.830   9.919  1.00 13.82           C  \\nATOM     98  N1   DC A   5      27.308  27.595  11.235  1.00 22.24           N  \\nATOM     99  C2   DC A   5      27.581  28.683  11.989  1.00  9.58           C  \\nATOM    100  O2   DC A   5      27.383  29.828  11.573  1.00 25.04           O  \\nATOM    101  N3   DC A   5      28.050  28.457  13.220  1.00 23.52           N  \\nATOM    102  C4   DC A   5      28.242  27.259  13.710  1.00  9.77           C  \\nATOM    103  N4   DC A   5      28.680  27.143  14.947  1.00 20.94           N  \\nATOM    104  C5   DC A   5      27.978  26.108  12.961  1.00 15.31           C  \\nATOM    105  C6   DC A   5      27.515  26.334  11.726  1.00 20.31           C  \\nATOM    106  H41  DC A   5      28.833  26.230  15.355  1.00  0.00           H  \\nATOM    107  H42  DC A   5      28.860  27.971  15.496  1.00  0.00           H  \\nATOM    108  P    DC A   6      28.699  25.257   6.381  1.00 26.27           P  \\nATOM    109  OP1  DC A   6      28.416  24.653   5.094  1.00 29.77           O  \\nATOM    110  OP2  DC A   6      29.457  24.544   7.410  1.00 23.78           O  \\nATOM    111  O5'  DC A   6      29.520  26.539   6.046  1.00 32.49           O  \\nATOM    112  C5'  DC A   6      29.282  27.692   5.226  1.00 30.24           C  \\nATOM    113  C4'  DC A   6      30.531  28.674   5.429  1.00 22.63           C  \\nATOM    114  O4'  DC A   6      30.781  28.988   6.818  1.00 18.93           O  \\nATOM    115  C3'  DC A   6      31.823  27.970   5.079  1.00 28.36           C  \\nATOM    116  O3'  DC A   6      32.102  28.065   3.709  1.00 27.07           O  \\nATOM    117  C2'  DC A   6      32.812  28.770   5.808  1.00 13.00           C  \\nATOM    118  C1'  DC A   6      32.139  29.157   7.068  1.00  9.92           C  \\nATOM    119  N1   DC A   6      32.607  28.338   8.228  1.00 22.07           N  \\nATOM    120  C2   DC A   6      33.155  29.092   9.255  1.00 26.49           C  \\nATOM    121  O2   DC A   6      33.154  30.333   9.252  1.00 26.06           O  \\nATOM    122  N3   DC A   6      33.673  28.433  10.285  1.00 17.53           N  \\nATOM    123  C4   DC A   6      33.677  27.152  10.362  1.00  7.84           C  \\nATOM    124  N4   DC A   6      34.255  26.711  11.432  1.00 17.65           N  \\nATOM    125  C5   DC A   6      33.130  26.339   9.369  1.00  9.69           C  \\nATOM    126  C6   DC A   6      32.596  26.974   8.310  1.00 25.15           C  \\nATOM    127 HO3'  DC A   6      31.406  27.590   3.246  1.00  0.00           H  \\nATOM    128  H41  DC A   6      34.358  25.716  11.590  1.00  0.00           H  \\nATOM    129  H42  DC A   6      34.605  27.355  12.121  1.00  0.00           H  \\nTER     130       DC A   6                                                      \\nATOM    131  O5'  DT B   7      11.293  29.851  15.858  1.00 28.85           O  \\nATOM    132  C5'  DT B   7      11.549  31.182  16.395  1.00 25.35           C  \\nATOM    133  C4'  DT B   7      12.980  31.877  16.290  1.00 39.66           C  \\nATOM    134  O4'  DT B   7      14.126  31.292  17.044  1.00 37.54           O  \\nATOM    135  C3'  DT B   7      13.456  31.813  14.854  1.00 39.01           C  \\nATOM    136  O3'  DT B   7      14.165  32.997  14.544  1.00 38.98           O  \\nATOM    137  C2'  DT B   7      14.424  30.692  14.753  1.00 20.58           C  \\nATOM    138  C1'  DT B   7      14.994  30.617  16.125  1.00 20.74           C  \\nATOM    139  N1   DT B   7      15.053  29.255  16.540  1.00 18.62           N  \\nATOM    140  C2   DT B   7      15.671  29.004  17.731  1.00 20.33           C  \\nATOM    141  O2   DT B   7      16.113  29.817  18.554  1.00 26.72           O  \\nATOM    142  N3   DT B   7      15.786  27.715  18.009  1.00 22.63           N  \\nATOM    143  C4   DT B   7      15.361  26.621  17.272  1.00 33.64           C  \\nATOM    144  O4   DT B   7      15.606  25.447  17.626  1.00 34.71           O  \\nATOM    145  C5   DT B   7      14.708  26.988  16.073  1.00 14.16           C  \\nATOM    146  C7   DT B   7      14.102  25.888  15.216  1.00 39.75           C  \\nATOM    147  C6   DT B   7      14.575  28.259  15.732  1.00 24.49           C  \\nATOM    148  H3   DT B   7      16.244  27.571  18.880  1.00  0.00           H  \\nATOM    149 HO5'  DT B   7      11.404  29.913  14.905  1.00  0.00           H  \\nATOM    150  P    DA B   8      13.931  33.345  13.024  1.00 39.22           P  \\nATOM    151  OP1  DA B   8      12.612  34.036  13.039  1.00 34.77           O  \\nATOM    152  OP2  DA B   8      14.206  32.074  12.263  1.00 39.31           O  \\nATOM    153  O5'  DA B   8      15.000  34.463  12.432  1.00 39.80           O  \\nATOM    154  C5'  DA B   8      16.303  34.113  11.867  1.00 39.02           C  \\nATOM    155  C4'  DA B   8      17.406  35.058  12.322  1.00 17.13           C  \\nATOM    156  O4'  DA B   8      16.819  36.368  12.502  1.00 22.28           O  \\nATOM    157  C3'  DA B   8      18.055  34.660  13.684  1.00 35.33           C  \\nATOM    158  O3'  DA B   8      19.437  35.008  13.398  1.00 39.28           O  \\nATOM    159  C2'  DA B   8      17.310  35.602  14.642  1.00 35.21           C  \\nATOM    160  C1'  DA B   8      17.143  36.915  13.808  1.00 14.63           C  \\nATOM    161  N9   DA B   8      16.003  37.727  14.228  1.00 20.81           N  \\nATOM    162  C8   DA B   8      14.676  37.344  14.239  1.00 37.43           C  \\nATOM    163  N7   DA B   8      13.839  38.247  14.635  1.00 23.05           N  \\nATOM    164  C5   DA B   8      14.647  39.318  14.912  1.00 19.42           C  \\nATOM    165  C6   DA B   8      14.326  40.578  15.398  1.00 14.47           C  \\nATOM    166  N6   DA B   8      13.065  41.064  15.553  1.00 20.96           N  \\nATOM    167  N1   DA B   8      15.373  41.351  15.597  1.00 33.53           N  \\nATOM    168  C2   DA B   8      16.622  40.929  15.326  1.00 27.49           C  \\nATOM    169  N3   DA B   8      17.042  39.778  14.845  1.00 27.07           N  \\nATOM    170  C4   DA B   8      15.968  39.007  14.666  1.00 10.79           C  \\nATOM    171  H61  DA B   8      12.269  40.498  15.302  1.00  0.00           H  \\nATOM    172  H62  DA B   8      12.942  41.998  15.918  1.00  0.00           H  \\nATOM    173  P    DA B   9      20.719  34.445  14.194  1.00 35.42           P  \\nATOM    174  OP1  DA B   9      21.959  35.022  13.557  1.00 39.82           O  \\nATOM    175  OP2  DA B   9      20.528  32.939  14.321  1.00 28.79           O  \\nATOM    176  O5'  DA B   9      20.556  35.250  15.553  1.00 39.94           O  \\nATOM    177  C5'  DA B   9      21.196  36.518  15.727  1.00 27.77           C  \\nATOM    178  C4'  DA B   9      20.828  37.045  17.056  1.00 16.24           C  \\nATOM    179  O4'  DA B   9      19.420  37.260  17.115  1.00 24.83           O  \\nATOM    180  C3'  DA B   9      21.131  36.015  18.071  1.00 22.96           C  \\nATOM    181  O3'  DA B   9      21.775  36.603  19.212  1.00 29.07           O  \\nATOM    182  C2'  DA B   9      19.740  35.550  18.407  1.00 26.00           C  \\nATOM    183  C1'  DA B   9      18.975  36.797  18.352  1.00 18.88           C  \\nATOM    184  N9   DA B   9      17.533  36.628  18.310  1.00 17.60           N  \\nATOM    185  C8   DA B   9      16.740  35.511  18.232  1.00 23.85           C  \\nATOM    186  N7   DA B   9      15.467  35.795  18.264  1.00 25.36           N  \\nATOM    187  C5   DA B   9      15.416  37.185  18.373  1.00 14.91           C  \\nATOM    188  C6   DA B   9      14.358  38.117  18.472  1.00 24.27           C  \\nATOM    189  N6   DA B   9      13.051  37.812  18.565  1.00 30.71           N  \\nATOM    190  N1   DA B   9      14.730  39.429  18.573  1.00 34.89           N  \\nATOM    191  C2   DA B   9      16.025  39.778  18.587  1.00 15.13           C  \\nATOM    192  N3   DA B   9      17.077  38.975  18.506  1.00 18.79           N  \\nATOM    193  C4   DA B   9      16.688  37.686  18.401  1.00 21.13           C  \\nATOM    194  H61  DA B   9      12.726  36.865  18.446  1.00  0.00           H  \\nATOM    195  H62  DA B   9      12.408  38.552  18.781  1.00  0.00           H  \\nATOM    196  P    DC B  10      23.405  36.738  19.343  1.00 28.67           P  \\nATOM    197  OP1  DC B  10      23.679  37.914  20.242  1.00 34.27           O  \\nATOM    198  OP2  DC B  10      23.910  36.731  17.961  1.00 38.84           O  \\nATOM    199  O5'  DC B  10      24.087  35.454  20.011  1.00 18.77           O  \\nATOM    200  C5'  DC B  10      24.087  35.100  21.368  1.00 30.20           C  \\nATOM    201  C4'  DC B  10      24.575  33.695  21.529  1.00  9.80           C  \\nATOM    202  O4'  DC B  10      23.800  32.700  20.879  1.00 18.83           O  \\nATOM    203  C3'  DC B  10      25.878  33.620  20.926  1.00 14.61           C  \\nATOM    204  O3'  DC B  10      26.615  32.959  21.958  1.00 36.36           O  \\nATOM    205  C2'  DC B  10      25.606  32.814  19.678  1.00 19.97           C  \\nATOM    206  C1'  DC B  10      24.481  31.842  19.992  1.00 15.38           C  \\nATOM    207  N1   DC B  10      23.686  31.565  18.773  1.00 11.72           N  \\nATOM    208  C2   DC B  10      23.456  30.308  18.325  1.00 11.51           C  \\nATOM    209  O2   DC B  10      23.748  29.333  18.974  1.00 25.69           O  \\nATOM    210  N3   DC B  10      22.811  30.099  17.201  1.00 14.49           N  \\nATOM    211  C4   DC B  10      22.370  31.109  16.513  1.00 12.31           C  \\nATOM    212  N4   DC B  10      21.662  30.932  15.398  1.00 25.78           N  \\nATOM    213  C5   DC B  10      22.573  32.400  16.939  1.00 14.33           C  \\nATOM    214  C6   DC B  10      23.237  32.570  18.074  1.00 13.01           C  \\nATOM    215  H41  DC B  10      21.523  29.994  15.042  1.00  0.00           H  \\nATOM    216  H42  DC B  10      21.261  31.720  14.909  1.00  0.00           H  \\nATOM    217  P    DC B  11      28.130  33.237  22.467  1.00 34.54           P  \\nATOM    218  OP1  DC B  11      28.322  32.261  23.594  1.00 29.17           O  \\nATOM    219  OP2  DC B  11      28.366  34.724  22.567  1.00 26.05           O  \\nATOM    220  O5'  DC B  11      28.851  32.747  21.146  1.00 29.65           O  \\nATOM    221  C5'  DC B  11      30.240  32.966  20.860  1.00 31.52           C  \\nATOM    222  C4'  DC B  11      30.646  31.847  19.984  1.00 13.84           C  \\nATOM    223  O4'  DC B  11      29.762  31.772  18.892  1.00 25.81           O  \\nATOM    224  C3'  DC B  11      32.016  32.042  19.492  1.00 15.90           C  \\nATOM    225  O3'  DC B  11      32.776  31.224  20.405  1.00 35.61           O  \\nATOM    226  C2'  DC B  11      31.886  31.528  18.055  1.00 19.50           C  \\nATOM    227  C1'  DC B  11      30.453  31.059  17.897  1.00 14.84           C  \\nATOM    228  N1   DC B  11      29.801  31.387  16.621  1.00  8.89           N  \\nATOM    229  C2   DC B  11      29.420  30.373  15.896  1.00 10.51           C  \\nATOM    230  O2   DC B  11      29.691  29.265  16.247  1.00 16.37           O  \\nATOM    231  N3   DC B  11      28.767  30.545  14.789  1.00 14.37           N  \\nATOM    232  C4   DC B  11      28.472  31.744  14.372  1.00  8.47           C  \\nATOM    233  N4   DC B  11      27.746  31.854  13.292  1.00 24.16           N  \\nATOM    234  C5   DC B  11      28.866  32.886  15.096  1.00 19.21           C  \\nATOM    235  C6   DC B  11      29.541  32.630  16.236  1.00 19.62           C  \\nATOM    236  H41  DC B  11      27.427  32.762  12.970  1.00  0.00           H  \\nATOM    237  H42  DC B  11      27.498  31.025  12.770  1.00  0.00           H  \\nATOM    238  P    DC B  12      34.436  31.147  20.578  1.00 35.23           P  \\nATOM    239  OP1  DC B  12      34.764  29.859  21.271  1.00 39.08           O  \\nATOM    240  OP2  DC B  12      34.907  32.477  21.122  1.00 36.25           O  \\nATOM    241  O5'  DC B  12      35.015  30.931  19.062  1.00 30.38           O  \\nATOM    242  C5'  DC B  12      36.391  30.665  18.739  1.00 30.68           C  \\nATOM    243  C4'  DC B  12      36.518  29.663  17.628  1.00 28.20           C  \\nATOM    244  O4'  DC B  12      35.889  30.201  16.528  1.00 29.99           O  \\nATOM    245  C3'  DC B  12      37.900  29.433  17.125  1.00 23.57           C  \\nATOM    246  O3'  DC B  12      38.418  28.232  17.727  1.00 39.77           O  \\nATOM    247  C2'  DC B  12      37.691  29.249  15.673  1.00 21.13           C  \\nATOM    248  C1'  DC B  12      36.254  29.370  15.451  1.00 17.03           C  \\nATOM    249  N1   DC B  12      35.819  30.088  14.254  1.00  9.75           N  \\nATOM    250  C2   DC B  12      35.372  29.340  13.243  1.00 20.35           C  \\nATOM    251  O2   DC B  12      35.274  28.103  13.297  1.00 21.58           O  \\nATOM    252  N3   DC B  12      34.928  30.008  12.187  1.00 14.37           N  \\nATOM    253  C4   DC B  12      34.893  31.316  12.097  1.00 11.85           C  \\nATOM    254  N4   DC B  12      34.397  31.868  10.987  1.00 17.73           N  \\nATOM    255  C5   DC B  12      35.351  32.092  13.152  1.00 12.20           C  \\nATOM    256  C6   DC B  12      35.811  31.425  14.209  1.00 15.76           C  \\nATOM    257 HO3'  DC B  12      38.451  28.385  18.677  1.00  0.00           H  \\nATOM    258  H41  DC B  12      34.400  32.875  10.890  1.00  0.00           H  \\nATOM    259  H42  DC B  12      34.013  31.291  10.256  1.00  0.00           H  \\nTER     260       DC B  12                                                      \\nHETATM  261  O   HOH A  13      28.213  22.100  27.530  1.00 42.22           O  \\nHETATM  262  O   HOH A  14      26.586  21.386   7.132  1.00 44.58           O  \\nHETATM  263  O   HOH A  15      21.619  32.763  24.469  1.00 36.66           O  \\nHETATM  264  O   HOH A  16      28.321  21.893  10.903  1.00 34.10           O  \\nHETATM  265  O   HOH A  20      22.484  26.473  28.343  1.00 38.32           O  \\nHETATM  266  O   HOH A  21      19.359  34.276  23.539  1.00 31.79           O  \\nHETATM  267  O   HOH A  22      20.296  23.598   9.052  1.00 39.71           O  \\nHETATM  268  O   HOH A  24      28.013  24.110  25.005  1.00 29.37           O  \\nHETATM  269  O   HOH A  30      29.595  24.355  16.471  1.00 35.57           O  \\nHETATM  270  O   HOH A  31      34.129  23.669  11.033  1.00 30.44           O  \\nHETATM  271  O   HOH A  32      26.296  25.396  21.262  1.00 22.48           O  \\nHETATM  272  O   HOH A  33      29.926  21.907   4.334  1.00 43.21           O  \\nHETATM  273  O   HOH A  34      25.342  21.361  22.578  1.00 27.71           O  \\nHETATM  274  O   HOH B  17      21.384  39.724  15.164  1.00 37.35           O  \\nHETATM  275  O   HOH B  18      15.873  22.366  19.329  1.00 48.06           O  \\nHETATM  276  O   HOH B  19      18.356  32.435  15.970  1.00 25.93           O  \\nHETATM  277  O   HOH B  23      41.100  27.441  15.831  1.00 27.63           O  \\nHETATM  278  O   HOH B  25      22.484  34.287  11.145  1.00 40.90           O  \\nHETATM  279  O   HOH B  26      12.592  34.741  18.612  1.00 23.80           O  \\nHETATM  280  O   HOH B  27      11.090  39.558  17.897  1.00 41.31           O  \\nHETATM  281  O   HOH B  28      13.559  32.507  19.884  1.00 29.78           O  \\nHETATM  282  O   HOH B  29      34.412  35.088  10.781  1.00 21.68           O  \\nHETATM  283  O   HOH B  35      19.923  40.404  18.291  1.00 31.79           O  \\nHETATM  284  O   HOH B  36      18.633  37.421  10.594  1.00 28.62           O  \\nHETATM  285  O   HOH B  37      41.530  30.514  14.417  1.00 36.59           O  \\nHETATM  286  O   HOH B  38      26.648  34.690  11.974  1.00 32.48           O  \\nMASTER      362    0    0    0    0    0    0    6  284    2    0    2          \\nEND                                                                             \\n\");\n",
       "\tviewer_1684696742117703.setStyle({\"model\": -1, \"chain\": \"B\"},{\"cartoon\": {\"color\": \"spectrum\"}});\n",
       "\tviewer_1684696742117703.zoomTo();\n",
       "viewer_1684696742117703.render();\n",
       "});\n",
       "</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Similarity score: 10.0\n",
      " Protein: structure of a dna in low salt conditions d(gaccgcggtc)(401D)\n",
      " Chain Id: A\n",
      " Sequence of the chain: DGDADCDCDGDCDGDGDTDC\n"
     ]
    },
    {
     "data": {
      "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_1684696742122504\"  style=\"position: relative; width: 600px; height: 400px\">\n        <p id=\"3dmolwarning_1684696742122504\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.0.1/3Dmol-min.js');\n}\n\nvar viewer_1684696742122504 = null;\nvar warn = document.getElementById(\"3dmolwarning_1684696742122504\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_1684696742122504 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_1684696742122504\"),{backgroundColor:\"white\"});\nviewer_1684696742122504.zoomTo();\n\tviewer_1684696742122504.addModelsAsFrames(\"HEADER    DNA                                     02-JUN-98   401D              \\nTITLE     STRUCTURE OF A DNA IN LOW SALT CONDITIONS D(GACCGCGGTC)               \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: DNA (5'-D(*GP*AP*CP*CP*GP*CP*GP*GP*TP*C)-3');              \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 SYNTHETIC: YES                                                       \\nKEYWDS    DNA DOUBLE HELIX, DNA                                                 \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    M.LUO,J.B.FINLEY                                                      \\nREVDAT   4   02-MAR-10 401D    1       HETATM                                   \\nREVDAT   3   24-FEB-09 401D    1       VERSN                                    \\nREVDAT   2   01-APR-03 401D    1       JRNL                                     \\nREVDAT   1   06-JUL-98 401D    0                                                \\nJRNL        AUTH   J.B.FINLEY,M.LUO                                             \\nJRNL        TITL   X-RAY CRYSTAL STRUCTURES OF HALF THE HUMAN                   \\nJRNL        TITL 2 PAPILLOMA VIRUS E2 BINDING SITE: D(GACCGCGGTC).              \\nJRNL        REF    NUCLEIC ACIDS RES.            V.  26  5719 1998              \\nJRNL        REFN                   ISSN 0305-1048                               \\nJRNL        PMID   9838004                                                      \\nJRNL        DOI    10.1093/NAR/26.24.5719                                       \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    2.20 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : X-PLOR                                               \\nREMARK   3   AUTHORS     : BRUNGER                                              \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 2.20                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 5.00                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : 3.000                          \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 1779                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : 0.219                           \\nREMARK   3   FREE R VALUE                     : 0.285                           \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 0                                       \\nREMARK   3   NUCLEIC ACID ATOMS       : 227                                     \\nREMARK   3   HETEROGEN ATOMS          : 0                                       \\nREMARK   3   SOLVENT ATOMS            : 102                                     \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : 0.005                           \\nREMARK   3   BOND ANGLES            (DEGREES) : 0.81                            \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : ISOTROPIC                                 \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : PARAM_NDBX_HIGH.DNA                            \\nREMARK   3  PARAMETER FILE  2  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : TOP_NDBX.DNA                                   \\nREMARK   3  TOPOLOGY FILE  2   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 401D COMPLIES WITH FORMAT V. 3.15, 01-DEC-08                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : 1                                  \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : N                                  \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : M                                  \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : AREA DETECTOR                      \\nREMARK 200  DETECTOR MANUFACTURER          : SIEMENS HI-STAR                    \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : SAINT                              \\nREMARK 200  DATA SCALING SOFTWARE          : SAINT                              \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : 1811                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : 2.200                              \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : 3.000                              \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : 82.3                               \\nREMARK 200  DATA REDUNDANCY                : 6.000                              \\nREMARK 200  R MERGE                    (I) : 0.03900                            \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 56.00                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.68                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: VAPOR DIFFUSION                          \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 61 2 2                         \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -Y,X-Y,Z+1/3                                            \\nREMARK 290       3555   -X+Y,-X,Z+2/3                                           \\nREMARK 290       4555   -X,-Y,Z+1/2                                             \\nREMARK 290       5555   Y,-X+Y,Z+5/6                                            \\nREMARK 290       6555   X-Y,X,Z+1/6                                             \\nREMARK 290       7555   Y,X,-Z+1/3                                              \\nREMARK 290       8555   X-Y,-Y,-Z                                               \\nREMARK 290       9555   -X,-X+Y,-Z+2/3                                          \\nREMARK 290      10555   -Y,-X,-Z+5/6                                            \\nREMARK 290      11555   -X+Y,Y,-Z+1/2                                           \\nREMARK 290      12555   X,X-Y,-Z+1/6                                            \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000       26.72500            \\nREMARK 290   SMTRY1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000       53.45000            \\nREMARK 290   SMTRY1   4 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   4  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000  1.000000       40.08750            \\nREMARK 290   SMTRY1   5  0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   5 -0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   5  0.000000  0.000000  1.000000       66.81250            \\nREMARK 290   SMTRY1   6  0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   6  0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   6  0.000000  0.000000  1.000000       13.36250            \\nREMARK 290   SMTRY1   7 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   7  0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   7  0.000000  0.000000 -1.000000       26.72500            \\nREMARK 290   SMTRY1   8  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   8  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   8  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   9 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   9 -0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   9  0.000000  0.000000 -1.000000       53.45000            \\nREMARK 290   SMTRY1  10  0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2  10 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3  10  0.000000  0.000000 -1.000000       66.81250            \\nREMARK 290   SMTRY1  11 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2  11  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3  11  0.000000  0.000000 -1.000000       40.08750            \\nREMARK 290   SMTRY1  12  0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2  12  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3  12  0.000000  0.000000 -1.000000       13.36250            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2 -1.000000  0.000000  0.000000       38.44400            \\nREMARK 350   BIOMT2   2  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000 -1.000000       40.08750            \\nREMARK 525                                                                      \\nREMARK 525 SOLVENT                                                              \\nREMARK 525                                                                      \\nREMARK 525 THE SOLVENT MOLECULES HAVE CHAIN IDENTIFIERS THAT                    \\nREMARK 525 INDICATE THE POLYMER CHAIN WITH WHICH THEY ARE MOST                  \\nREMARK 525 CLOSELY ASSOCIATED. THE REMARK LISTS ALL THE SOLVENT                 \\nREMARK 525 MOLECULES WHICH ARE MORE THAN 5A AWAY FROM THE                       \\nREMARK 525 NEAREST POLYMER CHAIN (M = MODEL NUMBER;                             \\nREMARK 525 RES=RESIDUE NAME; C=CHAIN IDENTIFIER; SSEQ=SEQUENCE                  \\nREMARK 525 NUMBER; I=INSERTION CODE):                                           \\nREMARK 525                                                                      \\nREMARK 525  M RES CSSEQI                                                        \\nREMARK 525    HOH A  30        DISTANCE =  6.82 ANGSTROMS                       \\nDBREF  401D A    1    10  PDB    401D     401D             1     10             \\nSEQRES   1 A   10   DG  DA  DC  DC  DG  DC  DG  DG  DT  DC                      \\nFORMUL   2  HOH   *34(H2 O)                                                     \\nCRYST1   38.444   38.444   80.175  90.00  90.00 120.00 P 61 2 2     12          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.026010  0.015020  0.000000        0.00000                         \\nSCALE2      0.000000  0.030040  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.012470        0.00000                         \\nATOM      1  O5'  DG A   1      28.445  12.780  16.727  1.00 19.94           O  \\nATOM      2  C5'  DG A   1      28.383  14.200  16.550  1.00 17.21           C  \\nATOM      3  C4'  DG A   1      27.495  14.612  15.397  1.00 17.12           C  \\nATOM      4  O4'  DG A   1      28.008  14.105  14.149  1.00 13.77           O  \\nATOM      5  C3'  DG A   1      26.063  14.109  15.475  1.00 16.70           C  \\nATOM      6  O3'  DG A   1      25.263  15.013  16.251  1.00 19.38           O  \\nATOM      7  C2'  DG A   1      25.641  14.146  14.019  1.00 15.75           C  \\nATOM      8  C1'  DG A   1      26.924  13.784  13.279  1.00 14.88           C  \\nATOM      9  N9   DG A   1      27.033  12.373  12.916  1.00 17.74           N  \\nATOM     10  C8   DG A   1      27.692  11.380  13.607  1.00 18.97           C  \\nATOM     11  N7   DG A   1      27.632  10.217  13.016  1.00 15.22           N  \\nATOM     12  C5   DG A   1      26.888  10.457  11.868  1.00 17.51           C  \\nATOM     13  C6   DG A   1      26.504   9.578  10.826  1.00 15.44           C  \\nATOM     14  O6   DG A   1      26.772   8.385  10.695  1.00 17.68           O  \\nATOM     15  N1   DG A   1      25.734  10.226   9.865  1.00 16.82           N  \\nATOM     16  C2   DG A   1      25.384  11.559   9.903  1.00 17.50           C  \\nATOM     17  N2   DG A   1      24.606  12.005   8.900  1.00 14.45           N  \\nATOM     18  N3   DG A   1      25.759  12.394  10.859  1.00 16.73           N  \\nATOM     19  C4   DG A   1      26.500  11.779  11.801  1.00 14.78           C  \\nATOM     20  H1   DG A   1      25.410   9.675   9.086  1.00  0.28           H  \\nATOM     21  H21  DG A   1      24.314  12.970   8.882  1.00 -3.63           H  \\nATOM     22  H22  DG A   1      24.313  11.376   8.166  1.00  1.60           H  \\nATOM     23 HO5'  DG A   1      29.035  12.620  17.467  1.00  0.85           H  \\nATOM     24  P    DA A   2      23.995  14.454  17.068  1.00 19.26           P  \\nATOM     25  OP1  DA A   2      23.559  15.503  18.022  1.00 23.59           O  \\nATOM     26  OP2  DA A   2      24.357  13.109  17.582  1.00 19.04           O  \\nATOM     27  O5'  DA A   2      22.870  14.266  15.954  1.00 16.06           O  \\nATOM     28  C5'  DA A   2      22.393  15.378  15.187  1.00 14.98           C  \\nATOM     29  C4'  DA A   2      21.620  14.889  13.982  1.00 17.07           C  \\nATOM     30  O4'  DA A   2      22.501  14.299  13.007  1.00 13.79           O  \\nATOM     31  C3'  DA A   2      20.582  13.818  14.274  1.00 17.14           C  \\nATOM     32  O3'  DA A   2      19.350  14.403  14.709  1.00 21.29           O  \\nATOM     33  C2'  DA A   2      20.415  13.185  12.909  1.00 15.76           C  \\nATOM     34  C1'  DA A   2      21.841  13.199  12.375  1.00 12.76           C  \\nATOM     35  N9   DA A   2      22.575  11.978  12.706  1.00 10.51           N  \\nATOM     36  C8   DA A   2      23.391  11.758  13.783  1.00 10.96           C  \\nATOM     37  N7   DA A   2      23.908  10.555  13.817  1.00 11.09           N  \\nATOM     38  C5   DA A   2      23.402   9.942  12.682  1.00  9.35           C  \\nATOM     39  C6   DA A   2      23.583   8.657  12.143  1.00 10.47           C  \\nATOM     40  N6   DA A   2      24.346   7.717  12.710  1.00 11.06           N  \\nATOM     41  N1   DA A   2      22.945   8.364  10.990  1.00 12.25           N  \\nATOM     42  C2   DA A   2      22.168   9.301  10.433  1.00  9.93           C  \\nATOM     43  N3   DA A   2      21.913  10.542  10.849  1.00  8.56           N  \\nATOM     44  C4   DA A   2      22.575  10.805  11.989  1.00 10.42           C  \\nATOM     45  H61  DA A   2      24.834   7.913  13.574  1.00 -2.75           H  \\nATOM     46  H62  DA A   2      24.439   6.810  12.276  1.00  0.55           H  \\nATOM     47  P    DC A   3      18.393  13.588  15.709  1.00 17.98           P  \\nATOM     48  OP1  DC A   3      17.261  14.479  16.049  1.00 21.44           O  \\nATOM     49  OP2  DC A   3      19.213  12.988  16.796  1.00 20.04           O  \\nATOM     50  O5'  DC A   3      17.825  12.406  14.816  1.00 16.92           O  \\nATOM     51  C5'  DC A   3      16.910  12.683  13.757  1.00 17.74           C  \\nATOM     52  C4'  DC A   3      16.750  11.466  12.876  1.00 16.62           C  \\nATOM     53  O4'  DC A   3      18.033  11.098  12.332  1.00 13.12           O  \\nATOM     54  C3'  DC A   3      16.247  10.216  13.576  1.00 16.85           C  \\nATOM     55  O3'  DC A   3      14.809  10.215  13.684  1.00 17.85           O  \\nATOM     56  C2'  DC A   3      16.736   9.137  12.632  1.00 16.58           C  \\nATOM     57  C1'  DC A   3      18.095   9.677  12.191  1.00 15.13           C  \\nATOM     58  N1   DC A   3      19.199   9.179  13.025  1.00 12.40           N  \\nATOM     59  C2   DC A   3      19.750   7.947  12.720  1.00 11.65           C  \\nATOM     60  O2   DC A   3      19.337   7.341  11.725  1.00 13.90           O  \\nATOM     61  N3   DC A   3      20.715   7.441  13.509  1.00  9.80           N  \\nATOM     62  C4   DC A   3      21.140   8.130  14.566  1.00  8.68           C  \\nATOM     63  N4   DC A   3      22.084   7.580  15.331  1.00  5.25           N  \\nATOM     64  C5   DC A   3      20.615   9.412  14.886  1.00  7.30           C  \\nATOM     65  C6   DC A   3      19.657   9.895  14.094  1.00 10.90           C  \\nATOM     66  H41  DC A   3      22.422   8.072  16.145  1.00 -0.22           H  \\nATOM     67  H42  DC A   3      22.458   6.672  15.095  1.00 -1.59           H  \\nATOM     68  P    DC A   4      14.095   9.365  14.853  1.00 14.97           P  \\nATOM     69  OP1  DC A   4      12.684   9.788  14.982  1.00 20.66           O  \\nATOM     70  OP2  DC A   4      14.979   9.383  16.047  1.00 16.36           O  \\nATOM     71  O5'  DC A   4      14.095   7.889  14.277  1.00 13.68           O  \\nATOM     72  C5'  DC A   4      13.442   7.604  13.046  1.00 15.28           C  \\nATOM     73  C4'  DC A   4      13.787   6.207  12.591  1.00 15.73           C  \\nATOM     74  O4'  DC A   4      15.171   6.141  12.211  1.00 15.49           O  \\nATOM     75  C3'  DC A   4      13.629   5.132  13.650  1.00 15.89           C  \\nATOM     76  O3'  DC A   4      12.278   4.702  13.789  1.00 17.36           O  \\nATOM     77  C2'  DC A   4      14.489   4.033  13.068  1.00 13.91           C  \\nATOM     78  C1'  DC A   4      15.660   4.825  12.494  1.00 15.50           C  \\nATOM     79  N1   DC A   4      16.778   4.936  13.439  1.00 12.78           N  \\nATOM     80  C2   DC A   4      17.669   3.875  13.515  1.00 14.08           C  \\nATOM     81  O2   DC A   4      17.482   2.886  12.789  1.00 13.67           O  \\nATOM     82  N3   DC A   4      18.704   3.938  14.373  1.00 14.40           N  \\nATOM     83  C4   DC A   4      18.861   5.002  15.150  1.00  9.25           C  \\nATOM     84  N4   DC A   4      19.901   4.995  15.996  1.00  6.68           N  \\nATOM     85  C5   DC A   4      17.964   6.110  15.101  1.00  8.65           C  \\nATOM     86  C6   DC A   4      16.946   6.037  14.234  1.00 12.29           C  \\nATOM     87  H41  DC A   4      20.058   5.780  16.611  1.00 -1.36           H  \\nATOM     88  H42  DC A   4      20.525   4.203  16.018  1.00  1.44           H  \\nATOM     89  P    DG A   5      11.805   4.044  15.171  1.00 18.31           P  \\nATOM     90  OP1  DG A   5      10.325   3.973  15.152  1.00 20.54           O  \\nATOM     91  OP2  DG A   5      12.495   4.736  16.289  1.00 17.66           O  \\nATOM     92  O5'  DG A   5      12.387   2.566  15.084  1.00 17.42           O  \\nATOM     93  C5'  DG A   5      11.908   1.654  14.082  1.00 15.38           C  \\nATOM     94  C4'  DG A   5      12.634   0.333  14.186  1.00 13.44           C  \\nATOM     95  O4'  DG A   5      14.018   0.492  13.825  1.00 12.19           O  \\nATOM     96  C3'  DG A   5      12.659  -0.239  15.591  1.00 14.60           C  \\nATOM     97  O3'  DG A   5      11.468  -0.967  15.893  1.00 13.65           O  \\nATOM     98  C2'  DG A   5      13.868  -1.148  15.527  1.00 11.58           C  \\nATOM     99  C1'  DG A   5      14.824  -0.321  14.678  1.00 10.77           C  \\nATOM    100  N9   DG A   5      15.674   0.560  15.470  1.00  7.38           N  \\nATOM    101  C8   DG A   5      15.557   1.921  15.626  1.00  3.29           C  \\nATOM    102  N7   DG A   5      16.486   2.428  16.390  1.00  6.42           N  \\nATOM    103  C5   DG A   5      17.260   1.335  16.760  1.00  5.20           C  \\nATOM    104  C6   DG A   5      18.418   1.261  17.568  1.00  4.02           C  \\nATOM    105  O6   DG A   5      19.024   2.176  18.116  1.00  5.31           O  \\nATOM    106  N1   DG A   5      18.871  -0.052  17.701  1.00  5.63           N  \\nATOM    107  C2   DG A   5      18.285  -1.149  17.117  1.00  2.64           C  \\nATOM    108  N2   DG A   5      18.846  -2.337  17.374  1.00  2.00           N  \\nATOM    109  N3   DG A   5      17.220  -1.086  16.342  1.00  6.00           N  \\nATOM    110  C4   DG A   5      16.763   0.175  16.210  1.00  2.95           C  \\nATOM    111  H1   DG A   5      19.690  -0.207  18.272  1.00  0.80           H  \\nATOM    112  H21  DG A   5      18.457  -3.181  16.979  1.00  0.70           H  \\nATOM    113  H22  DG A   5      19.663  -2.391  17.966  1.00 -1.96           H  \\nATOM    114  P    DC A   6      11.011  -1.100  17.420  1.00 13.94           P  \\nATOM    115  OP1  DC A   6       9.719  -1.821  17.488  1.00 13.64           O  \\nATOM    116  OP2  DC A   6      11.126   0.245  18.023  1.00 17.86           O  \\nATOM    117  O5'  DC A   6      12.159  -1.991  18.062  1.00 10.56           O  \\nATOM    118  C5'  DC A   6      12.274  -3.383  17.744  1.00 11.64           C  \\nATOM    119  C4'  DC A   6      13.366  -4.014  18.575  1.00  8.61           C  \\nATOM    120  O4'  DC A   6      14.656  -3.514  18.180  1.00 12.90           O  \\nATOM    121  C3'  DC A   6      13.294  -3.732  20.058  1.00  8.64           C  \\nATOM    122  O3'  DC A   6      12.364  -4.544  20.749  1.00 13.95           O  \\nATOM    123  C2'  DC A   6      14.706  -4.076  20.476  1.00 12.33           C  \\nATOM    124  C1'  DC A   6      15.509  -3.479  19.330  1.00 10.42           C  \\nATOM    125  N1   DC A   6      15.859  -2.080  19.604  1.00 11.65           N  \\nATOM    126  C2   DC A   6      16.983  -1.816  20.389  1.00 12.58           C  \\nATOM    127  O2   DC A   6      17.674  -2.763  20.785  1.00 14.76           O  \\nATOM    128  N3   DC A   6      17.292  -0.538  20.696  1.00 14.18           N  \\nATOM    129  C4   DC A   6      16.533   0.460  20.239  1.00 12.29           C  \\nATOM    130  N4   DC A   6      16.881   1.705  20.573  1.00 12.71           N  \\nATOM    131  C5   DC A   6      15.388   0.224  19.419  1.00 10.23           C  \\nATOM    132  C6   DC A   6      15.093  -1.052  19.126  1.00 12.78           C  \\nATOM    133  H41  DC A   6      16.338   2.490  20.244  1.00 -1.39           H  \\nATOM    134  H42  DC A   6      17.690   1.861  21.157  1.00  3.29           H  \\nATOM    135  P    DG A   7      11.798  -4.037  22.154  1.00 15.99           P  \\nATOM    136  OP1  DG A   7      10.852  -5.057  22.669  1.00 19.94           O  \\nATOM    137  OP2  DG A   7      11.349  -2.634  21.968  1.00 16.34           O  \\nATOM    138  O5'  DG A   7      13.076  -4.002  23.100  1.00 12.83           O  \\nATOM    139  C5'  DG A   7      13.791  -5.203  23.404  1.00 13.52           C  \\nATOM    140  C4'  DG A   7      14.975  -4.900  24.293  1.00 14.02           C  \\nATOM    141  O4'  DG A   7      15.912  -4.027  23.631  1.00 12.27           O  \\nATOM    142  C3'  DG A   7      14.663  -4.217  25.612  1.00 11.21           C  \\nATOM    143  O3'  DG A   7      14.306  -5.195  26.592  1.00 17.14           O  \\nATOM    144  C2'  DG A   7      16.010  -3.617  25.959  1.00 14.58           C  \\nATOM    145  C1'  DG A   7      16.564  -3.213  24.600  1.00  8.47           C  \\nATOM    146  N9   DG A   7      16.313  -1.820  24.263  1.00  8.09           N  \\nATOM    147  C8   DG A   7      15.326  -1.313  23.446  1.00  8.46           C  \\nATOM    148  N7   DG A   7      15.387  -0.014  23.318  1.00  9.75           N  \\nATOM    149  C5   DG A   7      16.479   0.357  24.101  1.00  7.36           C  \\nATOM    150  C6   DG A   7      17.052   1.630  24.340  1.00  8.34           C  \\nATOM    151  O6   DG A   7      16.712   2.726  23.886  1.00 13.98           O  \\nATOM    152  N1   DG A   7      18.140   1.550  25.202  1.00  6.12           N  \\nATOM    153  C2   DG A   7      18.617   0.396  25.766  1.00  4.94           C  \\nATOM    154  N2   DG A   7      19.668   0.532  26.589  1.00  8.08           N  \\nATOM    155  N3   DG A   7      18.104  -0.796  25.545  1.00  6.58           N  \\nATOM    156  C4   DG A   7      17.046  -0.744  24.705  1.00  8.86           C  \\nATOM    157  H1   DG A   7      18.621   2.409  25.428  1.00 -0.48           H  \\nATOM    158  H21  DG A   7      20.062  -0.282  27.037  1.00  0.54           H  \\nATOM    159  H22  DG A   7      20.065   1.445  26.761  1.00  1.09           H  \\nATOM    160  P    DG A   8      13.622  -4.729  27.967  1.00 13.18           P  \\nATOM    161  OP1  DG A   8      13.394  -5.938  28.784  1.00 19.66           O  \\nATOM    162  OP2  DG A   8      12.486  -3.834  27.630  1.00 15.17           O  \\nATOM    163  O5'  DG A   8      14.744  -3.862  28.679  1.00 14.95           O  \\nATOM    164  C5'  DG A   8      14.398  -2.765  29.536  1.00 12.43           C  \\nATOM    165  C4'  DG A   8      15.641  -1.972  29.857  1.00 14.30           C  \\nATOM    166  O4'  DG A   8      16.218  -1.475  28.635  1.00 11.37           O  \\nATOM    167  C3'  DG A   8      15.453  -0.745  30.733  1.00 15.04           C  \\nATOM    168  O3'  DG A   8      15.484  -1.116  32.113  1.00 16.48           O  \\nATOM    169  C2'  DG A   8      16.684   0.061  30.380  1.00 13.73           C  \\nATOM    170  C1'  DG A   8      16.824  -0.208  28.884  1.00 13.57           C  \\nATOM    171  N9   DG A   8      16.151   0.789  28.058  1.00 13.22           N  \\nATOM    172  C8   DG A   8      15.077   0.608  27.216  1.00 10.00           C  \\nATOM    173  N7   DG A   8      14.732   1.699  26.587  1.00  3.41           N  \\nATOM    174  C5   DG A   8      15.628   2.657  27.048  1.00  6.86           C  \\nATOM    175  C6   DG A   8      15.763   4.029  26.718  1.00  7.58           C  \\nATOM    176  O6   DG A   8      15.093   4.698  25.921  1.00 15.10           O  \\nATOM    177  N1   DG A   8      16.808   4.627  27.417  1.00  5.98           N  \\nATOM    178  C2   DG A   8      17.617   3.985  28.322  1.00  5.06           C  \\nATOM    179  N2   DG A   8      18.573   4.723  28.892  1.00  4.15           N  \\nATOM    180  N3   DG A   8      17.499   2.713  28.642  1.00  6.09           N  \\nATOM    181  C4   DG A   8      16.497   2.113  27.967  1.00  8.70           C  \\nATOM    182  H1   DG A   8      16.977   5.608  27.249  1.00 -0.37           H  \\nATOM    183  H21  DG A   8      19.198   4.303  29.564  1.00 -1.31           H  \\nATOM    184  H22  DG A   8      18.671   5.699  28.651  1.00 -0.47           H  \\nATOM    185  P    DT A   9      14.633  -0.275  33.178  1.00 16.60           P  \\nATOM    186  OP1  DT A   9      14.604  -1.033  34.451  1.00 21.64           O  \\nATOM    187  OP2  DT A   9      13.358   0.131  32.534  1.00 17.29           O  \\nATOM    188  O5'  DT A   9      15.509   1.033  33.407  1.00 20.09           O  \\nATOM    189  C5'  DT A   9      16.829   0.944  33.971  1.00 17.60           C  \\nATOM    190  C4'  DT A   9      17.458   2.315  34.025  1.00 14.16           C  \\nATOM    191  O4'  DT A   9      17.588   2.836  32.689  1.00 14.03           O  \\nATOM    192  C3'  DT A   9      16.594   3.319  34.765  1.00 17.72           C  \\nATOM    193  O3'  DT A   9      16.940   3.294  36.155  1.00 23.99           O  \\nATOM    194  C2'  DT A   9      17.009   4.629  34.126  1.00 14.15           C  \\nATOM    195  C1'  DT A   9      17.311   4.235  32.685  1.00 13.03           C  \\nATOM    196  N1   DT A   9      16.229   4.484  31.710  1.00 11.86           N  \\nATOM    197  C2   DT A   9      16.223   5.694  31.046  1.00 11.27           C  \\nATOM    198  O2   DT A   9      17.025   6.584  31.269  1.00 10.38           O  \\nATOM    199  N3   DT A   9      15.235   5.823  30.105  1.00 11.43           N  \\nATOM    200  C4   DT A   9      14.273   4.893  29.776  1.00 10.66           C  \\nATOM    201  O4   DT A   9      13.476   5.137  28.878  1.00 19.24           O  \\nATOM    202  C5   DT A   9      14.307   3.667  30.541  1.00  8.58           C  \\nATOM    203  C7   DT A   9      13.271   2.621  30.279  1.00  3.81           C  \\nATOM    204  C6   DT A   9      15.271   3.524  31.456  1.00 10.82           C  \\nATOM    205  H3   DT A   9      15.206   6.695  29.596  1.00 -0.14           H  \\nATOM    206  P    DC A  10      15.834   3.668  37.262  1.00 29.15           P  \\nATOM    207  OP1  DC A  10      16.460   3.493  38.598  1.00 34.10           O  \\nATOM    208  OP2  DC A  10      14.573   2.951  36.942  1.00 28.05           O  \\nATOM    209  O5'  DC A  10      15.596   5.224  37.042  1.00 24.96           O  \\nATOM    210  C5'  DC A  10      16.640   6.163  37.331  1.00 23.04           C  \\nATOM    211  C4'  DC A  10      16.272   7.531  36.805  1.00 20.09           C  \\nATOM    212  O4'  DC A  10      16.288   7.552  35.365  1.00 15.61           O  \\nATOM    213  C3'  DC A  10      14.889   8.025  37.194  1.00 19.99           C  \\nATOM    214  O3'  DC A  10      14.849   8.509  38.548  1.00 18.36           O  \\nATOM    215  C2'  DC A  10      14.633   9.075  36.131  1.00 17.70           C  \\nATOM    216  C1'  DC A  10      15.363   8.533  34.901  1.00 13.61           C  \\nATOM    217  N1   DC A  10      14.473   7.904  33.919  1.00 10.50           N  \\nATOM    218  C2   DC A  10      14.091   8.645  32.810  1.00  8.79           C  \\nATOM    219  O2   DC A  10      14.536   9.791  32.681  1.00 12.66           O  \\nATOM    220  N3   DC A  10      13.245   8.103  31.907  1.00 11.23           N  \\nATOM    221  C4   DC A  10      12.779   6.867  32.092  1.00  8.63           C  \\nATOM    222  N4   DC A  10      11.927   6.382  31.190  1.00  7.04           N  \\nATOM    223  C5   DC A  10      13.164   6.078  33.211  1.00  8.82           C  \\nATOM    224  C6   DC A  10      14.011   6.629  34.092  1.00 10.53           C  \\nATOM    225 HO3'  DC A  10      13.956   8.819  38.715  1.00 -1.18           H  \\nATOM    226  H41  DC A  10      11.589   5.437  31.285  1.00  3.12           H  \\nATOM    227  H42  DC A  10      11.619   6.947  30.414  1.00 -0.85           H  \\nTER     228       DC A  10                                                      \\nHETATM  229  O   HOH A  11      15.084   4.232  19.813  0.92 17.79           O  \\nHETATM  230  H1  HOH A  11      14.509   3.560  20.181  1.00  2.20           H  \\nHETATM  231  H2  HOH A  11      14.672   4.477  18.985  1.00 -3.70           H  \\nHETATM  232  O   HOH A  12       8.633  -5.205  20.617  0.92 19.91           O  \\nHETATM  233  H1  HOH A  12       7.748  -5.548  20.739  1.00  0.52           H  \\nHETATM  234  H2  HOH A  12       9.126  -5.929  20.236  1.00 -0.19           H  \\nHETATM  235  O   HOH A  13      12.832   3.510  35.090  0.92 12.43           O  \\nHETATM  236  H1  HOH A  13      12.323   3.313  34.303  1.00  0.34           H  \\nHETATM  237  H2  HOH A  13      13.428   2.767  35.184  1.00 -2.35           H  \\nHETATM  238  O   HOH A  14      25.078  10.081  16.137  0.92 14.98           O  \\nHETATM  239  H1  HOH A  14      24.928  10.949  16.513  1.00 -0.15           H  \\nHETATM  240  H2  HOH A  14      25.109   9.498  16.895  1.00  0.42           H  \\nHETATM  241  O   HOH A  15       9.991  -4.605  26.139  0.92 20.73           O  \\nHETATM  242  H1  HOH A  15       9.685  -4.683  27.045  1.00  1.05           H  \\nHETATM  243  H2  HOH A  15       9.315  -5.036  25.620  1.00  3.78           H  \\nHETATM  244  O   HOH A  16      12.977   2.543  17.987  0.92 24.71           O  \\nHETATM  245  H1  HOH A  16      13.532   1.993  18.541  1.00 -1.76           H  \\nHETATM  246  H2  HOH A  16      13.529   3.292  17.762  1.00 -1.61           H  \\nHETATM  247  O   HOH A  17      19.767   1.402  30.209  0.92 14.79           O  \\nHETATM  248  H1  HOH A  17      19.637   2.085  30.867  1.00 -3.35           H  \\nHETATM  249  H2  HOH A  17      20.447   0.839  30.578  1.00  0.18           H  \\nHETATM  250  O   HOH A  18      13.290   1.860  22.372  0.92 21.52           O  \\nHETATM  251  H1  HOH A  18      14.004   1.380  22.791  1.00  0.09           H  \\nHETATM  252  H2  HOH A  18      13.587   2.770  22.355  1.00  2.28           H  \\nHETATM  253  O   HOH A  19       9.092  -3.545  15.066  0.92 29.78           O  \\nHETATM  254  H1  HOH A  19       9.704  -2.996  14.579  1.00  1.80           H  \\nHETATM  255  H2  HOH A  19       9.123  -3.205  15.962  1.00 -0.80           H  \\nHETATM  256  O   HOH A  20      11.347  -0.301  20.759  0.92 22.64           O  \\nHETATM  257  H1  HOH A  20      11.438   0.541  20.315  1.00 -1.22           H  \\nHETATM  258  H2  HOH A  20      11.295  -0.945  20.052  1.00  0.80           H  \\nHETATM  259  O   HOH A  21      19.866   0.048  32.722  0.92 40.84           O  \\nHETATM  260  H1  HOH A  21      20.802   0.025  32.925  1.00  0.23           H  \\nHETATM  261  H2  HOH A  21      19.441  -0.331  33.490  1.00 -0.27           H  \\nHETATM  262  O   HOH A  22      10.434   2.601  28.103  0.92 36.31           O  \\nHETATM  263  H1  HOH A  22      10.686   3.217  28.792  1.00  2.55           H  \\nHETATM  264  H2  HOH A  22       9.581   2.267  28.376  1.00 -0.52           H  \\nHETATM  265  O   HOH A  23      15.670   1.222  11.204  0.92 38.36           O  \\nHETATM  266  H1  HOH A  23      15.191   0.549  11.687  1.00 -1.02           H  \\nHETATM  267  H2  HOH A  23      16.568   1.158  11.531  1.00  0.11           H  \\nHETATM  268  O   HOH A  24      12.633  -1.288  26.556  0.92 18.82           O  \\nHETATM  269  H1  HOH A  24      12.536  -0.798  25.740  1.00  1.10           H  \\nHETATM  270  H2  HOH A  24      12.478  -2.201  26.313  1.00  1.08           H  \\nHETATM  271  O   HOH A  25      16.750  17.104  15.219  0.92 13.67           O  \\nHETATM  272  H1  HOH A  25      17.225  17.932  15.137  1.00 -0.19           H  \\nHETATM  273  H2  HOH A  25      17.407  16.484  15.535  1.00  1.43           H  \\nHETATM  274  O   HOH A  26      11.183  -0.661  24.128  0.92 14.09           O  \\nHETATM  275  H1  HOH A  26      11.354   0.278  24.192  1.00 -0.32           H  \\nHETATM  276  H2  HOH A  26      11.050  -0.942  25.033  1.00 -0.66           H  \\nHETATM  277  O   HOH A  27      20.912   4.065  31.343  0.92 17.55           O  \\nHETATM  278  H1  HOH A  27      20.209   3.767  30.764  1.00 -1.40           H  \\nHETATM  279  H2  HOH A  27      21.619   3.435  31.202  1.00 -0.91           H  \\nHETATM  280  O   HOH A  28      17.680   7.449   9.662  0.92 34.97           O  \\nHETATM  281  H1  HOH A  28      17.544   6.604  10.095  1.00 -1.64           H  \\nHETATM  282  H2  HOH A  28      18.556   7.373   9.286  1.00  0.34           H  \\nHETATM  283  O   HOH A  29      10.361   4.046  31.687  0.92 21.54           O  \\nHETATM  284  H1  HOH A  29      11.035   4.285  32.323  1.00 -2.32           H  \\nHETATM  285  H2  HOH A  29      10.851   3.733  30.928  1.00  3.17           H  \\nHETATM  286  O   HOH A  30       5.775   3.565  32.021  0.92 54.10           O  \\nHETATM  287  H1  HOH A  30       5.777   4.522  32.044  1.00 -0.85           H  \\nHETATM  288  H2  HOH A  30       5.775   3.348  31.089  1.00  0.36           H  \\nHETATM  289  O   HOH A  31      19.894  12.183   9.431  0.92 35.18           O  \\nHETATM  290  H1  HOH A  31      20.562  11.820  10.003  1.00 -0.52           H  \\nHETATM  291  H2  HOH A  31      19.153  11.588   9.542  1.00  1.51           H  \\nHETATM  292  O   HOH A  32      31.037  13.244  19.339  0.92 32.52           O  \\nHETATM  293  H1  HOH A  32      30.100  13.238  19.139  1.00 -0.50           H  \\nHETATM  294  H2  HOH A  32      31.241  12.329  19.531  1.00  1.17           H  \\nHETATM  295  O   HOH A  33      11.558  -0.240  29.036  0.92 30.11           O  \\nHETATM  296  H1  HOH A  33      10.955   0.323  28.550  1.00 -0.33           H  \\nHETATM  297  H2  HOH A  33      11.635  -1.030  28.501  1.00 -2.52           H  \\nHETATM  298  O   HOH A  35      15.976  11.888  34.322  0.92 13.69           O  \\nHETATM  299  H1  HOH A  35      15.850  11.760  35.266  1.00  1.96           H  \\nHETATM  300  H2  HOH A  35      15.715  11.048  33.941  1.00 -0.38           H  \\nHETATM  301  O   HOH A  37      13.420   7.005  40.225  0.92 10.34           O  \\nHETATM  302  H1  HOH A  37      14.089   7.225  40.873  1.00 -0.69           H  \\nHETATM  303  H2  HOH A  37      12.710   7.624  40.394  1.00 -3.25           H  \\nHETATM  304  O   HOH A  38      17.284  -2.500  35.916  0.92 37.92           O  \\nHETATM  305  H1  HOH A  38      18.133  -2.832  35.624  1.00  3.16           H  \\nHETATM  306  H2  HOH A  38      16.878  -2.147  35.126  1.00  1.24           H  \\nHETATM  307  O   HOH A  39      20.933   3.007  34.529  0.92 49.67           O  \\nHETATM  308  H1  HOH A  39      20.827   2.704  35.430  1.00  2.93           H  \\nHETATM  309  H2  HOH A  39      21.823   2.751  34.287  1.00 -0.06           H  \\nHETATM  310  O   HOH A  40      10.697   7.276  11.276  0.92 23.22           O  \\nHETATM  311  H1  HOH A  40      10.550   6.505  11.825  1.00  1.08           H  \\nHETATM  312  H2  HOH A  40      11.123   7.906  11.857  1.00 -0.06           H  \\nHETATM  313  O   HOH A  41      23.494  14.870   8.728  0.92 19.82           O  \\nHETATM  314  H1  HOH A  41      23.653  15.799   8.907  1.00  1.41           H  \\nHETATM  315  H2  HOH A  41      22.746  14.866   8.128  1.00 -3.30           H  \\nHETATM  316  O   HOH A  42      12.066  11.232  12.195  0.96 36.07           O  \\nHETATM  317  H1  HOH A  42      11.248  11.727  12.164  1.00 -2.80           H  \\nHETATM  318  H2  HOH A  42      12.250  11.149  13.132  1.00  0.23           H  \\nHETATM  319  O   HOH A  43      26.672  12.712  18.951  0.96 42.39           O  \\nHETATM  320  H1  HOH A  43      27.559  13.063  18.900  1.00  0.23           H  \\nHETATM  321  H2  HOH A  43      26.468  12.449  18.052  1.00 -3.65           H  \\nHETATM  322  O   HOH A  44      14.854  11.227  19.213  0.96 36.50           O  \\nHETATM  323  H1  HOH A  44      14.000  11.658  19.156  1.00  3.66           H  \\nHETATM  324  H2  HOH A  44      14.988  11.082  20.149  1.00  1.26           H  \\nHETATM  325  O   HOH A  45      10.696   7.159  17.758  0.99 54.36           O  \\nHETATM  326  H1  HOH A  45       9.772   7.043  17.554  1.00  0.66           H  \\nHETATM  327  H2  HOH A  45      10.864   6.465  18.396  1.00  0.14           H  \\nHETATM  328  O   HOH A  46      15.287  13.783  18.124  0.99 32.92           O  \\nHETATM  329  H1  HOH A  46      15.021  14.146  17.276  1.00 -1.45           H  \\nHETATM  330  H2  HOH A  46      16.242  13.801  18.095  1.00  1.68           H  \\nMASTER      247    0    0    0    0    0    0    6  329    1    0    1          \\nEND                                                                             \\n\");\n\tviewer_1684696742122504.setStyle({\"model\": -1, \"chain\": \"A\"},{\"cartoon\": {\"color\": \"spectrum\"}});\n\tviewer_1684696742122504.zoomTo();\nviewer_1684696742122504.render();\n});\n</script>",
      "text/html": [
       "<div id=\"3dmolviewer_1684696742122504\"  style=\"position: relative; width: 600px; height: 400px\">\n",
       "        <p id=\"3dmolwarning_1684696742122504\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
       "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
       "        </div>\n",
       "<script>\n",
       "\n",
       "var loadScriptAsync = function(uri){\n",
       "  return new Promise((resolve, reject) => {\n",
       "    //this is to ignore the existence of requirejs amd\n",
       "    var savedexports, savedmodule;\n",
       "    if (typeof exports !== 'undefined') savedexports = exports;\n",
       "    else exports = {}\n",
       "    if (typeof module !== 'undefined') savedmodule = module;\n",
       "    else module = {}\n",
       "\n",
       "    var tag = document.createElement('script');\n",
       "    tag.src = uri;\n",
       "    tag.async = true;\n",
       "    tag.onload = () => {\n",
       "        exports = savedexports;\n",
       "        module = savedmodule;\n",
       "        resolve();\n",
       "    };\n",
       "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
       "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
       "});\n",
       "};\n",
       "\n",
       "if(typeof $3Dmolpromise === 'undefined') {\n",
       "$3Dmolpromise = null;\n",
       "  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.0.1/3Dmol-min.js');\n",
       "}\n",
       "\n",
       "var viewer_1684696742122504 = null;\n",
       "var warn = document.getElementById(\"3dmolwarning_1684696742122504\");\n",
       "if(warn) {\n",
       "    warn.parentNode.removeChild(warn);\n",
       "}\n",
       "$3Dmolpromise.then(function() {\n",
       "viewer_1684696742122504 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_1684696742122504\"),{backgroundColor:\"white\"});\n",
       "viewer_1684696742122504.zoomTo();\n",
       "\tviewer_1684696742122504.addModelsAsFrames(\"HEADER    DNA                                     02-JUN-98   401D              \\nTITLE     STRUCTURE OF A DNA IN LOW SALT CONDITIONS D(GACCGCGGTC)               \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: DNA (5'-D(*GP*AP*CP*CP*GP*CP*GP*GP*TP*C)-3');              \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 SYNTHETIC: YES                                                       \\nKEYWDS    DNA DOUBLE HELIX, DNA                                                 \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    M.LUO,J.B.FINLEY                                                      \\nREVDAT   4   02-MAR-10 401D    1       HETATM                                   \\nREVDAT   3   24-FEB-09 401D    1       VERSN                                    \\nREVDAT   2   01-APR-03 401D    1       JRNL                                     \\nREVDAT   1   06-JUL-98 401D    0                                                \\nJRNL        AUTH   J.B.FINLEY,M.LUO                                             \\nJRNL        TITL   X-RAY CRYSTAL STRUCTURES OF HALF THE HUMAN                   \\nJRNL        TITL 2 PAPILLOMA VIRUS E2 BINDING SITE: D(GACCGCGGTC).              \\nJRNL        REF    NUCLEIC ACIDS RES.            V.  26  5719 1998              \\nJRNL        REFN                   ISSN 0305-1048                               \\nJRNL        PMID   9838004                                                      \\nJRNL        DOI    10.1093/NAR/26.24.5719                                       \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    2.20 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : X-PLOR                                               \\nREMARK   3   AUTHORS     : BRUNGER                                              \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 2.20                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 5.00                           \\nREMARK   3   DATA CUTOFF            (SIGMA(F)) : 3.000                          \\nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \\nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \\nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : NULL                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 1779                           \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \\nREMARK   3   FREE R VALUE TEST SET SELECTION  : NULL                            \\nREMARK   3   R VALUE            (WORKING SET) : 0.219                           \\nREMARK   3   FREE R VALUE                     : 0.285                           \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \\nREMARK   3   FREE R VALUE TEST SET COUNT      : NULL                            \\nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \\nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : NULL                         \\nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : NULL                         \\nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : NULL                         \\nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \\nREMARK   3   BIN R VALUE           (WORKING SET) : NULL                         \\nREMARK   3   BIN FREE R VALUE                    : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \\nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : NULL                         \\nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : NULL                         \\nREMARK   3                                                                      \\nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \\nREMARK   3   PROTEIN ATOMS            : 0                                       \\nREMARK   3   NUCLEIC ACID ATOMS       : 227                                     \\nREMARK   3   HETEROGEN ATOMS          : 0                                       \\nREMARK   3   SOLVENT ATOMS            : 102                                     \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : NULL                                                 \\nREMARK   3    B22 (A**2) : NULL                                                 \\nREMARK   3    B33 (A**2) : NULL                                                 \\nREMARK   3    B12 (A**2) : NULL                                                 \\nREMARK   3    B13 (A**2) : NULL                                                 \\nREMARK   3    B23 (A**2) : NULL                                                 \\nREMARK   3                                                                      \\nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \\nREMARK   3   ESD FROM LUZZATI PLOT        (A) : NULL                            \\nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \\nREMARK   3   LOW RESOLUTION CUTOFF        (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \\nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : NULL                            \\nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \\nREMARK   3   BOND LENGTHS                 (A) : 0.005                           \\nREMARK   3   BOND ANGLES            (DEGREES) : 0.81                            \\nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \\nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL MODEL : ISOTROPIC                                 \\nREMARK   3                                                                      \\nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \\nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \\nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  NCS MODEL : NULL                                                    \\nREMARK   3                                                                      \\nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \\nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \\nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \\nREMARK   3                                                                      \\nREMARK   3  PARAMETER FILE  1  : PARAM_NDBX_HIGH.DNA                            \\nREMARK   3  PARAMETER FILE  2  : NULL                                           \\nREMARK   3  TOPOLOGY FILE  1   : TOP_NDBX.DNA                                   \\nREMARK   3  TOPOLOGY FILE  2   : NULL                                           \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 401D COMPLIES WITH FORMAT V. 3.15, 01-DEC-08                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : NULL                               \\nREMARK 200  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 200  PH                             : NULL                               \\nREMARK 200  NUMBER OF CRYSTALS USED        : 1                                  \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : N                                  \\nREMARK 200  RADIATION SOURCE               : NULL                               \\nREMARK 200  BEAMLINE                       : NULL                               \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : M                                  \\nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \\nREMARK 200  MONOCHROMATOR                  : NULL                               \\nREMARK 200  OPTICS                         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : AREA DETECTOR                      \\nREMARK 200  DETECTOR MANUFACTURER          : SIEMENS HI-STAR                    \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : SAINT                              \\nREMARK 200  DATA SCALING SOFTWARE          : SAINT                              \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : 1811                               \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : 2.200                              \\nREMARK 200  RESOLUTION RANGE LOW       (A) : NULL                               \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : 3.000                              \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : 82.3                               \\nREMARK 200  DATA REDUNDANCY                : 6.000                              \\nREMARK 200  R MERGE                    (I) : 0.03900                            \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: NULL                                           \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: NULL                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 56.00                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.68                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: VAPOR DIFFUSION                          \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 61 2 2                         \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -Y,X-Y,Z+1/3                                            \\nREMARK 290       3555   -X+Y,-X,Z+2/3                                           \\nREMARK 290       4555   -X,-Y,Z+1/2                                             \\nREMARK 290       5555   Y,-X+Y,Z+5/6                                            \\nREMARK 290       6555   X-Y,X,Z+1/6                                             \\nREMARK 290       7555   Y,X,-Z+1/3                                              \\nREMARK 290       8555   X-Y,-Y,-Z                                               \\nREMARK 290       9555   -X,-X+Y,-Z+2/3                                          \\nREMARK 290      10555   -Y,-X,-Z+5/6                                            \\nREMARK 290      11555   -X+Y,Y,-Z+1/2                                           \\nREMARK 290      12555   X,X-Y,-Z+1/6                                            \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000       26.72500            \\nREMARK 290   SMTRY1   3 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   3 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000       53.45000            \\nREMARK 290   SMTRY1   4 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   4  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   4  0.000000  0.000000  1.000000       40.08750            \\nREMARK 290   SMTRY1   5  0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   5 -0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   5  0.000000  0.000000  1.000000       66.81250            \\nREMARK 290   SMTRY1   6  0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   6  0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   6  0.000000  0.000000  1.000000       13.36250            \\nREMARK 290   SMTRY1   7 -0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   7  0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   7  0.000000  0.000000 -1.000000       26.72500            \\nREMARK 290   SMTRY1   8  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   8  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   8  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   9 -0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2   9 -0.866025  0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3   9  0.000000  0.000000 -1.000000       53.45000            \\nREMARK 290   SMTRY1  10  0.500000 -0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2  10 -0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3  10  0.000000  0.000000 -1.000000       66.81250            \\nREMARK 290   SMTRY1  11 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2  11  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3  11  0.000000  0.000000 -1.000000       40.08750            \\nREMARK 290   SMTRY1  12  0.500000  0.866025  0.000000        0.00000            \\nREMARK 290   SMTRY2  12  0.866025 -0.500000  0.000000        0.00000            \\nREMARK 290   SMTRY3  12  0.000000  0.000000 -1.000000       13.36250            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2 -1.000000  0.000000  0.000000       38.44400            \\nREMARK 350   BIOMT2   2  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000 -1.000000       40.08750            \\nREMARK 525                                                                      \\nREMARK 525 SOLVENT                                                              \\nREMARK 525                                                                      \\nREMARK 525 THE SOLVENT MOLECULES HAVE CHAIN IDENTIFIERS THAT                    \\nREMARK 525 INDICATE THE POLYMER CHAIN WITH WHICH THEY ARE MOST                  \\nREMARK 525 CLOSELY ASSOCIATED. THE REMARK LISTS ALL THE SOLVENT                 \\nREMARK 525 MOLECULES WHICH ARE MORE THAN 5A AWAY FROM THE                       \\nREMARK 525 NEAREST POLYMER CHAIN (M = MODEL NUMBER;                             \\nREMARK 525 RES=RESIDUE NAME; C=CHAIN IDENTIFIER; SSEQ=SEQUENCE                  \\nREMARK 525 NUMBER; I=INSERTION CODE):                                           \\nREMARK 525                                                                      \\nREMARK 525  M RES CSSEQI                                                        \\nREMARK 525    HOH A  30        DISTANCE =  6.82 ANGSTROMS                       \\nDBREF  401D A    1    10  PDB    401D     401D             1     10             \\nSEQRES   1 A   10   DG  DA  DC  DC  DG  DC  DG  DG  DT  DC                      \\nFORMUL   2  HOH   *34(H2 O)                                                     \\nCRYST1   38.444   38.444   80.175  90.00  90.00 120.00 P 61 2 2     12          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.026010  0.015020  0.000000        0.00000                         \\nSCALE2      0.000000  0.030040  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.012470        0.00000                         \\nATOM      1  O5'  DG A   1      28.445  12.780  16.727  1.00 19.94           O  \\nATOM      2  C5'  DG A   1      28.383  14.200  16.550  1.00 17.21           C  \\nATOM      3  C4'  DG A   1      27.495  14.612  15.397  1.00 17.12           C  \\nATOM      4  O4'  DG A   1      28.008  14.105  14.149  1.00 13.77           O  \\nATOM      5  C3'  DG A   1      26.063  14.109  15.475  1.00 16.70           C  \\nATOM      6  O3'  DG A   1      25.263  15.013  16.251  1.00 19.38           O  \\nATOM      7  C2'  DG A   1      25.641  14.146  14.019  1.00 15.75           C  \\nATOM      8  C1'  DG A   1      26.924  13.784  13.279  1.00 14.88           C  \\nATOM      9  N9   DG A   1      27.033  12.373  12.916  1.00 17.74           N  \\nATOM     10  C8   DG A   1      27.692  11.380  13.607  1.00 18.97           C  \\nATOM     11  N7   DG A   1      27.632  10.217  13.016  1.00 15.22           N  \\nATOM     12  C5   DG A   1      26.888  10.457  11.868  1.00 17.51           C  \\nATOM     13  C6   DG A   1      26.504   9.578  10.826  1.00 15.44           C  \\nATOM     14  O6   DG A   1      26.772   8.385  10.695  1.00 17.68           O  \\nATOM     15  N1   DG A   1      25.734  10.226   9.865  1.00 16.82           N  \\nATOM     16  C2   DG A   1      25.384  11.559   9.903  1.00 17.50           C  \\nATOM     17  N2   DG A   1      24.606  12.005   8.900  1.00 14.45           N  \\nATOM     18  N3   DG A   1      25.759  12.394  10.859  1.00 16.73           N  \\nATOM     19  C4   DG A   1      26.500  11.779  11.801  1.00 14.78           C  \\nATOM     20  H1   DG A   1      25.410   9.675   9.086  1.00  0.28           H  \\nATOM     21  H21  DG A   1      24.314  12.970   8.882  1.00 -3.63           H  \\nATOM     22  H22  DG A   1      24.313  11.376   8.166  1.00  1.60           H  \\nATOM     23 HO5'  DG A   1      29.035  12.620  17.467  1.00  0.85           H  \\nATOM     24  P    DA A   2      23.995  14.454  17.068  1.00 19.26           P  \\nATOM     25  OP1  DA A   2      23.559  15.503  18.022  1.00 23.59           O  \\nATOM     26  OP2  DA A   2      24.357  13.109  17.582  1.00 19.04           O  \\nATOM     27  O5'  DA A   2      22.870  14.266  15.954  1.00 16.06           O  \\nATOM     28  C5'  DA A   2      22.393  15.378  15.187  1.00 14.98           C  \\nATOM     29  C4'  DA A   2      21.620  14.889  13.982  1.00 17.07           C  \\nATOM     30  O4'  DA A   2      22.501  14.299  13.007  1.00 13.79           O  \\nATOM     31  C3'  DA A   2      20.582  13.818  14.274  1.00 17.14           C  \\nATOM     32  O3'  DA A   2      19.350  14.403  14.709  1.00 21.29           O  \\nATOM     33  C2'  DA A   2      20.415  13.185  12.909  1.00 15.76           C  \\nATOM     34  C1'  DA A   2      21.841  13.199  12.375  1.00 12.76           C  \\nATOM     35  N9   DA A   2      22.575  11.978  12.706  1.00 10.51           N  \\nATOM     36  C8   DA A   2      23.391  11.758  13.783  1.00 10.96           C  \\nATOM     37  N7   DA A   2      23.908  10.555  13.817  1.00 11.09           N  \\nATOM     38  C5   DA A   2      23.402   9.942  12.682  1.00  9.35           C  \\nATOM     39  C6   DA A   2      23.583   8.657  12.143  1.00 10.47           C  \\nATOM     40  N6   DA A   2      24.346   7.717  12.710  1.00 11.06           N  \\nATOM     41  N1   DA A   2      22.945   8.364  10.990  1.00 12.25           N  \\nATOM     42  C2   DA A   2      22.168   9.301  10.433  1.00  9.93           C  \\nATOM     43  N3   DA A   2      21.913  10.542  10.849  1.00  8.56           N  \\nATOM     44  C4   DA A   2      22.575  10.805  11.989  1.00 10.42           C  \\nATOM     45  H61  DA A   2      24.834   7.913  13.574  1.00 -2.75           H  \\nATOM     46  H62  DA A   2      24.439   6.810  12.276  1.00  0.55           H  \\nATOM     47  P    DC A   3      18.393  13.588  15.709  1.00 17.98           P  \\nATOM     48  OP1  DC A   3      17.261  14.479  16.049  1.00 21.44           O  \\nATOM     49  OP2  DC A   3      19.213  12.988  16.796  1.00 20.04           O  \\nATOM     50  O5'  DC A   3      17.825  12.406  14.816  1.00 16.92           O  \\nATOM     51  C5'  DC A   3      16.910  12.683  13.757  1.00 17.74           C  \\nATOM     52  C4'  DC A   3      16.750  11.466  12.876  1.00 16.62           C  \\nATOM     53  O4'  DC A   3      18.033  11.098  12.332  1.00 13.12           O  \\nATOM     54  C3'  DC A   3      16.247  10.216  13.576  1.00 16.85           C  \\nATOM     55  O3'  DC A   3      14.809  10.215  13.684  1.00 17.85           O  \\nATOM     56  C2'  DC A   3      16.736   9.137  12.632  1.00 16.58           C  \\nATOM     57  C1'  DC A   3      18.095   9.677  12.191  1.00 15.13           C  \\nATOM     58  N1   DC A   3      19.199   9.179  13.025  1.00 12.40           N  \\nATOM     59  C2   DC A   3      19.750   7.947  12.720  1.00 11.65           C  \\nATOM     60  O2   DC A   3      19.337   7.341  11.725  1.00 13.90           O  \\nATOM     61  N3   DC A   3      20.715   7.441  13.509  1.00  9.80           N  \\nATOM     62  C4   DC A   3      21.140   8.130  14.566  1.00  8.68           C  \\nATOM     63  N4   DC A   3      22.084   7.580  15.331  1.00  5.25           N  \\nATOM     64  C5   DC A   3      20.615   9.412  14.886  1.00  7.30           C  \\nATOM     65  C6   DC A   3      19.657   9.895  14.094  1.00 10.90           C  \\nATOM     66  H41  DC A   3      22.422   8.072  16.145  1.00 -0.22           H  \\nATOM     67  H42  DC A   3      22.458   6.672  15.095  1.00 -1.59           H  \\nATOM     68  P    DC A   4      14.095   9.365  14.853  1.00 14.97           P  \\nATOM     69  OP1  DC A   4      12.684   9.788  14.982  1.00 20.66           O  \\nATOM     70  OP2  DC A   4      14.979   9.383  16.047  1.00 16.36           O  \\nATOM     71  O5'  DC A   4      14.095   7.889  14.277  1.00 13.68           O  \\nATOM     72  C5'  DC A   4      13.442   7.604  13.046  1.00 15.28           C  \\nATOM     73  C4'  DC A   4      13.787   6.207  12.591  1.00 15.73           C  \\nATOM     74  O4'  DC A   4      15.171   6.141  12.211  1.00 15.49           O  \\nATOM     75  C3'  DC A   4      13.629   5.132  13.650  1.00 15.89           C  \\nATOM     76  O3'  DC A   4      12.278   4.702  13.789  1.00 17.36           O  \\nATOM     77  C2'  DC A   4      14.489   4.033  13.068  1.00 13.91           C  \\nATOM     78  C1'  DC A   4      15.660   4.825  12.494  1.00 15.50           C  \\nATOM     79  N1   DC A   4      16.778   4.936  13.439  1.00 12.78           N  \\nATOM     80  C2   DC A   4      17.669   3.875  13.515  1.00 14.08           C  \\nATOM     81  O2   DC A   4      17.482   2.886  12.789  1.00 13.67           O  \\nATOM     82  N3   DC A   4      18.704   3.938  14.373  1.00 14.40           N  \\nATOM     83  C4   DC A   4      18.861   5.002  15.150  1.00  9.25           C  \\nATOM     84  N4   DC A   4      19.901   4.995  15.996  1.00  6.68           N  \\nATOM     85  C5   DC A   4      17.964   6.110  15.101  1.00  8.65           C  \\nATOM     86  C6   DC A   4      16.946   6.037  14.234  1.00 12.29           C  \\nATOM     87  H41  DC A   4      20.058   5.780  16.611  1.00 -1.36           H  \\nATOM     88  H42  DC A   4      20.525   4.203  16.018  1.00  1.44           H  \\nATOM     89  P    DG A   5      11.805   4.044  15.171  1.00 18.31           P  \\nATOM     90  OP1  DG A   5      10.325   3.973  15.152  1.00 20.54           O  \\nATOM     91  OP2  DG A   5      12.495   4.736  16.289  1.00 17.66           O  \\nATOM     92  O5'  DG A   5      12.387   2.566  15.084  1.00 17.42           O  \\nATOM     93  C5'  DG A   5      11.908   1.654  14.082  1.00 15.38           C  \\nATOM     94  C4'  DG A   5      12.634   0.333  14.186  1.00 13.44           C  \\nATOM     95  O4'  DG A   5      14.018   0.492  13.825  1.00 12.19           O  \\nATOM     96  C3'  DG A   5      12.659  -0.239  15.591  1.00 14.60           C  \\nATOM     97  O3'  DG A   5      11.468  -0.967  15.893  1.00 13.65           O  \\nATOM     98  C2'  DG A   5      13.868  -1.148  15.527  1.00 11.58           C  \\nATOM     99  C1'  DG A   5      14.824  -0.321  14.678  1.00 10.77           C  \\nATOM    100  N9   DG A   5      15.674   0.560  15.470  1.00  7.38           N  \\nATOM    101  C8   DG A   5      15.557   1.921  15.626  1.00  3.29           C  \\nATOM    102  N7   DG A   5      16.486   2.428  16.390  1.00  6.42           N  \\nATOM    103  C5   DG A   5      17.260   1.335  16.760  1.00  5.20           C  \\nATOM    104  C6   DG A   5      18.418   1.261  17.568  1.00  4.02           C  \\nATOM    105  O6   DG A   5      19.024   2.176  18.116  1.00  5.31           O  \\nATOM    106  N1   DG A   5      18.871  -0.052  17.701  1.00  5.63           N  \\nATOM    107  C2   DG A   5      18.285  -1.149  17.117  1.00  2.64           C  \\nATOM    108  N2   DG A   5      18.846  -2.337  17.374  1.00  2.00           N  \\nATOM    109  N3   DG A   5      17.220  -1.086  16.342  1.00  6.00           N  \\nATOM    110  C4   DG A   5      16.763   0.175  16.210  1.00  2.95           C  \\nATOM    111  H1   DG A   5      19.690  -0.207  18.272  1.00  0.80           H  \\nATOM    112  H21  DG A   5      18.457  -3.181  16.979  1.00  0.70           H  \\nATOM    113  H22  DG A   5      19.663  -2.391  17.966  1.00 -1.96           H  \\nATOM    114  P    DC A   6      11.011  -1.100  17.420  1.00 13.94           P  \\nATOM    115  OP1  DC A   6       9.719  -1.821  17.488  1.00 13.64           O  \\nATOM    116  OP2  DC A   6      11.126   0.245  18.023  1.00 17.86           O  \\nATOM    117  O5'  DC A   6      12.159  -1.991  18.062  1.00 10.56           O  \\nATOM    118  C5'  DC A   6      12.274  -3.383  17.744  1.00 11.64           C  \\nATOM    119  C4'  DC A   6      13.366  -4.014  18.575  1.00  8.61           C  \\nATOM    120  O4'  DC A   6      14.656  -3.514  18.180  1.00 12.90           O  \\nATOM    121  C3'  DC A   6      13.294  -3.732  20.058  1.00  8.64           C  \\nATOM    122  O3'  DC A   6      12.364  -4.544  20.749  1.00 13.95           O  \\nATOM    123  C2'  DC A   6      14.706  -4.076  20.476  1.00 12.33           C  \\nATOM    124  C1'  DC A   6      15.509  -3.479  19.330  1.00 10.42           C  \\nATOM    125  N1   DC A   6      15.859  -2.080  19.604  1.00 11.65           N  \\nATOM    126  C2   DC A   6      16.983  -1.816  20.389  1.00 12.58           C  \\nATOM    127  O2   DC A   6      17.674  -2.763  20.785  1.00 14.76           O  \\nATOM    128  N3   DC A   6      17.292  -0.538  20.696  1.00 14.18           N  \\nATOM    129  C4   DC A   6      16.533   0.460  20.239  1.00 12.29           C  \\nATOM    130  N4   DC A   6      16.881   1.705  20.573  1.00 12.71           N  \\nATOM    131  C5   DC A   6      15.388   0.224  19.419  1.00 10.23           C  \\nATOM    132  C6   DC A   6      15.093  -1.052  19.126  1.00 12.78           C  \\nATOM    133  H41  DC A   6      16.338   2.490  20.244  1.00 -1.39           H  \\nATOM    134  H42  DC A   6      17.690   1.861  21.157  1.00  3.29           H  \\nATOM    135  P    DG A   7      11.798  -4.037  22.154  1.00 15.99           P  \\nATOM    136  OP1  DG A   7      10.852  -5.057  22.669  1.00 19.94           O  \\nATOM    137  OP2  DG A   7      11.349  -2.634  21.968  1.00 16.34           O  \\nATOM    138  O5'  DG A   7      13.076  -4.002  23.100  1.00 12.83           O  \\nATOM    139  C5'  DG A   7      13.791  -5.203  23.404  1.00 13.52           C  \\nATOM    140  C4'  DG A   7      14.975  -4.900  24.293  1.00 14.02           C  \\nATOM    141  O4'  DG A   7      15.912  -4.027  23.631  1.00 12.27           O  \\nATOM    142  C3'  DG A   7      14.663  -4.217  25.612  1.00 11.21           C  \\nATOM    143  O3'  DG A   7      14.306  -5.195  26.592  1.00 17.14           O  \\nATOM    144  C2'  DG A   7      16.010  -3.617  25.959  1.00 14.58           C  \\nATOM    145  C1'  DG A   7      16.564  -3.213  24.600  1.00  8.47           C  \\nATOM    146  N9   DG A   7      16.313  -1.820  24.263  1.00  8.09           N  \\nATOM    147  C8   DG A   7      15.326  -1.313  23.446  1.00  8.46           C  \\nATOM    148  N7   DG A   7      15.387  -0.014  23.318  1.00  9.75           N  \\nATOM    149  C5   DG A   7      16.479   0.357  24.101  1.00  7.36           C  \\nATOM    150  C6   DG A   7      17.052   1.630  24.340  1.00  8.34           C  \\nATOM    151  O6   DG A   7      16.712   2.726  23.886  1.00 13.98           O  \\nATOM    152  N1   DG A   7      18.140   1.550  25.202  1.00  6.12           N  \\nATOM    153  C2   DG A   7      18.617   0.396  25.766  1.00  4.94           C  \\nATOM    154  N2   DG A   7      19.668   0.532  26.589  1.00  8.08           N  \\nATOM    155  N3   DG A   7      18.104  -0.796  25.545  1.00  6.58           N  \\nATOM    156  C4   DG A   7      17.046  -0.744  24.705  1.00  8.86           C  \\nATOM    157  H1   DG A   7      18.621   2.409  25.428  1.00 -0.48           H  \\nATOM    158  H21  DG A   7      20.062  -0.282  27.037  1.00  0.54           H  \\nATOM    159  H22  DG A   7      20.065   1.445  26.761  1.00  1.09           H  \\nATOM    160  P    DG A   8      13.622  -4.729  27.967  1.00 13.18           P  \\nATOM    161  OP1  DG A   8      13.394  -5.938  28.784  1.00 19.66           O  \\nATOM    162  OP2  DG A   8      12.486  -3.834  27.630  1.00 15.17           O  \\nATOM    163  O5'  DG A   8      14.744  -3.862  28.679  1.00 14.95           O  \\nATOM    164  C5'  DG A   8      14.398  -2.765  29.536  1.00 12.43           C  \\nATOM    165  C4'  DG A   8      15.641  -1.972  29.857  1.00 14.30           C  \\nATOM    166  O4'  DG A   8      16.218  -1.475  28.635  1.00 11.37           O  \\nATOM    167  C3'  DG A   8      15.453  -0.745  30.733  1.00 15.04           C  \\nATOM    168  O3'  DG A   8      15.484  -1.116  32.113  1.00 16.48           O  \\nATOM    169  C2'  DG A   8      16.684   0.061  30.380  1.00 13.73           C  \\nATOM    170  C1'  DG A   8      16.824  -0.208  28.884  1.00 13.57           C  \\nATOM    171  N9   DG A   8      16.151   0.789  28.058  1.00 13.22           N  \\nATOM    172  C8   DG A   8      15.077   0.608  27.216  1.00 10.00           C  \\nATOM    173  N7   DG A   8      14.732   1.699  26.587  1.00  3.41           N  \\nATOM    174  C5   DG A   8      15.628   2.657  27.048  1.00  6.86           C  \\nATOM    175  C6   DG A   8      15.763   4.029  26.718  1.00  7.58           C  \\nATOM    176  O6   DG A   8      15.093   4.698  25.921  1.00 15.10           O  \\nATOM    177  N1   DG A   8      16.808   4.627  27.417  1.00  5.98           N  \\nATOM    178  C2   DG A   8      17.617   3.985  28.322  1.00  5.06           C  \\nATOM    179  N2   DG A   8      18.573   4.723  28.892  1.00  4.15           N  \\nATOM    180  N3   DG A   8      17.499   2.713  28.642  1.00  6.09           N  \\nATOM    181  C4   DG A   8      16.497   2.113  27.967  1.00  8.70           C  \\nATOM    182  H1   DG A   8      16.977   5.608  27.249  1.00 -0.37           H  \\nATOM    183  H21  DG A   8      19.198   4.303  29.564  1.00 -1.31           H  \\nATOM    184  H22  DG A   8      18.671   5.699  28.651  1.00 -0.47           H  \\nATOM    185  P    DT A   9      14.633  -0.275  33.178  1.00 16.60           P  \\nATOM    186  OP1  DT A   9      14.604  -1.033  34.451  1.00 21.64           O  \\nATOM    187  OP2  DT A   9      13.358   0.131  32.534  1.00 17.29           O  \\nATOM    188  O5'  DT A   9      15.509   1.033  33.407  1.00 20.09           O  \\nATOM    189  C5'  DT A   9      16.829   0.944  33.971  1.00 17.60           C  \\nATOM    190  C4'  DT A   9      17.458   2.315  34.025  1.00 14.16           C  \\nATOM    191  O4'  DT A   9      17.588   2.836  32.689  1.00 14.03           O  \\nATOM    192  C3'  DT A   9      16.594   3.319  34.765  1.00 17.72           C  \\nATOM    193  O3'  DT A   9      16.940   3.294  36.155  1.00 23.99           O  \\nATOM    194  C2'  DT A   9      17.009   4.629  34.126  1.00 14.15           C  \\nATOM    195  C1'  DT A   9      17.311   4.235  32.685  1.00 13.03           C  \\nATOM    196  N1   DT A   9      16.229   4.484  31.710  1.00 11.86           N  \\nATOM    197  C2   DT A   9      16.223   5.694  31.046  1.00 11.27           C  \\nATOM    198  O2   DT A   9      17.025   6.584  31.269  1.00 10.38           O  \\nATOM    199  N3   DT A   9      15.235   5.823  30.105  1.00 11.43           N  \\nATOM    200  C4   DT A   9      14.273   4.893  29.776  1.00 10.66           C  \\nATOM    201  O4   DT A   9      13.476   5.137  28.878  1.00 19.24           O  \\nATOM    202  C5   DT A   9      14.307   3.667  30.541  1.00  8.58           C  \\nATOM    203  C7   DT A   9      13.271   2.621  30.279  1.00  3.81           C  \\nATOM    204  C6   DT A   9      15.271   3.524  31.456  1.00 10.82           C  \\nATOM    205  H3   DT A   9      15.206   6.695  29.596  1.00 -0.14           H  \\nATOM    206  P    DC A  10      15.834   3.668  37.262  1.00 29.15           P  \\nATOM    207  OP1  DC A  10      16.460   3.493  38.598  1.00 34.10           O  \\nATOM    208  OP2  DC A  10      14.573   2.951  36.942  1.00 28.05           O  \\nATOM    209  O5'  DC A  10      15.596   5.224  37.042  1.00 24.96           O  \\nATOM    210  C5'  DC A  10      16.640   6.163  37.331  1.00 23.04           C  \\nATOM    211  C4'  DC A  10      16.272   7.531  36.805  1.00 20.09           C  \\nATOM    212  O4'  DC A  10      16.288   7.552  35.365  1.00 15.61           O  \\nATOM    213  C3'  DC A  10      14.889   8.025  37.194  1.00 19.99           C  \\nATOM    214  O3'  DC A  10      14.849   8.509  38.548  1.00 18.36           O  \\nATOM    215  C2'  DC A  10      14.633   9.075  36.131  1.00 17.70           C  \\nATOM    216  C1'  DC A  10      15.363   8.533  34.901  1.00 13.61           C  \\nATOM    217  N1   DC A  10      14.473   7.904  33.919  1.00 10.50           N  \\nATOM    218  C2   DC A  10      14.091   8.645  32.810  1.00  8.79           C  \\nATOM    219  O2   DC A  10      14.536   9.791  32.681  1.00 12.66           O  \\nATOM    220  N3   DC A  10      13.245   8.103  31.907  1.00 11.23           N  \\nATOM    221  C4   DC A  10      12.779   6.867  32.092  1.00  8.63           C  \\nATOM    222  N4   DC A  10      11.927   6.382  31.190  1.00  7.04           N  \\nATOM    223  C5   DC A  10      13.164   6.078  33.211  1.00  8.82           C  \\nATOM    224  C6   DC A  10      14.011   6.629  34.092  1.00 10.53           C  \\nATOM    225 HO3'  DC A  10      13.956   8.819  38.715  1.00 -1.18           H  \\nATOM    226  H41  DC A  10      11.589   5.437  31.285  1.00  3.12           H  \\nATOM    227  H42  DC A  10      11.619   6.947  30.414  1.00 -0.85           H  \\nTER     228       DC A  10                                                      \\nHETATM  229  O   HOH A  11      15.084   4.232  19.813  0.92 17.79           O  \\nHETATM  230  H1  HOH A  11      14.509   3.560  20.181  1.00  2.20           H  \\nHETATM  231  H2  HOH A  11      14.672   4.477  18.985  1.00 -3.70           H  \\nHETATM  232  O   HOH A  12       8.633  -5.205  20.617  0.92 19.91           O  \\nHETATM  233  H1  HOH A  12       7.748  -5.548  20.739  1.00  0.52           H  \\nHETATM  234  H2  HOH A  12       9.126  -5.929  20.236  1.00 -0.19           H  \\nHETATM  235  O   HOH A  13      12.832   3.510  35.090  0.92 12.43           O  \\nHETATM  236  H1  HOH A  13      12.323   3.313  34.303  1.00  0.34           H  \\nHETATM  237  H2  HOH A  13      13.428   2.767  35.184  1.00 -2.35           H  \\nHETATM  238  O   HOH A  14      25.078  10.081  16.137  0.92 14.98           O  \\nHETATM  239  H1  HOH A  14      24.928  10.949  16.513  1.00 -0.15           H  \\nHETATM  240  H2  HOH A  14      25.109   9.498  16.895  1.00  0.42           H  \\nHETATM  241  O   HOH A  15       9.991  -4.605  26.139  0.92 20.73           O  \\nHETATM  242  H1  HOH A  15       9.685  -4.683  27.045  1.00  1.05           H  \\nHETATM  243  H2  HOH A  15       9.315  -5.036  25.620  1.00  3.78           H  \\nHETATM  244  O   HOH A  16      12.977   2.543  17.987  0.92 24.71           O  \\nHETATM  245  H1  HOH A  16      13.532   1.993  18.541  1.00 -1.76           H  \\nHETATM  246  H2  HOH A  16      13.529   3.292  17.762  1.00 -1.61           H  \\nHETATM  247  O   HOH A  17      19.767   1.402  30.209  0.92 14.79           O  \\nHETATM  248  H1  HOH A  17      19.637   2.085  30.867  1.00 -3.35           H  \\nHETATM  249  H2  HOH A  17      20.447   0.839  30.578  1.00  0.18           H  \\nHETATM  250  O   HOH A  18      13.290   1.860  22.372  0.92 21.52           O  \\nHETATM  251  H1  HOH A  18      14.004   1.380  22.791  1.00  0.09           H  \\nHETATM  252  H2  HOH A  18      13.587   2.770  22.355  1.00  2.28           H  \\nHETATM  253  O   HOH A  19       9.092  -3.545  15.066  0.92 29.78           O  \\nHETATM  254  H1  HOH A  19       9.704  -2.996  14.579  1.00  1.80           H  \\nHETATM  255  H2  HOH A  19       9.123  -3.205  15.962  1.00 -0.80           H  \\nHETATM  256  O   HOH A  20      11.347  -0.301  20.759  0.92 22.64           O  \\nHETATM  257  H1  HOH A  20      11.438   0.541  20.315  1.00 -1.22           H  \\nHETATM  258  H2  HOH A  20      11.295  -0.945  20.052  1.00  0.80           H  \\nHETATM  259  O   HOH A  21      19.866   0.048  32.722  0.92 40.84           O  \\nHETATM  260  H1  HOH A  21      20.802   0.025  32.925  1.00  0.23           H  \\nHETATM  261  H2  HOH A  21      19.441  -0.331  33.490  1.00 -0.27           H  \\nHETATM  262  O   HOH A  22      10.434   2.601  28.103  0.92 36.31           O  \\nHETATM  263  H1  HOH A  22      10.686   3.217  28.792  1.00  2.55           H  \\nHETATM  264  H2  HOH A  22       9.581   2.267  28.376  1.00 -0.52           H  \\nHETATM  265  O   HOH A  23      15.670   1.222  11.204  0.92 38.36           O  \\nHETATM  266  H1  HOH A  23      15.191   0.549  11.687  1.00 -1.02           H  \\nHETATM  267  H2  HOH A  23      16.568   1.158  11.531  1.00  0.11           H  \\nHETATM  268  O   HOH A  24      12.633  -1.288  26.556  0.92 18.82           O  \\nHETATM  269  H1  HOH A  24      12.536  -0.798  25.740  1.00  1.10           H  \\nHETATM  270  H2  HOH A  24      12.478  -2.201  26.313  1.00  1.08           H  \\nHETATM  271  O   HOH A  25      16.750  17.104  15.219  0.92 13.67           O  \\nHETATM  272  H1  HOH A  25      17.225  17.932  15.137  1.00 -0.19           H  \\nHETATM  273  H2  HOH A  25      17.407  16.484  15.535  1.00  1.43           H  \\nHETATM  274  O   HOH A  26      11.183  -0.661  24.128  0.92 14.09           O  \\nHETATM  275  H1  HOH A  26      11.354   0.278  24.192  1.00 -0.32           H  \\nHETATM  276  H2  HOH A  26      11.050  -0.942  25.033  1.00 -0.66           H  \\nHETATM  277  O   HOH A  27      20.912   4.065  31.343  0.92 17.55           O  \\nHETATM  278  H1  HOH A  27      20.209   3.767  30.764  1.00 -1.40           H  \\nHETATM  279  H2  HOH A  27      21.619   3.435  31.202  1.00 -0.91           H  \\nHETATM  280  O   HOH A  28      17.680   7.449   9.662  0.92 34.97           O  \\nHETATM  281  H1  HOH A  28      17.544   6.604  10.095  1.00 -1.64           H  \\nHETATM  282  H2  HOH A  28      18.556   7.373   9.286  1.00  0.34           H  \\nHETATM  283  O   HOH A  29      10.361   4.046  31.687  0.92 21.54           O  \\nHETATM  284  H1  HOH A  29      11.035   4.285  32.323  1.00 -2.32           H  \\nHETATM  285  H2  HOH A  29      10.851   3.733  30.928  1.00  3.17           H  \\nHETATM  286  O   HOH A  30       5.775   3.565  32.021  0.92 54.10           O  \\nHETATM  287  H1  HOH A  30       5.777   4.522  32.044  1.00 -0.85           H  \\nHETATM  288  H2  HOH A  30       5.775   3.348  31.089  1.00  0.36           H  \\nHETATM  289  O   HOH A  31      19.894  12.183   9.431  0.92 35.18           O  \\nHETATM  290  H1  HOH A  31      20.562  11.820  10.003  1.00 -0.52           H  \\nHETATM  291  H2  HOH A  31      19.153  11.588   9.542  1.00  1.51           H  \\nHETATM  292  O   HOH A  32      31.037  13.244  19.339  0.92 32.52           O  \\nHETATM  293  H1  HOH A  32      30.100  13.238  19.139  1.00 -0.50           H  \\nHETATM  294  H2  HOH A  32      31.241  12.329  19.531  1.00  1.17           H  \\nHETATM  295  O   HOH A  33      11.558  -0.240  29.036  0.92 30.11           O  \\nHETATM  296  H1  HOH A  33      10.955   0.323  28.550  1.00 -0.33           H  \\nHETATM  297  H2  HOH A  33      11.635  -1.030  28.501  1.00 -2.52           H  \\nHETATM  298  O   HOH A  35      15.976  11.888  34.322  0.92 13.69           O  \\nHETATM  299  H1  HOH A  35      15.850  11.760  35.266  1.00  1.96           H  \\nHETATM  300  H2  HOH A  35      15.715  11.048  33.941  1.00 -0.38           H  \\nHETATM  301  O   HOH A  37      13.420   7.005  40.225  0.92 10.34           O  \\nHETATM  302  H1  HOH A  37      14.089   7.225  40.873  1.00 -0.69           H  \\nHETATM  303  H2  HOH A  37      12.710   7.624  40.394  1.00 -3.25           H  \\nHETATM  304  O   HOH A  38      17.284  -2.500  35.916  0.92 37.92           O  \\nHETATM  305  H1  HOH A  38      18.133  -2.832  35.624  1.00  3.16           H  \\nHETATM  306  H2  HOH A  38      16.878  -2.147  35.126  1.00  1.24           H  \\nHETATM  307  O   HOH A  39      20.933   3.007  34.529  0.92 49.67           O  \\nHETATM  308  H1  HOH A  39      20.827   2.704  35.430  1.00  2.93           H  \\nHETATM  309  H2  HOH A  39      21.823   2.751  34.287  1.00 -0.06           H  \\nHETATM  310  O   HOH A  40      10.697   7.276  11.276  0.92 23.22           O  \\nHETATM  311  H1  HOH A  40      10.550   6.505  11.825  1.00  1.08           H  \\nHETATM  312  H2  HOH A  40      11.123   7.906  11.857  1.00 -0.06           H  \\nHETATM  313  O   HOH A  41      23.494  14.870   8.728  0.92 19.82           O  \\nHETATM  314  H1  HOH A  41      23.653  15.799   8.907  1.00  1.41           H  \\nHETATM  315  H2  HOH A  41      22.746  14.866   8.128  1.00 -3.30           H  \\nHETATM  316  O   HOH A  42      12.066  11.232  12.195  0.96 36.07           O  \\nHETATM  317  H1  HOH A  42      11.248  11.727  12.164  1.00 -2.80           H  \\nHETATM  318  H2  HOH A  42      12.250  11.149  13.132  1.00  0.23           H  \\nHETATM  319  O   HOH A  43      26.672  12.712  18.951  0.96 42.39           O  \\nHETATM  320  H1  HOH A  43      27.559  13.063  18.900  1.00  0.23           H  \\nHETATM  321  H2  HOH A  43      26.468  12.449  18.052  1.00 -3.65           H  \\nHETATM  322  O   HOH A  44      14.854  11.227  19.213  0.96 36.50           O  \\nHETATM  323  H1  HOH A  44      14.000  11.658  19.156  1.00  3.66           H  \\nHETATM  324  H2  HOH A  44      14.988  11.082  20.149  1.00  1.26           H  \\nHETATM  325  O   HOH A  45      10.696   7.159  17.758  0.99 54.36           O  \\nHETATM  326  H1  HOH A  45       9.772   7.043  17.554  1.00  0.66           H  \\nHETATM  327  H2  HOH A  45      10.864   6.465  18.396  1.00  0.14           H  \\nHETATM  328  O   HOH A  46      15.287  13.783  18.124  0.99 32.92           O  \\nHETATM  329  H1  HOH A  46      15.021  14.146  17.276  1.00 -1.45           H  \\nHETATM  330  H2  HOH A  46      16.242  13.801  18.095  1.00  1.68           H  \\nMASTER      247    0    0    0    0    0    0    6  329    1    0    1          \\nEND                                                                             \\n\");\n",
       "\tviewer_1684696742122504.setStyle({\"model\": -1, \"chain\": \"A\"},{\"cartoon\": {\"color\": \"spectrum\"}});\n",
       "\tviewer_1684696742122504.zoomTo();\n",
       "viewer_1684696742122504.render();\n",
       "});\n",
       "</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Similarity score: 10.0\n",
      " Protein: the unusual structure of the human centromere (gga)2 motif: unpaired guanosine residues stacked between sheared g(dot)a pairs(103D)\n",
      " Chain Id: A\n",
      " Sequence of the chain: DGDTDGDGDADADTDGDGDADADC\n"
     ]
    },
    {
     "data": {
      "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_168469674212664\"  style=\"position: relative; width: 600px; height: 400px\">\n        <p id=\"3dmolwarning_168469674212664\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.0.1/3Dmol-min.js');\n}\n\nvar viewer_168469674212664 = null;\nvar warn = document.getElementById(\"3dmolwarning_168469674212664\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_168469674212664 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_168469674212664\"),{backgroundColor:\"white\"});\nviewer_168469674212664.zoomTo();\n\tviewer_168469674212664.addModelsAsFrames(\"HEADER    DNA                                     16-DEC-94   103D              \\nTITLE     THE UNUSUAL STRUCTURE OF THE HUMAN CENTROMERE (GGA)2 MOTIF: UNPAIRED  \\nTITLE    2 GUANOSINE RESIDUES STACKED BETWEEN SHEARED G(DOT)A PAIRS             \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: DNA (5'-D(*GP*TP*GP*GP*AP*AP*TP*GP*GP*AP*AP*C)-3');        \\nCOMPND   3 CHAIN: A, B;                                                         \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 SYNTHETIC: YES;                                                      \\nSOURCE   3 OTHER_DETAILS: CHEMICALLY SYNTHESIZED                                \\nKEYWDS    DNA, DOUBLE HELIX, G-G STACKING, G:A MISMATCH, HUMAN CENTROMERE       \\nKEYWDS   2 REPEAT, GA-BRACKETED G-STACK MOTIF                                   \\nEXPDTA    SOLUTION NMR                                                          \\nAUTHOR    S.-H.CHOU,L.ZHU,B.R.REID                                              \\nREVDAT   5   16-FEB-22 103D    1       REMARK                                   \\nREVDAT   4   07-APR-09 103D    1       REMARK                                   \\nREVDAT   3   24-FEB-09 103D    1       VERSN                                    \\nREVDAT   2   01-APR-03 103D    1       JRNL                                     \\nREVDAT   1   31-MAR-95 103D    0                                                \\nJRNL        AUTH   S.H.CHOU,L.ZHU,B.R.REID                                      \\nJRNL        TITL   THE UNUSUAL STRUCTURE OF THE HUMAN CENTROMERE (GGA)2 MOTIF.  \\nJRNL        TITL 2 UNPAIRED GUANOSINE RESIDUES STACKED BETWEEN SHEARED G.A      \\nJRNL        TITL 3 PAIRS.                                                       \\nJRNL        REF    J.MOL.BIOL.                   V. 244   259 1994              \\nJRNL        REFN                   ISSN 0022-2836                               \\nJRNL        PMID   7966337                                                      \\nJRNL        DOI    10.1006/JMBI.1994.1727                                       \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION. NOT APPLICABLE.                                          \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : AMBER                                                \\nREMARK   3   AUTHORS     : BIOSYM TECHNOLOGIES, INC.                            \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: R VALUE 0.250 FINAL RMS COORD. SHIFT      \\nREMARK   3  0.65 ANGSTROMS NUMBER OF ATOMS USED IN REFINEMENT. NUMBER OF        \\nREMARK   3  PROTEIN ATOMS 0 NUMBER OF NUCLEIC ACID ATOMS 772 NUMBER OF          \\nREMARK   3  HETEROGEN ATOMS 0 NUMBER OF SOLVENT ATOMS 0. THE DUPLEX CONTAINS    \\nREMARK   3  UNPAIRED STAGGERED GUANOSINE RESIDUES, WHICH CO-STACK BY            \\nREMARK   3  INTERCALATION BETWEEN SHEARED G(DOT)A AND A(DOT)G BASE-PAIRS TO     \\nREMARK   3  FORM AN INTERESTING NEW STRUCTURAL MOTIF, THE GA-BRACKETED G-       \\nREMARK   3  STACK. THE TGGAA REPEAT UNIT CONTAINS SIX \\\"STEPS\\\", FOUR OF WHICH    \\nREMARK   3  ARE NOT WATSON-CRICK BASE-PAIRS.                                    \\nREMARK   4                                                                      \\nREMARK   4 103D COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000170007.                                   \\nREMARK 210                                                                      \\nREMARK 210 EXPERIMENTAL DETAILS                                                 \\nREMARK 210  EXPERIMENT TYPE                : NMR                                \\nREMARK 210  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 210  PH                             : NULL                               \\nREMARK 210  IONIC STRENGTH                 : NULL                               \\nREMARK 210  PRESSURE                       : NULL                               \\nREMARK 210  SAMPLE CONTENTS                : NULL                               \\nREMARK 210                                                                      \\nREMARK 210  NMR EXPERIMENTS CONDUCTED      : NULL                               \\nREMARK 210  SPECTROMETER FIELD STRENGTH    : NULL                               \\nREMARK 210  SPECTROMETER MODEL             : NULL                               \\nREMARK 210  SPECTROMETER MANUFACTURER      : NULL                               \\nREMARK 210                                                                      \\nREMARK 210  STRUCTURE DETERMINATION.                                            \\nREMARK 210   SOFTWARE USED                 : NULL                               \\nREMARK 210   METHOD USED                   : NULL                               \\nREMARK 210                                                                      \\nREMARK 210 CONFORMERS, NUMBER CALCULATED   : NULL                               \\nREMARK 210 CONFORMERS, NUMBER SUBMITTED    : 1                                  \\nREMARK 210 CONFORMERS, SELECTION CRITERIA  : NULL                               \\nREMARK 210                                                                      \\nREMARK 210 BEST REPRESENTATIVE CONFORMER IN THIS ENSEMBLE : NULL                \\nREMARK 210                                                                      \\nREMARK 210 REMARK: NULL                                                         \\nREMARK 215                                                                      \\nREMARK 215 NMR STUDY                                                            \\nREMARK 215 THE COORDINATES IN THIS ENTRY WERE GENERATED FROM SOLUTION           \\nREMARK 215 NMR DATA.  PROTEIN DATA BANK CONVENTIONS REQUIRE THAT                \\nREMARK 215 CRYST1 AND SCALE RECORDS BE INCLUDED, BUT THE VALUES ON              \\nREMARK 215 THESE RECORDS ARE MEANINGLESS.                                       \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A, B                                  \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \\nREMARK 500     DG A   1   O4' -  C1' -  N9  ANGL. DEV. =   1.9 DEGREES          \\nREMARK 500     DA A   5   O4' -  C1' -  N9  ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500     DA A   6   O4' -  C1' -  N9  ANGL. DEV. =   2.2 DEGREES          \\nREMARK 500     DT A   7   O4' -  C1' -  N1  ANGL. DEV. =   2.5 DEGREES          \\nREMARK 500     DA A  10   O4' -  C1' -  N9  ANGL. DEV. =   2.7 DEGREES          \\nREMARK 500     DA A  11   O4' -  C1' -  N9  ANGL. DEV. =   2.0 DEGREES          \\nREMARK 500     DC A  12   O4' -  C1' -  N1  ANGL. DEV. =   2.5 DEGREES          \\nREMARK 500     DG B  13   O4' -  C1' -  N9  ANGL. DEV. =   1.9 DEGREES          \\nREMARK 500     DT B  14   O4' -  C1' -  N1  ANGL. DEV. =   1.8 DEGREES          \\nREMARK 500     DA B  17   O4' -  C1' -  N9  ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500     DA B  18   O4' -  C1' -  N9  ANGL. DEV. =   2.3 DEGREES          \\nREMARK 500     DT B  19   O4' -  C1' -  N1  ANGL. DEV. =   2.5 DEGREES          \\nREMARK 500     DA B  22   O4' -  C1' -  N9  ANGL. DEV. =   2.7 DEGREES          \\nREMARK 500     DA B  23   O4' -  C1' -  N9  ANGL. DEV. =   1.9 DEGREES          \\nREMARK 500     DC B  24   O4' -  C1' -  N1  ANGL. DEV. =   2.4 DEGREES          \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: PLANAR GROUPS                                              \\nREMARK 500                                                                      \\nREMARK 500 PLANAR GROUPS IN THE FOLLOWING RESIDUES HAVE A TOTAL                 \\nREMARK 500 RMS DISTANCE OF ALL ATOMS FROM THE BEST-FIT PLANE                    \\nREMARK 500 BY MORE THAN AN EXPECTED VALUE OF 6*RMSD, WITH AN                    \\nREMARK 500 RMSD 0.02 ANGSTROMS, OR AT LEAST ONE ATOM HAS                        \\nREMARK 500 AN RMSD GREATER THAN THIS VALUE                                      \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        RMS     TYPE                                    \\nREMARK 500     DT A   2         0.06    SIDE CHAIN                              \\nREMARK 500     DT B  14         0.06    SIDE CHAIN                              \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nDBREF  103D A    1    12  PDB    103D     103D             1     12             \\nDBREF  103D B   13    24  PDB    103D     103D            13     24             \\nSEQRES   1 A   12   DG  DT  DG  DG  DA  DA  DT  DG  DG  DA  DA  DC              \\nSEQRES   1 B   12   DG  DT  DG  DG  DA  DA  DT  DG  DG  DA  DA  DC              \\nCRYST1    1.000    1.000    1.000  90.00  90.00  90.00 P 1           1          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      1.000000  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  1.000000  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  1.000000        0.00000                         \\nATOM      1  O5'  DG A   1      -7.643 -22.640  -1.099  1.00  0.00           O  \\nATOM      2  C5'  DG A   1      -8.313 -23.137   0.040  1.00  0.00           C  \\nATOM      3  C4'  DG A   1      -7.935 -22.349   1.300  1.00  0.00           C  \\nATOM      4  O4'  DG A   1      -6.537 -22.463   1.524  1.00  0.00           O  \\nATOM      5  C3'  DG A   1      -8.273 -20.854   1.190  1.00  0.00           C  \\nATOM      6  O3'  DG A   1      -8.832 -20.418   2.419  1.00  0.00           O  \\nATOM      7  C2'  DG A   1      -6.902 -20.240   0.924  1.00  0.00           C  \\nATOM      8  C1'  DG A   1      -5.995 -21.171   1.722  1.00  0.00           C  \\nATOM      9  N9   DG A   1      -4.593 -21.126   1.247  1.00  0.00           N  \\nATOM     10  C8   DG A   1      -4.107 -21.429  -0.001  1.00  0.00           C  \\nATOM     11  N7   DG A   1      -2.812 -21.335  -0.110  1.00  0.00           N  \\nATOM     12  C5   DG A   1      -2.397 -20.963   1.163  1.00  0.00           C  \\nATOM     13  C6   DG A   1      -1.080 -20.733   1.666  1.00  0.00           C  \\nATOM     14  O6   DG A   1      -0.017 -20.780   1.053  1.00  0.00           O  \\nATOM     15  N1   DG A   1      -1.077 -20.427   3.024  1.00  0.00           N  \\nATOM     16  C2   DG A   1      -2.214 -20.317   3.803  1.00  0.00           C  \\nATOM     17  N2   DG A   1      -2.045 -20.010   5.092  1.00  0.00           N  \\nATOM     18  N3   DG A   1      -3.455 -20.507   3.325  1.00  0.00           N  \\nATOM     19  C4   DG A   1      -3.479 -20.835   2.004  1.00  0.00           C  \\nATOM     20  H5'  DG A   1      -8.042 -24.185   0.178  1.00  0.00           H  \\nATOM     21 H5''  DG A   1      -9.389 -23.072  -0.122  1.00  0.00           H  \\nATOM     22  H4'  DG A   1      -8.473 -22.789   2.142  1.00  0.00           H  \\nATOM     23  H3'  DG A   1      -8.969 -20.658   0.374  1.00  0.00           H  \\nATOM     24  H2'  DG A   1      -6.682 -20.310  -0.141  1.00  0.00           H  \\nATOM     25 H2''  DG A   1      -6.821 -19.205   1.250  1.00  0.00           H  \\nATOM     26  H1'  DG A   1      -6.058 -20.912   2.780  1.00  0.00           H  \\nATOM     27  H8   DG A   1      -4.739 -21.718  -0.828  1.00  0.00           H  \\nATOM     28  H1   DG A   1      -0.169 -20.285   3.449  1.00  0.00           H  \\nATOM     29  H21  DG A   1      -1.114 -19.925   5.477  1.00  0.00           H  \\nATOM     30  H22  DG A   1      -2.853 -19.902   5.687  1.00  0.00           H  \\nATOM     31 HO5'  DG A   1      -7.929 -21.737  -1.254  1.00  0.00           H  \\nATOM     32  P    DT A   2      -9.356 -18.904   2.648  1.00  0.00           P  \\nATOM     33  OP1  DT A   2     -10.579 -18.963   3.478  1.00  0.00           O  \\nATOM     34  OP2  DT A   2      -9.388 -18.212   1.340  1.00  0.00           O  \\nATOM     35  O5'  DT A   2      -8.175 -18.250   3.532  1.00  0.00           O  \\nATOM     36  C5'  DT A   2      -7.959 -18.655   4.871  1.00  0.00           C  \\nATOM     37  C4'  DT A   2      -6.761 -17.920   5.479  1.00  0.00           C  \\nATOM     38  O4'  DT A   2      -5.603 -18.173   4.699  1.00  0.00           O  \\nATOM     39  C3'  DT A   2      -6.955 -16.396   5.550  1.00  0.00           C  \\nATOM     40  O3'  DT A   2      -6.507 -15.961   6.826  1.00  0.00           O  \\nATOM     41  C2'  DT A   2      -6.062 -15.914   4.406  1.00  0.00           C  \\nATOM     42  C1'  DT A   2      -4.938 -16.944   4.495  1.00  0.00           C  \\nATOM     43  N1   DT A   2      -4.057 -17.032   3.297  1.00  0.00           N  \\nATOM     44  C2   DT A   2      -2.677 -17.088   3.499  1.00  0.00           C  \\nATOM     45  O2   DT A   2      -2.146 -16.945   4.598  1.00  0.00           O  \\nATOM     46  N3   DT A   2      -1.900 -17.352   2.381  1.00  0.00           N  \\nATOM     47  C4   DT A   2      -2.368 -17.603   1.102  1.00  0.00           C  \\nATOM     48  O4   DT A   2      -1.578 -17.886   0.204  1.00  0.00           O  \\nATOM     49  C5   DT A   2      -3.808 -17.502   0.964  1.00  0.00           C  \\nATOM     50  C7   DT A   2      -4.451 -17.702  -0.395  1.00  0.00           C  \\nATOM     51  C6   DT A   2      -4.587 -17.231   2.042  1.00  0.00           C  \\nATOM     52  H5'  DT A   2      -7.763 -19.727   4.902  1.00  0.00           H  \\nATOM     53 H5''  DT A   2      -8.845 -18.442   5.472  1.00  0.00           H  \\nATOM     54  H4'  DT A   2      -6.601 -18.314   6.483  1.00  0.00           H  \\nATOM     55  H3'  DT A   2      -7.997 -16.110   5.395  1.00  0.00           H  \\nATOM     56  H2'  DT A   2      -6.630 -15.995   3.481  1.00  0.00           H  \\nATOM     57 H2''  DT A   2      -5.703 -14.894   4.522  1.00  0.00           H  \\nATOM     58  H1'  DT A   2      -4.373 -16.711   5.397  1.00  0.00           H  \\nATOM     59  H3   DT A   2      -0.903 -17.421   2.534  1.00  0.00           H  \\nATOM     60  H71  DT A   2      -3.753 -17.435  -1.188  1.00  0.00           H  \\nATOM     61  H72  DT A   2      -4.736 -18.745  -0.514  1.00  0.00           H  \\nATOM     62  H73  DT A   2      -5.340 -17.077  -0.489  1.00  0.00           H  \\nATOM     63  H6   DT A   2      -5.655 -17.199   1.905  1.00  0.00           H  \\nATOM     64  P    DG A   3      -6.534 -14.414   7.292  1.00  0.00           P  \\nATOM     65  OP1  DG A   3      -6.475 -14.378   8.770  1.00  0.00           O  \\nATOM     66  OP2  DG A   3      -7.637 -13.728   6.584  1.00  0.00           O  \\nATOM     67  O5'  DG A   3      -5.139 -13.848   6.720  1.00  0.00           O  \\nATOM     68  C5'  DG A   3      -3.896 -14.287   7.234  1.00  0.00           C  \\nATOM     69  C4'  DG A   3      -2.748 -13.705   6.405  1.00  0.00           C  \\nATOM     70  O4'  DG A   3      -2.863 -14.119   5.053  1.00  0.00           O  \\nATOM     71  C3'  DG A   3      -2.751 -12.171   6.416  1.00  0.00           C  \\nATOM     72  O3'  DG A   3      -1.406 -11.747   6.547  1.00  0.00           O  \\nATOM     73  C2'  DG A   3      -3.335 -11.844   5.041  1.00  0.00           C  \\nATOM     74  C1'  DG A   3      -2.741 -12.983   4.218  1.00  0.00           C  \\nATOM     75  N9   DG A   3      -3.405 -13.249   2.921  1.00  0.00           N  \\nATOM     76  C8   DG A   3      -4.715 -13.058   2.555  1.00  0.00           C  \\nATOM     77  N7   DG A   3      -4.994 -13.444   1.343  1.00  0.00           N  \\nATOM     78  C5   DG A   3      -3.784 -13.924   0.857  1.00  0.00           C  \\nATOM     79  C6   DG A   3      -3.457 -14.458  -0.426  1.00  0.00           C  \\nATOM     80  O6   DG A   3      -4.194 -14.609  -1.397  1.00  0.00           O  \\nATOM     81  N1   DG A   3      -2.119 -14.829  -0.523  1.00  0.00           N  \\nATOM     82  C2   DG A   3      -1.196 -14.693   0.496  1.00  0.00           C  \\nATOM     83  N2   DG A   3       0.052 -15.094   0.239  1.00  0.00           N  \\nATOM     84  N3   DG A   3      -1.501 -14.190   1.703  1.00  0.00           N  \\nATOM     85  C4   DG A   3      -2.807 -13.820   1.820  1.00  0.00           C  \\nATOM     86  H5'  DG A   3      -3.836 -15.375   7.202  1.00  0.00           H  \\nATOM     87 H5''  DG A   3      -3.792 -13.963   8.271  1.00  0.00           H  \\nATOM     88  H4'  DG A   3      -1.809 -14.080   6.816  1.00  0.00           H  \\nATOM     89  H3'  DG A   3      -3.366 -11.782   7.227  1.00  0.00           H  \\nATOM     90  H2'  DG A   3      -4.421 -11.917   5.096  1.00  0.00           H  \\nATOM     91 H2''  DG A   3      -3.043 -10.865   4.661  1.00  0.00           H  \\nATOM     92  H1'  DG A   3      -1.692 -12.735   4.043  1.00  0.00           H  \\nATOM     93  H8   DG A   3      -5.463 -12.632   3.207  1.00  0.00           H  \\nATOM     94  H1   DG A   3      -1.821 -15.221  -1.404  1.00  0.00           H  \\nATOM     95  H21  DG A   3       0.298 -15.465  -0.666  1.00  0.00           H  \\nATOM     96  H22  DG A   3       0.772 -14.962   0.941  1.00  0.00           H  \\nATOM     97  P    DG A   4      -0.997 -10.224   6.884  1.00  0.00           P  \\nATOM     98  OP1  DG A   4      -0.439 -10.175   8.254  1.00  0.00           O  \\nATOM     99  OP2  DG A   4      -2.112  -9.328   6.505  1.00  0.00           O  \\nATOM    100  O5'  DG A   4       0.196 -10.007   5.828  1.00  0.00           O  \\nATOM    101  C5'  DG A   4       1.508 -10.482   6.064  1.00  0.00           C  \\nATOM    102  C4'  DG A   4       2.254 -10.592   4.729  1.00  0.00           C  \\nATOM    103  O4'  DG A   4       1.640 -11.522   3.855  1.00  0.00           O  \\nATOM    104  C3'  DG A   4       2.288  -9.262   3.978  1.00  0.00           C  \\nATOM    105  O3'  DG A   4       3.326  -8.456   4.505  1.00  0.00           O  \\nATOM    106  C2'  DG A   4       2.518  -9.741   2.552  1.00  0.00           C  \\nATOM    107  C1'  DG A   4       1.787 -11.086   2.512  1.00  0.00           C  \\nATOM    108  N9   DG A   4       0.429 -11.010   1.915  1.00  0.00           N  \\nATOM    109  C8   DG A   4      -0.746 -10.615   2.504  1.00  0.00           C  \\nATOM    110  N7   DG A   4      -1.792 -10.688   1.732  1.00  0.00           N  \\nATOM    111  C5   DG A   4      -1.287 -11.185   0.538  1.00  0.00           C  \\nATOM    112  C6   DG A   4      -1.954 -11.480  -0.691  1.00  0.00           C  \\nATOM    113  O6   DG A   4      -3.141 -11.323  -0.965  1.00  0.00           O  \\nATOM    114  N1   DG A   4      -1.096 -12.013  -1.650  1.00  0.00           N  \\nATOM    115  C2   DG A   4       0.251 -12.248  -1.443  1.00  0.00           C  \\nATOM    116  N2   DG A   4       0.952 -12.759  -2.460  1.00  0.00           N  \\nATOM    117  N3   DG A   4       0.873 -11.977  -0.287  1.00  0.00           N  \\nATOM    118  C4   DG A   4       0.061 -11.433   0.657  1.00  0.00           C  \\nATOM    119  H5'  DG A   4       1.495 -11.457   6.552  1.00  0.00           H  \\nATOM    120 H5''  DG A   4       2.027  -9.777   6.714  1.00  0.00           H  \\nATOM    121  H4'  DG A   4       3.271 -10.944   4.919  1.00  0.00           H  \\nATOM    122  H3'  DG A   4       1.317  -8.768   4.033  1.00  0.00           H  \\nATOM    123  H2'  DG A   4       2.150  -9.039   1.815  1.00  0.00           H  \\nATOM    124 H2''  DG A   4       3.578  -9.904   2.373  1.00  0.00           H  \\nATOM    125  H1'  DG A   4       2.409 -11.766   1.933  1.00  0.00           H  \\nATOM    126  H8   DG A   4      -0.813 -10.291   3.528  1.00  0.00           H  \\nATOM    127  H1   DG A   4      -1.500 -12.225  -2.549  1.00  0.00           H  \\nATOM    128  H21  DG A   4       0.523 -12.928  -3.357  1.00  0.00           H  \\nATOM    129  H22  DG A   4       1.950 -12.906  -2.334  1.00  0.00           H  \\nATOM    130  P    DA A   5       3.562  -6.924   4.058  1.00  0.00           P  \\nATOM    131  OP1  DA A   5       4.379  -6.268   5.103  1.00  0.00           O  \\nATOM    132  OP2  DA A   5       2.259  -6.334   3.677  1.00  0.00           O  \\nATOM    133  O5'  DA A   5       4.464  -7.037   2.730  1.00  0.00           O  \\nATOM    134  C5'  DA A   5       5.721  -7.693   2.730  1.00  0.00           C  \\nATOM    135  C4'  DA A   5       6.616  -7.135   1.615  1.00  0.00           C  \\nATOM    136  O4'  DA A   5       5.843  -6.978   0.432  1.00  0.00           O  \\nATOM    137  C3'  DA A   5       7.201  -5.758   1.969  1.00  0.00           C  \\nATOM    138  O3'  DA A   5       8.471  -5.625   1.344  1.00  0.00           O  \\nATOM    139  C2'  DA A   5       6.152  -4.834   1.366  1.00  0.00           C  \\nATOM    140  C1'  DA A   5       5.814  -5.603   0.093  1.00  0.00           C  \\nATOM    141  N9   DA A   5       4.510  -5.219  -0.496  1.00  0.00           N  \\nATOM    142  C8   DA A   5       3.302  -4.987   0.119  1.00  0.00           C  \\nATOM    143  N7   DA A   5       2.320  -4.747  -0.706  1.00  0.00           N  \\nATOM    144  C5   DA A   5       2.915  -4.819  -1.959  1.00  0.00           C  \\nATOM    145  C6   DA A   5       2.422  -4.678  -3.273  1.00  0.00           C  \\nATOM    146  N6   DA A   5       1.142  -4.414  -3.547  1.00  0.00           N  \\nATOM    147  N1   DA A   5       3.285  -4.794  -4.301  1.00  0.00           N  \\nATOM    148  C2   DA A   5       4.562  -5.063  -4.040  1.00  0.00           C  \\nATOM    149  N3   DA A   5       5.145  -5.238  -2.858  1.00  0.00           N  \\nATOM    150  C4   DA A   5       4.254  -5.095  -1.840  1.00  0.00           C  \\nATOM    151  H5'  DA A   5       5.562  -8.757   2.560  1.00  0.00           H  \\nATOM    152 H5''  DA A   5       6.232  -7.572   3.686  1.00  0.00           H  \\nATOM    153  H4'  DA A   5       7.427  -7.844   1.444  1.00  0.00           H  \\nATOM    154  H3'  DA A   5       7.295  -5.615   3.047  1.00  0.00           H  \\nATOM    155  H2'  DA A   5       5.300  -4.777   2.042  1.00  0.00           H  \\nATOM    156 H2''  DA A   5       6.527  -3.836   1.162  1.00  0.00           H  \\nATOM    157  H1'  DA A   5       6.610  -5.423  -0.628  1.00  0.00           H  \\nATOM    158  H8   DA A   5       3.159  -5.017   1.188  1.00  0.00           H  \\nATOM    159  H61  DA A   5       0.850  -4.267  -4.502  1.00  0.00           H  \\nATOM    160  H62  DA A   5       0.451  -4.437  -2.806  1.00  0.00           H  \\nATOM    161  H2   DA A   5       5.213  -5.162  -4.894  1.00  0.00           H  \\nATOM    162  P    DA A   6       9.363  -4.276   1.441  1.00  0.00           P  \\nATOM    163  OP1  DA A   6      10.782  -4.649   1.246  1.00  0.00           O  \\nATOM    164  OP2  DA A   6       8.960  -3.530   2.654  1.00  0.00           O  \\nATOM    165  O5'  DA A   6       8.887  -3.431   0.150  1.00  0.00           O  \\nATOM    166  C5'  DA A   6       9.220  -3.846  -1.162  1.00  0.00           C  \\nATOM    167  C4'  DA A   6       8.472  -3.000  -2.195  1.00  0.00           C  \\nATOM    168  O4'  DA A   6       7.079  -3.068  -1.929  1.00  0.00           O  \\nATOM    169  C3'  DA A   6       8.883  -1.518  -2.181  1.00  0.00           C  \\nATOM    170  O3'  DA A   6       9.162  -1.103  -3.510  1.00  0.00           O  \\nATOM    171  C2'  DA A   6       7.638  -0.843  -1.609  1.00  0.00           C  \\nATOM    172  C1'  DA A   6       6.534  -1.778  -2.095  1.00  0.00           C  \\nATOM    173  N9   DA A   6       5.266  -1.649  -1.337  1.00  0.00           N  \\nATOM    174  C8   DA A   6       5.062  -1.659   0.023  1.00  0.00           C  \\nATOM    175  N7   DA A   6       3.811  -1.603   0.382  1.00  0.00           N  \\nATOM    176  C5   DA A   6       3.128  -1.551  -0.825  1.00  0.00           C  \\nATOM    177  C6   DA A   6       1.757  -1.487  -1.146  1.00  0.00           C  \\nATOM    178  N6   DA A   6       0.795  -1.473  -0.218  1.00  0.00           N  \\nATOM    179  N1   DA A   6       1.406  -1.437  -2.443  1.00  0.00           N  \\nATOM    180  C2   DA A   6       2.358  -1.447  -3.373  1.00  0.00           C  \\nATOM    181  N3   DA A   6       3.677  -1.511  -3.202  1.00  0.00           N  \\nATOM    182  C4   DA A   6       4.005  -1.564  -1.882  1.00  0.00           C  \\nATOM    183  H5'  DA A   6       8.939  -4.890  -1.301  1.00  0.00           H  \\nATOM    184 H5''  DA A   6      10.295  -3.750  -1.321  1.00  0.00           H  \\nATOM    185  H4'  DA A   6       8.666  -3.422  -3.183  1.00  0.00           H  \\nATOM    186  H3'  DA A   6       9.752  -1.349  -1.542  1.00  0.00           H  \\nATOM    187  H2'  DA A   6       7.729  -0.858  -0.523  1.00  0.00           H  \\nATOM    188 H2''  DA A   6       7.497   0.178  -1.959  1.00  0.00           H  \\nATOM    189  H1'  DA A   6       6.357  -1.604  -3.158  1.00  0.00           H  \\nATOM    190  H8   DA A   6       5.858  -1.702   0.751  1.00  0.00           H  \\nATOM    191  H61  DA A   6      -0.176  -1.429  -0.493  1.00  0.00           H  \\nATOM    192  H62  DA A   6       1.049  -1.512   0.759  1.00  0.00           H  \\nATOM    193  H2   DA A   6       2.014  -1.412  -4.395  1.00  0.00           H  \\nATOM    194  P    DT A   7       9.713   0.376  -3.872  1.00  0.00           P  \\nATOM    195  OP1  DT A   7      10.872   0.227  -4.781  1.00  0.00           O  \\nATOM    196  OP2  DT A   7       9.854   1.156  -2.622  1.00  0.00           O  \\nATOM    197  O5'  DT A   7       8.487   0.997  -4.715  1.00  0.00           O  \\nATOM    198  C5'  DT A   7       8.196   0.539  -6.023  1.00  0.00           C  \\nATOM    199  C4'  DT A   7       6.841   1.074  -6.496  1.00  0.00           C  \\nATOM    200  O4'  DT A   7       5.858   0.807  -5.506  1.00  0.00           O  \\nATOM    201  C3'  DT A   7       6.839   2.587  -6.766  1.00  0.00           C  \\nATOM    202  O3'  DT A   7       6.155   2.808  -7.992  1.00  0.00           O  \\nATOM    203  C2'  DT A   7       6.075   3.121  -5.556  1.00  0.00           C  \\nATOM    204  C1'  DT A   7       5.086   1.977  -5.335  1.00  0.00           C  \\nATOM    205  N1   DT A   7       4.411   1.977  -4.004  1.00  0.00           N  \\nATOM    206  C2   DT A   7       3.021   1.846  -3.966  1.00  0.00           C  \\nATOM    207  O2   DT A   7       2.314   1.799  -4.970  1.00  0.00           O  \\nATOM    208  N3   DT A   7       2.445   1.746  -2.708  1.00  0.00           N  \\nATOM    209  C4   DT A   7       3.120   1.734  -1.498  1.00  0.00           C  \\nATOM    210  O4   DT A   7       2.496   1.594  -0.449  1.00  0.00           O  \\nATOM    211  C5   DT A   7       4.558   1.885  -1.617  1.00  0.00           C  \\nATOM    212  C7   DT A   7       5.422   1.939  -0.371  1.00  0.00           C  \\nATOM    213  C6   DT A   7       5.142   1.992  -2.837  1.00  0.00           C  \\nATOM    214  H5'  DT A   7       8.155  -0.551  -6.034  1.00  0.00           H  \\nATOM    215 H5''  DT A   7       8.978   0.859  -6.713  1.00  0.00           H  \\nATOM    216  H4'  DT A   7       6.575   0.544  -7.412  1.00  0.00           H  \\nATOM    217  H3'  DT A   7       7.851   2.993  -6.814  1.00  0.00           H  \\nATOM    218  H2'  DT A   7       6.781   3.227  -4.736  1.00  0.00           H  \\nATOM    219 H2''  DT A   7       5.578   4.072  -5.733  1.00  0.00           H  \\nATOM    220  H1'  DT A   7       4.370   2.022  -6.154  1.00  0.00           H  \\nATOM    221  H3   DT A   7       1.442   1.618  -2.682  1.00  0.00           H  \\nATOM    222  H71  DT A   7       4.825   2.215   0.499  1.00  0.00           H  \\nATOM    223  H72  DT A   7       5.875   0.967  -0.192  1.00  0.00           H  \\nATOM    224  H73  DT A   7       6.214   2.678  -0.498  1.00  0.00           H  \\nATOM    225  H6   DT A   7       6.217   2.067  -2.881  1.00  0.00           H  \\nATOM    226  P    DG A   8       5.963   4.274  -8.646  1.00  0.00           P  \\nATOM    227  OP1  DG A   8       5.644   4.099 -10.080  1.00  0.00           O  \\nATOM    228  OP2  DG A   8       7.110   5.120  -8.246  1.00  0.00           O  \\nATOM    229  O5'  DG A   8       4.646   4.830  -7.903  1.00  0.00           O  \\nATOM    230  C5'  DG A   8       3.371   4.257  -8.123  1.00  0.00           C  \\nATOM    231  C4'  DG A   8       2.339   4.891  -7.185  1.00  0.00           C  \\nATOM    232  O4'  DG A   8       2.698   4.651  -5.832  1.00  0.00           O  \\nATOM    233  C3'  DG A   8       2.247   6.411  -7.372  1.00  0.00           C  \\nATOM    234  O3'  DG A   8       0.879   6.770  -7.293  1.00  0.00           O  \\nATOM    235  C2'  DG A   8       3.050   6.921  -6.174  1.00  0.00           C  \\nATOM    236  C1'  DG A   8       2.659   5.879  -5.130  1.00  0.00           C  \\nATOM    237  N9   DG A   8       3.547   5.807  -3.945  1.00  0.00           N  \\nATOM    238  C8   DG A   8       4.888   6.082  -3.843  1.00  0.00           C  \\nATOM    239  N7   DG A   8       5.390   5.876  -2.658  1.00  0.00           N  \\nATOM    240  C5   DG A   8       4.305   5.439  -1.907  1.00  0.00           C  \\nATOM    241  C6   DG A   8       4.229   5.086  -0.526  1.00  0.00           C  \\nATOM    242  O6   DG A   8       5.128   5.089   0.311  1.00  0.00           O  \\nATOM    243  N1   DG A   8       2.946   4.704  -0.143  1.00  0.00           N  \\nATOM    244  C2   DG A   8       1.856   4.677  -0.992  1.00  0.00           C  \\nATOM    245  N2   DG A   8       0.685   4.316  -0.461  1.00  0.00           N  \\nATOM    246  N3   DG A   8       1.926   5.007  -2.291  1.00  0.00           N  \\nATOM    247  C4   DG A   8       3.174   5.384  -2.687  1.00  0.00           C  \\nATOM    248  H5'  DG A   8       3.402   3.183  -7.939  1.00  0.00           H  \\nATOM    249 H5''  DG A   8       3.067   4.426  -9.157  1.00  0.00           H  \\nATOM    250  H4'  DG A   8       1.370   4.430  -7.385  1.00  0.00           H  \\nATOM    251  H3'  DG A   8       2.676   6.727  -8.324  1.00  0.00           H  \\nATOM    252  H2'  DG A   8       4.113   6.867  -6.415  1.00  0.00           H  \\nATOM    253 H2''  DG A   8       2.783   7.932  -5.866  1.00  0.00           H  \\nATOM    254  H1'  DG A   8       1.641   6.111  -4.807  1.00  0.00           H  \\nATOM    255  H8   DG A   8       5.491   6.434  -4.667  1.00  0.00           H  \\nATOM    256  H1   DG A   8       2.822   4.440   0.822  1.00  0.00           H  \\nATOM    257  H21  DG A   8       0.614   4.079   0.517  1.00  0.00           H  \\nATOM    258  H22  DG A   8      -0.150   4.335  -1.037  1.00  0.00           H  \\nATOM    259  P    DG A   9       0.330   8.218  -7.744  1.00  0.00           P  \\nATOM    260  OP1  DG A   9      -0.439   8.061  -8.998  1.00  0.00           O  \\nATOM    261  OP2  DG A   9       1.438   9.197  -7.672  1.00  0.00           O  \\nATOM    262  O5'  DG A   9      -0.690   8.535  -6.543  1.00  0.00           O  \\nATOM    263  C5'  DG A   9      -2.012   8.031  -6.519  1.00  0.00           C  \\nATOM    264  C4'  DG A   9      -2.566   8.145  -5.094  1.00  0.00           C  \\nATOM    265  O4'  DG A   9      -1.825   7.353  -4.184  1.00  0.00           O  \\nATOM    266  C3'  DG A   9      -2.506   9.580  -4.566  1.00  0.00           C  \\nATOM    267  O3'  DG A   9      -3.619  10.313  -5.044  1.00  0.00           O  \\nATOM    268  C2'  DG A   9      -2.497   9.337  -3.063  1.00  0.00           C  \\nATOM    269  C1'  DG A   9      -1.771   7.994  -2.919  1.00  0.00           C  \\nATOM    270  N9   DG A   9      -0.338   8.108  -2.538  1.00  0.00           N  \\nATOM    271  C8   DG A   9       0.746   8.359  -3.343  1.00  0.00           C  \\nATOM    272  N7   DG A   9       1.893   8.366  -2.727  1.00  0.00           N  \\nATOM    273  C5   DG A   9       1.561   8.070  -1.412  1.00  0.00           C  \\nATOM    274  C6   DG A   9       2.401   7.935  -0.265  1.00  0.00           C  \\nATOM    275  O6   DG A   9       3.614   8.107  -0.190  1.00  0.00           O  \\nATOM    276  N1   DG A   9       1.693   7.555   0.872  1.00  0.00           N  \\nATOM    277  C2   DG A   9       0.328   7.335   0.900  1.00  0.00           C  \\nATOM    278  N2   DG A   9      -0.210   6.988   2.072  1.00  0.00           N  \\nATOM    279  N3   DG A   9      -0.460   7.469  -0.177  1.00  0.00           N  \\nATOM    280  C4   DG A   9       0.207   7.851  -1.298  1.00  0.00           C  \\nATOM    281  H5'  DG A   9      -2.045   6.989  -6.838  1.00  0.00           H  \\nATOM    282 H5''  DG A   9      -2.628   8.623  -7.197  1.00  0.00           H  \\nATOM    283  H4'  DG A   9      -3.595   7.780  -5.082  1.00  0.00           H  \\nATOM    284  H3'  DG A   9      -1.573  10.062  -4.854  1.00  0.00           H  \\nATOM    285  H2'  DG A   9      -2.015  10.137  -2.518  1.00  0.00           H  \\nATOM    286 H2''  DG A   9      -3.515   9.237  -2.692  1.00  0.00           H  \\nATOM    287  H1'  DG A   9      -2.311   7.429  -2.161  1.00  0.00           H  \\nATOM    288  H8   DG A   9       0.678   8.518  -4.403  1.00  0.00           H  \\nATOM    289  H1   DG A   9       2.224   7.458   1.723  1.00  0.00           H  \\nATOM    290  H21  DG A   9       0.352   6.926   2.907  1.00  0.00           H  \\nATOM    291  H22  DG A   9      -1.215   6.851   2.123  1.00  0.00           H  \\nATOM    292  P    DA A  10      -3.768  11.908  -4.844  1.00  0.00           P  \\nATOM    293  OP1  DA A  10      -4.648  12.420  -5.917  1.00  0.00           O  \\nATOM    294  OP2  DA A  10      -2.422  12.498  -4.662  1.00  0.00           O  \\nATOM    295  O5'  DA A  10      -4.564  12.045  -3.452  1.00  0.00           O  \\nATOM    296  C5'  DA A  10      -5.795  11.380  -3.232  1.00  0.00           C  \\nATOM    297  C4'  DA A  10      -6.574  12.044  -2.090  1.00  0.00           C  \\nATOM    298  O4'  DA A  10      -5.681  12.349  -1.028  1.00  0.00           O  \\nATOM    299  C3'  DA A  10      -7.245  13.358  -2.515  1.00  0.00           C  \\nATOM    300  O3'  DA A  10      -8.433  13.513  -1.751  1.00  0.00           O  \\nATOM    301  C2'  DA A  10      -6.168  14.372  -2.152  1.00  0.00           C  \\nATOM    302  C1'  DA A  10      -5.636  13.757  -0.862  1.00  0.00           C  \\nATOM    303  N9   DA A  10      -4.269  14.207  -0.519  1.00  0.00           N  \\nATOM    304  C8   DA A  10      -3.167  14.362  -1.327  1.00  0.00           C  \\nATOM    305  N7   DA A  10      -2.080  14.697  -0.690  1.00  0.00           N  \\nATOM    306  C5   DA A  10      -2.485  14.770   0.637  1.00  0.00           C  \\nATOM    307  C6   DA A  10      -1.807  15.061   1.839  1.00  0.00           C  \\nATOM    308  N6   DA A  10      -0.504  15.345   1.885  1.00  0.00           N  \\nATOM    309  N1   DA A  10      -2.508  15.070   2.989  1.00  0.00           N  \\nATOM    310  C2   DA A  10      -3.806  14.776   2.951  1.00  0.00           C  \\nATOM    311  N3   DA A  10      -4.552  14.464   1.896  1.00  0.00           N  \\nATOM    312  C4   DA A  10      -3.822  14.485   0.749  1.00  0.00           C  \\nATOM    313  H5'  DA A  10      -5.589  10.343  -2.966  1.00  0.00           H  \\nATOM    314 H5''  DA A  10      -6.412  11.388  -4.132  1.00  0.00           H  \\nATOM    315  H4'  DA A  10      -7.333  11.339  -1.748  1.00  0.00           H  \\nATOM    316  H3'  DA A  10      -7.475  13.385  -3.581  1.00  0.00           H  \\nATOM    317  H2'  DA A  10      -5.414  14.386  -2.937  1.00  0.00           H  \\nATOM    318 H2''  DA A  10      -6.554  15.376  -2.003  1.00  0.00           H  \\nATOM    319  H1'  DA A  10      -6.323  14.024  -0.061  1.00  0.00           H  \\nATOM    320  H8   DA A  10      -3.184  14.211  -2.396  1.00  0.00           H  \\nATOM    321  H61  DA A  10      -0.075  15.609   2.761  1.00  0.00           H  \\nATOM    322  H62  DA A  10       0.069  15.235   1.057  1.00  0.00           H  \\nATOM    323  H2   DA A  10      -4.325  14.780   3.896  1.00  0.00           H  \\nATOM    324  P    DA A  11      -9.394  14.810  -1.869  1.00  0.00           P  \\nATOM    325  OP1  DA A  11     -10.740  14.433  -1.386  1.00  0.00           O  \\nATOM    326  OP2  DA A  11      -9.234  15.394  -3.219  1.00  0.00           O  \\nATOM    327  O5'  DA A  11      -8.742  15.826  -0.800  1.00  0.00           O  \\nATOM    328  C5'  DA A  11      -8.827  15.586   0.592  1.00  0.00           C  \\nATOM    329  C4'  DA A  11      -7.934  16.582   1.339  1.00  0.00           C  \\nATOM    330  O4'  DA A  11      -6.600  16.494   0.866  1.00  0.00           O  \\nATOM    331  C3'  DA A  11      -8.405  18.029   1.139  1.00  0.00           C  \\nATOM    332  O3'  DA A  11      -8.526  18.646   2.413  1.00  0.00           O  \\nATOM    333  C2'  DA A  11      -7.288  18.627   0.284  1.00  0.00           C  \\nATOM    334  C1'  DA A  11      -6.088  17.802   0.739  1.00  0.00           C  \\nATOM    335  N9   DA A  11      -4.945  17.818  -0.203  1.00  0.00           N  \\nATOM    336  C8   DA A  11      -4.941  17.661  -1.568  1.00  0.00           C  \\nATOM    337  N7   DA A  11      -3.754  17.670  -2.108  1.00  0.00           N  \\nATOM    338  C5   DA A  11      -2.903  17.845  -1.024  1.00  0.00           C  \\nATOM    339  C6   DA A  11      -1.500  17.949  -0.911  1.00  0.00           C  \\nATOM    340  N6   DA A  11      -0.681  17.883  -1.966  1.00  0.00           N  \\nATOM    341  N1   DA A  11      -0.968  18.143   0.310  1.00  0.00           N  \\nATOM    342  C2   DA A  11      -1.779  18.218   1.364  1.00  0.00           C  \\nATOM    343  N3   DA A  11      -3.106  18.128   1.391  1.00  0.00           N  \\nATOM    344  C4   DA A  11      -3.618  17.939   0.144  1.00  0.00           C  \\nATOM    345  H5'  DA A  11      -8.503  14.572   0.827  1.00  0.00           H  \\nATOM    346 H5''  DA A  11      -9.859  15.703   0.925  1.00  0.00           H  \\nATOM    347  H4'  DA A  11      -7.947  16.329   2.400  1.00  0.00           H  \\nATOM    348  H3'  DA A  11      -9.358  18.055   0.610  1.00  0.00           H  \\nATOM    349  H2'  DA A  11      -7.517  18.432  -0.763  1.00  0.00           H  \\nATOM    350 H2''  DA A  11      -7.149  19.695   0.441  1.00  0.00           H  \\nATOM    351  H1'  DA A  11      -5.765  18.170   1.712  1.00  0.00           H  \\nATOM    352  H8   DA A  11      -5.838  17.538  -2.157  1.00  0.00           H  \\nATOM    353  H61  DA A  11       0.317  17.992  -1.843  1.00  0.00           H  \\nATOM    354  H62  DA A  11      -1.069  17.748  -2.888  1.00  0.00           H  \\nATOM    355  H2   DA A  11      -1.298  18.375   2.317  1.00  0.00           H  \\nATOM    356  P    DC A  12      -9.152  20.124   2.613  1.00  0.00           P  \\nATOM    357  OP1  DC A  12     -10.116  20.076   3.734  1.00  0.00           O  \\nATOM    358  OP2  DC A  12      -9.578  20.645   1.295  1.00  0.00           O  \\nATOM    359  O5'  DC A  12      -7.866  20.968   3.088  1.00  0.00           O  \\nATOM    360  C5'  DC A  12      -7.409  20.939   4.430  1.00  0.00           C  \\nATOM    361  C4'  DC A  12      -5.979  21.486   4.471  1.00  0.00           C  \\nATOM    362  O4'  DC A  12      -5.178  20.822   3.513  1.00  0.00           O  \\nATOM    363  C3'  DC A  12      -5.913  22.975   4.122  1.00  0.00           C  \\nATOM    364  O3'  DC A  12      -5.844  23.770   5.287  1.00  0.00           O  \\nATOM    365  C2'  DC A  12      -4.638  23.102   3.292  1.00  0.00           C  \\nATOM    366  C1'  DC A  12      -4.097  21.674   3.191  1.00  0.00           C  \\nATOM    367  N1   DC A  12      -3.608  21.408   1.811  1.00  0.00           N  \\nATOM    368  C2   DC A  12      -2.235  21.410   1.557  1.00  0.00           C  \\nATOM    369  O2   DC A  12      -1.416  21.525   2.466  1.00  0.00           O  \\nATOM    370  N3   DC A  12      -1.814  21.268   0.267  1.00  0.00           N  \\nATOM    371  C4   DC A  12      -2.689  21.136  -0.741  1.00  0.00           C  \\nATOM    372  N4   DC A  12      -2.217  20.980  -1.981  1.00  0.00           N  \\nATOM    373  C5   DC A  12      -4.105  21.125  -0.502  1.00  0.00           C  \\nATOM    374  C6   DC A  12      -4.508  21.262   0.783  1.00  0.00           C  \\nATOM    375  H5'  DC A  12      -7.415  19.923   4.828  1.00  0.00           H  \\nATOM    376 H5''  DC A  12      -8.057  21.555   5.054  1.00  0.00           H  \\nATOM    377  H4'  DC A  12      -5.546  21.325   5.460  1.00  0.00           H  \\nATOM    378  H3'  DC A  12      -6.768  23.275   3.514  1.00  0.00           H  \\nATOM    379 HO3'  DC A  12      -6.657  23.652   5.783  1.00  0.00           H  \\nATOM    380  H2'  DC A  12      -4.895  23.497   2.309  1.00  0.00           H  \\nATOM    381 H2''  DC A  12      -3.902  23.753   3.761  1.00  0.00           H  \\nATOM    382  H1'  DC A  12      -3.331  21.535   3.953  1.00  0.00           H  \\nATOM    383  H41  DC A  12      -1.216  20.953  -2.129  1.00  0.00           H  \\nATOM    384  H42  DC A  12      -2.850  20.847  -2.756  1.00  0.00           H  \\nATOM    385  H5   DC A  12      -4.842  21.010  -1.284  1.00  0.00           H  \\nATOM    386  H6   DC A  12      -5.562  21.282   1.013  1.00  0.00           H  \\nTER     387       DC A  12                                                      \\nATOM    388  O5'  DG B  13       8.692  22.125  -1.628  1.00  0.00           O  \\nATOM    389  C5'  DG B  13       8.994  22.802  -0.426  1.00  0.00           C  \\nATOM    390  C4'  DG B  13       8.201  22.220   0.750  1.00  0.00           C  \\nATOM    391  O4'  DG B  13       6.811  22.374   0.501  1.00  0.00           O  \\nATOM    392  C3'  DG B  13       8.479  20.725   0.965  1.00  0.00           C  \\nATOM    393  O3'  DG B  13       8.592  20.475   2.357  1.00  0.00           O  \\nATOM    394  C2'  DG B  13       7.237  20.084   0.356  1.00  0.00           C  \\nATOM    395  C1'  DG B  13       6.171  21.124   0.683  1.00  0.00           C  \\nATOM    396  N9   DG B  13       4.994  21.017  -0.212  1.00  0.00           N  \\nATOM    397  C8   DG B  13       4.951  21.135  -1.579  1.00  0.00           C  \\nATOM    398  N7   DG B  13       3.756  21.031  -2.089  1.00  0.00           N  \\nATOM    399  C5   DG B  13       2.936  20.853  -0.981  1.00  0.00           C  \\nATOM    400  C6   DG B  13       1.518  20.705  -0.906  1.00  0.00           C  \\nATOM    401  O6   DG B  13       0.711  20.664  -1.832  1.00  0.00           O  \\nATOM    402  N1   DG B  13       1.063  20.603   0.405  1.00  0.00           N  \\nATOM    403  C2   DG B  13       1.884  20.606   1.519  1.00  0.00           C  \\nATOM    404  N2   DG B  13       1.294  20.495   2.713  1.00  0.00           N  \\nATOM    405  N3   DG B  13       3.220  20.717   1.448  1.00  0.00           N  \\nATOM    406  C4   DG B  13       3.684  20.847   0.175  1.00  0.00           C  \\nATOM    407  H5'  DG B  13       8.746  23.858  -0.542  1.00  0.00           H  \\nATOM    408 H5''  DG B  13      10.062  22.711  -0.224  1.00  0.00           H  \\nATOM    409  H4'  DG B  13       8.472  22.781   1.647  1.00  0.00           H  \\nATOM    410  H3'  DG B  13       9.390  20.408   0.453  1.00  0.00           H  \\nATOM    411  H2'  DG B  13       7.377  20.000  -0.722  1.00  0.00           H  \\nATOM    412 H2''  DG B  13       7.005  19.107   0.775  1.00  0.00           H  \\nATOM    413  H1'  DG B  13       5.878  21.020   1.728  1.00  0.00           H  \\nATOM    414  H8   DG B  13       5.829  21.295  -2.187  1.00  0.00           H  \\nATOM    415  H1   DG B  13       0.061  20.531   0.528  1.00  0.00           H  \\nATOM    416  H21  DG B  13       0.285  20.475   2.782  1.00  0.00           H  \\nATOM    417  H22  DG B  13       1.862  20.474   3.547  1.00  0.00           H  \\nATOM    418 HO5'  DG B  13       9.205  22.519  -2.337  1.00  0.00           H  \\nATOM    419  P    DT B  14       8.934  19.011   2.956  1.00  0.00           P  \\nATOM    420  OP1  DT B  14       9.823  19.191   4.125  1.00  0.00           O  \\nATOM    421  OP2  DT B  14       9.354  18.131   1.842  1.00  0.00           O  \\nATOM    422  O5'  DT B  14       7.499  18.499   3.486  1.00  0.00           O  \\nATOM    423  C5'  DT B  14       6.884  19.096   4.612  1.00  0.00           C  \\nATOM    424  C4'  DT B  14       5.519  18.461   4.891  1.00  0.00           C  \\nATOM    425  O4'  DT B  14       4.686  18.604   3.750  1.00  0.00           O  \\nATOM    426  C3'  DT B  14       5.603  16.962   5.226  1.00  0.00           C  \\nATOM    427  O3'  DT B  14       4.750  16.719   6.336  1.00  0.00           O  \\nATOM    428  C2'  DT B  14       5.097  16.325   3.932  1.00  0.00           C  \\nATOM    429  C1'  DT B  14       4.057  17.362   3.513  1.00  0.00           C  \\nATOM    430  N1   DT B  14       3.606  17.278   2.096  1.00  0.00           N  \\nATOM    431  C2   DT B  14       2.237  17.366   1.835  1.00  0.00           C  \\nATOM    432  O2   DT B  14       1.379  17.386   2.714  1.00  0.00           O  \\nATOM    433  N3   DT B  14       1.869  17.464   0.502  1.00  0.00           N  \\nATOM    434  C4   DT B  14       2.730  17.523  -0.582  1.00  0.00           C  \\nATOM    435  O4   DT B  14       2.281  17.673  -1.715  1.00  0.00           O  \\nATOM    436  C5   DT B  14       4.133  17.400  -0.235  1.00  0.00           C  \\nATOM    437  C7   DT B  14       5.185  17.396  -1.328  1.00  0.00           C  \\nATOM    438  C6   DT B  14       4.516  17.289   1.062  1.00  0.00           C  \\nATOM    439  H5'  DT B  14       6.743  20.162   4.431  1.00  0.00           H  \\nATOM    440 H5''  DT B  14       7.517  18.970   5.492  1.00  0.00           H  \\nATOM    441  H4'  DT B  14       5.064  18.997   5.726  1.00  0.00           H  \\nATOM    442  H3'  DT B  14       6.624  16.653   5.456  1.00  0.00           H  \\nATOM    443  H2'  DT B  14       5.932  16.270   3.237  1.00  0.00           H  \\nATOM    444 H2''  DT B  14       4.669  15.334   4.062  1.00  0.00           H  \\nATOM    445  H1'  DT B  14       3.224  17.267   4.210  1.00  0.00           H  \\nATOM    446  H3   DT B  14       0.880  17.558   0.315  1.00  0.00           H  \\nATOM    447  H71  DT B  14       4.756  17.043  -2.266  1.00  0.00           H  \\nATOM    448  H72  DT B  14       5.565  18.405  -1.473  1.00  0.00           H  \\nATOM    449  H73  DT B  14       6.011  16.740  -1.054  1.00  0.00           H  \\nATOM    450  H6   DT B  14       5.569  17.235   1.281  1.00  0.00           H  \\nATOM    451  P    DG B  15       4.549  15.254   6.990  1.00  0.00           P  \\nATOM    452  OP1  DG B  15       4.020  15.432   8.361  1.00  0.00           O  \\nATOM    453  OP2  DG B  15       5.783  14.468   6.774  1.00  0.00           O  \\nATOM    454  O5'  DG B  15       3.381  14.619   6.080  1.00  0.00           O  \\nATOM    455  C5'  DG B  15       2.063  15.133   6.103  1.00  0.00           C  \\nATOM    456  C4'  DG B  15       1.210  14.441   5.036  1.00  0.00           C  \\nATOM    457  O4'  DG B  15       1.766  14.658   3.749  1.00  0.00           O  \\nATOM    458  C3'  DG B  15       1.137  12.925   5.254  1.00  0.00           C  \\nATOM    459  O3'  DG B  15      -0.200  12.527   5.011  1.00  0.00           O  \\nATOM    460  C2'  DG B  15       2.104  12.403   4.190  1.00  0.00           C  \\nATOM    461  C1'  DG B  15       1.853  13.416   3.078  1.00  0.00           C  \\nATOM    462  N9   DG B  15       2.902  13.491   2.034  1.00  0.00           N  \\nATOM    463  C8   DG B  15       4.250  13.246   2.131  1.00  0.00           C  \\nATOM    464  N7   DG B  15       4.913  13.452   1.028  1.00  0.00           N  \\nATOM    465  C5   DG B  15       3.939  13.858   0.124  1.00  0.00           C  \\nATOM    466  C6   DG B  15       4.056  14.199  -1.258  1.00  0.00           C  \\nATOM    467  O6   DG B  15       5.066  14.207  -1.957  1.00  0.00           O  \\nATOM    468  N1   DG B  15       2.835  14.554  -1.824  1.00  0.00           N  \\nATOM    469  C2   DG B  15       1.633  14.569  -1.140  1.00  0.00           C  \\nATOM    470  N2   DG B  15       0.548  14.927  -1.832  1.00  0.00           N  \\nATOM    471  N3   DG B  15       1.521  14.247   0.158  1.00  0.00           N  \\nATOM    472  C4   DG B  15       2.706  13.896   0.732  1.00  0.00           C  \\nATOM    473  H5'  DG B  15       2.071  16.205   5.906  1.00  0.00           H  \\nATOM    474 H5''  DG B  15       1.619  14.963   7.085  1.00  0.00           H  \\nATOM    475  H4'  DG B  15       0.208  14.873   5.072  1.00  0.00           H  \\nATOM    476  H3'  DG B  15       1.450  12.654   6.263  1.00  0.00           H  \\nATOM    477  H2'  DG B  15       3.120  12.479   4.576  1.00  0.00           H  \\nATOM    478 H2''  DG B  15       1.898  11.382   3.872  1.00  0.00           H  \\nATOM    479  H1'  DG B  15       0.900  13.151   2.615  1.00  0.00           H  \\nATOM    480  H8   DG B  15       4.735  12.918   3.038  1.00  0.00           H  \\nATOM    481  H1   DG B  15       2.846  14.814  -2.798  1.00  0.00           H  \\nATOM    482  H21  DG B  15       0.616  15.164  -2.810  1.00  0.00           H  \\nATOM    483  H22  DG B  15      -0.361  14.900  -1.383  1.00  0.00           H  \\nATOM    484  P    DG B  16      -0.764  11.068   5.404  1.00  0.00           P  \\nATOM    485  OP1  DG B  16      -1.722  11.216   6.523  1.00  0.00           O  \\nATOM    486  OP2  DG B  16       0.371  10.125   5.520  1.00  0.00           O  \\nATOM    487  O5'  DG B  16      -1.578  10.705   4.067  1.00  0.00           O  \\nATOM    488  C5'  DG B  16      -2.876  11.208   3.813  1.00  0.00           C  \\nATOM    489  C4'  DG B  16      -3.169  11.119   2.311  1.00  0.00           C  \\nATOM    490  O4'  DG B  16      -2.280  11.917   1.551  1.00  0.00           O  \\nATOM    491  C3'  DG B  16      -3.020   9.694   1.781  1.00  0.00           C  \\nATOM    492  O3'  DG B  16      -4.198   8.962   2.067  1.00  0.00           O  \\nATOM    493  C2'  DG B  16      -2.782   9.959   0.301  1.00  0.00           C  \\nATOM    494  C1'  DG B  16      -2.023  11.290   0.303  1.00  0.00           C  \\nATOM    495  N9   DG B  16      -0.551  11.139   0.173  1.00  0.00           N  \\nATOM    496  C8   DG B  16       0.368  10.841   1.147  1.00  0.00           C  \\nATOM    497  N7   DG B  16       1.604  10.808   0.735  1.00  0.00           N  \\nATOM    498  C5   DG B  16       1.510  11.123  -0.614  1.00  0.00           C  \\nATOM    499  C6   DG B  16       2.534  11.240  -1.603  1.00  0.00           C  \\nATOM    500  O6   DG B  16       3.740  11.050  -1.471  1.00  0.00           O  \\nATOM    501  N1   DG B  16       2.035  11.623  -2.845  1.00  0.00           N  \\nATOM    502  C2   DG B  16       0.701  11.877  -3.102  1.00  0.00           C  \\nATOM    503  N2   DG B  16       0.368  12.229  -4.348  1.00  0.00           N  \\nATOM    504  N3   DG B  16      -0.258  11.773  -2.170  1.00  0.00           N  \\nATOM    505  C4   DG B  16       0.202  11.377  -0.955  1.00  0.00           C  \\nATOM    506  H5'  DG B  16      -2.970  12.244   4.140  1.00  0.00           H  \\nATOM    507 H5''  DG B  16      -3.600  10.605   4.362  1.00  0.00           H  \\nATOM    508  H4'  DG B  16      -4.180  11.488   2.125  1.00  0.00           H  \\nATOM    509  H3'  DG B  16      -2.132   9.221   2.201  1.00  0.00           H  \\nATOM    510  H2'  DG B  16      -2.231   9.161  -0.180  1.00  0.00           H  \\nATOM    511 H2''  DG B  16      -3.729  10.087  -0.218  1.00  0.00           H  \\nATOM    512  H1'  DG B  16      -2.411  11.875  -0.530  1.00  0.00           H  \\nATOM    513  H8   DG B  16       0.105  10.668   2.176  1.00  0.00           H  \\nATOM    514  H1   DG B  16       2.705  11.704  -3.595  1.00  0.00           H  \\nATOM    515  H21  DG B  16       1.060  12.268  -5.081  1.00  0.00           H  \\nATOM    516  H22  DG B  16      -0.613  12.385  -4.562  1.00  0.00           H  \\nATOM    517  P    DA B  17      -4.335   7.377   1.798  1.00  0.00           P  \\nATOM    518  OP1  DA B  17      -5.461   6.874   2.615  1.00  0.00           O  \\nATOM    519  OP2  DA B  17      -3.000   6.754   1.940  1.00  0.00           O  \\nATOM    520  O5'  DA B  17      -4.768   7.278   0.251  1.00  0.00           O  \\nATOM    521  C5'  DA B  17      -5.940   7.904  -0.242  1.00  0.00           C  \\nATOM    522  C4'  DA B  17      -6.463   7.161  -1.478  1.00  0.00           C  \\nATOM    523  O4'  DA B  17      -5.368   6.847  -2.328  1.00  0.00           O  \\nATOM    524  C3'  DA B  17      -7.159   5.840  -1.116  1.00  0.00           C  \\nATOM    525  O3'  DA B  17      -8.183   5.587  -2.069  1.00  0.00           O  \\nATOM    526  C2'  DA B  17      -6.000   4.859  -1.228  1.00  0.00           C  \\nATOM    527  C1'  DA B  17      -5.273   5.437  -2.438  1.00  0.00           C  \\nATOM    528  N9   DA B  17      -3.863   4.998  -2.544  1.00  0.00           N  \\nATOM    529  C8   DA B  17      -2.903   4.877  -1.566  1.00  0.00           C  \\nATOM    530  N7   DA B  17      -1.726   4.537  -2.013  1.00  0.00           N  \\nATOM    531  C5   DA B  17      -1.913   4.421  -3.384  1.00  0.00           C  \\nATOM    532  C6   DA B  17      -1.051   4.100  -4.455  1.00  0.00           C  \\nATOM    533  N6   DA B  17       0.243   3.819  -4.289  1.00  0.00           N  \\nATOM    534  N1   DA B  17      -1.560   4.059  -5.701  1.00  0.00           N  \\nATOM    535  C2   DA B  17      -2.848   4.344  -5.879  1.00  0.00           C  \\nATOM    536  N3   DA B  17      -3.755   4.674  -4.965  1.00  0.00           N  \\nATOM    537  C4   DA B  17      -3.217   4.689  -3.716  1.00  0.00           C  \\nATOM    538  H5'  DA B  17      -5.702   8.931  -0.517  1.00  0.00           H  \\nATOM    539 H5''  DA B  17      -6.727   7.926   0.514  1.00  0.00           H  \\nATOM    540  H4'  DA B  17      -7.165   7.816  -1.997  1.00  0.00           H  \\nATOM    541  H3'  DA B  17      -7.579   5.858  -0.109  1.00  0.00           H  \\nATOM    542  H2'  DA B  17      -5.394   4.922  -0.325  1.00  0.00           H  \\nATOM    543 H2''  DA B  17      -6.320   3.833  -1.385  1.00  0.00           H  \\nATOM    544  H1'  DA B  17      -5.820   5.137  -3.330  1.00  0.00           H  \\nATOM    545  H8   DA B  17      -3.088   5.063  -0.519  1.00  0.00           H  \\nATOM    546  H61  DA B  17       0.806   3.543  -5.082  1.00  0.00           H  \\nATOM    547  H62  DA B  17       0.680   3.957  -3.385  1.00  0.00           H  \\nATOM    548  H2   DA B  17      -3.208   4.315  -6.895  1.00  0.00           H  \\nATOM    549  P    DA B  18      -9.096   4.248  -2.047  1.00  0.00           P  \\nATOM    550  OP1  DA B  18     -10.379   4.557  -2.718  1.00  0.00           O  \\nATOM    551  OP2  DA B  18      -9.100   3.703  -0.672  1.00  0.00           O  \\nATOM    552  O5'  DA B  18      -8.271   3.230  -2.990  1.00  0.00           O  \\nATOM    553  C5'  DA B  18      -8.181   3.435  -4.388  1.00  0.00           C  \\nATOM    554  C4'  DA B  18      -7.174   2.461  -5.006  1.00  0.00           C  \\nATOM    555  O4'  DA B  18      -5.925   2.601  -4.346  1.00  0.00           O  \\nATOM    556  C3'  DA B  18      -7.604   0.989  -4.895  1.00  0.00           C  \\nATOM    557  O3'  DA B  18      -7.484   0.375  -6.170  1.00  0.00           O  \\nATOM    558  C2'  DA B  18      -6.601   0.434  -3.886  1.00  0.00           C  \\nATOM    559  C1'  DA B  18      -5.382   1.315  -4.152  1.00  0.00           C  \\nATOM    560  N9   DA B  18      -4.402   1.326  -3.039  1.00  0.00           N  \\nATOM    561  C8   DA B  18      -4.614   1.542  -1.697  1.00  0.00           C  \\nATOM    562  N7   DA B  18      -3.529   1.560  -0.976  1.00  0.00           N  \\nATOM    563  C5   DA B  18      -2.519   1.342  -1.902  1.00  0.00           C  \\nATOM    564  C6   DA B  18      -1.117   1.255  -1.784  1.00  0.00           C  \\nATOM    565  N6   DA B  18      -0.476   1.392  -0.618  1.00  0.00           N  \\nATOM    566  N1   DA B  18      -0.395   1.024  -2.896  1.00  0.00           N  \\nATOM    567  C2   DA B  18      -1.026   0.881  -4.059  1.00  0.00           C  \\nATOM    568  N3   DA B  18      -2.333   0.946  -4.302  1.00  0.00           N  \\nATOM    569  C4   DA B  18      -3.039   1.185  -3.163  1.00  0.00           C  \\nATOM    570  H5'  DA B  18      -7.845   4.453  -4.590  1.00  0.00           H  \\nATOM    571 H5''  DA B  18      -9.158   3.290  -4.850  1.00  0.00           H  \\nATOM    572  H4'  DA B  18      -7.051   2.726  -6.058  1.00  0.00           H  \\nATOM    573  H3'  DA B  18      -8.627   0.896  -4.526  1.00  0.00           H  \\nATOM    574  H2'  DA B  18      -7.010   0.605  -2.891  1.00  0.00           H  \\nATOM    575 H2''  DA B  18      -6.382  -0.623  -4.025  1.00  0.00           H  \\nATOM    576  H1'  DA B  18      -4.899   0.992  -5.075  1.00  0.00           H  \\nATOM    577  H8   DA B  18      -5.589   1.678  -1.256  1.00  0.00           H  \\nATOM    578  H61  DA B  18       0.532   1.327  -0.581  1.00  0.00           H  \\nATOM    579  H62  DA B  18      -1.009   1.569   0.221  1.00  0.00           H  \\nATOM    580  H2   DA B  18      -0.392   0.704  -4.915  1.00  0.00           H  \\nATOM    581  P    DT B  19      -7.936  -1.153  -6.457  1.00  0.00           P  \\nATOM    582  OP1  DT B  19      -8.773  -1.166  -7.678  1.00  0.00           O  \\nATOM    583  OP2  DT B  19      -8.456  -1.744  -5.203  1.00  0.00           O  \\nATOM    584  O5'  DT B  19      -6.529  -1.863  -6.800  1.00  0.00           O  \\nATOM    585  C5'  DT B  19      -5.855  -1.593  -8.016  1.00  0.00           C  \\nATOM    586  C4'  DT B  19      -4.433  -2.160  -7.984  1.00  0.00           C  \\nATOM    587  O4'  DT B  19      -3.781  -1.731  -6.798  1.00  0.00           O  \\nATOM    588  C3'  DT B  19      -4.383  -3.697  -8.021  1.00  0.00           C  \\nATOM    589  O3'  DT B  19      -3.372  -4.076  -8.945  1.00  0.00           O  \\nATOM    590  C2'  DT B  19      -4.022  -4.033  -6.575  1.00  0.00           C  \\nATOM    591  C1'  DT B  19      -3.120  -2.847  -6.238  1.00  0.00           C  \\nATOM    592  N1   DT B  19      -2.873  -2.640  -4.781  1.00  0.00           N  \\nATOM    593  C2   DT B  19      -1.555  -2.477  -4.349  1.00  0.00           C  \\nATOM    594  O2   DT B  19      -0.580  -2.560  -5.093  1.00  0.00           O  \\nATOM    595  N3   DT B  19      -1.378  -2.186  -3.004  1.00  0.00           N  \\nATOM    596  C4   DT B  19      -2.383  -2.014  -2.066  1.00  0.00           C  \\nATOM    597  O4   DT B  19      -2.096  -1.712  -0.910  1.00  0.00           O  \\nATOM    598  C5   DT B  19      -3.723  -2.209  -2.586  1.00  0.00           C  \\nATOM    599  C7   DT B  19      -4.919  -2.099  -1.660  1.00  0.00           C  \\nATOM    600  C6   DT B  19      -3.919  -2.501  -3.897  1.00  0.00           C  \\nATOM    601  H5'  DT B  19      -5.789  -0.516  -8.171  1.00  0.00           H  \\nATOM    602 H5''  DT B  19      -6.406  -2.027  -8.852  1.00  0.00           H  \\nATOM    603  H4'  DT B  19      -3.896  -1.762  -8.847  1.00  0.00           H  \\nATOM    604  H3'  DT B  19      -5.344  -4.128  -8.307  1.00  0.00           H  \\nATOM    605  H2'  DT B  19      -4.942  -4.038  -5.994  1.00  0.00           H  \\nATOM    606 H2''  DT B  19      -3.515  -4.987  -6.458  1.00  0.00           H  \\nATOM    607  H1'  DT B  19      -2.194  -2.994  -6.792  1.00  0.00           H  \\nATOM    608  H3   DT B  19      -0.426  -2.036  -2.699  1.00  0.00           H  \\nATOM    609  H71  DT B  19      -5.373  -1.115  -1.755  1.00  0.00           H  \\nATOM    610  H72  DT B  19      -5.662  -2.855  -1.918  1.00  0.00           H  \\nATOM    611  H73  DT B  19      -4.617  -2.249  -0.623  1.00  0.00           H  \\nATOM    612  H6   DT B  19      -4.933  -2.601  -4.249  1.00  0.00           H  \\nATOM    613  P    DG B  20      -3.026  -5.616  -9.294  1.00  0.00           P  \\nATOM    614  OP1  DG B  20      -2.294  -5.641 -10.580  1.00  0.00           O  \\nATOM    615  OP2  DG B  20      -4.256  -6.422  -9.133  1.00  0.00           O  \\nATOM    616  O5'  DG B  20      -1.998  -6.029  -8.124  1.00  0.00           O  \\nATOM    617  C5'  DG B  20      -0.704  -5.463  -8.038  1.00  0.00           C  \\nATOM    618  C4'  DG B  20      -0.007  -5.932  -6.756  1.00  0.00           C  \\nATOM    619  O4'  DG B  20      -0.749  -5.516  -5.619  1.00  0.00           O  \\nATOM    620  C3'  DG B  20       0.111  -7.460  -6.691  1.00  0.00           C  \\nATOM    621  O3'  DG B  20       1.386  -7.767  -6.152  1.00  0.00           O  \\nATOM    622  C2'  DG B  20      -1.024  -7.820  -5.730  1.00  0.00           C  \\nATOM    623  C1'  DG B  20      -0.947  -6.633  -4.774  1.00  0.00           C  \\nATOM    624  N9   DG B  20      -2.149  -6.417  -3.934  1.00  0.00           N  \\nATOM    625  C8   DG B  20      -3.464  -6.707  -4.203  1.00  0.00           C  \\nATOM    626  N7   DG B  20      -4.295  -6.345  -3.267  1.00  0.00           N  \\nATOM    627  C5   DG B  20      -3.479  -5.781  -2.295  1.00  0.00           C  \\nATOM    628  C6   DG B  20      -3.817  -5.231  -1.021  1.00  0.00           C  \\nATOM    629  O6   DG B  20      -4.927  -5.135  -0.503  1.00  0.00           O  \\nATOM    630  N1   DG B  20      -2.701  -4.768  -0.328  1.00  0.00           N  \\nATOM    631  C2   DG B  20      -1.405  -4.837  -0.802  1.00  0.00           C  \\nATOM    632  N2   DG B  20      -0.442  -4.372  -0.002  1.00  0.00           N  \\nATOM    633  N3   DG B  20      -1.086  -5.351  -2.000  1.00  0.00           N  \\nATOM    634  C4   DG B  20      -2.164  -5.811  -2.696  1.00  0.00           C  \\nATOM    635  H5'  DG B  20      -0.767  -4.375  -8.026  1.00  0.00           H  \\nATOM    636 H5''  DG B  20      -0.111  -5.770  -8.901  1.00  0.00           H  \\nATOM    637  H4'  DG B  20       0.985  -5.478  -6.723  1.00  0.00           H  \\nATOM    638  H3'  DG B  20      -0.015  -7.917  -7.673  1.00  0.00           H  \\nATOM    639  H2'  DG B  20      -1.964  -7.827  -6.285  1.00  0.00           H  \\nATOM    640 H2''  DG B  20      -0.880  -8.771  -5.219  1.00  0.00           H  \\nATOM    641  H1'  DG B  20      -0.078  -6.793  -4.131  1.00  0.00           H  \\nATOM    642  H8   DG B  20      -3.797  -7.185  -5.112  1.00  0.00           H  \\nATOM    643  H1   DG B  20      -2.869  -4.365   0.581  1.00  0.00           H  \\nATOM    644  H21  DG B  20      -0.665  -3.996   0.907  1.00  0.00           H  \\nATOM    645  H22  DG B  20       0.527  -4.453  -0.290  1.00  0.00           H  \\nATOM    646  P    DG B  21       2.024  -9.247  -6.211  1.00  0.00           P  \\nATOM    647  OP1  DG B  21       3.136  -9.244  -7.188  1.00  0.00           O  \\nATOM    648  OP2  DG B  21       0.932 -10.238  -6.335  1.00  0.00           O  \\nATOM    649  O5'  DG B  21       2.634  -9.366  -4.728  1.00  0.00           O  \\nATOM    650  C5'  DG B  21       3.892  -8.820  -4.379  1.00  0.00           C  \\nATOM    651  C4'  DG B  21       3.984  -8.711  -2.853  1.00  0.00           C  \\nATOM    652  O4'  DG B  21       3.006  -7.827  -2.336  1.00  0.00           O  \\nATOM    653  C3'  DG B  21       3.757 -10.058  -2.164  1.00  0.00           C  \\nATOM    654  O3'  DG B  21       4.960 -10.804  -2.168  1.00  0.00           O  \\nATOM    655  C2'  DG B  21       3.289  -9.606  -0.788  1.00  0.00           C  \\nATOM    656  C1'  DG B  21       2.562  -8.286  -1.068  1.00  0.00           C  \\nATOM    657  N9   DG B  21       1.080  -8.399  -1.129  1.00  0.00           N  \\nATOM    658  C8   DG B  21       0.293  -8.799  -2.181  1.00  0.00           C  \\nATOM    659  N7   DG B  21      -0.987  -8.757  -1.951  1.00  0.00           N  \\nATOM    660  C5   DG B  21      -1.072  -8.267  -0.656  1.00  0.00           C  \\nATOM    661  C6   DG B  21      -2.223  -7.998   0.146  1.00  0.00           C  \\nATOM    662  O6   DG B  21      -3.403  -8.196  -0.130  1.00  0.00           O  \\nATOM    663  N1   DG B  21      -1.895  -7.440   1.378  1.00  0.00           N  \\nATOM    664  C2   DG B  21      -0.602  -7.172   1.790  1.00  0.00           C  \\nATOM    665  N2   DG B  21      -0.446  -6.644   3.007  1.00  0.00           N  \\nATOM    666  N3   DG B  21       0.477  -7.430   1.037  1.00  0.00           N  \\nATOM    667  C4   DG B  21       0.183  -7.989  -0.167  1.00  0.00           C  \\nATOM    668  H5'  DG B  21       4.032  -7.833  -4.822  1.00  0.00           H  \\nATOM    669 H5''  DG B  21       4.678  -9.482  -4.745  1.00  0.00           H  \\nATOM    670  H4'  DG B  21       4.963  -8.310  -2.583  1.00  0.00           H  \\nATOM    671  H3'  DG B  21       2.955 -10.613  -2.649  1.00  0.00           H  \\nATOM    672  H2'  DG B  21       2.659 -10.339  -0.303  1.00  0.00           H  \\nATOM    673 H2''  DG B  21       4.145  -9.415  -0.143  1.00  0.00           H  \\nATOM    674  H1'  DG B  21       2.846  -7.600  -0.272  1.00  0.00           H  \\nATOM    675  H8   DG B  21       0.683  -9.104  -3.135  1.00  0.00           H  \\nATOM    676  H1   DG B  21      -2.661  -7.241   2.003  1.00  0.00           H  \\nATOM    677  H21  DG B  21      -1.236  -6.484   3.613  1.00  0.00           H  \\nATOM    678  H22  DG B  21       0.497  -6.466   3.341  1.00  0.00           H  \\nATOM    679  P    DA B  22       5.037 -12.344  -1.689  1.00  0.00           P  \\nATOM    680  OP1  DA B  22       6.199 -12.970  -2.359  1.00  0.00           O  \\nATOM    681  OP2  DA B  22       3.697 -12.958  -1.830  1.00  0.00           O  \\nATOM    682  O5'  DA B  22       5.377 -12.240  -0.119  1.00  0.00           O  \\nATOM    683  C5'  DA B  22       6.485 -11.495   0.353  1.00  0.00           C  \\nATOM    684  C4'  DA B  22       6.880 -11.944   1.766  1.00  0.00           C  \\nATOM    685  O4'  DA B  22       5.707 -12.137   2.543  1.00  0.00           O  \\nATOM    686  C3'  DA B  22       7.656 -13.269   1.773  1.00  0.00           C  \\nATOM    687  O3'  DA B  22       8.548 -13.254   2.879  1.00  0.00           O  \\nATOM    688  C2'  DA B  22       6.523 -14.273   1.937  1.00  0.00           C  \\nATOM    689  C1'  DA B  22       5.614 -13.508   2.894  1.00  0.00           C  \\nATOM    690  N9   DA B  22       4.209 -13.970   2.861  1.00  0.00           N  \\nATOM    691  C8   DA B  22       3.416 -14.290   1.784  1.00  0.00           C  \\nATOM    692  N7   DA B  22       2.184 -14.584   2.096  1.00  0.00           N  \\nATOM    693  C5   DA B  22       2.154 -14.452   3.478  1.00  0.00           C  \\nATOM    694  C6   DA B  22       1.134 -14.604   4.440  1.00  0.00           C  \\nATOM    695  N6   DA B  22      -0.118 -14.938   4.121  1.00  0.00           N  \\nATOM    696  N1   DA B  22       1.441 -14.424   5.739  1.00  0.00           N  \\nATOM    697  C2   DA B  22       2.685 -14.079   6.062  1.00  0.00           C  \\nATOM    698  N3   DA B  22       3.723 -13.880   5.257  1.00  0.00           N  \\nATOM    699  C4   DA B  22       3.389 -14.092   3.956  1.00  0.00           C  \\nATOM    700  H5'  DA B  22       6.211 -10.440   0.380  1.00  0.00           H  \\nATOM    701 H5''  DA B  22       7.346 -11.609  -0.308  1.00  0.00           H  \\nATOM    702  H4'  DA B  22       7.495 -11.158   2.207  1.00  0.00           H  \\nATOM    703  H3'  DA B  22       8.206 -13.437   0.846  1.00  0.00           H  \\nATOM    704  H2'  DA B  22       6.050 -14.437   0.971  1.00  0.00           H  \\nATOM    705 H2''  DA B  22       6.847 -15.224   2.347  1.00  0.00           H  \\nATOM    706  H1'  DA B  22       6.017 -13.627   3.898  1.00  0.00           H  \\nATOM    707  H8   DA B  22       3.765 -14.289   0.763  1.00  0.00           H  \\nATOM    708  H61  DA B  22      -0.799 -15.098   4.849  1.00  0.00           H  \\nATOM    709  H62  DA B  22      -0.403 -14.971   3.149  1.00  0.00           H  \\nATOM    710  H2   DA B  22       2.882 -13.931   7.112  1.00  0.00           H  \\nATOM    711  P    DA B  23       9.498 -14.507   3.262  1.00  0.00           P  \\nATOM    712  OP1  DA B  23      10.631 -13.997   4.067  1.00  0.00           O  \\nATOM    713  OP2  DA B  23       9.762 -15.291   2.035  1.00  0.00           O  \\nATOM    714  O5'  DA B  23       8.550 -15.384   4.227  1.00  0.00           O  \\nATOM    715  C5'  DA B  23       8.201 -14.937   5.523  1.00  0.00           C  \\nATOM    716  C4'  DA B  23       7.125 -15.855   6.111  1.00  0.00           C  \\nATOM    717  O4'  DA B  23       6.004 -15.909   5.243  1.00  0.00           O  \\nATOM    718  C3'  DA B  23       7.640 -17.289   6.298  1.00  0.00           C  \\nATOM    719  O3'  DA B  23       7.347 -17.705   7.624  1.00  0.00           O  \\nATOM    720  C2'  DA B  23       6.857 -18.064   5.238  1.00  0.00           C  \\nATOM    721  C1'  DA B  23       5.569 -17.250   5.162  1.00  0.00           C  \\nATOM    722  N9   DA B  23       4.787 -17.460   3.922  1.00  0.00           N  \\nATOM    723  C8   DA B  23       5.218 -17.504   2.618  1.00  0.00           C  \\nATOM    724  N7   DA B  23       4.265 -17.645   1.740  1.00  0.00           N  \\nATOM    725  C5   DA B  23       3.113 -17.702   2.513  1.00  0.00           C  \\nATOM    726  C6   DA B  23       1.748 -17.856   2.192  1.00  0.00           C  \\nATOM    727  N6   DA B  23       1.307 -17.978   0.936  1.00  0.00           N  \\nATOM    728  N1   DA B  23       0.854 -17.899   3.198  1.00  0.00           N  \\nATOM    729  C2   DA B  23       1.287 -17.784   4.452  1.00  0.00           C  \\nATOM    730  N3   DA B  23       2.536 -17.627   4.883  1.00  0.00           N  \\nATOM    731  C4   DA B  23       3.419 -17.594   3.847  1.00  0.00           C  \\nATOM    732  H5'  DA B  23       7.817 -13.917   5.486  1.00  0.00           H  \\nATOM    733 H5''  DA B  23       9.080 -14.949   6.168  1.00  0.00           H  \\nATOM    734  H4'  DA B  23       6.806 -15.445   7.070  1.00  0.00           H  \\nATOM    735  H3'  DA B  23       8.712 -17.341   6.108  1.00  0.00           H  \\nATOM    736  H2'  DA B  23       7.407 -18.009   4.298  1.00  0.00           H  \\nATOM    737 H2''  DA B  23       6.682 -19.104   5.506  1.00  0.00           H  \\nATOM    738  H1'  DA B  23       4.956 -17.492   6.028  1.00  0.00           H  \\nATOM    739  H8   DA B  23       6.256 -17.425   2.330  1.00  0.00           H  \\nATOM    740  H61  DA B  23       0.323 -18.116   0.752  1.00  0.00           H  \\nATOM    741  H62  DA B  23       1.969 -17.957   0.174  1.00  0.00           H  \\nATOM    742  H2   DA B  23       0.527 -17.826   5.218  1.00  0.00           H  \\nATOM    743  P    DC B  24       7.880 -19.103   8.239  1.00  0.00           P  \\nATOM    744  OP1  DC B  24       8.444 -18.837   9.581  1.00  0.00           O  \\nATOM    745  OP2  DC B  24       8.699 -19.794   7.218  1.00  0.00           O  \\nATOM    746  O5'  DC B  24       6.510 -19.929   8.421  1.00  0.00           O  \\nATOM    747  C5'  DC B  24       5.652 -19.718   9.530  1.00  0.00           C  \\nATOM    748  C4'  DC B  24       4.280 -20.317   9.208  1.00  0.00           C  \\nATOM    749  O4'  DC B  24       3.831 -19.851   7.950  1.00  0.00           O  \\nATOM    750  C3'  DC B  24       4.318 -21.845   9.110  1.00  0.00           C  \\nATOM    751  O3'  DC B  24       3.855 -22.448  10.301  1.00  0.00           O  \\nATOM    752  C2'  DC B  24       3.395 -22.164   7.937  1.00  0.00           C  \\nATOM    753  C1'  DC B  24       2.914 -20.799   7.442  1.00  0.00           C  \\nATOM    754  N1   DC B  24       2.896 -20.769   5.955  1.00  0.00           N  \\nATOM    755  C2   DC B  24       1.679 -20.878   5.280  1.00  0.00           C  \\nATOM    756  O2   DC B  24       0.610 -20.900   5.887  1.00  0.00           O  \\nATOM    757  N3   DC B  24       1.696 -20.948   3.918  1.00  0.00           N  \\nATOM    758  C4   DC B  24       2.848 -20.923   3.233  1.00  0.00           C  \\nATOM    759  N4   DC B  24       2.801 -20.973   1.898  1.00  0.00           N  \\nATOM    760  C5   DC B  24       4.112 -20.809   3.906  1.00  0.00           C  \\nATOM    761  C6   DC B  24       4.079 -20.734   5.257  1.00  0.00           C  \\nATOM    762  H5'  DC B  24       5.537 -18.654   9.743  1.00  0.00           H  \\nATOM    763 H5''  DC B  24       6.068 -20.202  10.415  1.00  0.00           H  \\nATOM    764  H4'  DC B  24       3.557 -20.018   9.969  1.00  0.00           H  \\nATOM    765  H3'  DC B  24       5.324 -22.197   8.879  1.00  0.00           H  \\nATOM    766 HO3'  DC B  24       2.968 -22.129  10.481  1.00  0.00           H  \\nATOM    767  H2'  DC B  24       3.970 -22.680   7.169  1.00  0.00           H  \\nATOM    768 H2''  DC B  24       2.548 -22.782   8.229  1.00  0.00           H  \\nATOM    769  H1'  DC B  24       1.943 -20.587   7.888  1.00  0.00           H  \\nATOM    770  H41  DC B  24       1.902 -21.016   1.437  1.00  0.00           H  \\nATOM    771  H42  DC B  24       3.650 -20.922   1.355  1.00  0.00           H  \\nATOM    772  H5   DC B  24       5.062 -20.776   3.392  1.00  0.00           H  \\nATOM    773  H6   DC B  24       5.004 -20.670   5.810  1.00  0.00           H  \\nTER     774       DC B  24                                                      \\nMASTER      121    0    0    0    0    0    0    6  500    2    0    2          \\nEND                                                                             \\n\");\n\tviewer_168469674212664.setStyle({\"model\": -1, \"chain\": \"A\"},{\"cartoon\": {\"color\": \"spectrum\"}});\n\tviewer_168469674212664.zoomTo();\nviewer_168469674212664.render();\n});\n</script>",
      "text/html": [
       "<div id=\"3dmolviewer_168469674212664\"  style=\"position: relative; width: 600px; height: 400px\">\n",
       "        <p id=\"3dmolwarning_168469674212664\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
       "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
       "        </div>\n",
       "<script>\n",
       "\n",
       "var loadScriptAsync = function(uri){\n",
       "  return new Promise((resolve, reject) => {\n",
       "    //this is to ignore the existence of requirejs amd\n",
       "    var savedexports, savedmodule;\n",
       "    if (typeof exports !== 'undefined') savedexports = exports;\n",
       "    else exports = {}\n",
       "    if (typeof module !== 'undefined') savedmodule = module;\n",
       "    else module = {}\n",
       "\n",
       "    var tag = document.createElement('script');\n",
       "    tag.src = uri;\n",
       "    tag.async = true;\n",
       "    tag.onload = () => {\n",
       "        exports = savedexports;\n",
       "        module = savedmodule;\n",
       "        resolve();\n",
       "    };\n",
       "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
       "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
       "});\n",
       "};\n",
       "\n",
       "if(typeof $3Dmolpromise === 'undefined') {\n",
       "$3Dmolpromise = null;\n",
       "  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.0.1/3Dmol-min.js');\n",
       "}\n",
       "\n",
       "var viewer_168469674212664 = null;\n",
       "var warn = document.getElementById(\"3dmolwarning_168469674212664\");\n",
       "if(warn) {\n",
       "    warn.parentNode.removeChild(warn);\n",
       "}\n",
       "$3Dmolpromise.then(function() {\n",
       "viewer_168469674212664 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_168469674212664\"),{backgroundColor:\"white\"});\n",
       "viewer_168469674212664.zoomTo();\n",
       "\tviewer_168469674212664.addModelsAsFrames(\"HEADER    DNA                                     16-DEC-94   103D              \\nTITLE     THE UNUSUAL STRUCTURE OF THE HUMAN CENTROMERE (GGA)2 MOTIF: UNPAIRED  \\nTITLE    2 GUANOSINE RESIDUES STACKED BETWEEN SHEARED G(DOT)A PAIRS             \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: DNA (5'-D(*GP*TP*GP*GP*AP*AP*TP*GP*GP*AP*AP*C)-3');        \\nCOMPND   3 CHAIN: A, B;                                                         \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 SYNTHETIC: YES;                                                      \\nSOURCE   3 OTHER_DETAILS: CHEMICALLY SYNTHESIZED                                \\nKEYWDS    DNA, DOUBLE HELIX, G-G STACKING, G:A MISMATCH, HUMAN CENTROMERE       \\nKEYWDS   2 REPEAT, GA-BRACKETED G-STACK MOTIF                                   \\nEXPDTA    SOLUTION NMR                                                          \\nAUTHOR    S.-H.CHOU,L.ZHU,B.R.REID                                              \\nREVDAT   5   16-FEB-22 103D    1       REMARK                                   \\nREVDAT   4   07-APR-09 103D    1       REMARK                                   \\nREVDAT   3   24-FEB-09 103D    1       VERSN                                    \\nREVDAT   2   01-APR-03 103D    1       JRNL                                     \\nREVDAT   1   31-MAR-95 103D    0                                                \\nJRNL        AUTH   S.H.CHOU,L.ZHU,B.R.REID                                      \\nJRNL        TITL   THE UNUSUAL STRUCTURE OF THE HUMAN CENTROMERE (GGA)2 MOTIF.  \\nJRNL        TITL 2 UNPAIRED GUANOSINE RESIDUES STACKED BETWEEN SHEARED G.A      \\nJRNL        TITL 3 PAIRS.                                                       \\nJRNL        REF    J.MOL.BIOL.                   V. 244   259 1994              \\nJRNL        REFN                   ISSN 0022-2836                               \\nJRNL        PMID   7966337                                                      \\nJRNL        DOI    10.1006/JMBI.1994.1727                                       \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION. NOT APPLICABLE.                                          \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : AMBER                                                \\nREMARK   3   AUTHORS     : BIOSYM TECHNOLOGIES, INC.                            \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: R VALUE 0.250 FINAL RMS COORD. SHIFT      \\nREMARK   3  0.65 ANGSTROMS NUMBER OF ATOMS USED IN REFINEMENT. NUMBER OF        \\nREMARK   3  PROTEIN ATOMS 0 NUMBER OF NUCLEIC ACID ATOMS 772 NUMBER OF          \\nREMARK   3  HETEROGEN ATOMS 0 NUMBER OF SOLVENT ATOMS 0. THE DUPLEX CONTAINS    \\nREMARK   3  UNPAIRED STAGGERED GUANOSINE RESIDUES, WHICH CO-STACK BY            \\nREMARK   3  INTERCALATION BETWEEN SHEARED G(DOT)A AND A(DOT)G BASE-PAIRS TO     \\nREMARK   3  FORM AN INTERESTING NEW STRUCTURAL MOTIF, THE GA-BRACKETED G-       \\nREMARK   3  STACK. THE TGGAA REPEAT UNIT CONTAINS SIX \\\"STEPS\\\", FOUR OF WHICH    \\nREMARK   3  ARE NOT WATSON-CRICK BASE-PAIRS.                                    \\nREMARK   4                                                                      \\nREMARK   4 103D COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000170007.                                   \\nREMARK 210                                                                      \\nREMARK 210 EXPERIMENTAL DETAILS                                                 \\nREMARK 210  EXPERIMENT TYPE                : NMR                                \\nREMARK 210  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 210  PH                             : NULL                               \\nREMARK 210  IONIC STRENGTH                 : NULL                               \\nREMARK 210  PRESSURE                       : NULL                               \\nREMARK 210  SAMPLE CONTENTS                : NULL                               \\nREMARK 210                                                                      \\nREMARK 210  NMR EXPERIMENTS CONDUCTED      : NULL                               \\nREMARK 210  SPECTROMETER FIELD STRENGTH    : NULL                               \\nREMARK 210  SPECTROMETER MODEL             : NULL                               \\nREMARK 210  SPECTROMETER MANUFACTURER      : NULL                               \\nREMARK 210                                                                      \\nREMARK 210  STRUCTURE DETERMINATION.                                            \\nREMARK 210   SOFTWARE USED                 : NULL                               \\nREMARK 210   METHOD USED                   : NULL                               \\nREMARK 210                                                                      \\nREMARK 210 CONFORMERS, NUMBER CALCULATED   : NULL                               \\nREMARK 210 CONFORMERS, NUMBER SUBMITTED    : 1                                  \\nREMARK 210 CONFORMERS, SELECTION CRITERIA  : NULL                               \\nREMARK 210                                                                      \\nREMARK 210 BEST REPRESENTATIVE CONFORMER IN THIS ENSEMBLE : NULL                \\nREMARK 210                                                                      \\nREMARK 210 REMARK: NULL                                                         \\nREMARK 215                                                                      \\nREMARK 215 NMR STUDY                                                            \\nREMARK 215 THE COORDINATES IN THIS ENTRY WERE GENERATED FROM SOLUTION           \\nREMARK 215 NMR DATA.  PROTEIN DATA BANK CONVENTIONS REQUIRE THAT                \\nREMARK 215 CRYST1 AND SCALE RECORDS BE INCLUDED, BUT THE VALUES ON              \\nREMARK 215 THESE RECORDS ARE MEANINGLESS.                                       \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A, B                                  \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \\nREMARK 500     DG A   1   O4' -  C1' -  N9  ANGL. DEV. =   1.9 DEGREES          \\nREMARK 500     DA A   5   O4' -  C1' -  N9  ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500     DA A   6   O4' -  C1' -  N9  ANGL. DEV. =   2.2 DEGREES          \\nREMARK 500     DT A   7   O4' -  C1' -  N1  ANGL. DEV. =   2.5 DEGREES          \\nREMARK 500     DA A  10   O4' -  C1' -  N9  ANGL. DEV. =   2.7 DEGREES          \\nREMARK 500     DA A  11   O4' -  C1' -  N9  ANGL. DEV. =   2.0 DEGREES          \\nREMARK 500     DC A  12   O4' -  C1' -  N1  ANGL. DEV. =   2.5 DEGREES          \\nREMARK 500     DG B  13   O4' -  C1' -  N9  ANGL. DEV. =   1.9 DEGREES          \\nREMARK 500     DT B  14   O4' -  C1' -  N1  ANGL. DEV. =   1.8 DEGREES          \\nREMARK 500     DA B  17   O4' -  C1' -  N9  ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500     DA B  18   O4' -  C1' -  N9  ANGL. DEV. =   2.3 DEGREES          \\nREMARK 500     DT B  19   O4' -  C1' -  N1  ANGL. DEV. =   2.5 DEGREES          \\nREMARK 500     DA B  22   O4' -  C1' -  N9  ANGL. DEV. =   2.7 DEGREES          \\nREMARK 500     DA B  23   O4' -  C1' -  N9  ANGL. DEV. =   1.9 DEGREES          \\nREMARK 500     DC B  24   O4' -  C1' -  N1  ANGL. DEV. =   2.4 DEGREES          \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: PLANAR GROUPS                                              \\nREMARK 500                                                                      \\nREMARK 500 PLANAR GROUPS IN THE FOLLOWING RESIDUES HAVE A TOTAL                 \\nREMARK 500 RMS DISTANCE OF ALL ATOMS FROM THE BEST-FIT PLANE                    \\nREMARK 500 BY MORE THAN AN EXPECTED VALUE OF 6*RMSD, WITH AN                    \\nREMARK 500 RMSD 0.02 ANGSTROMS, OR AT LEAST ONE ATOM HAS                        \\nREMARK 500 AN RMSD GREATER THAN THIS VALUE                                      \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        RMS     TYPE                                    \\nREMARK 500     DT A   2         0.06    SIDE CHAIN                              \\nREMARK 500     DT B  14         0.06    SIDE CHAIN                              \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nDBREF  103D A    1    12  PDB    103D     103D             1     12             \\nDBREF  103D B   13    24  PDB    103D     103D            13     24             \\nSEQRES   1 A   12   DG  DT  DG  DG  DA  DA  DT  DG  DG  DA  DA  DC              \\nSEQRES   1 B   12   DG  DT  DG  DG  DA  DA  DT  DG  DG  DA  DA  DC              \\nCRYST1    1.000    1.000    1.000  90.00  90.00  90.00 P 1           1          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      1.000000  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  1.000000  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  1.000000        0.00000                         \\nATOM      1  O5'  DG A   1      -7.643 -22.640  -1.099  1.00  0.00           O  \\nATOM      2  C5'  DG A   1      -8.313 -23.137   0.040  1.00  0.00           C  \\nATOM      3  C4'  DG A   1      -7.935 -22.349   1.300  1.00  0.00           C  \\nATOM      4  O4'  DG A   1      -6.537 -22.463   1.524  1.00  0.00           O  \\nATOM      5  C3'  DG A   1      -8.273 -20.854   1.190  1.00  0.00           C  \\nATOM      6  O3'  DG A   1      -8.832 -20.418   2.419  1.00  0.00           O  \\nATOM      7  C2'  DG A   1      -6.902 -20.240   0.924  1.00  0.00           C  \\nATOM      8  C1'  DG A   1      -5.995 -21.171   1.722  1.00  0.00           C  \\nATOM      9  N9   DG A   1      -4.593 -21.126   1.247  1.00  0.00           N  \\nATOM     10  C8   DG A   1      -4.107 -21.429  -0.001  1.00  0.00           C  \\nATOM     11  N7   DG A   1      -2.812 -21.335  -0.110  1.00  0.00           N  \\nATOM     12  C5   DG A   1      -2.397 -20.963   1.163  1.00  0.00           C  \\nATOM     13  C6   DG A   1      -1.080 -20.733   1.666  1.00  0.00           C  \\nATOM     14  O6   DG A   1      -0.017 -20.780   1.053  1.00  0.00           O  \\nATOM     15  N1   DG A   1      -1.077 -20.427   3.024  1.00  0.00           N  \\nATOM     16  C2   DG A   1      -2.214 -20.317   3.803  1.00  0.00           C  \\nATOM     17  N2   DG A   1      -2.045 -20.010   5.092  1.00  0.00           N  \\nATOM     18  N3   DG A   1      -3.455 -20.507   3.325  1.00  0.00           N  \\nATOM     19  C4   DG A   1      -3.479 -20.835   2.004  1.00  0.00           C  \\nATOM     20  H5'  DG A   1      -8.042 -24.185   0.178  1.00  0.00           H  \\nATOM     21 H5''  DG A   1      -9.389 -23.072  -0.122  1.00  0.00           H  \\nATOM     22  H4'  DG A   1      -8.473 -22.789   2.142  1.00  0.00           H  \\nATOM     23  H3'  DG A   1      -8.969 -20.658   0.374  1.00  0.00           H  \\nATOM     24  H2'  DG A   1      -6.682 -20.310  -0.141  1.00  0.00           H  \\nATOM     25 H2''  DG A   1      -6.821 -19.205   1.250  1.00  0.00           H  \\nATOM     26  H1'  DG A   1      -6.058 -20.912   2.780  1.00  0.00           H  \\nATOM     27  H8   DG A   1      -4.739 -21.718  -0.828  1.00  0.00           H  \\nATOM     28  H1   DG A   1      -0.169 -20.285   3.449  1.00  0.00           H  \\nATOM     29  H21  DG A   1      -1.114 -19.925   5.477  1.00  0.00           H  \\nATOM     30  H22  DG A   1      -2.853 -19.902   5.687  1.00  0.00           H  \\nATOM     31 HO5'  DG A   1      -7.929 -21.737  -1.254  1.00  0.00           H  \\nATOM     32  P    DT A   2      -9.356 -18.904   2.648  1.00  0.00           P  \\nATOM     33  OP1  DT A   2     -10.579 -18.963   3.478  1.00  0.00           O  \\nATOM     34  OP2  DT A   2      -9.388 -18.212   1.340  1.00  0.00           O  \\nATOM     35  O5'  DT A   2      -8.175 -18.250   3.532  1.00  0.00           O  \\nATOM     36  C5'  DT A   2      -7.959 -18.655   4.871  1.00  0.00           C  \\nATOM     37  C4'  DT A   2      -6.761 -17.920   5.479  1.00  0.00           C  \\nATOM     38  O4'  DT A   2      -5.603 -18.173   4.699  1.00  0.00           O  \\nATOM     39  C3'  DT A   2      -6.955 -16.396   5.550  1.00  0.00           C  \\nATOM     40  O3'  DT A   2      -6.507 -15.961   6.826  1.00  0.00           O  \\nATOM     41  C2'  DT A   2      -6.062 -15.914   4.406  1.00  0.00           C  \\nATOM     42  C1'  DT A   2      -4.938 -16.944   4.495  1.00  0.00           C  \\nATOM     43  N1   DT A   2      -4.057 -17.032   3.297  1.00  0.00           N  \\nATOM     44  C2   DT A   2      -2.677 -17.088   3.499  1.00  0.00           C  \\nATOM     45  O2   DT A   2      -2.146 -16.945   4.598  1.00  0.00           O  \\nATOM     46  N3   DT A   2      -1.900 -17.352   2.381  1.00  0.00           N  \\nATOM     47  C4   DT A   2      -2.368 -17.603   1.102  1.00  0.00           C  \\nATOM     48  O4   DT A   2      -1.578 -17.886   0.204  1.00  0.00           O  \\nATOM     49  C5   DT A   2      -3.808 -17.502   0.964  1.00  0.00           C  \\nATOM     50  C7   DT A   2      -4.451 -17.702  -0.395  1.00  0.00           C  \\nATOM     51  C6   DT A   2      -4.587 -17.231   2.042  1.00  0.00           C  \\nATOM     52  H5'  DT A   2      -7.763 -19.727   4.902  1.00  0.00           H  \\nATOM     53 H5''  DT A   2      -8.845 -18.442   5.472  1.00  0.00           H  \\nATOM     54  H4'  DT A   2      -6.601 -18.314   6.483  1.00  0.00           H  \\nATOM     55  H3'  DT A   2      -7.997 -16.110   5.395  1.00  0.00           H  \\nATOM     56  H2'  DT A   2      -6.630 -15.995   3.481  1.00  0.00           H  \\nATOM     57 H2''  DT A   2      -5.703 -14.894   4.522  1.00  0.00           H  \\nATOM     58  H1'  DT A   2      -4.373 -16.711   5.397  1.00  0.00           H  \\nATOM     59  H3   DT A   2      -0.903 -17.421   2.534  1.00  0.00           H  \\nATOM     60  H71  DT A   2      -3.753 -17.435  -1.188  1.00  0.00           H  \\nATOM     61  H72  DT A   2      -4.736 -18.745  -0.514  1.00  0.00           H  \\nATOM     62  H73  DT A   2      -5.340 -17.077  -0.489  1.00  0.00           H  \\nATOM     63  H6   DT A   2      -5.655 -17.199   1.905  1.00  0.00           H  \\nATOM     64  P    DG A   3      -6.534 -14.414   7.292  1.00  0.00           P  \\nATOM     65  OP1  DG A   3      -6.475 -14.378   8.770  1.00  0.00           O  \\nATOM     66  OP2  DG A   3      -7.637 -13.728   6.584  1.00  0.00           O  \\nATOM     67  O5'  DG A   3      -5.139 -13.848   6.720  1.00  0.00           O  \\nATOM     68  C5'  DG A   3      -3.896 -14.287   7.234  1.00  0.00           C  \\nATOM     69  C4'  DG A   3      -2.748 -13.705   6.405  1.00  0.00           C  \\nATOM     70  O4'  DG A   3      -2.863 -14.119   5.053  1.00  0.00           O  \\nATOM     71  C3'  DG A   3      -2.751 -12.171   6.416  1.00  0.00           C  \\nATOM     72  O3'  DG A   3      -1.406 -11.747   6.547  1.00  0.00           O  \\nATOM     73  C2'  DG A   3      -3.335 -11.844   5.041  1.00  0.00           C  \\nATOM     74  C1'  DG A   3      -2.741 -12.983   4.218  1.00  0.00           C  \\nATOM     75  N9   DG A   3      -3.405 -13.249   2.921  1.00  0.00           N  \\nATOM     76  C8   DG A   3      -4.715 -13.058   2.555  1.00  0.00           C  \\nATOM     77  N7   DG A   3      -4.994 -13.444   1.343  1.00  0.00           N  \\nATOM     78  C5   DG A   3      -3.784 -13.924   0.857  1.00  0.00           C  \\nATOM     79  C6   DG A   3      -3.457 -14.458  -0.426  1.00  0.00           C  \\nATOM     80  O6   DG A   3      -4.194 -14.609  -1.397  1.00  0.00           O  \\nATOM     81  N1   DG A   3      -2.119 -14.829  -0.523  1.00  0.00           N  \\nATOM     82  C2   DG A   3      -1.196 -14.693   0.496  1.00  0.00           C  \\nATOM     83  N2   DG A   3       0.052 -15.094   0.239  1.00  0.00           N  \\nATOM     84  N3   DG A   3      -1.501 -14.190   1.703  1.00  0.00           N  \\nATOM     85  C4   DG A   3      -2.807 -13.820   1.820  1.00  0.00           C  \\nATOM     86  H5'  DG A   3      -3.836 -15.375   7.202  1.00  0.00           H  \\nATOM     87 H5''  DG A   3      -3.792 -13.963   8.271  1.00  0.00           H  \\nATOM     88  H4'  DG A   3      -1.809 -14.080   6.816  1.00  0.00           H  \\nATOM     89  H3'  DG A   3      -3.366 -11.782   7.227  1.00  0.00           H  \\nATOM     90  H2'  DG A   3      -4.421 -11.917   5.096  1.00  0.00           H  \\nATOM     91 H2''  DG A   3      -3.043 -10.865   4.661  1.00  0.00           H  \\nATOM     92  H1'  DG A   3      -1.692 -12.735   4.043  1.00  0.00           H  \\nATOM     93  H8   DG A   3      -5.463 -12.632   3.207  1.00  0.00           H  \\nATOM     94  H1   DG A   3      -1.821 -15.221  -1.404  1.00  0.00           H  \\nATOM     95  H21  DG A   3       0.298 -15.465  -0.666  1.00  0.00           H  \\nATOM     96  H22  DG A   3       0.772 -14.962   0.941  1.00  0.00           H  \\nATOM     97  P    DG A   4      -0.997 -10.224   6.884  1.00  0.00           P  \\nATOM     98  OP1  DG A   4      -0.439 -10.175   8.254  1.00  0.00           O  \\nATOM     99  OP2  DG A   4      -2.112  -9.328   6.505  1.00  0.00           O  \\nATOM    100  O5'  DG A   4       0.196 -10.007   5.828  1.00  0.00           O  \\nATOM    101  C5'  DG A   4       1.508 -10.482   6.064  1.00  0.00           C  \\nATOM    102  C4'  DG A   4       2.254 -10.592   4.729  1.00  0.00           C  \\nATOM    103  O4'  DG A   4       1.640 -11.522   3.855  1.00  0.00           O  \\nATOM    104  C3'  DG A   4       2.288  -9.262   3.978  1.00  0.00           C  \\nATOM    105  O3'  DG A   4       3.326  -8.456   4.505  1.00  0.00           O  \\nATOM    106  C2'  DG A   4       2.518  -9.741   2.552  1.00  0.00           C  \\nATOM    107  C1'  DG A   4       1.787 -11.086   2.512  1.00  0.00           C  \\nATOM    108  N9   DG A   4       0.429 -11.010   1.915  1.00  0.00           N  \\nATOM    109  C8   DG A   4      -0.746 -10.615   2.504  1.00  0.00           C  \\nATOM    110  N7   DG A   4      -1.792 -10.688   1.732  1.00  0.00           N  \\nATOM    111  C5   DG A   4      -1.287 -11.185   0.538  1.00  0.00           C  \\nATOM    112  C6   DG A   4      -1.954 -11.480  -0.691  1.00  0.00           C  \\nATOM    113  O6   DG A   4      -3.141 -11.323  -0.965  1.00  0.00           O  \\nATOM    114  N1   DG A   4      -1.096 -12.013  -1.650  1.00  0.00           N  \\nATOM    115  C2   DG A   4       0.251 -12.248  -1.443  1.00  0.00           C  \\nATOM    116  N2   DG A   4       0.952 -12.759  -2.460  1.00  0.00           N  \\nATOM    117  N3   DG A   4       0.873 -11.977  -0.287  1.00  0.00           N  \\nATOM    118  C4   DG A   4       0.061 -11.433   0.657  1.00  0.00           C  \\nATOM    119  H5'  DG A   4       1.495 -11.457   6.552  1.00  0.00           H  \\nATOM    120 H5''  DG A   4       2.027  -9.777   6.714  1.00  0.00           H  \\nATOM    121  H4'  DG A   4       3.271 -10.944   4.919  1.00  0.00           H  \\nATOM    122  H3'  DG A   4       1.317  -8.768   4.033  1.00  0.00           H  \\nATOM    123  H2'  DG A   4       2.150  -9.039   1.815  1.00  0.00           H  \\nATOM    124 H2''  DG A   4       3.578  -9.904   2.373  1.00  0.00           H  \\nATOM    125  H1'  DG A   4       2.409 -11.766   1.933  1.00  0.00           H  \\nATOM    126  H8   DG A   4      -0.813 -10.291   3.528  1.00  0.00           H  \\nATOM    127  H1   DG A   4      -1.500 -12.225  -2.549  1.00  0.00           H  \\nATOM    128  H21  DG A   4       0.523 -12.928  -3.357  1.00  0.00           H  \\nATOM    129  H22  DG A   4       1.950 -12.906  -2.334  1.00  0.00           H  \\nATOM    130  P    DA A   5       3.562  -6.924   4.058  1.00  0.00           P  \\nATOM    131  OP1  DA A   5       4.379  -6.268   5.103  1.00  0.00           O  \\nATOM    132  OP2  DA A   5       2.259  -6.334   3.677  1.00  0.00           O  \\nATOM    133  O5'  DA A   5       4.464  -7.037   2.730  1.00  0.00           O  \\nATOM    134  C5'  DA A   5       5.721  -7.693   2.730  1.00  0.00           C  \\nATOM    135  C4'  DA A   5       6.616  -7.135   1.615  1.00  0.00           C  \\nATOM    136  O4'  DA A   5       5.843  -6.978   0.432  1.00  0.00           O  \\nATOM    137  C3'  DA A   5       7.201  -5.758   1.969  1.00  0.00           C  \\nATOM    138  O3'  DA A   5       8.471  -5.625   1.344  1.00  0.00           O  \\nATOM    139  C2'  DA A   5       6.152  -4.834   1.366  1.00  0.00           C  \\nATOM    140  C1'  DA A   5       5.814  -5.603   0.093  1.00  0.00           C  \\nATOM    141  N9   DA A   5       4.510  -5.219  -0.496  1.00  0.00           N  \\nATOM    142  C8   DA A   5       3.302  -4.987   0.119  1.00  0.00           C  \\nATOM    143  N7   DA A   5       2.320  -4.747  -0.706  1.00  0.00           N  \\nATOM    144  C5   DA A   5       2.915  -4.819  -1.959  1.00  0.00           C  \\nATOM    145  C6   DA A   5       2.422  -4.678  -3.273  1.00  0.00           C  \\nATOM    146  N6   DA A   5       1.142  -4.414  -3.547  1.00  0.00           N  \\nATOM    147  N1   DA A   5       3.285  -4.794  -4.301  1.00  0.00           N  \\nATOM    148  C2   DA A   5       4.562  -5.063  -4.040  1.00  0.00           C  \\nATOM    149  N3   DA A   5       5.145  -5.238  -2.858  1.00  0.00           N  \\nATOM    150  C4   DA A   5       4.254  -5.095  -1.840  1.00  0.00           C  \\nATOM    151  H5'  DA A   5       5.562  -8.757   2.560  1.00  0.00           H  \\nATOM    152 H5''  DA A   5       6.232  -7.572   3.686  1.00  0.00           H  \\nATOM    153  H4'  DA A   5       7.427  -7.844   1.444  1.00  0.00           H  \\nATOM    154  H3'  DA A   5       7.295  -5.615   3.047  1.00  0.00           H  \\nATOM    155  H2'  DA A   5       5.300  -4.777   2.042  1.00  0.00           H  \\nATOM    156 H2''  DA A   5       6.527  -3.836   1.162  1.00  0.00           H  \\nATOM    157  H1'  DA A   5       6.610  -5.423  -0.628  1.00  0.00           H  \\nATOM    158  H8   DA A   5       3.159  -5.017   1.188  1.00  0.00           H  \\nATOM    159  H61  DA A   5       0.850  -4.267  -4.502  1.00  0.00           H  \\nATOM    160  H62  DA A   5       0.451  -4.437  -2.806  1.00  0.00           H  \\nATOM    161  H2   DA A   5       5.213  -5.162  -4.894  1.00  0.00           H  \\nATOM    162  P    DA A   6       9.363  -4.276   1.441  1.00  0.00           P  \\nATOM    163  OP1  DA A   6      10.782  -4.649   1.246  1.00  0.00           O  \\nATOM    164  OP2  DA A   6       8.960  -3.530   2.654  1.00  0.00           O  \\nATOM    165  O5'  DA A   6       8.887  -3.431   0.150  1.00  0.00           O  \\nATOM    166  C5'  DA A   6       9.220  -3.846  -1.162  1.00  0.00           C  \\nATOM    167  C4'  DA A   6       8.472  -3.000  -2.195  1.00  0.00           C  \\nATOM    168  O4'  DA A   6       7.079  -3.068  -1.929  1.00  0.00           O  \\nATOM    169  C3'  DA A   6       8.883  -1.518  -2.181  1.00  0.00           C  \\nATOM    170  O3'  DA A   6       9.162  -1.103  -3.510  1.00  0.00           O  \\nATOM    171  C2'  DA A   6       7.638  -0.843  -1.609  1.00  0.00           C  \\nATOM    172  C1'  DA A   6       6.534  -1.778  -2.095  1.00  0.00           C  \\nATOM    173  N9   DA A   6       5.266  -1.649  -1.337  1.00  0.00           N  \\nATOM    174  C8   DA A   6       5.062  -1.659   0.023  1.00  0.00           C  \\nATOM    175  N7   DA A   6       3.811  -1.603   0.382  1.00  0.00           N  \\nATOM    176  C5   DA A   6       3.128  -1.551  -0.825  1.00  0.00           C  \\nATOM    177  C6   DA A   6       1.757  -1.487  -1.146  1.00  0.00           C  \\nATOM    178  N6   DA A   6       0.795  -1.473  -0.218  1.00  0.00           N  \\nATOM    179  N1   DA A   6       1.406  -1.437  -2.443  1.00  0.00           N  \\nATOM    180  C2   DA A   6       2.358  -1.447  -3.373  1.00  0.00           C  \\nATOM    181  N3   DA A   6       3.677  -1.511  -3.202  1.00  0.00           N  \\nATOM    182  C4   DA A   6       4.005  -1.564  -1.882  1.00  0.00           C  \\nATOM    183  H5'  DA A   6       8.939  -4.890  -1.301  1.00  0.00           H  \\nATOM    184 H5''  DA A   6      10.295  -3.750  -1.321  1.00  0.00           H  \\nATOM    185  H4'  DA A   6       8.666  -3.422  -3.183  1.00  0.00           H  \\nATOM    186  H3'  DA A   6       9.752  -1.349  -1.542  1.00  0.00           H  \\nATOM    187  H2'  DA A   6       7.729  -0.858  -0.523  1.00  0.00           H  \\nATOM    188 H2''  DA A   6       7.497   0.178  -1.959  1.00  0.00           H  \\nATOM    189  H1'  DA A   6       6.357  -1.604  -3.158  1.00  0.00           H  \\nATOM    190  H8   DA A   6       5.858  -1.702   0.751  1.00  0.00           H  \\nATOM    191  H61  DA A   6      -0.176  -1.429  -0.493  1.00  0.00           H  \\nATOM    192  H62  DA A   6       1.049  -1.512   0.759  1.00  0.00           H  \\nATOM    193  H2   DA A   6       2.014  -1.412  -4.395  1.00  0.00           H  \\nATOM    194  P    DT A   7       9.713   0.376  -3.872  1.00  0.00           P  \\nATOM    195  OP1  DT A   7      10.872   0.227  -4.781  1.00  0.00           O  \\nATOM    196  OP2  DT A   7       9.854   1.156  -2.622  1.00  0.00           O  \\nATOM    197  O5'  DT A   7       8.487   0.997  -4.715  1.00  0.00           O  \\nATOM    198  C5'  DT A   7       8.196   0.539  -6.023  1.00  0.00           C  \\nATOM    199  C4'  DT A   7       6.841   1.074  -6.496  1.00  0.00           C  \\nATOM    200  O4'  DT A   7       5.858   0.807  -5.506  1.00  0.00           O  \\nATOM    201  C3'  DT A   7       6.839   2.587  -6.766  1.00  0.00           C  \\nATOM    202  O3'  DT A   7       6.155   2.808  -7.992  1.00  0.00           O  \\nATOM    203  C2'  DT A   7       6.075   3.121  -5.556  1.00  0.00           C  \\nATOM    204  C1'  DT A   7       5.086   1.977  -5.335  1.00  0.00           C  \\nATOM    205  N1   DT A   7       4.411   1.977  -4.004  1.00  0.00           N  \\nATOM    206  C2   DT A   7       3.021   1.846  -3.966  1.00  0.00           C  \\nATOM    207  O2   DT A   7       2.314   1.799  -4.970  1.00  0.00           O  \\nATOM    208  N3   DT A   7       2.445   1.746  -2.708  1.00  0.00           N  \\nATOM    209  C4   DT A   7       3.120   1.734  -1.498  1.00  0.00           C  \\nATOM    210  O4   DT A   7       2.496   1.594  -0.449  1.00  0.00           O  \\nATOM    211  C5   DT A   7       4.558   1.885  -1.617  1.00  0.00           C  \\nATOM    212  C7   DT A   7       5.422   1.939  -0.371  1.00  0.00           C  \\nATOM    213  C6   DT A   7       5.142   1.992  -2.837  1.00  0.00           C  \\nATOM    214  H5'  DT A   7       8.155  -0.551  -6.034  1.00  0.00           H  \\nATOM    215 H5''  DT A   7       8.978   0.859  -6.713  1.00  0.00           H  \\nATOM    216  H4'  DT A   7       6.575   0.544  -7.412  1.00  0.00           H  \\nATOM    217  H3'  DT A   7       7.851   2.993  -6.814  1.00  0.00           H  \\nATOM    218  H2'  DT A   7       6.781   3.227  -4.736  1.00  0.00           H  \\nATOM    219 H2''  DT A   7       5.578   4.072  -5.733  1.00  0.00           H  \\nATOM    220  H1'  DT A   7       4.370   2.022  -6.154  1.00  0.00           H  \\nATOM    221  H3   DT A   7       1.442   1.618  -2.682  1.00  0.00           H  \\nATOM    222  H71  DT A   7       4.825   2.215   0.499  1.00  0.00           H  \\nATOM    223  H72  DT A   7       5.875   0.967  -0.192  1.00  0.00           H  \\nATOM    224  H73  DT A   7       6.214   2.678  -0.498  1.00  0.00           H  \\nATOM    225  H6   DT A   7       6.217   2.067  -2.881  1.00  0.00           H  \\nATOM    226  P    DG A   8       5.963   4.274  -8.646  1.00  0.00           P  \\nATOM    227  OP1  DG A   8       5.644   4.099 -10.080  1.00  0.00           O  \\nATOM    228  OP2  DG A   8       7.110   5.120  -8.246  1.00  0.00           O  \\nATOM    229  O5'  DG A   8       4.646   4.830  -7.903  1.00  0.00           O  \\nATOM    230  C5'  DG A   8       3.371   4.257  -8.123  1.00  0.00           C  \\nATOM    231  C4'  DG A   8       2.339   4.891  -7.185  1.00  0.00           C  \\nATOM    232  O4'  DG A   8       2.698   4.651  -5.832  1.00  0.00           O  \\nATOM    233  C3'  DG A   8       2.247   6.411  -7.372  1.00  0.00           C  \\nATOM    234  O3'  DG A   8       0.879   6.770  -7.293  1.00  0.00           O  \\nATOM    235  C2'  DG A   8       3.050   6.921  -6.174  1.00  0.00           C  \\nATOM    236  C1'  DG A   8       2.659   5.879  -5.130  1.00  0.00           C  \\nATOM    237  N9   DG A   8       3.547   5.807  -3.945  1.00  0.00           N  \\nATOM    238  C8   DG A   8       4.888   6.082  -3.843  1.00  0.00           C  \\nATOM    239  N7   DG A   8       5.390   5.876  -2.658  1.00  0.00           N  \\nATOM    240  C5   DG A   8       4.305   5.439  -1.907  1.00  0.00           C  \\nATOM    241  C6   DG A   8       4.229   5.086  -0.526  1.00  0.00           C  \\nATOM    242  O6   DG A   8       5.128   5.089   0.311  1.00  0.00           O  \\nATOM    243  N1   DG A   8       2.946   4.704  -0.143  1.00  0.00           N  \\nATOM    244  C2   DG A   8       1.856   4.677  -0.992  1.00  0.00           C  \\nATOM    245  N2   DG A   8       0.685   4.316  -0.461  1.00  0.00           N  \\nATOM    246  N3   DG A   8       1.926   5.007  -2.291  1.00  0.00           N  \\nATOM    247  C4   DG A   8       3.174   5.384  -2.687  1.00  0.00           C  \\nATOM    248  H5'  DG A   8       3.402   3.183  -7.939  1.00  0.00           H  \\nATOM    249 H5''  DG A   8       3.067   4.426  -9.157  1.00  0.00           H  \\nATOM    250  H4'  DG A   8       1.370   4.430  -7.385  1.00  0.00           H  \\nATOM    251  H3'  DG A   8       2.676   6.727  -8.324  1.00  0.00           H  \\nATOM    252  H2'  DG A   8       4.113   6.867  -6.415  1.00  0.00           H  \\nATOM    253 H2''  DG A   8       2.783   7.932  -5.866  1.00  0.00           H  \\nATOM    254  H1'  DG A   8       1.641   6.111  -4.807  1.00  0.00           H  \\nATOM    255  H8   DG A   8       5.491   6.434  -4.667  1.00  0.00           H  \\nATOM    256  H1   DG A   8       2.822   4.440   0.822  1.00  0.00           H  \\nATOM    257  H21  DG A   8       0.614   4.079   0.517  1.00  0.00           H  \\nATOM    258  H22  DG A   8      -0.150   4.335  -1.037  1.00  0.00           H  \\nATOM    259  P    DG A   9       0.330   8.218  -7.744  1.00  0.00           P  \\nATOM    260  OP1  DG A   9      -0.439   8.061  -8.998  1.00  0.00           O  \\nATOM    261  OP2  DG A   9       1.438   9.197  -7.672  1.00  0.00           O  \\nATOM    262  O5'  DG A   9      -0.690   8.535  -6.543  1.00  0.00           O  \\nATOM    263  C5'  DG A   9      -2.012   8.031  -6.519  1.00  0.00           C  \\nATOM    264  C4'  DG A   9      -2.566   8.145  -5.094  1.00  0.00           C  \\nATOM    265  O4'  DG A   9      -1.825   7.353  -4.184  1.00  0.00           O  \\nATOM    266  C3'  DG A   9      -2.506   9.580  -4.566  1.00  0.00           C  \\nATOM    267  O3'  DG A   9      -3.619  10.313  -5.044  1.00  0.00           O  \\nATOM    268  C2'  DG A   9      -2.497   9.337  -3.063  1.00  0.00           C  \\nATOM    269  C1'  DG A   9      -1.771   7.994  -2.919  1.00  0.00           C  \\nATOM    270  N9   DG A   9      -0.338   8.108  -2.538  1.00  0.00           N  \\nATOM    271  C8   DG A   9       0.746   8.359  -3.343  1.00  0.00           C  \\nATOM    272  N7   DG A   9       1.893   8.366  -2.727  1.00  0.00           N  \\nATOM    273  C5   DG A   9       1.561   8.070  -1.412  1.00  0.00           C  \\nATOM    274  C6   DG A   9       2.401   7.935  -0.265  1.00  0.00           C  \\nATOM    275  O6   DG A   9       3.614   8.107  -0.190  1.00  0.00           O  \\nATOM    276  N1   DG A   9       1.693   7.555   0.872  1.00  0.00           N  \\nATOM    277  C2   DG A   9       0.328   7.335   0.900  1.00  0.00           C  \\nATOM    278  N2   DG A   9      -0.210   6.988   2.072  1.00  0.00           N  \\nATOM    279  N3   DG A   9      -0.460   7.469  -0.177  1.00  0.00           N  \\nATOM    280  C4   DG A   9       0.207   7.851  -1.298  1.00  0.00           C  \\nATOM    281  H5'  DG A   9      -2.045   6.989  -6.838  1.00  0.00           H  \\nATOM    282 H5''  DG A   9      -2.628   8.623  -7.197  1.00  0.00           H  \\nATOM    283  H4'  DG A   9      -3.595   7.780  -5.082  1.00  0.00           H  \\nATOM    284  H3'  DG A   9      -1.573  10.062  -4.854  1.00  0.00           H  \\nATOM    285  H2'  DG A   9      -2.015  10.137  -2.518  1.00  0.00           H  \\nATOM    286 H2''  DG A   9      -3.515   9.237  -2.692  1.00  0.00           H  \\nATOM    287  H1'  DG A   9      -2.311   7.429  -2.161  1.00  0.00           H  \\nATOM    288  H8   DG A   9       0.678   8.518  -4.403  1.00  0.00           H  \\nATOM    289  H1   DG A   9       2.224   7.458   1.723  1.00  0.00           H  \\nATOM    290  H21  DG A   9       0.352   6.926   2.907  1.00  0.00           H  \\nATOM    291  H22  DG A   9      -1.215   6.851   2.123  1.00  0.00           H  \\nATOM    292  P    DA A  10      -3.768  11.908  -4.844  1.00  0.00           P  \\nATOM    293  OP1  DA A  10      -4.648  12.420  -5.917  1.00  0.00           O  \\nATOM    294  OP2  DA A  10      -2.422  12.498  -4.662  1.00  0.00           O  \\nATOM    295  O5'  DA A  10      -4.564  12.045  -3.452  1.00  0.00           O  \\nATOM    296  C5'  DA A  10      -5.795  11.380  -3.232  1.00  0.00           C  \\nATOM    297  C4'  DA A  10      -6.574  12.044  -2.090  1.00  0.00           C  \\nATOM    298  O4'  DA A  10      -5.681  12.349  -1.028  1.00  0.00           O  \\nATOM    299  C3'  DA A  10      -7.245  13.358  -2.515  1.00  0.00           C  \\nATOM    300  O3'  DA A  10      -8.433  13.513  -1.751  1.00  0.00           O  \\nATOM    301  C2'  DA A  10      -6.168  14.372  -2.152  1.00  0.00           C  \\nATOM    302  C1'  DA A  10      -5.636  13.757  -0.862  1.00  0.00           C  \\nATOM    303  N9   DA A  10      -4.269  14.207  -0.519  1.00  0.00           N  \\nATOM    304  C8   DA A  10      -3.167  14.362  -1.327  1.00  0.00           C  \\nATOM    305  N7   DA A  10      -2.080  14.697  -0.690  1.00  0.00           N  \\nATOM    306  C5   DA A  10      -2.485  14.770   0.637  1.00  0.00           C  \\nATOM    307  C6   DA A  10      -1.807  15.061   1.839  1.00  0.00           C  \\nATOM    308  N6   DA A  10      -0.504  15.345   1.885  1.00  0.00           N  \\nATOM    309  N1   DA A  10      -2.508  15.070   2.989  1.00  0.00           N  \\nATOM    310  C2   DA A  10      -3.806  14.776   2.951  1.00  0.00           C  \\nATOM    311  N3   DA A  10      -4.552  14.464   1.896  1.00  0.00           N  \\nATOM    312  C4   DA A  10      -3.822  14.485   0.749  1.00  0.00           C  \\nATOM    313  H5'  DA A  10      -5.589  10.343  -2.966  1.00  0.00           H  \\nATOM    314 H5''  DA A  10      -6.412  11.388  -4.132  1.00  0.00           H  \\nATOM    315  H4'  DA A  10      -7.333  11.339  -1.748  1.00  0.00           H  \\nATOM    316  H3'  DA A  10      -7.475  13.385  -3.581  1.00  0.00           H  \\nATOM    317  H2'  DA A  10      -5.414  14.386  -2.937  1.00  0.00           H  \\nATOM    318 H2''  DA A  10      -6.554  15.376  -2.003  1.00  0.00           H  \\nATOM    319  H1'  DA A  10      -6.323  14.024  -0.061  1.00  0.00           H  \\nATOM    320  H8   DA A  10      -3.184  14.211  -2.396  1.00  0.00           H  \\nATOM    321  H61  DA A  10      -0.075  15.609   2.761  1.00  0.00           H  \\nATOM    322  H62  DA A  10       0.069  15.235   1.057  1.00  0.00           H  \\nATOM    323  H2   DA A  10      -4.325  14.780   3.896  1.00  0.00           H  \\nATOM    324  P    DA A  11      -9.394  14.810  -1.869  1.00  0.00           P  \\nATOM    325  OP1  DA A  11     -10.740  14.433  -1.386  1.00  0.00           O  \\nATOM    326  OP2  DA A  11      -9.234  15.394  -3.219  1.00  0.00           O  \\nATOM    327  O5'  DA A  11      -8.742  15.826  -0.800  1.00  0.00           O  \\nATOM    328  C5'  DA A  11      -8.827  15.586   0.592  1.00  0.00           C  \\nATOM    329  C4'  DA A  11      -7.934  16.582   1.339  1.00  0.00           C  \\nATOM    330  O4'  DA A  11      -6.600  16.494   0.866  1.00  0.00           O  \\nATOM    331  C3'  DA A  11      -8.405  18.029   1.139  1.00  0.00           C  \\nATOM    332  O3'  DA A  11      -8.526  18.646   2.413  1.00  0.00           O  \\nATOM    333  C2'  DA A  11      -7.288  18.627   0.284  1.00  0.00           C  \\nATOM    334  C1'  DA A  11      -6.088  17.802   0.739  1.00  0.00           C  \\nATOM    335  N9   DA A  11      -4.945  17.818  -0.203  1.00  0.00           N  \\nATOM    336  C8   DA A  11      -4.941  17.661  -1.568  1.00  0.00           C  \\nATOM    337  N7   DA A  11      -3.754  17.670  -2.108  1.00  0.00           N  \\nATOM    338  C5   DA A  11      -2.903  17.845  -1.024  1.00  0.00           C  \\nATOM    339  C6   DA A  11      -1.500  17.949  -0.911  1.00  0.00           C  \\nATOM    340  N6   DA A  11      -0.681  17.883  -1.966  1.00  0.00           N  \\nATOM    341  N1   DA A  11      -0.968  18.143   0.310  1.00  0.00           N  \\nATOM    342  C2   DA A  11      -1.779  18.218   1.364  1.00  0.00           C  \\nATOM    343  N3   DA A  11      -3.106  18.128   1.391  1.00  0.00           N  \\nATOM    344  C4   DA A  11      -3.618  17.939   0.144  1.00  0.00           C  \\nATOM    345  H5'  DA A  11      -8.503  14.572   0.827  1.00  0.00           H  \\nATOM    346 H5''  DA A  11      -9.859  15.703   0.925  1.00  0.00           H  \\nATOM    347  H4'  DA A  11      -7.947  16.329   2.400  1.00  0.00           H  \\nATOM    348  H3'  DA A  11      -9.358  18.055   0.610  1.00  0.00           H  \\nATOM    349  H2'  DA A  11      -7.517  18.432  -0.763  1.00  0.00           H  \\nATOM    350 H2''  DA A  11      -7.149  19.695   0.441  1.00  0.00           H  \\nATOM    351  H1'  DA A  11      -5.765  18.170   1.712  1.00  0.00           H  \\nATOM    352  H8   DA A  11      -5.838  17.538  -2.157  1.00  0.00           H  \\nATOM    353  H61  DA A  11       0.317  17.992  -1.843  1.00  0.00           H  \\nATOM    354  H62  DA A  11      -1.069  17.748  -2.888  1.00  0.00           H  \\nATOM    355  H2   DA A  11      -1.298  18.375   2.317  1.00  0.00           H  \\nATOM    356  P    DC A  12      -9.152  20.124   2.613  1.00  0.00           P  \\nATOM    357  OP1  DC A  12     -10.116  20.076   3.734  1.00  0.00           O  \\nATOM    358  OP2  DC A  12      -9.578  20.645   1.295  1.00  0.00           O  \\nATOM    359  O5'  DC A  12      -7.866  20.968   3.088  1.00  0.00           O  \\nATOM    360  C5'  DC A  12      -7.409  20.939   4.430  1.00  0.00           C  \\nATOM    361  C4'  DC A  12      -5.979  21.486   4.471  1.00  0.00           C  \\nATOM    362  O4'  DC A  12      -5.178  20.822   3.513  1.00  0.00           O  \\nATOM    363  C3'  DC A  12      -5.913  22.975   4.122  1.00  0.00           C  \\nATOM    364  O3'  DC A  12      -5.844  23.770   5.287  1.00  0.00           O  \\nATOM    365  C2'  DC A  12      -4.638  23.102   3.292  1.00  0.00           C  \\nATOM    366  C1'  DC A  12      -4.097  21.674   3.191  1.00  0.00           C  \\nATOM    367  N1   DC A  12      -3.608  21.408   1.811  1.00  0.00           N  \\nATOM    368  C2   DC A  12      -2.235  21.410   1.557  1.00  0.00           C  \\nATOM    369  O2   DC A  12      -1.416  21.525   2.466  1.00  0.00           O  \\nATOM    370  N3   DC A  12      -1.814  21.268   0.267  1.00  0.00           N  \\nATOM    371  C4   DC A  12      -2.689  21.136  -0.741  1.00  0.00           C  \\nATOM    372  N4   DC A  12      -2.217  20.980  -1.981  1.00  0.00           N  \\nATOM    373  C5   DC A  12      -4.105  21.125  -0.502  1.00  0.00           C  \\nATOM    374  C6   DC A  12      -4.508  21.262   0.783  1.00  0.00           C  \\nATOM    375  H5'  DC A  12      -7.415  19.923   4.828  1.00  0.00           H  \\nATOM    376 H5''  DC A  12      -8.057  21.555   5.054  1.00  0.00           H  \\nATOM    377  H4'  DC A  12      -5.546  21.325   5.460  1.00  0.00           H  \\nATOM    378  H3'  DC A  12      -6.768  23.275   3.514  1.00  0.00           H  \\nATOM    379 HO3'  DC A  12      -6.657  23.652   5.783  1.00  0.00           H  \\nATOM    380  H2'  DC A  12      -4.895  23.497   2.309  1.00  0.00           H  \\nATOM    381 H2''  DC A  12      -3.902  23.753   3.761  1.00  0.00           H  \\nATOM    382  H1'  DC A  12      -3.331  21.535   3.953  1.00  0.00           H  \\nATOM    383  H41  DC A  12      -1.216  20.953  -2.129  1.00  0.00           H  \\nATOM    384  H42  DC A  12      -2.850  20.847  -2.756  1.00  0.00           H  \\nATOM    385  H5   DC A  12      -4.842  21.010  -1.284  1.00  0.00           H  \\nATOM    386  H6   DC A  12      -5.562  21.282   1.013  1.00  0.00           H  \\nTER     387       DC A  12                                                      \\nATOM    388  O5'  DG B  13       8.692  22.125  -1.628  1.00  0.00           O  \\nATOM    389  C5'  DG B  13       8.994  22.802  -0.426  1.00  0.00           C  \\nATOM    390  C4'  DG B  13       8.201  22.220   0.750  1.00  0.00           C  \\nATOM    391  O4'  DG B  13       6.811  22.374   0.501  1.00  0.00           O  \\nATOM    392  C3'  DG B  13       8.479  20.725   0.965  1.00  0.00           C  \\nATOM    393  O3'  DG B  13       8.592  20.475   2.357  1.00  0.00           O  \\nATOM    394  C2'  DG B  13       7.237  20.084   0.356  1.00  0.00           C  \\nATOM    395  C1'  DG B  13       6.171  21.124   0.683  1.00  0.00           C  \\nATOM    396  N9   DG B  13       4.994  21.017  -0.212  1.00  0.00           N  \\nATOM    397  C8   DG B  13       4.951  21.135  -1.579  1.00  0.00           C  \\nATOM    398  N7   DG B  13       3.756  21.031  -2.089  1.00  0.00           N  \\nATOM    399  C5   DG B  13       2.936  20.853  -0.981  1.00  0.00           C  \\nATOM    400  C6   DG B  13       1.518  20.705  -0.906  1.00  0.00           C  \\nATOM    401  O6   DG B  13       0.711  20.664  -1.832  1.00  0.00           O  \\nATOM    402  N1   DG B  13       1.063  20.603   0.405  1.00  0.00           N  \\nATOM    403  C2   DG B  13       1.884  20.606   1.519  1.00  0.00           C  \\nATOM    404  N2   DG B  13       1.294  20.495   2.713  1.00  0.00           N  \\nATOM    405  N3   DG B  13       3.220  20.717   1.448  1.00  0.00           N  \\nATOM    406  C4   DG B  13       3.684  20.847   0.175  1.00  0.00           C  \\nATOM    407  H5'  DG B  13       8.746  23.858  -0.542  1.00  0.00           H  \\nATOM    408 H5''  DG B  13      10.062  22.711  -0.224  1.00  0.00           H  \\nATOM    409  H4'  DG B  13       8.472  22.781   1.647  1.00  0.00           H  \\nATOM    410  H3'  DG B  13       9.390  20.408   0.453  1.00  0.00           H  \\nATOM    411  H2'  DG B  13       7.377  20.000  -0.722  1.00  0.00           H  \\nATOM    412 H2''  DG B  13       7.005  19.107   0.775  1.00  0.00           H  \\nATOM    413  H1'  DG B  13       5.878  21.020   1.728  1.00  0.00           H  \\nATOM    414  H8   DG B  13       5.829  21.295  -2.187  1.00  0.00           H  \\nATOM    415  H1   DG B  13       0.061  20.531   0.528  1.00  0.00           H  \\nATOM    416  H21  DG B  13       0.285  20.475   2.782  1.00  0.00           H  \\nATOM    417  H22  DG B  13       1.862  20.474   3.547  1.00  0.00           H  \\nATOM    418 HO5'  DG B  13       9.205  22.519  -2.337  1.00  0.00           H  \\nATOM    419  P    DT B  14       8.934  19.011   2.956  1.00  0.00           P  \\nATOM    420  OP1  DT B  14       9.823  19.191   4.125  1.00  0.00           O  \\nATOM    421  OP2  DT B  14       9.354  18.131   1.842  1.00  0.00           O  \\nATOM    422  O5'  DT B  14       7.499  18.499   3.486  1.00  0.00           O  \\nATOM    423  C5'  DT B  14       6.884  19.096   4.612  1.00  0.00           C  \\nATOM    424  C4'  DT B  14       5.519  18.461   4.891  1.00  0.00           C  \\nATOM    425  O4'  DT B  14       4.686  18.604   3.750  1.00  0.00           O  \\nATOM    426  C3'  DT B  14       5.603  16.962   5.226  1.00  0.00           C  \\nATOM    427  O3'  DT B  14       4.750  16.719   6.336  1.00  0.00           O  \\nATOM    428  C2'  DT B  14       5.097  16.325   3.932  1.00  0.00           C  \\nATOM    429  C1'  DT B  14       4.057  17.362   3.513  1.00  0.00           C  \\nATOM    430  N1   DT B  14       3.606  17.278   2.096  1.00  0.00           N  \\nATOM    431  C2   DT B  14       2.237  17.366   1.835  1.00  0.00           C  \\nATOM    432  O2   DT B  14       1.379  17.386   2.714  1.00  0.00           O  \\nATOM    433  N3   DT B  14       1.869  17.464   0.502  1.00  0.00           N  \\nATOM    434  C4   DT B  14       2.730  17.523  -0.582  1.00  0.00           C  \\nATOM    435  O4   DT B  14       2.281  17.673  -1.715  1.00  0.00           O  \\nATOM    436  C5   DT B  14       4.133  17.400  -0.235  1.00  0.00           C  \\nATOM    437  C7   DT B  14       5.185  17.396  -1.328  1.00  0.00           C  \\nATOM    438  C6   DT B  14       4.516  17.289   1.062  1.00  0.00           C  \\nATOM    439  H5'  DT B  14       6.743  20.162   4.431  1.00  0.00           H  \\nATOM    440 H5''  DT B  14       7.517  18.970   5.492  1.00  0.00           H  \\nATOM    441  H4'  DT B  14       5.064  18.997   5.726  1.00  0.00           H  \\nATOM    442  H3'  DT B  14       6.624  16.653   5.456  1.00  0.00           H  \\nATOM    443  H2'  DT B  14       5.932  16.270   3.237  1.00  0.00           H  \\nATOM    444 H2''  DT B  14       4.669  15.334   4.062  1.00  0.00           H  \\nATOM    445  H1'  DT B  14       3.224  17.267   4.210  1.00  0.00           H  \\nATOM    446  H3   DT B  14       0.880  17.558   0.315  1.00  0.00           H  \\nATOM    447  H71  DT B  14       4.756  17.043  -2.266  1.00  0.00           H  \\nATOM    448  H72  DT B  14       5.565  18.405  -1.473  1.00  0.00           H  \\nATOM    449  H73  DT B  14       6.011  16.740  -1.054  1.00  0.00           H  \\nATOM    450  H6   DT B  14       5.569  17.235   1.281  1.00  0.00           H  \\nATOM    451  P    DG B  15       4.549  15.254   6.990  1.00  0.00           P  \\nATOM    452  OP1  DG B  15       4.020  15.432   8.361  1.00  0.00           O  \\nATOM    453  OP2  DG B  15       5.783  14.468   6.774  1.00  0.00           O  \\nATOM    454  O5'  DG B  15       3.381  14.619   6.080  1.00  0.00           O  \\nATOM    455  C5'  DG B  15       2.063  15.133   6.103  1.00  0.00           C  \\nATOM    456  C4'  DG B  15       1.210  14.441   5.036  1.00  0.00           C  \\nATOM    457  O4'  DG B  15       1.766  14.658   3.749  1.00  0.00           O  \\nATOM    458  C3'  DG B  15       1.137  12.925   5.254  1.00  0.00           C  \\nATOM    459  O3'  DG B  15      -0.200  12.527   5.011  1.00  0.00           O  \\nATOM    460  C2'  DG B  15       2.104  12.403   4.190  1.00  0.00           C  \\nATOM    461  C1'  DG B  15       1.853  13.416   3.078  1.00  0.00           C  \\nATOM    462  N9   DG B  15       2.902  13.491   2.034  1.00  0.00           N  \\nATOM    463  C8   DG B  15       4.250  13.246   2.131  1.00  0.00           C  \\nATOM    464  N7   DG B  15       4.913  13.452   1.028  1.00  0.00           N  \\nATOM    465  C5   DG B  15       3.939  13.858   0.124  1.00  0.00           C  \\nATOM    466  C6   DG B  15       4.056  14.199  -1.258  1.00  0.00           C  \\nATOM    467  O6   DG B  15       5.066  14.207  -1.957  1.00  0.00           O  \\nATOM    468  N1   DG B  15       2.835  14.554  -1.824  1.00  0.00           N  \\nATOM    469  C2   DG B  15       1.633  14.569  -1.140  1.00  0.00           C  \\nATOM    470  N2   DG B  15       0.548  14.927  -1.832  1.00  0.00           N  \\nATOM    471  N3   DG B  15       1.521  14.247   0.158  1.00  0.00           N  \\nATOM    472  C4   DG B  15       2.706  13.896   0.732  1.00  0.00           C  \\nATOM    473  H5'  DG B  15       2.071  16.205   5.906  1.00  0.00           H  \\nATOM    474 H5''  DG B  15       1.619  14.963   7.085  1.00  0.00           H  \\nATOM    475  H4'  DG B  15       0.208  14.873   5.072  1.00  0.00           H  \\nATOM    476  H3'  DG B  15       1.450  12.654   6.263  1.00  0.00           H  \\nATOM    477  H2'  DG B  15       3.120  12.479   4.576  1.00  0.00           H  \\nATOM    478 H2''  DG B  15       1.898  11.382   3.872  1.00  0.00           H  \\nATOM    479  H1'  DG B  15       0.900  13.151   2.615  1.00  0.00           H  \\nATOM    480  H8   DG B  15       4.735  12.918   3.038  1.00  0.00           H  \\nATOM    481  H1   DG B  15       2.846  14.814  -2.798  1.00  0.00           H  \\nATOM    482  H21  DG B  15       0.616  15.164  -2.810  1.00  0.00           H  \\nATOM    483  H22  DG B  15      -0.361  14.900  -1.383  1.00  0.00           H  \\nATOM    484  P    DG B  16      -0.764  11.068   5.404  1.00  0.00           P  \\nATOM    485  OP1  DG B  16      -1.722  11.216   6.523  1.00  0.00           O  \\nATOM    486  OP2  DG B  16       0.371  10.125   5.520  1.00  0.00           O  \\nATOM    487  O5'  DG B  16      -1.578  10.705   4.067  1.00  0.00           O  \\nATOM    488  C5'  DG B  16      -2.876  11.208   3.813  1.00  0.00           C  \\nATOM    489  C4'  DG B  16      -3.169  11.119   2.311  1.00  0.00           C  \\nATOM    490  O4'  DG B  16      -2.280  11.917   1.551  1.00  0.00           O  \\nATOM    491  C3'  DG B  16      -3.020   9.694   1.781  1.00  0.00           C  \\nATOM    492  O3'  DG B  16      -4.198   8.962   2.067  1.00  0.00           O  \\nATOM    493  C2'  DG B  16      -2.782   9.959   0.301  1.00  0.00           C  \\nATOM    494  C1'  DG B  16      -2.023  11.290   0.303  1.00  0.00           C  \\nATOM    495  N9   DG B  16      -0.551  11.139   0.173  1.00  0.00           N  \\nATOM    496  C8   DG B  16       0.368  10.841   1.147  1.00  0.00           C  \\nATOM    497  N7   DG B  16       1.604  10.808   0.735  1.00  0.00           N  \\nATOM    498  C5   DG B  16       1.510  11.123  -0.614  1.00  0.00           C  \\nATOM    499  C6   DG B  16       2.534  11.240  -1.603  1.00  0.00           C  \\nATOM    500  O6   DG B  16       3.740  11.050  -1.471  1.00  0.00           O  \\nATOM    501  N1   DG B  16       2.035  11.623  -2.845  1.00  0.00           N  \\nATOM    502  C2   DG B  16       0.701  11.877  -3.102  1.00  0.00           C  \\nATOM    503  N2   DG B  16       0.368  12.229  -4.348  1.00  0.00           N  \\nATOM    504  N3   DG B  16      -0.258  11.773  -2.170  1.00  0.00           N  \\nATOM    505  C4   DG B  16       0.202  11.377  -0.955  1.00  0.00           C  \\nATOM    506  H5'  DG B  16      -2.970  12.244   4.140  1.00  0.00           H  \\nATOM    507 H5''  DG B  16      -3.600  10.605   4.362  1.00  0.00           H  \\nATOM    508  H4'  DG B  16      -4.180  11.488   2.125  1.00  0.00           H  \\nATOM    509  H3'  DG B  16      -2.132   9.221   2.201  1.00  0.00           H  \\nATOM    510  H2'  DG B  16      -2.231   9.161  -0.180  1.00  0.00           H  \\nATOM    511 H2''  DG B  16      -3.729  10.087  -0.218  1.00  0.00           H  \\nATOM    512  H1'  DG B  16      -2.411  11.875  -0.530  1.00  0.00           H  \\nATOM    513  H8   DG B  16       0.105  10.668   2.176  1.00  0.00           H  \\nATOM    514  H1   DG B  16       2.705  11.704  -3.595  1.00  0.00           H  \\nATOM    515  H21  DG B  16       1.060  12.268  -5.081  1.00  0.00           H  \\nATOM    516  H22  DG B  16      -0.613  12.385  -4.562  1.00  0.00           H  \\nATOM    517  P    DA B  17      -4.335   7.377   1.798  1.00  0.00           P  \\nATOM    518  OP1  DA B  17      -5.461   6.874   2.615  1.00  0.00           O  \\nATOM    519  OP2  DA B  17      -3.000   6.754   1.940  1.00  0.00           O  \\nATOM    520  O5'  DA B  17      -4.768   7.278   0.251  1.00  0.00           O  \\nATOM    521  C5'  DA B  17      -5.940   7.904  -0.242  1.00  0.00           C  \\nATOM    522  C4'  DA B  17      -6.463   7.161  -1.478  1.00  0.00           C  \\nATOM    523  O4'  DA B  17      -5.368   6.847  -2.328  1.00  0.00           O  \\nATOM    524  C3'  DA B  17      -7.159   5.840  -1.116  1.00  0.00           C  \\nATOM    525  O3'  DA B  17      -8.183   5.587  -2.069  1.00  0.00           O  \\nATOM    526  C2'  DA B  17      -6.000   4.859  -1.228  1.00  0.00           C  \\nATOM    527  C1'  DA B  17      -5.273   5.437  -2.438  1.00  0.00           C  \\nATOM    528  N9   DA B  17      -3.863   4.998  -2.544  1.00  0.00           N  \\nATOM    529  C8   DA B  17      -2.903   4.877  -1.566  1.00  0.00           C  \\nATOM    530  N7   DA B  17      -1.726   4.537  -2.013  1.00  0.00           N  \\nATOM    531  C5   DA B  17      -1.913   4.421  -3.384  1.00  0.00           C  \\nATOM    532  C6   DA B  17      -1.051   4.100  -4.455  1.00  0.00           C  \\nATOM    533  N6   DA B  17       0.243   3.819  -4.289  1.00  0.00           N  \\nATOM    534  N1   DA B  17      -1.560   4.059  -5.701  1.00  0.00           N  \\nATOM    535  C2   DA B  17      -2.848   4.344  -5.879  1.00  0.00           C  \\nATOM    536  N3   DA B  17      -3.755   4.674  -4.965  1.00  0.00           N  \\nATOM    537  C4   DA B  17      -3.217   4.689  -3.716  1.00  0.00           C  \\nATOM    538  H5'  DA B  17      -5.702   8.931  -0.517  1.00  0.00           H  \\nATOM    539 H5''  DA B  17      -6.727   7.926   0.514  1.00  0.00           H  \\nATOM    540  H4'  DA B  17      -7.165   7.816  -1.997  1.00  0.00           H  \\nATOM    541  H3'  DA B  17      -7.579   5.858  -0.109  1.00  0.00           H  \\nATOM    542  H2'  DA B  17      -5.394   4.922  -0.325  1.00  0.00           H  \\nATOM    543 H2''  DA B  17      -6.320   3.833  -1.385  1.00  0.00           H  \\nATOM    544  H1'  DA B  17      -5.820   5.137  -3.330  1.00  0.00           H  \\nATOM    545  H8   DA B  17      -3.088   5.063  -0.519  1.00  0.00           H  \\nATOM    546  H61  DA B  17       0.806   3.543  -5.082  1.00  0.00           H  \\nATOM    547  H62  DA B  17       0.680   3.957  -3.385  1.00  0.00           H  \\nATOM    548  H2   DA B  17      -3.208   4.315  -6.895  1.00  0.00           H  \\nATOM    549  P    DA B  18      -9.096   4.248  -2.047  1.00  0.00           P  \\nATOM    550  OP1  DA B  18     -10.379   4.557  -2.718  1.00  0.00           O  \\nATOM    551  OP2  DA B  18      -9.100   3.703  -0.672  1.00  0.00           O  \\nATOM    552  O5'  DA B  18      -8.271   3.230  -2.990  1.00  0.00           O  \\nATOM    553  C5'  DA B  18      -8.181   3.435  -4.388  1.00  0.00           C  \\nATOM    554  C4'  DA B  18      -7.174   2.461  -5.006  1.00  0.00           C  \\nATOM    555  O4'  DA B  18      -5.925   2.601  -4.346  1.00  0.00           O  \\nATOM    556  C3'  DA B  18      -7.604   0.989  -4.895  1.00  0.00           C  \\nATOM    557  O3'  DA B  18      -7.484   0.375  -6.170  1.00  0.00           O  \\nATOM    558  C2'  DA B  18      -6.601   0.434  -3.886  1.00  0.00           C  \\nATOM    559  C1'  DA B  18      -5.382   1.315  -4.152  1.00  0.00           C  \\nATOM    560  N9   DA B  18      -4.402   1.326  -3.039  1.00  0.00           N  \\nATOM    561  C8   DA B  18      -4.614   1.542  -1.697  1.00  0.00           C  \\nATOM    562  N7   DA B  18      -3.529   1.560  -0.976  1.00  0.00           N  \\nATOM    563  C5   DA B  18      -2.519   1.342  -1.902  1.00  0.00           C  \\nATOM    564  C6   DA B  18      -1.117   1.255  -1.784  1.00  0.00           C  \\nATOM    565  N6   DA B  18      -0.476   1.392  -0.618  1.00  0.00           N  \\nATOM    566  N1   DA B  18      -0.395   1.024  -2.896  1.00  0.00           N  \\nATOM    567  C2   DA B  18      -1.026   0.881  -4.059  1.00  0.00           C  \\nATOM    568  N3   DA B  18      -2.333   0.946  -4.302  1.00  0.00           N  \\nATOM    569  C4   DA B  18      -3.039   1.185  -3.163  1.00  0.00           C  \\nATOM    570  H5'  DA B  18      -7.845   4.453  -4.590  1.00  0.00           H  \\nATOM    571 H5''  DA B  18      -9.158   3.290  -4.850  1.00  0.00           H  \\nATOM    572  H4'  DA B  18      -7.051   2.726  -6.058  1.00  0.00           H  \\nATOM    573  H3'  DA B  18      -8.627   0.896  -4.526  1.00  0.00           H  \\nATOM    574  H2'  DA B  18      -7.010   0.605  -2.891  1.00  0.00           H  \\nATOM    575 H2''  DA B  18      -6.382  -0.623  -4.025  1.00  0.00           H  \\nATOM    576  H1'  DA B  18      -4.899   0.992  -5.075  1.00  0.00           H  \\nATOM    577  H8   DA B  18      -5.589   1.678  -1.256  1.00  0.00           H  \\nATOM    578  H61  DA B  18       0.532   1.327  -0.581  1.00  0.00           H  \\nATOM    579  H62  DA B  18      -1.009   1.569   0.221  1.00  0.00           H  \\nATOM    580  H2   DA B  18      -0.392   0.704  -4.915  1.00  0.00           H  \\nATOM    581  P    DT B  19      -7.936  -1.153  -6.457  1.00  0.00           P  \\nATOM    582  OP1  DT B  19      -8.773  -1.166  -7.678  1.00  0.00           O  \\nATOM    583  OP2  DT B  19      -8.456  -1.744  -5.203  1.00  0.00           O  \\nATOM    584  O5'  DT B  19      -6.529  -1.863  -6.800  1.00  0.00           O  \\nATOM    585  C5'  DT B  19      -5.855  -1.593  -8.016  1.00  0.00           C  \\nATOM    586  C4'  DT B  19      -4.433  -2.160  -7.984  1.00  0.00           C  \\nATOM    587  O4'  DT B  19      -3.781  -1.731  -6.798  1.00  0.00           O  \\nATOM    588  C3'  DT B  19      -4.383  -3.697  -8.021  1.00  0.00           C  \\nATOM    589  O3'  DT B  19      -3.372  -4.076  -8.945  1.00  0.00           O  \\nATOM    590  C2'  DT B  19      -4.022  -4.033  -6.575  1.00  0.00           C  \\nATOM    591  C1'  DT B  19      -3.120  -2.847  -6.238  1.00  0.00           C  \\nATOM    592  N1   DT B  19      -2.873  -2.640  -4.781  1.00  0.00           N  \\nATOM    593  C2   DT B  19      -1.555  -2.477  -4.349  1.00  0.00           C  \\nATOM    594  O2   DT B  19      -0.580  -2.560  -5.093  1.00  0.00           O  \\nATOM    595  N3   DT B  19      -1.378  -2.186  -3.004  1.00  0.00           N  \\nATOM    596  C4   DT B  19      -2.383  -2.014  -2.066  1.00  0.00           C  \\nATOM    597  O4   DT B  19      -2.096  -1.712  -0.910  1.00  0.00           O  \\nATOM    598  C5   DT B  19      -3.723  -2.209  -2.586  1.00  0.00           C  \\nATOM    599  C7   DT B  19      -4.919  -2.099  -1.660  1.00  0.00           C  \\nATOM    600  C6   DT B  19      -3.919  -2.501  -3.897  1.00  0.00           C  \\nATOM    601  H5'  DT B  19      -5.789  -0.516  -8.171  1.00  0.00           H  \\nATOM    602 H5''  DT B  19      -6.406  -2.027  -8.852  1.00  0.00           H  \\nATOM    603  H4'  DT B  19      -3.896  -1.762  -8.847  1.00  0.00           H  \\nATOM    604  H3'  DT B  19      -5.344  -4.128  -8.307  1.00  0.00           H  \\nATOM    605  H2'  DT B  19      -4.942  -4.038  -5.994  1.00  0.00           H  \\nATOM    606 H2''  DT B  19      -3.515  -4.987  -6.458  1.00  0.00           H  \\nATOM    607  H1'  DT B  19      -2.194  -2.994  -6.792  1.00  0.00           H  \\nATOM    608  H3   DT B  19      -0.426  -2.036  -2.699  1.00  0.00           H  \\nATOM    609  H71  DT B  19      -5.373  -1.115  -1.755  1.00  0.00           H  \\nATOM    610  H72  DT B  19      -5.662  -2.855  -1.918  1.00  0.00           H  \\nATOM    611  H73  DT B  19      -4.617  -2.249  -0.623  1.00  0.00           H  \\nATOM    612  H6   DT B  19      -4.933  -2.601  -4.249  1.00  0.00           H  \\nATOM    613  P    DG B  20      -3.026  -5.616  -9.294  1.00  0.00           P  \\nATOM    614  OP1  DG B  20      -2.294  -5.641 -10.580  1.00  0.00           O  \\nATOM    615  OP2  DG B  20      -4.256  -6.422  -9.133  1.00  0.00           O  \\nATOM    616  O5'  DG B  20      -1.998  -6.029  -8.124  1.00  0.00           O  \\nATOM    617  C5'  DG B  20      -0.704  -5.463  -8.038  1.00  0.00           C  \\nATOM    618  C4'  DG B  20      -0.007  -5.932  -6.756  1.00  0.00           C  \\nATOM    619  O4'  DG B  20      -0.749  -5.516  -5.619  1.00  0.00           O  \\nATOM    620  C3'  DG B  20       0.111  -7.460  -6.691  1.00  0.00           C  \\nATOM    621  O3'  DG B  20       1.386  -7.767  -6.152  1.00  0.00           O  \\nATOM    622  C2'  DG B  20      -1.024  -7.820  -5.730  1.00  0.00           C  \\nATOM    623  C1'  DG B  20      -0.947  -6.633  -4.774  1.00  0.00           C  \\nATOM    624  N9   DG B  20      -2.149  -6.417  -3.934  1.00  0.00           N  \\nATOM    625  C8   DG B  20      -3.464  -6.707  -4.203  1.00  0.00           C  \\nATOM    626  N7   DG B  20      -4.295  -6.345  -3.267  1.00  0.00           N  \\nATOM    627  C5   DG B  20      -3.479  -5.781  -2.295  1.00  0.00           C  \\nATOM    628  C6   DG B  20      -3.817  -5.231  -1.021  1.00  0.00           C  \\nATOM    629  O6   DG B  20      -4.927  -5.135  -0.503  1.00  0.00           O  \\nATOM    630  N1   DG B  20      -2.701  -4.768  -0.328  1.00  0.00           N  \\nATOM    631  C2   DG B  20      -1.405  -4.837  -0.802  1.00  0.00           C  \\nATOM    632  N2   DG B  20      -0.442  -4.372  -0.002  1.00  0.00           N  \\nATOM    633  N3   DG B  20      -1.086  -5.351  -2.000  1.00  0.00           N  \\nATOM    634  C4   DG B  20      -2.164  -5.811  -2.696  1.00  0.00           C  \\nATOM    635  H5'  DG B  20      -0.767  -4.375  -8.026  1.00  0.00           H  \\nATOM    636 H5''  DG B  20      -0.111  -5.770  -8.901  1.00  0.00           H  \\nATOM    637  H4'  DG B  20       0.985  -5.478  -6.723  1.00  0.00           H  \\nATOM    638  H3'  DG B  20      -0.015  -7.917  -7.673  1.00  0.00           H  \\nATOM    639  H2'  DG B  20      -1.964  -7.827  -6.285  1.00  0.00           H  \\nATOM    640 H2''  DG B  20      -0.880  -8.771  -5.219  1.00  0.00           H  \\nATOM    641  H1'  DG B  20      -0.078  -6.793  -4.131  1.00  0.00           H  \\nATOM    642  H8   DG B  20      -3.797  -7.185  -5.112  1.00  0.00           H  \\nATOM    643  H1   DG B  20      -2.869  -4.365   0.581  1.00  0.00           H  \\nATOM    644  H21  DG B  20      -0.665  -3.996   0.907  1.00  0.00           H  \\nATOM    645  H22  DG B  20       0.527  -4.453  -0.290  1.00  0.00           H  \\nATOM    646  P    DG B  21       2.024  -9.247  -6.211  1.00  0.00           P  \\nATOM    647  OP1  DG B  21       3.136  -9.244  -7.188  1.00  0.00           O  \\nATOM    648  OP2  DG B  21       0.932 -10.238  -6.335  1.00  0.00           O  \\nATOM    649  O5'  DG B  21       2.634  -9.366  -4.728  1.00  0.00           O  \\nATOM    650  C5'  DG B  21       3.892  -8.820  -4.379  1.00  0.00           C  \\nATOM    651  C4'  DG B  21       3.984  -8.711  -2.853  1.00  0.00           C  \\nATOM    652  O4'  DG B  21       3.006  -7.827  -2.336  1.00  0.00           O  \\nATOM    653  C3'  DG B  21       3.757 -10.058  -2.164  1.00  0.00           C  \\nATOM    654  O3'  DG B  21       4.960 -10.804  -2.168  1.00  0.00           O  \\nATOM    655  C2'  DG B  21       3.289  -9.606  -0.788  1.00  0.00           C  \\nATOM    656  C1'  DG B  21       2.562  -8.286  -1.068  1.00  0.00           C  \\nATOM    657  N9   DG B  21       1.080  -8.399  -1.129  1.00  0.00           N  \\nATOM    658  C8   DG B  21       0.293  -8.799  -2.181  1.00  0.00           C  \\nATOM    659  N7   DG B  21      -0.987  -8.757  -1.951  1.00  0.00           N  \\nATOM    660  C5   DG B  21      -1.072  -8.267  -0.656  1.00  0.00           C  \\nATOM    661  C6   DG B  21      -2.223  -7.998   0.146  1.00  0.00           C  \\nATOM    662  O6   DG B  21      -3.403  -8.196  -0.130  1.00  0.00           O  \\nATOM    663  N1   DG B  21      -1.895  -7.440   1.378  1.00  0.00           N  \\nATOM    664  C2   DG B  21      -0.602  -7.172   1.790  1.00  0.00           C  \\nATOM    665  N2   DG B  21      -0.446  -6.644   3.007  1.00  0.00           N  \\nATOM    666  N3   DG B  21       0.477  -7.430   1.037  1.00  0.00           N  \\nATOM    667  C4   DG B  21       0.183  -7.989  -0.167  1.00  0.00           C  \\nATOM    668  H5'  DG B  21       4.032  -7.833  -4.822  1.00  0.00           H  \\nATOM    669 H5''  DG B  21       4.678  -9.482  -4.745  1.00  0.00           H  \\nATOM    670  H4'  DG B  21       4.963  -8.310  -2.583  1.00  0.00           H  \\nATOM    671  H3'  DG B  21       2.955 -10.613  -2.649  1.00  0.00           H  \\nATOM    672  H2'  DG B  21       2.659 -10.339  -0.303  1.00  0.00           H  \\nATOM    673 H2''  DG B  21       4.145  -9.415  -0.143  1.00  0.00           H  \\nATOM    674  H1'  DG B  21       2.846  -7.600  -0.272  1.00  0.00           H  \\nATOM    675  H8   DG B  21       0.683  -9.104  -3.135  1.00  0.00           H  \\nATOM    676  H1   DG B  21      -2.661  -7.241   2.003  1.00  0.00           H  \\nATOM    677  H21  DG B  21      -1.236  -6.484   3.613  1.00  0.00           H  \\nATOM    678  H22  DG B  21       0.497  -6.466   3.341  1.00  0.00           H  \\nATOM    679  P    DA B  22       5.037 -12.344  -1.689  1.00  0.00           P  \\nATOM    680  OP1  DA B  22       6.199 -12.970  -2.359  1.00  0.00           O  \\nATOM    681  OP2  DA B  22       3.697 -12.958  -1.830  1.00  0.00           O  \\nATOM    682  O5'  DA B  22       5.377 -12.240  -0.119  1.00  0.00           O  \\nATOM    683  C5'  DA B  22       6.485 -11.495   0.353  1.00  0.00           C  \\nATOM    684  C4'  DA B  22       6.880 -11.944   1.766  1.00  0.00           C  \\nATOM    685  O4'  DA B  22       5.707 -12.137   2.543  1.00  0.00           O  \\nATOM    686  C3'  DA B  22       7.656 -13.269   1.773  1.00  0.00           C  \\nATOM    687  O3'  DA B  22       8.548 -13.254   2.879  1.00  0.00           O  \\nATOM    688  C2'  DA B  22       6.523 -14.273   1.937  1.00  0.00           C  \\nATOM    689  C1'  DA B  22       5.614 -13.508   2.894  1.00  0.00           C  \\nATOM    690  N9   DA B  22       4.209 -13.970   2.861  1.00  0.00           N  \\nATOM    691  C8   DA B  22       3.416 -14.290   1.784  1.00  0.00           C  \\nATOM    692  N7   DA B  22       2.184 -14.584   2.096  1.00  0.00           N  \\nATOM    693  C5   DA B  22       2.154 -14.452   3.478  1.00  0.00           C  \\nATOM    694  C6   DA B  22       1.134 -14.604   4.440  1.00  0.00           C  \\nATOM    695  N6   DA B  22      -0.118 -14.938   4.121  1.00  0.00           N  \\nATOM    696  N1   DA B  22       1.441 -14.424   5.739  1.00  0.00           N  \\nATOM    697  C2   DA B  22       2.685 -14.079   6.062  1.00  0.00           C  \\nATOM    698  N3   DA B  22       3.723 -13.880   5.257  1.00  0.00           N  \\nATOM    699  C4   DA B  22       3.389 -14.092   3.956  1.00  0.00           C  \\nATOM    700  H5'  DA B  22       6.211 -10.440   0.380  1.00  0.00           H  \\nATOM    701 H5''  DA B  22       7.346 -11.609  -0.308  1.00  0.00           H  \\nATOM    702  H4'  DA B  22       7.495 -11.158   2.207  1.00  0.00           H  \\nATOM    703  H3'  DA B  22       8.206 -13.437   0.846  1.00  0.00           H  \\nATOM    704  H2'  DA B  22       6.050 -14.437   0.971  1.00  0.00           H  \\nATOM    705 H2''  DA B  22       6.847 -15.224   2.347  1.00  0.00           H  \\nATOM    706  H1'  DA B  22       6.017 -13.627   3.898  1.00  0.00           H  \\nATOM    707  H8   DA B  22       3.765 -14.289   0.763  1.00  0.00           H  \\nATOM    708  H61  DA B  22      -0.799 -15.098   4.849  1.00  0.00           H  \\nATOM    709  H62  DA B  22      -0.403 -14.971   3.149  1.00  0.00           H  \\nATOM    710  H2   DA B  22       2.882 -13.931   7.112  1.00  0.00           H  \\nATOM    711  P    DA B  23       9.498 -14.507   3.262  1.00  0.00           P  \\nATOM    712  OP1  DA B  23      10.631 -13.997   4.067  1.00  0.00           O  \\nATOM    713  OP2  DA B  23       9.762 -15.291   2.035  1.00  0.00           O  \\nATOM    714  O5'  DA B  23       8.550 -15.384   4.227  1.00  0.00           O  \\nATOM    715  C5'  DA B  23       8.201 -14.937   5.523  1.00  0.00           C  \\nATOM    716  C4'  DA B  23       7.125 -15.855   6.111  1.00  0.00           C  \\nATOM    717  O4'  DA B  23       6.004 -15.909   5.243  1.00  0.00           O  \\nATOM    718  C3'  DA B  23       7.640 -17.289   6.298  1.00  0.00           C  \\nATOM    719  O3'  DA B  23       7.347 -17.705   7.624  1.00  0.00           O  \\nATOM    720  C2'  DA B  23       6.857 -18.064   5.238  1.00  0.00           C  \\nATOM    721  C1'  DA B  23       5.569 -17.250   5.162  1.00  0.00           C  \\nATOM    722  N9   DA B  23       4.787 -17.460   3.922  1.00  0.00           N  \\nATOM    723  C8   DA B  23       5.218 -17.504   2.618  1.00  0.00           C  \\nATOM    724  N7   DA B  23       4.265 -17.645   1.740  1.00  0.00           N  \\nATOM    725  C5   DA B  23       3.113 -17.702   2.513  1.00  0.00           C  \\nATOM    726  C6   DA B  23       1.748 -17.856   2.192  1.00  0.00           C  \\nATOM    727  N6   DA B  23       1.307 -17.978   0.936  1.00  0.00           N  \\nATOM    728  N1   DA B  23       0.854 -17.899   3.198  1.00  0.00           N  \\nATOM    729  C2   DA B  23       1.287 -17.784   4.452  1.00  0.00           C  \\nATOM    730  N3   DA B  23       2.536 -17.627   4.883  1.00  0.00           N  \\nATOM    731  C4   DA B  23       3.419 -17.594   3.847  1.00  0.00           C  \\nATOM    732  H5'  DA B  23       7.817 -13.917   5.486  1.00  0.00           H  \\nATOM    733 H5''  DA B  23       9.080 -14.949   6.168  1.00  0.00           H  \\nATOM    734  H4'  DA B  23       6.806 -15.445   7.070  1.00  0.00           H  \\nATOM    735  H3'  DA B  23       8.712 -17.341   6.108  1.00  0.00           H  \\nATOM    736  H2'  DA B  23       7.407 -18.009   4.298  1.00  0.00           H  \\nATOM    737 H2''  DA B  23       6.682 -19.104   5.506  1.00  0.00           H  \\nATOM    738  H1'  DA B  23       4.956 -17.492   6.028  1.00  0.00           H  \\nATOM    739  H8   DA B  23       6.256 -17.425   2.330  1.00  0.00           H  \\nATOM    740  H61  DA B  23       0.323 -18.116   0.752  1.00  0.00           H  \\nATOM    741  H62  DA B  23       1.969 -17.957   0.174  1.00  0.00           H  \\nATOM    742  H2   DA B  23       0.527 -17.826   5.218  1.00  0.00           H  \\nATOM    743  P    DC B  24       7.880 -19.103   8.239  1.00  0.00           P  \\nATOM    744  OP1  DC B  24       8.444 -18.837   9.581  1.00  0.00           O  \\nATOM    745  OP2  DC B  24       8.699 -19.794   7.218  1.00  0.00           O  \\nATOM    746  O5'  DC B  24       6.510 -19.929   8.421  1.00  0.00           O  \\nATOM    747  C5'  DC B  24       5.652 -19.718   9.530  1.00  0.00           C  \\nATOM    748  C4'  DC B  24       4.280 -20.317   9.208  1.00  0.00           C  \\nATOM    749  O4'  DC B  24       3.831 -19.851   7.950  1.00  0.00           O  \\nATOM    750  C3'  DC B  24       4.318 -21.845   9.110  1.00  0.00           C  \\nATOM    751  O3'  DC B  24       3.855 -22.448  10.301  1.00  0.00           O  \\nATOM    752  C2'  DC B  24       3.395 -22.164   7.937  1.00  0.00           C  \\nATOM    753  C1'  DC B  24       2.914 -20.799   7.442  1.00  0.00           C  \\nATOM    754  N1   DC B  24       2.896 -20.769   5.955  1.00  0.00           N  \\nATOM    755  C2   DC B  24       1.679 -20.878   5.280  1.00  0.00           C  \\nATOM    756  O2   DC B  24       0.610 -20.900   5.887  1.00  0.00           O  \\nATOM    757  N3   DC B  24       1.696 -20.948   3.918  1.00  0.00           N  \\nATOM    758  C4   DC B  24       2.848 -20.923   3.233  1.00  0.00           C  \\nATOM    759  N4   DC B  24       2.801 -20.973   1.898  1.00  0.00           N  \\nATOM    760  C5   DC B  24       4.112 -20.809   3.906  1.00  0.00           C  \\nATOM    761  C6   DC B  24       4.079 -20.734   5.257  1.00  0.00           C  \\nATOM    762  H5'  DC B  24       5.537 -18.654   9.743  1.00  0.00           H  \\nATOM    763 H5''  DC B  24       6.068 -20.202  10.415  1.00  0.00           H  \\nATOM    764  H4'  DC B  24       3.557 -20.018   9.969  1.00  0.00           H  \\nATOM    765  H3'  DC B  24       5.324 -22.197   8.879  1.00  0.00           H  \\nATOM    766 HO3'  DC B  24       2.968 -22.129  10.481  1.00  0.00           H  \\nATOM    767  H2'  DC B  24       3.970 -22.680   7.169  1.00  0.00           H  \\nATOM    768 H2''  DC B  24       2.548 -22.782   8.229  1.00  0.00           H  \\nATOM    769  H1'  DC B  24       1.943 -20.587   7.888  1.00  0.00           H  \\nATOM    770  H41  DC B  24       1.902 -21.016   1.437  1.00  0.00           H  \\nATOM    771  H42  DC B  24       3.650 -20.922   1.355  1.00  0.00           H  \\nATOM    772  H5   DC B  24       5.062 -20.776   3.392  1.00  0.00           H  \\nATOM    773  H6   DC B  24       5.004 -20.670   5.810  1.00  0.00           H  \\nTER     774       DC B  24                                                      \\nMASTER      121    0    0    0    0    0    0    6  500    2    0    2          \\nEND                                                                             \\n\");\n",
       "\tviewer_168469674212664.setStyle({\"model\": -1, \"chain\": \"A\"},{\"cartoon\": {\"color\": \"spectrum\"}});\n",
       "\tviewer_168469674212664.zoomTo();\n",
       "viewer_168469674212664.render();\n",
       "});\n",
       "</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Similarity score: 10.0\n",
      " Protein: the unusual structure of the human centromere (gga)2 motif: unpaired guanosine residues stacked between sheared g(dot)a pairs(103D)\n",
      " Chain Id: B\n",
      " Sequence of the chain: DGDTDGDGDADADTDGDGDADADC\n"
     ]
    },
    {
     "data": {
      "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_1684696742130174\"  style=\"position: relative; width: 600px; height: 400px\">\n        <p id=\"3dmolwarning_1684696742130174\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.0.1/3Dmol-min.js');\n}\n\nvar viewer_1684696742130174 = null;\nvar warn = document.getElementById(\"3dmolwarning_1684696742130174\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_1684696742130174 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_1684696742130174\"),{backgroundColor:\"white\"});\nviewer_1684696742130174.zoomTo();\n\tviewer_1684696742130174.addModelsAsFrames(\"HEADER    DNA                                     16-DEC-94   103D              \\nTITLE     THE UNUSUAL STRUCTURE OF THE HUMAN CENTROMERE (GGA)2 MOTIF: UNPAIRED  \\nTITLE    2 GUANOSINE RESIDUES STACKED BETWEEN SHEARED G(DOT)A PAIRS             \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: DNA (5'-D(*GP*TP*GP*GP*AP*AP*TP*GP*GP*AP*AP*C)-3');        \\nCOMPND   3 CHAIN: A, B;                                                         \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 SYNTHETIC: YES;                                                      \\nSOURCE   3 OTHER_DETAILS: CHEMICALLY SYNTHESIZED                                \\nKEYWDS    DNA, DOUBLE HELIX, G-G STACKING, G:A MISMATCH, HUMAN CENTROMERE       \\nKEYWDS   2 REPEAT, GA-BRACKETED G-STACK MOTIF                                   \\nEXPDTA    SOLUTION NMR                                                          \\nAUTHOR    S.-H.CHOU,L.ZHU,B.R.REID                                              \\nREVDAT   5   16-FEB-22 103D    1       REMARK                                   \\nREVDAT   4   07-APR-09 103D    1       REMARK                                   \\nREVDAT   3   24-FEB-09 103D    1       VERSN                                    \\nREVDAT   2   01-APR-03 103D    1       JRNL                                     \\nREVDAT   1   31-MAR-95 103D    0                                                \\nJRNL        AUTH   S.H.CHOU,L.ZHU,B.R.REID                                      \\nJRNL        TITL   THE UNUSUAL STRUCTURE OF THE HUMAN CENTROMERE (GGA)2 MOTIF.  \\nJRNL        TITL 2 UNPAIRED GUANOSINE RESIDUES STACKED BETWEEN SHEARED G.A      \\nJRNL        TITL 3 PAIRS.                                                       \\nJRNL        REF    J.MOL.BIOL.                   V. 244   259 1994              \\nJRNL        REFN                   ISSN 0022-2836                               \\nJRNL        PMID   7966337                                                      \\nJRNL        DOI    10.1006/JMBI.1994.1727                                       \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION. NOT APPLICABLE.                                          \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : AMBER                                                \\nREMARK   3   AUTHORS     : BIOSYM TECHNOLOGIES, INC.                            \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: R VALUE 0.250 FINAL RMS COORD. SHIFT      \\nREMARK   3  0.65 ANGSTROMS NUMBER OF ATOMS USED IN REFINEMENT. NUMBER OF        \\nREMARK   3  PROTEIN ATOMS 0 NUMBER OF NUCLEIC ACID ATOMS 772 NUMBER OF          \\nREMARK   3  HETEROGEN ATOMS 0 NUMBER OF SOLVENT ATOMS 0. THE DUPLEX CONTAINS    \\nREMARK   3  UNPAIRED STAGGERED GUANOSINE RESIDUES, WHICH CO-STACK BY            \\nREMARK   3  INTERCALATION BETWEEN SHEARED G(DOT)A AND A(DOT)G BASE-PAIRS TO     \\nREMARK   3  FORM AN INTERESTING NEW STRUCTURAL MOTIF, THE GA-BRACKETED G-       \\nREMARK   3  STACK. THE TGGAA REPEAT UNIT CONTAINS SIX \\\"STEPS\\\", FOUR OF WHICH    \\nREMARK   3  ARE NOT WATSON-CRICK BASE-PAIRS.                                    \\nREMARK   4                                                                      \\nREMARK   4 103D COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000170007.                                   \\nREMARK 210                                                                      \\nREMARK 210 EXPERIMENTAL DETAILS                                                 \\nREMARK 210  EXPERIMENT TYPE                : NMR                                \\nREMARK 210  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 210  PH                             : NULL                               \\nREMARK 210  IONIC STRENGTH                 : NULL                               \\nREMARK 210  PRESSURE                       : NULL                               \\nREMARK 210  SAMPLE CONTENTS                : NULL                               \\nREMARK 210                                                                      \\nREMARK 210  NMR EXPERIMENTS CONDUCTED      : NULL                               \\nREMARK 210  SPECTROMETER FIELD STRENGTH    : NULL                               \\nREMARK 210  SPECTROMETER MODEL             : NULL                               \\nREMARK 210  SPECTROMETER MANUFACTURER      : NULL                               \\nREMARK 210                                                                      \\nREMARK 210  STRUCTURE DETERMINATION.                                            \\nREMARK 210   SOFTWARE USED                 : NULL                               \\nREMARK 210   METHOD USED                   : NULL                               \\nREMARK 210                                                                      \\nREMARK 210 CONFORMERS, NUMBER CALCULATED   : NULL                               \\nREMARK 210 CONFORMERS, NUMBER SUBMITTED    : 1                                  \\nREMARK 210 CONFORMERS, SELECTION CRITERIA  : NULL                               \\nREMARK 210                                                                      \\nREMARK 210 BEST REPRESENTATIVE CONFORMER IN THIS ENSEMBLE : NULL                \\nREMARK 210                                                                      \\nREMARK 210 REMARK: NULL                                                         \\nREMARK 215                                                                      \\nREMARK 215 NMR STUDY                                                            \\nREMARK 215 THE COORDINATES IN THIS ENTRY WERE GENERATED FROM SOLUTION           \\nREMARK 215 NMR DATA.  PROTEIN DATA BANK CONVENTIONS REQUIRE THAT                \\nREMARK 215 CRYST1 AND SCALE RECORDS BE INCLUDED, BUT THE VALUES ON              \\nREMARK 215 THESE RECORDS ARE MEANINGLESS.                                       \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A, B                                  \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \\nREMARK 500     DG A   1   O4' -  C1' -  N9  ANGL. DEV. =   1.9 DEGREES          \\nREMARK 500     DA A   5   O4' -  C1' -  N9  ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500     DA A   6   O4' -  C1' -  N9  ANGL. DEV. =   2.2 DEGREES          \\nREMARK 500     DT A   7   O4' -  C1' -  N1  ANGL. DEV. =   2.5 DEGREES          \\nREMARK 500     DA A  10   O4' -  C1' -  N9  ANGL. DEV. =   2.7 DEGREES          \\nREMARK 500     DA A  11   O4' -  C1' -  N9  ANGL. DEV. =   2.0 DEGREES          \\nREMARK 500     DC A  12   O4' -  C1' -  N1  ANGL. DEV. =   2.5 DEGREES          \\nREMARK 500     DG B  13   O4' -  C1' -  N9  ANGL. DEV. =   1.9 DEGREES          \\nREMARK 500     DT B  14   O4' -  C1' -  N1  ANGL. DEV. =   1.8 DEGREES          \\nREMARK 500     DA B  17   O4' -  C1' -  N9  ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500     DA B  18   O4' -  C1' -  N9  ANGL. DEV. =   2.3 DEGREES          \\nREMARK 500     DT B  19   O4' -  C1' -  N1  ANGL. DEV. =   2.5 DEGREES          \\nREMARK 500     DA B  22   O4' -  C1' -  N9  ANGL. DEV. =   2.7 DEGREES          \\nREMARK 500     DA B  23   O4' -  C1' -  N9  ANGL. DEV. =   1.9 DEGREES          \\nREMARK 500     DC B  24   O4' -  C1' -  N1  ANGL. DEV. =   2.4 DEGREES          \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: PLANAR GROUPS                                              \\nREMARK 500                                                                      \\nREMARK 500 PLANAR GROUPS IN THE FOLLOWING RESIDUES HAVE A TOTAL                 \\nREMARK 500 RMS DISTANCE OF ALL ATOMS FROM THE BEST-FIT PLANE                    \\nREMARK 500 BY MORE THAN AN EXPECTED VALUE OF 6*RMSD, WITH AN                    \\nREMARK 500 RMSD 0.02 ANGSTROMS, OR AT LEAST ONE ATOM HAS                        \\nREMARK 500 AN RMSD GREATER THAN THIS VALUE                                      \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        RMS     TYPE                                    \\nREMARK 500     DT A   2         0.06    SIDE CHAIN                              \\nREMARK 500     DT B  14         0.06    SIDE CHAIN                              \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nDBREF  103D A    1    12  PDB    103D     103D             1     12             \\nDBREF  103D B   13    24  PDB    103D     103D            13     24             \\nSEQRES   1 A   12   DG  DT  DG  DG  DA  DA  DT  DG  DG  DA  DA  DC              \\nSEQRES   1 B   12   DG  DT  DG  DG  DA  DA  DT  DG  DG  DA  DA  DC              \\nCRYST1    1.000    1.000    1.000  90.00  90.00  90.00 P 1           1          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      1.000000  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  1.000000  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  1.000000        0.00000                         \\nATOM      1  O5'  DG A   1      -7.643 -22.640  -1.099  1.00  0.00           O  \\nATOM      2  C5'  DG A   1      -8.313 -23.137   0.040  1.00  0.00           C  \\nATOM      3  C4'  DG A   1      -7.935 -22.349   1.300  1.00  0.00           C  \\nATOM      4  O4'  DG A   1      -6.537 -22.463   1.524  1.00  0.00           O  \\nATOM      5  C3'  DG A   1      -8.273 -20.854   1.190  1.00  0.00           C  \\nATOM      6  O3'  DG A   1      -8.832 -20.418   2.419  1.00  0.00           O  \\nATOM      7  C2'  DG A   1      -6.902 -20.240   0.924  1.00  0.00           C  \\nATOM      8  C1'  DG A   1      -5.995 -21.171   1.722  1.00  0.00           C  \\nATOM      9  N9   DG A   1      -4.593 -21.126   1.247  1.00  0.00           N  \\nATOM     10  C8   DG A   1      -4.107 -21.429  -0.001  1.00  0.00           C  \\nATOM     11  N7   DG A   1      -2.812 -21.335  -0.110  1.00  0.00           N  \\nATOM     12  C5   DG A   1      -2.397 -20.963   1.163  1.00  0.00           C  \\nATOM     13  C6   DG A   1      -1.080 -20.733   1.666  1.00  0.00           C  \\nATOM     14  O6   DG A   1      -0.017 -20.780   1.053  1.00  0.00           O  \\nATOM     15  N1   DG A   1      -1.077 -20.427   3.024  1.00  0.00           N  \\nATOM     16  C2   DG A   1      -2.214 -20.317   3.803  1.00  0.00           C  \\nATOM     17  N2   DG A   1      -2.045 -20.010   5.092  1.00  0.00           N  \\nATOM     18  N3   DG A   1      -3.455 -20.507   3.325  1.00  0.00           N  \\nATOM     19  C4   DG A   1      -3.479 -20.835   2.004  1.00  0.00           C  \\nATOM     20  H5'  DG A   1      -8.042 -24.185   0.178  1.00  0.00           H  \\nATOM     21 H5''  DG A   1      -9.389 -23.072  -0.122  1.00  0.00           H  \\nATOM     22  H4'  DG A   1      -8.473 -22.789   2.142  1.00  0.00           H  \\nATOM     23  H3'  DG A   1      -8.969 -20.658   0.374  1.00  0.00           H  \\nATOM     24  H2'  DG A   1      -6.682 -20.310  -0.141  1.00  0.00           H  \\nATOM     25 H2''  DG A   1      -6.821 -19.205   1.250  1.00  0.00           H  \\nATOM     26  H1'  DG A   1      -6.058 -20.912   2.780  1.00  0.00           H  \\nATOM     27  H8   DG A   1      -4.739 -21.718  -0.828  1.00  0.00           H  \\nATOM     28  H1   DG A   1      -0.169 -20.285   3.449  1.00  0.00           H  \\nATOM     29  H21  DG A   1      -1.114 -19.925   5.477  1.00  0.00           H  \\nATOM     30  H22  DG A   1      -2.853 -19.902   5.687  1.00  0.00           H  \\nATOM     31 HO5'  DG A   1      -7.929 -21.737  -1.254  1.00  0.00           H  \\nATOM     32  P    DT A   2      -9.356 -18.904   2.648  1.00  0.00           P  \\nATOM     33  OP1  DT A   2     -10.579 -18.963   3.478  1.00  0.00           O  \\nATOM     34  OP2  DT A   2      -9.388 -18.212   1.340  1.00  0.00           O  \\nATOM     35  O5'  DT A   2      -8.175 -18.250   3.532  1.00  0.00           O  \\nATOM     36  C5'  DT A   2      -7.959 -18.655   4.871  1.00  0.00           C  \\nATOM     37  C4'  DT A   2      -6.761 -17.920   5.479  1.00  0.00           C  \\nATOM     38  O4'  DT A   2      -5.603 -18.173   4.699  1.00  0.00           O  \\nATOM     39  C3'  DT A   2      -6.955 -16.396   5.550  1.00  0.00           C  \\nATOM     40  O3'  DT A   2      -6.507 -15.961   6.826  1.00  0.00           O  \\nATOM     41  C2'  DT A   2      -6.062 -15.914   4.406  1.00  0.00           C  \\nATOM     42  C1'  DT A   2      -4.938 -16.944   4.495  1.00  0.00           C  \\nATOM     43  N1   DT A   2      -4.057 -17.032   3.297  1.00  0.00           N  \\nATOM     44  C2   DT A   2      -2.677 -17.088   3.499  1.00  0.00           C  \\nATOM     45  O2   DT A   2      -2.146 -16.945   4.598  1.00  0.00           O  \\nATOM     46  N3   DT A   2      -1.900 -17.352   2.381  1.00  0.00           N  \\nATOM     47  C4   DT A   2      -2.368 -17.603   1.102  1.00  0.00           C  \\nATOM     48  O4   DT A   2      -1.578 -17.886   0.204  1.00  0.00           O  \\nATOM     49  C5   DT A   2      -3.808 -17.502   0.964  1.00  0.00           C  \\nATOM     50  C7   DT A   2      -4.451 -17.702  -0.395  1.00  0.00           C  \\nATOM     51  C6   DT A   2      -4.587 -17.231   2.042  1.00  0.00           C  \\nATOM     52  H5'  DT A   2      -7.763 -19.727   4.902  1.00  0.00           H  \\nATOM     53 H5''  DT A   2      -8.845 -18.442   5.472  1.00  0.00           H  \\nATOM     54  H4'  DT A   2      -6.601 -18.314   6.483  1.00  0.00           H  \\nATOM     55  H3'  DT A   2      -7.997 -16.110   5.395  1.00  0.00           H  \\nATOM     56  H2'  DT A   2      -6.630 -15.995   3.481  1.00  0.00           H  \\nATOM     57 H2''  DT A   2      -5.703 -14.894   4.522  1.00  0.00           H  \\nATOM     58  H1'  DT A   2      -4.373 -16.711   5.397  1.00  0.00           H  \\nATOM     59  H3   DT A   2      -0.903 -17.421   2.534  1.00  0.00           H  \\nATOM     60  H71  DT A   2      -3.753 -17.435  -1.188  1.00  0.00           H  \\nATOM     61  H72  DT A   2      -4.736 -18.745  -0.514  1.00  0.00           H  \\nATOM     62  H73  DT A   2      -5.340 -17.077  -0.489  1.00  0.00           H  \\nATOM     63  H6   DT A   2      -5.655 -17.199   1.905  1.00  0.00           H  \\nATOM     64  P    DG A   3      -6.534 -14.414   7.292  1.00  0.00           P  \\nATOM     65  OP1  DG A   3      -6.475 -14.378   8.770  1.00  0.00           O  \\nATOM     66  OP2  DG A   3      -7.637 -13.728   6.584  1.00  0.00           O  \\nATOM     67  O5'  DG A   3      -5.139 -13.848   6.720  1.00  0.00           O  \\nATOM     68  C5'  DG A   3      -3.896 -14.287   7.234  1.00  0.00           C  \\nATOM     69  C4'  DG A   3      -2.748 -13.705   6.405  1.00  0.00           C  \\nATOM     70  O4'  DG A   3      -2.863 -14.119   5.053  1.00  0.00           O  \\nATOM     71  C3'  DG A   3      -2.751 -12.171   6.416  1.00  0.00           C  \\nATOM     72  O3'  DG A   3      -1.406 -11.747   6.547  1.00  0.00           O  \\nATOM     73  C2'  DG A   3      -3.335 -11.844   5.041  1.00  0.00           C  \\nATOM     74  C1'  DG A   3      -2.741 -12.983   4.218  1.00  0.00           C  \\nATOM     75  N9   DG A   3      -3.405 -13.249   2.921  1.00  0.00           N  \\nATOM     76  C8   DG A   3      -4.715 -13.058   2.555  1.00  0.00           C  \\nATOM     77  N7   DG A   3      -4.994 -13.444   1.343  1.00  0.00           N  \\nATOM     78  C5   DG A   3      -3.784 -13.924   0.857  1.00  0.00           C  \\nATOM     79  C6   DG A   3      -3.457 -14.458  -0.426  1.00  0.00           C  \\nATOM     80  O6   DG A   3      -4.194 -14.609  -1.397  1.00  0.00           O  \\nATOM     81  N1   DG A   3      -2.119 -14.829  -0.523  1.00  0.00           N  \\nATOM     82  C2   DG A   3      -1.196 -14.693   0.496  1.00  0.00           C  \\nATOM     83  N2   DG A   3       0.052 -15.094   0.239  1.00  0.00           N  \\nATOM     84  N3   DG A   3      -1.501 -14.190   1.703  1.00  0.00           N  \\nATOM     85  C4   DG A   3      -2.807 -13.820   1.820  1.00  0.00           C  \\nATOM     86  H5'  DG A   3      -3.836 -15.375   7.202  1.00  0.00           H  \\nATOM     87 H5''  DG A   3      -3.792 -13.963   8.271  1.00  0.00           H  \\nATOM     88  H4'  DG A   3      -1.809 -14.080   6.816  1.00  0.00           H  \\nATOM     89  H3'  DG A   3      -3.366 -11.782   7.227  1.00  0.00           H  \\nATOM     90  H2'  DG A   3      -4.421 -11.917   5.096  1.00  0.00           H  \\nATOM     91 H2''  DG A   3      -3.043 -10.865   4.661  1.00  0.00           H  \\nATOM     92  H1'  DG A   3      -1.692 -12.735   4.043  1.00  0.00           H  \\nATOM     93  H8   DG A   3      -5.463 -12.632   3.207  1.00  0.00           H  \\nATOM     94  H1   DG A   3      -1.821 -15.221  -1.404  1.00  0.00           H  \\nATOM     95  H21  DG A   3       0.298 -15.465  -0.666  1.00  0.00           H  \\nATOM     96  H22  DG A   3       0.772 -14.962   0.941  1.00  0.00           H  \\nATOM     97  P    DG A   4      -0.997 -10.224   6.884  1.00  0.00           P  \\nATOM     98  OP1  DG A   4      -0.439 -10.175   8.254  1.00  0.00           O  \\nATOM     99  OP2  DG A   4      -2.112  -9.328   6.505  1.00  0.00           O  \\nATOM    100  O5'  DG A   4       0.196 -10.007   5.828  1.00  0.00           O  \\nATOM    101  C5'  DG A   4       1.508 -10.482   6.064  1.00  0.00           C  \\nATOM    102  C4'  DG A   4       2.254 -10.592   4.729  1.00  0.00           C  \\nATOM    103  O4'  DG A   4       1.640 -11.522   3.855  1.00  0.00           O  \\nATOM    104  C3'  DG A   4       2.288  -9.262   3.978  1.00  0.00           C  \\nATOM    105  O3'  DG A   4       3.326  -8.456   4.505  1.00  0.00           O  \\nATOM    106  C2'  DG A   4       2.518  -9.741   2.552  1.00  0.00           C  \\nATOM    107  C1'  DG A   4       1.787 -11.086   2.512  1.00  0.00           C  \\nATOM    108  N9   DG A   4       0.429 -11.010   1.915  1.00  0.00           N  \\nATOM    109  C8   DG A   4      -0.746 -10.615   2.504  1.00  0.00           C  \\nATOM    110  N7   DG A   4      -1.792 -10.688   1.732  1.00  0.00           N  \\nATOM    111  C5   DG A   4      -1.287 -11.185   0.538  1.00  0.00           C  \\nATOM    112  C6   DG A   4      -1.954 -11.480  -0.691  1.00  0.00           C  \\nATOM    113  O6   DG A   4      -3.141 -11.323  -0.965  1.00  0.00           O  \\nATOM    114  N1   DG A   4      -1.096 -12.013  -1.650  1.00  0.00           N  \\nATOM    115  C2   DG A   4       0.251 -12.248  -1.443  1.00  0.00           C  \\nATOM    116  N2   DG A   4       0.952 -12.759  -2.460  1.00  0.00           N  \\nATOM    117  N3   DG A   4       0.873 -11.977  -0.287  1.00  0.00           N  \\nATOM    118  C4   DG A   4       0.061 -11.433   0.657  1.00  0.00           C  \\nATOM    119  H5'  DG A   4       1.495 -11.457   6.552  1.00  0.00           H  \\nATOM    120 H5''  DG A   4       2.027  -9.777   6.714  1.00  0.00           H  \\nATOM    121  H4'  DG A   4       3.271 -10.944   4.919  1.00  0.00           H  \\nATOM    122  H3'  DG A   4       1.317  -8.768   4.033  1.00  0.00           H  \\nATOM    123  H2'  DG A   4       2.150  -9.039   1.815  1.00  0.00           H  \\nATOM    124 H2''  DG A   4       3.578  -9.904   2.373  1.00  0.00           H  \\nATOM    125  H1'  DG A   4       2.409 -11.766   1.933  1.00  0.00           H  \\nATOM    126  H8   DG A   4      -0.813 -10.291   3.528  1.00  0.00           H  \\nATOM    127  H1   DG A   4      -1.500 -12.225  -2.549  1.00  0.00           H  \\nATOM    128  H21  DG A   4       0.523 -12.928  -3.357  1.00  0.00           H  \\nATOM    129  H22  DG A   4       1.950 -12.906  -2.334  1.00  0.00           H  \\nATOM    130  P    DA A   5       3.562  -6.924   4.058  1.00  0.00           P  \\nATOM    131  OP1  DA A   5       4.379  -6.268   5.103  1.00  0.00           O  \\nATOM    132  OP2  DA A   5       2.259  -6.334   3.677  1.00  0.00           O  \\nATOM    133  O5'  DA A   5       4.464  -7.037   2.730  1.00  0.00           O  \\nATOM    134  C5'  DA A   5       5.721  -7.693   2.730  1.00  0.00           C  \\nATOM    135  C4'  DA A   5       6.616  -7.135   1.615  1.00  0.00           C  \\nATOM    136  O4'  DA A   5       5.843  -6.978   0.432  1.00  0.00           O  \\nATOM    137  C3'  DA A   5       7.201  -5.758   1.969  1.00  0.00           C  \\nATOM    138  O3'  DA A   5       8.471  -5.625   1.344  1.00  0.00           O  \\nATOM    139  C2'  DA A   5       6.152  -4.834   1.366  1.00  0.00           C  \\nATOM    140  C1'  DA A   5       5.814  -5.603   0.093  1.00  0.00           C  \\nATOM    141  N9   DA A   5       4.510  -5.219  -0.496  1.00  0.00           N  \\nATOM    142  C8   DA A   5       3.302  -4.987   0.119  1.00  0.00           C  \\nATOM    143  N7   DA A   5       2.320  -4.747  -0.706  1.00  0.00           N  \\nATOM    144  C5   DA A   5       2.915  -4.819  -1.959  1.00  0.00           C  \\nATOM    145  C6   DA A   5       2.422  -4.678  -3.273  1.00  0.00           C  \\nATOM    146  N6   DA A   5       1.142  -4.414  -3.547  1.00  0.00           N  \\nATOM    147  N1   DA A   5       3.285  -4.794  -4.301  1.00  0.00           N  \\nATOM    148  C2   DA A   5       4.562  -5.063  -4.040  1.00  0.00           C  \\nATOM    149  N3   DA A   5       5.145  -5.238  -2.858  1.00  0.00           N  \\nATOM    150  C4   DA A   5       4.254  -5.095  -1.840  1.00  0.00           C  \\nATOM    151  H5'  DA A   5       5.562  -8.757   2.560  1.00  0.00           H  \\nATOM    152 H5''  DA A   5       6.232  -7.572   3.686  1.00  0.00           H  \\nATOM    153  H4'  DA A   5       7.427  -7.844   1.444  1.00  0.00           H  \\nATOM    154  H3'  DA A   5       7.295  -5.615   3.047  1.00  0.00           H  \\nATOM    155  H2'  DA A   5       5.300  -4.777   2.042  1.00  0.00           H  \\nATOM    156 H2''  DA A   5       6.527  -3.836   1.162  1.00  0.00           H  \\nATOM    157  H1'  DA A   5       6.610  -5.423  -0.628  1.00  0.00           H  \\nATOM    158  H8   DA A   5       3.159  -5.017   1.188  1.00  0.00           H  \\nATOM    159  H61  DA A   5       0.850  -4.267  -4.502  1.00  0.00           H  \\nATOM    160  H62  DA A   5       0.451  -4.437  -2.806  1.00  0.00           H  \\nATOM    161  H2   DA A   5       5.213  -5.162  -4.894  1.00  0.00           H  \\nATOM    162  P    DA A   6       9.363  -4.276   1.441  1.00  0.00           P  \\nATOM    163  OP1  DA A   6      10.782  -4.649   1.246  1.00  0.00           O  \\nATOM    164  OP2  DA A   6       8.960  -3.530   2.654  1.00  0.00           O  \\nATOM    165  O5'  DA A   6       8.887  -3.431   0.150  1.00  0.00           O  \\nATOM    166  C5'  DA A   6       9.220  -3.846  -1.162  1.00  0.00           C  \\nATOM    167  C4'  DA A   6       8.472  -3.000  -2.195  1.00  0.00           C  \\nATOM    168  O4'  DA A   6       7.079  -3.068  -1.929  1.00  0.00           O  \\nATOM    169  C3'  DA A   6       8.883  -1.518  -2.181  1.00  0.00           C  \\nATOM    170  O3'  DA A   6       9.162  -1.103  -3.510  1.00  0.00           O  \\nATOM    171  C2'  DA A   6       7.638  -0.843  -1.609  1.00  0.00           C  \\nATOM    172  C1'  DA A   6       6.534  -1.778  -2.095  1.00  0.00           C  \\nATOM    173  N9   DA A   6       5.266  -1.649  -1.337  1.00  0.00           N  \\nATOM    174  C8   DA A   6       5.062  -1.659   0.023  1.00  0.00           C  \\nATOM    175  N7   DA A   6       3.811  -1.603   0.382  1.00  0.00           N  \\nATOM    176  C5   DA A   6       3.128  -1.551  -0.825  1.00  0.00           C  \\nATOM    177  C6   DA A   6       1.757  -1.487  -1.146  1.00  0.00           C  \\nATOM    178  N6   DA A   6       0.795  -1.473  -0.218  1.00  0.00           N  \\nATOM    179  N1   DA A   6       1.406  -1.437  -2.443  1.00  0.00           N  \\nATOM    180  C2   DA A   6       2.358  -1.447  -3.373  1.00  0.00           C  \\nATOM    181  N3   DA A   6       3.677  -1.511  -3.202  1.00  0.00           N  \\nATOM    182  C4   DA A   6       4.005  -1.564  -1.882  1.00  0.00           C  \\nATOM    183  H5'  DA A   6       8.939  -4.890  -1.301  1.00  0.00           H  \\nATOM    184 H5''  DA A   6      10.295  -3.750  -1.321  1.00  0.00           H  \\nATOM    185  H4'  DA A   6       8.666  -3.422  -3.183  1.00  0.00           H  \\nATOM    186  H3'  DA A   6       9.752  -1.349  -1.542  1.00  0.00           H  \\nATOM    187  H2'  DA A   6       7.729  -0.858  -0.523  1.00  0.00           H  \\nATOM    188 H2''  DA A   6       7.497   0.178  -1.959  1.00  0.00           H  \\nATOM    189  H1'  DA A   6       6.357  -1.604  -3.158  1.00  0.00           H  \\nATOM    190  H8   DA A   6       5.858  -1.702   0.751  1.00  0.00           H  \\nATOM    191  H61  DA A   6      -0.176  -1.429  -0.493  1.00  0.00           H  \\nATOM    192  H62  DA A   6       1.049  -1.512   0.759  1.00  0.00           H  \\nATOM    193  H2   DA A   6       2.014  -1.412  -4.395  1.00  0.00           H  \\nATOM    194  P    DT A   7       9.713   0.376  -3.872  1.00  0.00           P  \\nATOM    195  OP1  DT A   7      10.872   0.227  -4.781  1.00  0.00           O  \\nATOM    196  OP2  DT A   7       9.854   1.156  -2.622  1.00  0.00           O  \\nATOM    197  O5'  DT A   7       8.487   0.997  -4.715  1.00  0.00           O  \\nATOM    198  C5'  DT A   7       8.196   0.539  -6.023  1.00  0.00           C  \\nATOM    199  C4'  DT A   7       6.841   1.074  -6.496  1.00  0.00           C  \\nATOM    200  O4'  DT A   7       5.858   0.807  -5.506  1.00  0.00           O  \\nATOM    201  C3'  DT A   7       6.839   2.587  -6.766  1.00  0.00           C  \\nATOM    202  O3'  DT A   7       6.155   2.808  -7.992  1.00  0.00           O  \\nATOM    203  C2'  DT A   7       6.075   3.121  -5.556  1.00  0.00           C  \\nATOM    204  C1'  DT A   7       5.086   1.977  -5.335  1.00  0.00           C  \\nATOM    205  N1   DT A   7       4.411   1.977  -4.004  1.00  0.00           N  \\nATOM    206  C2   DT A   7       3.021   1.846  -3.966  1.00  0.00           C  \\nATOM    207  O2   DT A   7       2.314   1.799  -4.970  1.00  0.00           O  \\nATOM    208  N3   DT A   7       2.445   1.746  -2.708  1.00  0.00           N  \\nATOM    209  C4   DT A   7       3.120   1.734  -1.498  1.00  0.00           C  \\nATOM    210  O4   DT A   7       2.496   1.594  -0.449  1.00  0.00           O  \\nATOM    211  C5   DT A   7       4.558   1.885  -1.617  1.00  0.00           C  \\nATOM    212  C7   DT A   7       5.422   1.939  -0.371  1.00  0.00           C  \\nATOM    213  C6   DT A   7       5.142   1.992  -2.837  1.00  0.00           C  \\nATOM    214  H5'  DT A   7       8.155  -0.551  -6.034  1.00  0.00           H  \\nATOM    215 H5''  DT A   7       8.978   0.859  -6.713  1.00  0.00           H  \\nATOM    216  H4'  DT A   7       6.575   0.544  -7.412  1.00  0.00           H  \\nATOM    217  H3'  DT A   7       7.851   2.993  -6.814  1.00  0.00           H  \\nATOM    218  H2'  DT A   7       6.781   3.227  -4.736  1.00  0.00           H  \\nATOM    219 H2''  DT A   7       5.578   4.072  -5.733  1.00  0.00           H  \\nATOM    220  H1'  DT A   7       4.370   2.022  -6.154  1.00  0.00           H  \\nATOM    221  H3   DT A   7       1.442   1.618  -2.682  1.00  0.00           H  \\nATOM    222  H71  DT A   7       4.825   2.215   0.499  1.00  0.00           H  \\nATOM    223  H72  DT A   7       5.875   0.967  -0.192  1.00  0.00           H  \\nATOM    224  H73  DT A   7       6.214   2.678  -0.498  1.00  0.00           H  \\nATOM    225  H6   DT A   7       6.217   2.067  -2.881  1.00  0.00           H  \\nATOM    226  P    DG A   8       5.963   4.274  -8.646  1.00  0.00           P  \\nATOM    227  OP1  DG A   8       5.644   4.099 -10.080  1.00  0.00           O  \\nATOM    228  OP2  DG A   8       7.110   5.120  -8.246  1.00  0.00           O  \\nATOM    229  O5'  DG A   8       4.646   4.830  -7.903  1.00  0.00           O  \\nATOM    230  C5'  DG A   8       3.371   4.257  -8.123  1.00  0.00           C  \\nATOM    231  C4'  DG A   8       2.339   4.891  -7.185  1.00  0.00           C  \\nATOM    232  O4'  DG A   8       2.698   4.651  -5.832  1.00  0.00           O  \\nATOM    233  C3'  DG A   8       2.247   6.411  -7.372  1.00  0.00           C  \\nATOM    234  O3'  DG A   8       0.879   6.770  -7.293  1.00  0.00           O  \\nATOM    235  C2'  DG A   8       3.050   6.921  -6.174  1.00  0.00           C  \\nATOM    236  C1'  DG A   8       2.659   5.879  -5.130  1.00  0.00           C  \\nATOM    237  N9   DG A   8       3.547   5.807  -3.945  1.00  0.00           N  \\nATOM    238  C8   DG A   8       4.888   6.082  -3.843  1.00  0.00           C  \\nATOM    239  N7   DG A   8       5.390   5.876  -2.658  1.00  0.00           N  \\nATOM    240  C5   DG A   8       4.305   5.439  -1.907  1.00  0.00           C  \\nATOM    241  C6   DG A   8       4.229   5.086  -0.526  1.00  0.00           C  \\nATOM    242  O6   DG A   8       5.128   5.089   0.311  1.00  0.00           O  \\nATOM    243  N1   DG A   8       2.946   4.704  -0.143  1.00  0.00           N  \\nATOM    244  C2   DG A   8       1.856   4.677  -0.992  1.00  0.00           C  \\nATOM    245  N2   DG A   8       0.685   4.316  -0.461  1.00  0.00           N  \\nATOM    246  N3   DG A   8       1.926   5.007  -2.291  1.00  0.00           N  \\nATOM    247  C4   DG A   8       3.174   5.384  -2.687  1.00  0.00           C  \\nATOM    248  H5'  DG A   8       3.402   3.183  -7.939  1.00  0.00           H  \\nATOM    249 H5''  DG A   8       3.067   4.426  -9.157  1.00  0.00           H  \\nATOM    250  H4'  DG A   8       1.370   4.430  -7.385  1.00  0.00           H  \\nATOM    251  H3'  DG A   8       2.676   6.727  -8.324  1.00  0.00           H  \\nATOM    252  H2'  DG A   8       4.113   6.867  -6.415  1.00  0.00           H  \\nATOM    253 H2''  DG A   8       2.783   7.932  -5.866  1.00  0.00           H  \\nATOM    254  H1'  DG A   8       1.641   6.111  -4.807  1.00  0.00           H  \\nATOM    255  H8   DG A   8       5.491   6.434  -4.667  1.00  0.00           H  \\nATOM    256  H1   DG A   8       2.822   4.440   0.822  1.00  0.00           H  \\nATOM    257  H21  DG A   8       0.614   4.079   0.517  1.00  0.00           H  \\nATOM    258  H22  DG A   8      -0.150   4.335  -1.037  1.00  0.00           H  \\nATOM    259  P    DG A   9       0.330   8.218  -7.744  1.00  0.00           P  \\nATOM    260  OP1  DG A   9      -0.439   8.061  -8.998  1.00  0.00           O  \\nATOM    261  OP2  DG A   9       1.438   9.197  -7.672  1.00  0.00           O  \\nATOM    262  O5'  DG A   9      -0.690   8.535  -6.543  1.00  0.00           O  \\nATOM    263  C5'  DG A   9      -2.012   8.031  -6.519  1.00  0.00           C  \\nATOM    264  C4'  DG A   9      -2.566   8.145  -5.094  1.00  0.00           C  \\nATOM    265  O4'  DG A   9      -1.825   7.353  -4.184  1.00  0.00           O  \\nATOM    266  C3'  DG A   9      -2.506   9.580  -4.566  1.00  0.00           C  \\nATOM    267  O3'  DG A   9      -3.619  10.313  -5.044  1.00  0.00           O  \\nATOM    268  C2'  DG A   9      -2.497   9.337  -3.063  1.00  0.00           C  \\nATOM    269  C1'  DG A   9      -1.771   7.994  -2.919  1.00  0.00           C  \\nATOM    270  N9   DG A   9      -0.338   8.108  -2.538  1.00  0.00           N  \\nATOM    271  C8   DG A   9       0.746   8.359  -3.343  1.00  0.00           C  \\nATOM    272  N7   DG A   9       1.893   8.366  -2.727  1.00  0.00           N  \\nATOM    273  C5   DG A   9       1.561   8.070  -1.412  1.00  0.00           C  \\nATOM    274  C6   DG A   9       2.401   7.935  -0.265  1.00  0.00           C  \\nATOM    275  O6   DG A   9       3.614   8.107  -0.190  1.00  0.00           O  \\nATOM    276  N1   DG A   9       1.693   7.555   0.872  1.00  0.00           N  \\nATOM    277  C2   DG A   9       0.328   7.335   0.900  1.00  0.00           C  \\nATOM    278  N2   DG A   9      -0.210   6.988   2.072  1.00  0.00           N  \\nATOM    279  N3   DG A   9      -0.460   7.469  -0.177  1.00  0.00           N  \\nATOM    280  C4   DG A   9       0.207   7.851  -1.298  1.00  0.00           C  \\nATOM    281  H5'  DG A   9      -2.045   6.989  -6.838  1.00  0.00           H  \\nATOM    282 H5''  DG A   9      -2.628   8.623  -7.197  1.00  0.00           H  \\nATOM    283  H4'  DG A   9      -3.595   7.780  -5.082  1.00  0.00           H  \\nATOM    284  H3'  DG A   9      -1.573  10.062  -4.854  1.00  0.00           H  \\nATOM    285  H2'  DG A   9      -2.015  10.137  -2.518  1.00  0.00           H  \\nATOM    286 H2''  DG A   9      -3.515   9.237  -2.692  1.00  0.00           H  \\nATOM    287  H1'  DG A   9      -2.311   7.429  -2.161  1.00  0.00           H  \\nATOM    288  H8   DG A   9       0.678   8.518  -4.403  1.00  0.00           H  \\nATOM    289  H1   DG A   9       2.224   7.458   1.723  1.00  0.00           H  \\nATOM    290  H21  DG A   9       0.352   6.926   2.907  1.00  0.00           H  \\nATOM    291  H22  DG A   9      -1.215   6.851   2.123  1.00  0.00           H  \\nATOM    292  P    DA A  10      -3.768  11.908  -4.844  1.00  0.00           P  \\nATOM    293  OP1  DA A  10      -4.648  12.420  -5.917  1.00  0.00           O  \\nATOM    294  OP2  DA A  10      -2.422  12.498  -4.662  1.00  0.00           O  \\nATOM    295  O5'  DA A  10      -4.564  12.045  -3.452  1.00  0.00           O  \\nATOM    296  C5'  DA A  10      -5.795  11.380  -3.232  1.00  0.00           C  \\nATOM    297  C4'  DA A  10      -6.574  12.044  -2.090  1.00  0.00           C  \\nATOM    298  O4'  DA A  10      -5.681  12.349  -1.028  1.00  0.00           O  \\nATOM    299  C3'  DA A  10      -7.245  13.358  -2.515  1.00  0.00           C  \\nATOM    300  O3'  DA A  10      -8.433  13.513  -1.751  1.00  0.00           O  \\nATOM    301  C2'  DA A  10      -6.168  14.372  -2.152  1.00  0.00           C  \\nATOM    302  C1'  DA A  10      -5.636  13.757  -0.862  1.00  0.00           C  \\nATOM    303  N9   DA A  10      -4.269  14.207  -0.519  1.00  0.00           N  \\nATOM    304  C8   DA A  10      -3.167  14.362  -1.327  1.00  0.00           C  \\nATOM    305  N7   DA A  10      -2.080  14.697  -0.690  1.00  0.00           N  \\nATOM    306  C5   DA A  10      -2.485  14.770   0.637  1.00  0.00           C  \\nATOM    307  C6   DA A  10      -1.807  15.061   1.839  1.00  0.00           C  \\nATOM    308  N6   DA A  10      -0.504  15.345   1.885  1.00  0.00           N  \\nATOM    309  N1   DA A  10      -2.508  15.070   2.989  1.00  0.00           N  \\nATOM    310  C2   DA A  10      -3.806  14.776   2.951  1.00  0.00           C  \\nATOM    311  N3   DA A  10      -4.552  14.464   1.896  1.00  0.00           N  \\nATOM    312  C4   DA A  10      -3.822  14.485   0.749  1.00  0.00           C  \\nATOM    313  H5'  DA A  10      -5.589  10.343  -2.966  1.00  0.00           H  \\nATOM    314 H5''  DA A  10      -6.412  11.388  -4.132  1.00  0.00           H  \\nATOM    315  H4'  DA A  10      -7.333  11.339  -1.748  1.00  0.00           H  \\nATOM    316  H3'  DA A  10      -7.475  13.385  -3.581  1.00  0.00           H  \\nATOM    317  H2'  DA A  10      -5.414  14.386  -2.937  1.00  0.00           H  \\nATOM    318 H2''  DA A  10      -6.554  15.376  -2.003  1.00  0.00           H  \\nATOM    319  H1'  DA A  10      -6.323  14.024  -0.061  1.00  0.00           H  \\nATOM    320  H8   DA A  10      -3.184  14.211  -2.396  1.00  0.00           H  \\nATOM    321  H61  DA A  10      -0.075  15.609   2.761  1.00  0.00           H  \\nATOM    322  H62  DA A  10       0.069  15.235   1.057  1.00  0.00           H  \\nATOM    323  H2   DA A  10      -4.325  14.780   3.896  1.00  0.00           H  \\nATOM    324  P    DA A  11      -9.394  14.810  -1.869  1.00  0.00           P  \\nATOM    325  OP1  DA A  11     -10.740  14.433  -1.386  1.00  0.00           O  \\nATOM    326  OP2  DA A  11      -9.234  15.394  -3.219  1.00  0.00           O  \\nATOM    327  O5'  DA A  11      -8.742  15.826  -0.800  1.00  0.00           O  \\nATOM    328  C5'  DA A  11      -8.827  15.586   0.592  1.00  0.00           C  \\nATOM    329  C4'  DA A  11      -7.934  16.582   1.339  1.00  0.00           C  \\nATOM    330  O4'  DA A  11      -6.600  16.494   0.866  1.00  0.00           O  \\nATOM    331  C3'  DA A  11      -8.405  18.029   1.139  1.00  0.00           C  \\nATOM    332  O3'  DA A  11      -8.526  18.646   2.413  1.00  0.00           O  \\nATOM    333  C2'  DA A  11      -7.288  18.627   0.284  1.00  0.00           C  \\nATOM    334  C1'  DA A  11      -6.088  17.802   0.739  1.00  0.00           C  \\nATOM    335  N9   DA A  11      -4.945  17.818  -0.203  1.00  0.00           N  \\nATOM    336  C8   DA A  11      -4.941  17.661  -1.568  1.00  0.00           C  \\nATOM    337  N7   DA A  11      -3.754  17.670  -2.108  1.00  0.00           N  \\nATOM    338  C5   DA A  11      -2.903  17.845  -1.024  1.00  0.00           C  \\nATOM    339  C6   DA A  11      -1.500  17.949  -0.911  1.00  0.00           C  \\nATOM    340  N6   DA A  11      -0.681  17.883  -1.966  1.00  0.00           N  \\nATOM    341  N1   DA A  11      -0.968  18.143   0.310  1.00  0.00           N  \\nATOM    342  C2   DA A  11      -1.779  18.218   1.364  1.00  0.00           C  \\nATOM    343  N3   DA A  11      -3.106  18.128   1.391  1.00  0.00           N  \\nATOM    344  C4   DA A  11      -3.618  17.939   0.144  1.00  0.00           C  \\nATOM    345  H5'  DA A  11      -8.503  14.572   0.827  1.00  0.00           H  \\nATOM    346 H5''  DA A  11      -9.859  15.703   0.925  1.00  0.00           H  \\nATOM    347  H4'  DA A  11      -7.947  16.329   2.400  1.00  0.00           H  \\nATOM    348  H3'  DA A  11      -9.358  18.055   0.610  1.00  0.00           H  \\nATOM    349  H2'  DA A  11      -7.517  18.432  -0.763  1.00  0.00           H  \\nATOM    350 H2''  DA A  11      -7.149  19.695   0.441  1.00  0.00           H  \\nATOM    351  H1'  DA A  11      -5.765  18.170   1.712  1.00  0.00           H  \\nATOM    352  H8   DA A  11      -5.838  17.538  -2.157  1.00  0.00           H  \\nATOM    353  H61  DA A  11       0.317  17.992  -1.843  1.00  0.00           H  \\nATOM    354  H62  DA A  11      -1.069  17.748  -2.888  1.00  0.00           H  \\nATOM    355  H2   DA A  11      -1.298  18.375   2.317  1.00  0.00           H  \\nATOM    356  P    DC A  12      -9.152  20.124   2.613  1.00  0.00           P  \\nATOM    357  OP1  DC A  12     -10.116  20.076   3.734  1.00  0.00           O  \\nATOM    358  OP2  DC A  12      -9.578  20.645   1.295  1.00  0.00           O  \\nATOM    359  O5'  DC A  12      -7.866  20.968   3.088  1.00  0.00           O  \\nATOM    360  C5'  DC A  12      -7.409  20.939   4.430  1.00  0.00           C  \\nATOM    361  C4'  DC A  12      -5.979  21.486   4.471  1.00  0.00           C  \\nATOM    362  O4'  DC A  12      -5.178  20.822   3.513  1.00  0.00           O  \\nATOM    363  C3'  DC A  12      -5.913  22.975   4.122  1.00  0.00           C  \\nATOM    364  O3'  DC A  12      -5.844  23.770   5.287  1.00  0.00           O  \\nATOM    365  C2'  DC A  12      -4.638  23.102   3.292  1.00  0.00           C  \\nATOM    366  C1'  DC A  12      -4.097  21.674   3.191  1.00  0.00           C  \\nATOM    367  N1   DC A  12      -3.608  21.408   1.811  1.00  0.00           N  \\nATOM    368  C2   DC A  12      -2.235  21.410   1.557  1.00  0.00           C  \\nATOM    369  O2   DC A  12      -1.416  21.525   2.466  1.00  0.00           O  \\nATOM    370  N3   DC A  12      -1.814  21.268   0.267  1.00  0.00           N  \\nATOM    371  C4   DC A  12      -2.689  21.136  -0.741  1.00  0.00           C  \\nATOM    372  N4   DC A  12      -2.217  20.980  -1.981  1.00  0.00           N  \\nATOM    373  C5   DC A  12      -4.105  21.125  -0.502  1.00  0.00           C  \\nATOM    374  C6   DC A  12      -4.508  21.262   0.783  1.00  0.00           C  \\nATOM    375  H5'  DC A  12      -7.415  19.923   4.828  1.00  0.00           H  \\nATOM    376 H5''  DC A  12      -8.057  21.555   5.054  1.00  0.00           H  \\nATOM    377  H4'  DC A  12      -5.546  21.325   5.460  1.00  0.00           H  \\nATOM    378  H3'  DC A  12      -6.768  23.275   3.514  1.00  0.00           H  \\nATOM    379 HO3'  DC A  12      -6.657  23.652   5.783  1.00  0.00           H  \\nATOM    380  H2'  DC A  12      -4.895  23.497   2.309  1.00  0.00           H  \\nATOM    381 H2''  DC A  12      -3.902  23.753   3.761  1.00  0.00           H  \\nATOM    382  H1'  DC A  12      -3.331  21.535   3.953  1.00  0.00           H  \\nATOM    383  H41  DC A  12      -1.216  20.953  -2.129  1.00  0.00           H  \\nATOM    384  H42  DC A  12      -2.850  20.847  -2.756  1.00  0.00           H  \\nATOM    385  H5   DC A  12      -4.842  21.010  -1.284  1.00  0.00           H  \\nATOM    386  H6   DC A  12      -5.562  21.282   1.013  1.00  0.00           H  \\nTER     387       DC A  12                                                      \\nATOM    388  O5'  DG B  13       8.692  22.125  -1.628  1.00  0.00           O  \\nATOM    389  C5'  DG B  13       8.994  22.802  -0.426  1.00  0.00           C  \\nATOM    390  C4'  DG B  13       8.201  22.220   0.750  1.00  0.00           C  \\nATOM    391  O4'  DG B  13       6.811  22.374   0.501  1.00  0.00           O  \\nATOM    392  C3'  DG B  13       8.479  20.725   0.965  1.00  0.00           C  \\nATOM    393  O3'  DG B  13       8.592  20.475   2.357  1.00  0.00           O  \\nATOM    394  C2'  DG B  13       7.237  20.084   0.356  1.00  0.00           C  \\nATOM    395  C1'  DG B  13       6.171  21.124   0.683  1.00  0.00           C  \\nATOM    396  N9   DG B  13       4.994  21.017  -0.212  1.00  0.00           N  \\nATOM    397  C8   DG B  13       4.951  21.135  -1.579  1.00  0.00           C  \\nATOM    398  N7   DG B  13       3.756  21.031  -2.089  1.00  0.00           N  \\nATOM    399  C5   DG B  13       2.936  20.853  -0.981  1.00  0.00           C  \\nATOM    400  C6   DG B  13       1.518  20.705  -0.906  1.00  0.00           C  \\nATOM    401  O6   DG B  13       0.711  20.664  -1.832  1.00  0.00           O  \\nATOM    402  N1   DG B  13       1.063  20.603   0.405  1.00  0.00           N  \\nATOM    403  C2   DG B  13       1.884  20.606   1.519  1.00  0.00           C  \\nATOM    404  N2   DG B  13       1.294  20.495   2.713  1.00  0.00           N  \\nATOM    405  N3   DG B  13       3.220  20.717   1.448  1.00  0.00           N  \\nATOM    406  C4   DG B  13       3.684  20.847   0.175  1.00  0.00           C  \\nATOM    407  H5'  DG B  13       8.746  23.858  -0.542  1.00  0.00           H  \\nATOM    408 H5''  DG B  13      10.062  22.711  -0.224  1.00  0.00           H  \\nATOM    409  H4'  DG B  13       8.472  22.781   1.647  1.00  0.00           H  \\nATOM    410  H3'  DG B  13       9.390  20.408   0.453  1.00  0.00           H  \\nATOM    411  H2'  DG B  13       7.377  20.000  -0.722  1.00  0.00           H  \\nATOM    412 H2''  DG B  13       7.005  19.107   0.775  1.00  0.00           H  \\nATOM    413  H1'  DG B  13       5.878  21.020   1.728  1.00  0.00           H  \\nATOM    414  H8   DG B  13       5.829  21.295  -2.187  1.00  0.00           H  \\nATOM    415  H1   DG B  13       0.061  20.531   0.528  1.00  0.00           H  \\nATOM    416  H21  DG B  13       0.285  20.475   2.782  1.00  0.00           H  \\nATOM    417  H22  DG B  13       1.862  20.474   3.547  1.00  0.00           H  \\nATOM    418 HO5'  DG B  13       9.205  22.519  -2.337  1.00  0.00           H  \\nATOM    419  P    DT B  14       8.934  19.011   2.956  1.00  0.00           P  \\nATOM    420  OP1  DT B  14       9.823  19.191   4.125  1.00  0.00           O  \\nATOM    421  OP2  DT B  14       9.354  18.131   1.842  1.00  0.00           O  \\nATOM    422  O5'  DT B  14       7.499  18.499   3.486  1.00  0.00           O  \\nATOM    423  C5'  DT B  14       6.884  19.096   4.612  1.00  0.00           C  \\nATOM    424  C4'  DT B  14       5.519  18.461   4.891  1.00  0.00           C  \\nATOM    425  O4'  DT B  14       4.686  18.604   3.750  1.00  0.00           O  \\nATOM    426  C3'  DT B  14       5.603  16.962   5.226  1.00  0.00           C  \\nATOM    427  O3'  DT B  14       4.750  16.719   6.336  1.00  0.00           O  \\nATOM    428  C2'  DT B  14       5.097  16.325   3.932  1.00  0.00           C  \\nATOM    429  C1'  DT B  14       4.057  17.362   3.513  1.00  0.00           C  \\nATOM    430  N1   DT B  14       3.606  17.278   2.096  1.00  0.00           N  \\nATOM    431  C2   DT B  14       2.237  17.366   1.835  1.00  0.00           C  \\nATOM    432  O2   DT B  14       1.379  17.386   2.714  1.00  0.00           O  \\nATOM    433  N3   DT B  14       1.869  17.464   0.502  1.00  0.00           N  \\nATOM    434  C4   DT B  14       2.730  17.523  -0.582  1.00  0.00           C  \\nATOM    435  O4   DT B  14       2.281  17.673  -1.715  1.00  0.00           O  \\nATOM    436  C5   DT B  14       4.133  17.400  -0.235  1.00  0.00           C  \\nATOM    437  C7   DT B  14       5.185  17.396  -1.328  1.00  0.00           C  \\nATOM    438  C6   DT B  14       4.516  17.289   1.062  1.00  0.00           C  \\nATOM    439  H5'  DT B  14       6.743  20.162   4.431  1.00  0.00           H  \\nATOM    440 H5''  DT B  14       7.517  18.970   5.492  1.00  0.00           H  \\nATOM    441  H4'  DT B  14       5.064  18.997   5.726  1.00  0.00           H  \\nATOM    442  H3'  DT B  14       6.624  16.653   5.456  1.00  0.00           H  \\nATOM    443  H2'  DT B  14       5.932  16.270   3.237  1.00  0.00           H  \\nATOM    444 H2''  DT B  14       4.669  15.334   4.062  1.00  0.00           H  \\nATOM    445  H1'  DT B  14       3.224  17.267   4.210  1.00  0.00           H  \\nATOM    446  H3   DT B  14       0.880  17.558   0.315  1.00  0.00           H  \\nATOM    447  H71  DT B  14       4.756  17.043  -2.266  1.00  0.00           H  \\nATOM    448  H72  DT B  14       5.565  18.405  -1.473  1.00  0.00           H  \\nATOM    449  H73  DT B  14       6.011  16.740  -1.054  1.00  0.00           H  \\nATOM    450  H6   DT B  14       5.569  17.235   1.281  1.00  0.00           H  \\nATOM    451  P    DG B  15       4.549  15.254   6.990  1.00  0.00           P  \\nATOM    452  OP1  DG B  15       4.020  15.432   8.361  1.00  0.00           O  \\nATOM    453  OP2  DG B  15       5.783  14.468   6.774  1.00  0.00           O  \\nATOM    454  O5'  DG B  15       3.381  14.619   6.080  1.00  0.00           O  \\nATOM    455  C5'  DG B  15       2.063  15.133   6.103  1.00  0.00           C  \\nATOM    456  C4'  DG B  15       1.210  14.441   5.036  1.00  0.00           C  \\nATOM    457  O4'  DG B  15       1.766  14.658   3.749  1.00  0.00           O  \\nATOM    458  C3'  DG B  15       1.137  12.925   5.254  1.00  0.00           C  \\nATOM    459  O3'  DG B  15      -0.200  12.527   5.011  1.00  0.00           O  \\nATOM    460  C2'  DG B  15       2.104  12.403   4.190  1.00  0.00           C  \\nATOM    461  C1'  DG B  15       1.853  13.416   3.078  1.00  0.00           C  \\nATOM    462  N9   DG B  15       2.902  13.491   2.034  1.00  0.00           N  \\nATOM    463  C8   DG B  15       4.250  13.246   2.131  1.00  0.00           C  \\nATOM    464  N7   DG B  15       4.913  13.452   1.028  1.00  0.00           N  \\nATOM    465  C5   DG B  15       3.939  13.858   0.124  1.00  0.00           C  \\nATOM    466  C6   DG B  15       4.056  14.199  -1.258  1.00  0.00           C  \\nATOM    467  O6   DG B  15       5.066  14.207  -1.957  1.00  0.00           O  \\nATOM    468  N1   DG B  15       2.835  14.554  -1.824  1.00  0.00           N  \\nATOM    469  C2   DG B  15       1.633  14.569  -1.140  1.00  0.00           C  \\nATOM    470  N2   DG B  15       0.548  14.927  -1.832  1.00  0.00           N  \\nATOM    471  N3   DG B  15       1.521  14.247   0.158  1.00  0.00           N  \\nATOM    472  C4   DG B  15       2.706  13.896   0.732  1.00  0.00           C  \\nATOM    473  H5'  DG B  15       2.071  16.205   5.906  1.00  0.00           H  \\nATOM    474 H5''  DG B  15       1.619  14.963   7.085  1.00  0.00           H  \\nATOM    475  H4'  DG B  15       0.208  14.873   5.072  1.00  0.00           H  \\nATOM    476  H3'  DG B  15       1.450  12.654   6.263  1.00  0.00           H  \\nATOM    477  H2'  DG B  15       3.120  12.479   4.576  1.00  0.00           H  \\nATOM    478 H2''  DG B  15       1.898  11.382   3.872  1.00  0.00           H  \\nATOM    479  H1'  DG B  15       0.900  13.151   2.615  1.00  0.00           H  \\nATOM    480  H8   DG B  15       4.735  12.918   3.038  1.00  0.00           H  \\nATOM    481  H1   DG B  15       2.846  14.814  -2.798  1.00  0.00           H  \\nATOM    482  H21  DG B  15       0.616  15.164  -2.810  1.00  0.00           H  \\nATOM    483  H22  DG B  15      -0.361  14.900  -1.383  1.00  0.00           H  \\nATOM    484  P    DG B  16      -0.764  11.068   5.404  1.00  0.00           P  \\nATOM    485  OP1  DG B  16      -1.722  11.216   6.523  1.00  0.00           O  \\nATOM    486  OP2  DG B  16       0.371  10.125   5.520  1.00  0.00           O  \\nATOM    487  O5'  DG B  16      -1.578  10.705   4.067  1.00  0.00           O  \\nATOM    488  C5'  DG B  16      -2.876  11.208   3.813  1.00  0.00           C  \\nATOM    489  C4'  DG B  16      -3.169  11.119   2.311  1.00  0.00           C  \\nATOM    490  O4'  DG B  16      -2.280  11.917   1.551  1.00  0.00           O  \\nATOM    491  C3'  DG B  16      -3.020   9.694   1.781  1.00  0.00           C  \\nATOM    492  O3'  DG B  16      -4.198   8.962   2.067  1.00  0.00           O  \\nATOM    493  C2'  DG B  16      -2.782   9.959   0.301  1.00  0.00           C  \\nATOM    494  C1'  DG B  16      -2.023  11.290   0.303  1.00  0.00           C  \\nATOM    495  N9   DG B  16      -0.551  11.139   0.173  1.00  0.00           N  \\nATOM    496  C8   DG B  16       0.368  10.841   1.147  1.00  0.00           C  \\nATOM    497  N7   DG B  16       1.604  10.808   0.735  1.00  0.00           N  \\nATOM    498  C5   DG B  16       1.510  11.123  -0.614  1.00  0.00           C  \\nATOM    499  C6   DG B  16       2.534  11.240  -1.603  1.00  0.00           C  \\nATOM    500  O6   DG B  16       3.740  11.050  -1.471  1.00  0.00           O  \\nATOM    501  N1   DG B  16       2.035  11.623  -2.845  1.00  0.00           N  \\nATOM    502  C2   DG B  16       0.701  11.877  -3.102  1.00  0.00           C  \\nATOM    503  N2   DG B  16       0.368  12.229  -4.348  1.00  0.00           N  \\nATOM    504  N3   DG B  16      -0.258  11.773  -2.170  1.00  0.00           N  \\nATOM    505  C4   DG B  16       0.202  11.377  -0.955  1.00  0.00           C  \\nATOM    506  H5'  DG B  16      -2.970  12.244   4.140  1.00  0.00           H  \\nATOM    507 H5''  DG B  16      -3.600  10.605   4.362  1.00  0.00           H  \\nATOM    508  H4'  DG B  16      -4.180  11.488   2.125  1.00  0.00           H  \\nATOM    509  H3'  DG B  16      -2.132   9.221   2.201  1.00  0.00           H  \\nATOM    510  H2'  DG B  16      -2.231   9.161  -0.180  1.00  0.00           H  \\nATOM    511 H2''  DG B  16      -3.729  10.087  -0.218  1.00  0.00           H  \\nATOM    512  H1'  DG B  16      -2.411  11.875  -0.530  1.00  0.00           H  \\nATOM    513  H8   DG B  16       0.105  10.668   2.176  1.00  0.00           H  \\nATOM    514  H1   DG B  16       2.705  11.704  -3.595  1.00  0.00           H  \\nATOM    515  H21  DG B  16       1.060  12.268  -5.081  1.00  0.00           H  \\nATOM    516  H22  DG B  16      -0.613  12.385  -4.562  1.00  0.00           H  \\nATOM    517  P    DA B  17      -4.335   7.377   1.798  1.00  0.00           P  \\nATOM    518  OP1  DA B  17      -5.461   6.874   2.615  1.00  0.00           O  \\nATOM    519  OP2  DA B  17      -3.000   6.754   1.940  1.00  0.00           O  \\nATOM    520  O5'  DA B  17      -4.768   7.278   0.251  1.00  0.00           O  \\nATOM    521  C5'  DA B  17      -5.940   7.904  -0.242  1.00  0.00           C  \\nATOM    522  C4'  DA B  17      -6.463   7.161  -1.478  1.00  0.00           C  \\nATOM    523  O4'  DA B  17      -5.368   6.847  -2.328  1.00  0.00           O  \\nATOM    524  C3'  DA B  17      -7.159   5.840  -1.116  1.00  0.00           C  \\nATOM    525  O3'  DA B  17      -8.183   5.587  -2.069  1.00  0.00           O  \\nATOM    526  C2'  DA B  17      -6.000   4.859  -1.228  1.00  0.00           C  \\nATOM    527  C1'  DA B  17      -5.273   5.437  -2.438  1.00  0.00           C  \\nATOM    528  N9   DA B  17      -3.863   4.998  -2.544  1.00  0.00           N  \\nATOM    529  C8   DA B  17      -2.903   4.877  -1.566  1.00  0.00           C  \\nATOM    530  N7   DA B  17      -1.726   4.537  -2.013  1.00  0.00           N  \\nATOM    531  C5   DA B  17      -1.913   4.421  -3.384  1.00  0.00           C  \\nATOM    532  C6   DA B  17      -1.051   4.100  -4.455  1.00  0.00           C  \\nATOM    533  N6   DA B  17       0.243   3.819  -4.289  1.00  0.00           N  \\nATOM    534  N1   DA B  17      -1.560   4.059  -5.701  1.00  0.00           N  \\nATOM    535  C2   DA B  17      -2.848   4.344  -5.879  1.00  0.00           C  \\nATOM    536  N3   DA B  17      -3.755   4.674  -4.965  1.00  0.00           N  \\nATOM    537  C4   DA B  17      -3.217   4.689  -3.716  1.00  0.00           C  \\nATOM    538  H5'  DA B  17      -5.702   8.931  -0.517  1.00  0.00           H  \\nATOM    539 H5''  DA B  17      -6.727   7.926   0.514  1.00  0.00           H  \\nATOM    540  H4'  DA B  17      -7.165   7.816  -1.997  1.00  0.00           H  \\nATOM    541  H3'  DA B  17      -7.579   5.858  -0.109  1.00  0.00           H  \\nATOM    542  H2'  DA B  17      -5.394   4.922  -0.325  1.00  0.00           H  \\nATOM    543 H2''  DA B  17      -6.320   3.833  -1.385  1.00  0.00           H  \\nATOM    544  H1'  DA B  17      -5.820   5.137  -3.330  1.00  0.00           H  \\nATOM    545  H8   DA B  17      -3.088   5.063  -0.519  1.00  0.00           H  \\nATOM    546  H61  DA B  17       0.806   3.543  -5.082  1.00  0.00           H  \\nATOM    547  H62  DA B  17       0.680   3.957  -3.385  1.00  0.00           H  \\nATOM    548  H2   DA B  17      -3.208   4.315  -6.895  1.00  0.00           H  \\nATOM    549  P    DA B  18      -9.096   4.248  -2.047  1.00  0.00           P  \\nATOM    550  OP1  DA B  18     -10.379   4.557  -2.718  1.00  0.00           O  \\nATOM    551  OP2  DA B  18      -9.100   3.703  -0.672  1.00  0.00           O  \\nATOM    552  O5'  DA B  18      -8.271   3.230  -2.990  1.00  0.00           O  \\nATOM    553  C5'  DA B  18      -8.181   3.435  -4.388  1.00  0.00           C  \\nATOM    554  C4'  DA B  18      -7.174   2.461  -5.006  1.00  0.00           C  \\nATOM    555  O4'  DA B  18      -5.925   2.601  -4.346  1.00  0.00           O  \\nATOM    556  C3'  DA B  18      -7.604   0.989  -4.895  1.00  0.00           C  \\nATOM    557  O3'  DA B  18      -7.484   0.375  -6.170  1.00  0.00           O  \\nATOM    558  C2'  DA B  18      -6.601   0.434  -3.886  1.00  0.00           C  \\nATOM    559  C1'  DA B  18      -5.382   1.315  -4.152  1.00  0.00           C  \\nATOM    560  N9   DA B  18      -4.402   1.326  -3.039  1.00  0.00           N  \\nATOM    561  C8   DA B  18      -4.614   1.542  -1.697  1.00  0.00           C  \\nATOM    562  N7   DA B  18      -3.529   1.560  -0.976  1.00  0.00           N  \\nATOM    563  C5   DA B  18      -2.519   1.342  -1.902  1.00  0.00           C  \\nATOM    564  C6   DA B  18      -1.117   1.255  -1.784  1.00  0.00           C  \\nATOM    565  N6   DA B  18      -0.476   1.392  -0.618  1.00  0.00           N  \\nATOM    566  N1   DA B  18      -0.395   1.024  -2.896  1.00  0.00           N  \\nATOM    567  C2   DA B  18      -1.026   0.881  -4.059  1.00  0.00           C  \\nATOM    568  N3   DA B  18      -2.333   0.946  -4.302  1.00  0.00           N  \\nATOM    569  C4   DA B  18      -3.039   1.185  -3.163  1.00  0.00           C  \\nATOM    570  H5'  DA B  18      -7.845   4.453  -4.590  1.00  0.00           H  \\nATOM    571 H5''  DA B  18      -9.158   3.290  -4.850  1.00  0.00           H  \\nATOM    572  H4'  DA B  18      -7.051   2.726  -6.058  1.00  0.00           H  \\nATOM    573  H3'  DA B  18      -8.627   0.896  -4.526  1.00  0.00           H  \\nATOM    574  H2'  DA B  18      -7.010   0.605  -2.891  1.00  0.00           H  \\nATOM    575 H2''  DA B  18      -6.382  -0.623  -4.025  1.00  0.00           H  \\nATOM    576  H1'  DA B  18      -4.899   0.992  -5.075  1.00  0.00           H  \\nATOM    577  H8   DA B  18      -5.589   1.678  -1.256  1.00  0.00           H  \\nATOM    578  H61  DA B  18       0.532   1.327  -0.581  1.00  0.00           H  \\nATOM    579  H62  DA B  18      -1.009   1.569   0.221  1.00  0.00           H  \\nATOM    580  H2   DA B  18      -0.392   0.704  -4.915  1.00  0.00           H  \\nATOM    581  P    DT B  19      -7.936  -1.153  -6.457  1.00  0.00           P  \\nATOM    582  OP1  DT B  19      -8.773  -1.166  -7.678  1.00  0.00           O  \\nATOM    583  OP2  DT B  19      -8.456  -1.744  -5.203  1.00  0.00           O  \\nATOM    584  O5'  DT B  19      -6.529  -1.863  -6.800  1.00  0.00           O  \\nATOM    585  C5'  DT B  19      -5.855  -1.593  -8.016  1.00  0.00           C  \\nATOM    586  C4'  DT B  19      -4.433  -2.160  -7.984  1.00  0.00           C  \\nATOM    587  O4'  DT B  19      -3.781  -1.731  -6.798  1.00  0.00           O  \\nATOM    588  C3'  DT B  19      -4.383  -3.697  -8.021  1.00  0.00           C  \\nATOM    589  O3'  DT B  19      -3.372  -4.076  -8.945  1.00  0.00           O  \\nATOM    590  C2'  DT B  19      -4.022  -4.033  -6.575  1.00  0.00           C  \\nATOM    591  C1'  DT B  19      -3.120  -2.847  -6.238  1.00  0.00           C  \\nATOM    592  N1   DT B  19      -2.873  -2.640  -4.781  1.00  0.00           N  \\nATOM    593  C2   DT B  19      -1.555  -2.477  -4.349  1.00  0.00           C  \\nATOM    594  O2   DT B  19      -0.580  -2.560  -5.093  1.00  0.00           O  \\nATOM    595  N3   DT B  19      -1.378  -2.186  -3.004  1.00  0.00           N  \\nATOM    596  C4   DT B  19      -2.383  -2.014  -2.066  1.00  0.00           C  \\nATOM    597  O4   DT B  19      -2.096  -1.712  -0.910  1.00  0.00           O  \\nATOM    598  C5   DT B  19      -3.723  -2.209  -2.586  1.00  0.00           C  \\nATOM    599  C7   DT B  19      -4.919  -2.099  -1.660  1.00  0.00           C  \\nATOM    600  C6   DT B  19      -3.919  -2.501  -3.897  1.00  0.00           C  \\nATOM    601  H5'  DT B  19      -5.789  -0.516  -8.171  1.00  0.00           H  \\nATOM    602 H5''  DT B  19      -6.406  -2.027  -8.852  1.00  0.00           H  \\nATOM    603  H4'  DT B  19      -3.896  -1.762  -8.847  1.00  0.00           H  \\nATOM    604  H3'  DT B  19      -5.344  -4.128  -8.307  1.00  0.00           H  \\nATOM    605  H2'  DT B  19      -4.942  -4.038  -5.994  1.00  0.00           H  \\nATOM    606 H2''  DT B  19      -3.515  -4.987  -6.458  1.00  0.00           H  \\nATOM    607  H1'  DT B  19      -2.194  -2.994  -6.792  1.00  0.00           H  \\nATOM    608  H3   DT B  19      -0.426  -2.036  -2.699  1.00  0.00           H  \\nATOM    609  H71  DT B  19      -5.373  -1.115  -1.755  1.00  0.00           H  \\nATOM    610  H72  DT B  19      -5.662  -2.855  -1.918  1.00  0.00           H  \\nATOM    611  H73  DT B  19      -4.617  -2.249  -0.623  1.00  0.00           H  \\nATOM    612  H6   DT B  19      -4.933  -2.601  -4.249  1.00  0.00           H  \\nATOM    613  P    DG B  20      -3.026  -5.616  -9.294  1.00  0.00           P  \\nATOM    614  OP1  DG B  20      -2.294  -5.641 -10.580  1.00  0.00           O  \\nATOM    615  OP2  DG B  20      -4.256  -6.422  -9.133  1.00  0.00           O  \\nATOM    616  O5'  DG B  20      -1.998  -6.029  -8.124  1.00  0.00           O  \\nATOM    617  C5'  DG B  20      -0.704  -5.463  -8.038  1.00  0.00           C  \\nATOM    618  C4'  DG B  20      -0.007  -5.932  -6.756  1.00  0.00           C  \\nATOM    619  O4'  DG B  20      -0.749  -5.516  -5.619  1.00  0.00           O  \\nATOM    620  C3'  DG B  20       0.111  -7.460  -6.691  1.00  0.00           C  \\nATOM    621  O3'  DG B  20       1.386  -7.767  -6.152  1.00  0.00           O  \\nATOM    622  C2'  DG B  20      -1.024  -7.820  -5.730  1.00  0.00           C  \\nATOM    623  C1'  DG B  20      -0.947  -6.633  -4.774  1.00  0.00           C  \\nATOM    624  N9   DG B  20      -2.149  -6.417  -3.934  1.00  0.00           N  \\nATOM    625  C8   DG B  20      -3.464  -6.707  -4.203  1.00  0.00           C  \\nATOM    626  N7   DG B  20      -4.295  -6.345  -3.267  1.00  0.00           N  \\nATOM    627  C5   DG B  20      -3.479  -5.781  -2.295  1.00  0.00           C  \\nATOM    628  C6   DG B  20      -3.817  -5.231  -1.021  1.00  0.00           C  \\nATOM    629  O6   DG B  20      -4.927  -5.135  -0.503  1.00  0.00           O  \\nATOM    630  N1   DG B  20      -2.701  -4.768  -0.328  1.00  0.00           N  \\nATOM    631  C2   DG B  20      -1.405  -4.837  -0.802  1.00  0.00           C  \\nATOM    632  N2   DG B  20      -0.442  -4.372  -0.002  1.00  0.00           N  \\nATOM    633  N3   DG B  20      -1.086  -5.351  -2.000  1.00  0.00           N  \\nATOM    634  C4   DG B  20      -2.164  -5.811  -2.696  1.00  0.00           C  \\nATOM    635  H5'  DG B  20      -0.767  -4.375  -8.026  1.00  0.00           H  \\nATOM    636 H5''  DG B  20      -0.111  -5.770  -8.901  1.00  0.00           H  \\nATOM    637  H4'  DG B  20       0.985  -5.478  -6.723  1.00  0.00           H  \\nATOM    638  H3'  DG B  20      -0.015  -7.917  -7.673  1.00  0.00           H  \\nATOM    639  H2'  DG B  20      -1.964  -7.827  -6.285  1.00  0.00           H  \\nATOM    640 H2''  DG B  20      -0.880  -8.771  -5.219  1.00  0.00           H  \\nATOM    641  H1'  DG B  20      -0.078  -6.793  -4.131  1.00  0.00           H  \\nATOM    642  H8   DG B  20      -3.797  -7.185  -5.112  1.00  0.00           H  \\nATOM    643  H1   DG B  20      -2.869  -4.365   0.581  1.00  0.00           H  \\nATOM    644  H21  DG B  20      -0.665  -3.996   0.907  1.00  0.00           H  \\nATOM    645  H22  DG B  20       0.527  -4.453  -0.290  1.00  0.00           H  \\nATOM    646  P    DG B  21       2.024  -9.247  -6.211  1.00  0.00           P  \\nATOM    647  OP1  DG B  21       3.136  -9.244  -7.188  1.00  0.00           O  \\nATOM    648  OP2  DG B  21       0.932 -10.238  -6.335  1.00  0.00           O  \\nATOM    649  O5'  DG B  21       2.634  -9.366  -4.728  1.00  0.00           O  \\nATOM    650  C5'  DG B  21       3.892  -8.820  -4.379  1.00  0.00           C  \\nATOM    651  C4'  DG B  21       3.984  -8.711  -2.853  1.00  0.00           C  \\nATOM    652  O4'  DG B  21       3.006  -7.827  -2.336  1.00  0.00           O  \\nATOM    653  C3'  DG B  21       3.757 -10.058  -2.164  1.00  0.00           C  \\nATOM    654  O3'  DG B  21       4.960 -10.804  -2.168  1.00  0.00           O  \\nATOM    655  C2'  DG B  21       3.289  -9.606  -0.788  1.00  0.00           C  \\nATOM    656  C1'  DG B  21       2.562  -8.286  -1.068  1.00  0.00           C  \\nATOM    657  N9   DG B  21       1.080  -8.399  -1.129  1.00  0.00           N  \\nATOM    658  C8   DG B  21       0.293  -8.799  -2.181  1.00  0.00           C  \\nATOM    659  N7   DG B  21      -0.987  -8.757  -1.951  1.00  0.00           N  \\nATOM    660  C5   DG B  21      -1.072  -8.267  -0.656  1.00  0.00           C  \\nATOM    661  C6   DG B  21      -2.223  -7.998   0.146  1.00  0.00           C  \\nATOM    662  O6   DG B  21      -3.403  -8.196  -0.130  1.00  0.00           O  \\nATOM    663  N1   DG B  21      -1.895  -7.440   1.378  1.00  0.00           N  \\nATOM    664  C2   DG B  21      -0.602  -7.172   1.790  1.00  0.00           C  \\nATOM    665  N2   DG B  21      -0.446  -6.644   3.007  1.00  0.00           N  \\nATOM    666  N3   DG B  21       0.477  -7.430   1.037  1.00  0.00           N  \\nATOM    667  C4   DG B  21       0.183  -7.989  -0.167  1.00  0.00           C  \\nATOM    668  H5'  DG B  21       4.032  -7.833  -4.822  1.00  0.00           H  \\nATOM    669 H5''  DG B  21       4.678  -9.482  -4.745  1.00  0.00           H  \\nATOM    670  H4'  DG B  21       4.963  -8.310  -2.583  1.00  0.00           H  \\nATOM    671  H3'  DG B  21       2.955 -10.613  -2.649  1.00  0.00           H  \\nATOM    672  H2'  DG B  21       2.659 -10.339  -0.303  1.00  0.00           H  \\nATOM    673 H2''  DG B  21       4.145  -9.415  -0.143  1.00  0.00           H  \\nATOM    674  H1'  DG B  21       2.846  -7.600  -0.272  1.00  0.00           H  \\nATOM    675  H8   DG B  21       0.683  -9.104  -3.135  1.00  0.00           H  \\nATOM    676  H1   DG B  21      -2.661  -7.241   2.003  1.00  0.00           H  \\nATOM    677  H21  DG B  21      -1.236  -6.484   3.613  1.00  0.00           H  \\nATOM    678  H22  DG B  21       0.497  -6.466   3.341  1.00  0.00           H  \\nATOM    679  P    DA B  22       5.037 -12.344  -1.689  1.00  0.00           P  \\nATOM    680  OP1  DA B  22       6.199 -12.970  -2.359  1.00  0.00           O  \\nATOM    681  OP2  DA B  22       3.697 -12.958  -1.830  1.00  0.00           O  \\nATOM    682  O5'  DA B  22       5.377 -12.240  -0.119  1.00  0.00           O  \\nATOM    683  C5'  DA B  22       6.485 -11.495   0.353  1.00  0.00           C  \\nATOM    684  C4'  DA B  22       6.880 -11.944   1.766  1.00  0.00           C  \\nATOM    685  O4'  DA B  22       5.707 -12.137   2.543  1.00  0.00           O  \\nATOM    686  C3'  DA B  22       7.656 -13.269   1.773  1.00  0.00           C  \\nATOM    687  O3'  DA B  22       8.548 -13.254   2.879  1.00  0.00           O  \\nATOM    688  C2'  DA B  22       6.523 -14.273   1.937  1.00  0.00           C  \\nATOM    689  C1'  DA B  22       5.614 -13.508   2.894  1.00  0.00           C  \\nATOM    690  N9   DA B  22       4.209 -13.970   2.861  1.00  0.00           N  \\nATOM    691  C8   DA B  22       3.416 -14.290   1.784  1.00  0.00           C  \\nATOM    692  N7   DA B  22       2.184 -14.584   2.096  1.00  0.00           N  \\nATOM    693  C5   DA B  22       2.154 -14.452   3.478  1.00  0.00           C  \\nATOM    694  C6   DA B  22       1.134 -14.604   4.440  1.00  0.00           C  \\nATOM    695  N6   DA B  22      -0.118 -14.938   4.121  1.00  0.00           N  \\nATOM    696  N1   DA B  22       1.441 -14.424   5.739  1.00  0.00           N  \\nATOM    697  C2   DA B  22       2.685 -14.079   6.062  1.00  0.00           C  \\nATOM    698  N3   DA B  22       3.723 -13.880   5.257  1.00  0.00           N  \\nATOM    699  C4   DA B  22       3.389 -14.092   3.956  1.00  0.00           C  \\nATOM    700  H5'  DA B  22       6.211 -10.440   0.380  1.00  0.00           H  \\nATOM    701 H5''  DA B  22       7.346 -11.609  -0.308  1.00  0.00           H  \\nATOM    702  H4'  DA B  22       7.495 -11.158   2.207  1.00  0.00           H  \\nATOM    703  H3'  DA B  22       8.206 -13.437   0.846  1.00  0.00           H  \\nATOM    704  H2'  DA B  22       6.050 -14.437   0.971  1.00  0.00           H  \\nATOM    705 H2''  DA B  22       6.847 -15.224   2.347  1.00  0.00           H  \\nATOM    706  H1'  DA B  22       6.017 -13.627   3.898  1.00  0.00           H  \\nATOM    707  H8   DA B  22       3.765 -14.289   0.763  1.00  0.00           H  \\nATOM    708  H61  DA B  22      -0.799 -15.098   4.849  1.00  0.00           H  \\nATOM    709  H62  DA B  22      -0.403 -14.971   3.149  1.00  0.00           H  \\nATOM    710  H2   DA B  22       2.882 -13.931   7.112  1.00  0.00           H  \\nATOM    711  P    DA B  23       9.498 -14.507   3.262  1.00  0.00           P  \\nATOM    712  OP1  DA B  23      10.631 -13.997   4.067  1.00  0.00           O  \\nATOM    713  OP2  DA B  23       9.762 -15.291   2.035  1.00  0.00           O  \\nATOM    714  O5'  DA B  23       8.550 -15.384   4.227  1.00  0.00           O  \\nATOM    715  C5'  DA B  23       8.201 -14.937   5.523  1.00  0.00           C  \\nATOM    716  C4'  DA B  23       7.125 -15.855   6.111  1.00  0.00           C  \\nATOM    717  O4'  DA B  23       6.004 -15.909   5.243  1.00  0.00           O  \\nATOM    718  C3'  DA B  23       7.640 -17.289   6.298  1.00  0.00           C  \\nATOM    719  O3'  DA B  23       7.347 -17.705   7.624  1.00  0.00           O  \\nATOM    720  C2'  DA B  23       6.857 -18.064   5.238  1.00  0.00           C  \\nATOM    721  C1'  DA B  23       5.569 -17.250   5.162  1.00  0.00           C  \\nATOM    722  N9   DA B  23       4.787 -17.460   3.922  1.00  0.00           N  \\nATOM    723  C8   DA B  23       5.218 -17.504   2.618  1.00  0.00           C  \\nATOM    724  N7   DA B  23       4.265 -17.645   1.740  1.00  0.00           N  \\nATOM    725  C5   DA B  23       3.113 -17.702   2.513  1.00  0.00           C  \\nATOM    726  C6   DA B  23       1.748 -17.856   2.192  1.00  0.00           C  \\nATOM    727  N6   DA B  23       1.307 -17.978   0.936  1.00  0.00           N  \\nATOM    728  N1   DA B  23       0.854 -17.899   3.198  1.00  0.00           N  \\nATOM    729  C2   DA B  23       1.287 -17.784   4.452  1.00  0.00           C  \\nATOM    730  N3   DA B  23       2.536 -17.627   4.883  1.00  0.00           N  \\nATOM    731  C4   DA B  23       3.419 -17.594   3.847  1.00  0.00           C  \\nATOM    732  H5'  DA B  23       7.817 -13.917   5.486  1.00  0.00           H  \\nATOM    733 H5''  DA B  23       9.080 -14.949   6.168  1.00  0.00           H  \\nATOM    734  H4'  DA B  23       6.806 -15.445   7.070  1.00  0.00           H  \\nATOM    735  H3'  DA B  23       8.712 -17.341   6.108  1.00  0.00           H  \\nATOM    736  H2'  DA B  23       7.407 -18.009   4.298  1.00  0.00           H  \\nATOM    737 H2''  DA B  23       6.682 -19.104   5.506  1.00  0.00           H  \\nATOM    738  H1'  DA B  23       4.956 -17.492   6.028  1.00  0.00           H  \\nATOM    739  H8   DA B  23       6.256 -17.425   2.330  1.00  0.00           H  \\nATOM    740  H61  DA B  23       0.323 -18.116   0.752  1.00  0.00           H  \\nATOM    741  H62  DA B  23       1.969 -17.957   0.174  1.00  0.00           H  \\nATOM    742  H2   DA B  23       0.527 -17.826   5.218  1.00  0.00           H  \\nATOM    743  P    DC B  24       7.880 -19.103   8.239  1.00  0.00           P  \\nATOM    744  OP1  DC B  24       8.444 -18.837   9.581  1.00  0.00           O  \\nATOM    745  OP2  DC B  24       8.699 -19.794   7.218  1.00  0.00           O  \\nATOM    746  O5'  DC B  24       6.510 -19.929   8.421  1.00  0.00           O  \\nATOM    747  C5'  DC B  24       5.652 -19.718   9.530  1.00  0.00           C  \\nATOM    748  C4'  DC B  24       4.280 -20.317   9.208  1.00  0.00           C  \\nATOM    749  O4'  DC B  24       3.831 -19.851   7.950  1.00  0.00           O  \\nATOM    750  C3'  DC B  24       4.318 -21.845   9.110  1.00  0.00           C  \\nATOM    751  O3'  DC B  24       3.855 -22.448  10.301  1.00  0.00           O  \\nATOM    752  C2'  DC B  24       3.395 -22.164   7.937  1.00  0.00           C  \\nATOM    753  C1'  DC B  24       2.914 -20.799   7.442  1.00  0.00           C  \\nATOM    754  N1   DC B  24       2.896 -20.769   5.955  1.00  0.00           N  \\nATOM    755  C2   DC B  24       1.679 -20.878   5.280  1.00  0.00           C  \\nATOM    756  O2   DC B  24       0.610 -20.900   5.887  1.00  0.00           O  \\nATOM    757  N3   DC B  24       1.696 -20.948   3.918  1.00  0.00           N  \\nATOM    758  C4   DC B  24       2.848 -20.923   3.233  1.00  0.00           C  \\nATOM    759  N4   DC B  24       2.801 -20.973   1.898  1.00  0.00           N  \\nATOM    760  C5   DC B  24       4.112 -20.809   3.906  1.00  0.00           C  \\nATOM    761  C6   DC B  24       4.079 -20.734   5.257  1.00  0.00           C  \\nATOM    762  H5'  DC B  24       5.537 -18.654   9.743  1.00  0.00           H  \\nATOM    763 H5''  DC B  24       6.068 -20.202  10.415  1.00  0.00           H  \\nATOM    764  H4'  DC B  24       3.557 -20.018   9.969  1.00  0.00           H  \\nATOM    765  H3'  DC B  24       5.324 -22.197   8.879  1.00  0.00           H  \\nATOM    766 HO3'  DC B  24       2.968 -22.129  10.481  1.00  0.00           H  \\nATOM    767  H2'  DC B  24       3.970 -22.680   7.169  1.00  0.00           H  \\nATOM    768 H2''  DC B  24       2.548 -22.782   8.229  1.00  0.00           H  \\nATOM    769  H1'  DC B  24       1.943 -20.587   7.888  1.00  0.00           H  \\nATOM    770  H41  DC B  24       1.902 -21.016   1.437  1.00  0.00           H  \\nATOM    771  H42  DC B  24       3.650 -20.922   1.355  1.00  0.00           H  \\nATOM    772  H5   DC B  24       5.062 -20.776   3.392  1.00  0.00           H  \\nATOM    773  H6   DC B  24       5.004 -20.670   5.810  1.00  0.00           H  \\nTER     774       DC B  24                                                      \\nMASTER      121    0    0    0    0    0    0    6  500    2    0    2          \\nEND                                                                             \\n\");\n\tviewer_1684696742130174.setStyle({\"model\": -1, \"chain\": \"B\"},{\"cartoon\": {\"color\": \"spectrum\"}});\n\tviewer_1684696742130174.zoomTo();\nviewer_1684696742130174.render();\n});\n</script>",
      "text/html": [
       "<div id=\"3dmolviewer_1684696742130174\"  style=\"position: relative; width: 600px; height: 400px\">\n",
       "        <p id=\"3dmolwarning_1684696742130174\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
       "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
       "        </div>\n",
       "<script>\n",
       "\n",
       "var loadScriptAsync = function(uri){\n",
       "  return new Promise((resolve, reject) => {\n",
       "    //this is to ignore the existence of requirejs amd\n",
       "    var savedexports, savedmodule;\n",
       "    if (typeof exports !== 'undefined') savedexports = exports;\n",
       "    else exports = {}\n",
       "    if (typeof module !== 'undefined') savedmodule = module;\n",
       "    else module = {}\n",
       "\n",
       "    var tag = document.createElement('script');\n",
       "    tag.src = uri;\n",
       "    tag.async = true;\n",
       "    tag.onload = () => {\n",
       "        exports = savedexports;\n",
       "        module = savedmodule;\n",
       "        resolve();\n",
       "    };\n",
       "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
       "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
       "});\n",
       "};\n",
       "\n",
       "if(typeof $3Dmolpromise === 'undefined') {\n",
       "$3Dmolpromise = null;\n",
       "  $3Dmolpromise = loadScriptAsync('https://cdnjs.cloudflare.com/ajax/libs/3Dmol/2.0.1/3Dmol-min.js');\n",
       "}\n",
       "\n",
       "var viewer_1684696742130174 = null;\n",
       "var warn = document.getElementById(\"3dmolwarning_1684696742130174\");\n",
       "if(warn) {\n",
       "    warn.parentNode.removeChild(warn);\n",
       "}\n",
       "$3Dmolpromise.then(function() {\n",
       "viewer_1684696742130174 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_1684696742130174\"),{backgroundColor:\"white\"});\n",
       "viewer_1684696742130174.zoomTo();\n",
       "\tviewer_1684696742130174.addModelsAsFrames(\"HEADER    DNA                                     16-DEC-94   103D              \\nTITLE     THE UNUSUAL STRUCTURE OF THE HUMAN CENTROMERE (GGA)2 MOTIF: UNPAIRED  \\nTITLE    2 GUANOSINE RESIDUES STACKED BETWEEN SHEARED G(DOT)A PAIRS             \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: DNA (5'-D(*GP*TP*GP*GP*AP*AP*TP*GP*GP*AP*AP*C)-3');        \\nCOMPND   3 CHAIN: A, B;                                                         \\nCOMPND   4 ENGINEERED: YES                                                      \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 SYNTHETIC: YES;                                                      \\nSOURCE   3 OTHER_DETAILS: CHEMICALLY SYNTHESIZED                                \\nKEYWDS    DNA, DOUBLE HELIX, G-G STACKING, G:A MISMATCH, HUMAN CENTROMERE       \\nKEYWDS   2 REPEAT, GA-BRACKETED G-STACK MOTIF                                   \\nEXPDTA    SOLUTION NMR                                                          \\nAUTHOR    S.-H.CHOU,L.ZHU,B.R.REID                                              \\nREVDAT   5   16-FEB-22 103D    1       REMARK                                   \\nREVDAT   4   07-APR-09 103D    1       REMARK                                   \\nREVDAT   3   24-FEB-09 103D    1       VERSN                                    \\nREVDAT   2   01-APR-03 103D    1       JRNL                                     \\nREVDAT   1   31-MAR-95 103D    0                                                \\nJRNL        AUTH   S.H.CHOU,L.ZHU,B.R.REID                                      \\nJRNL        TITL   THE UNUSUAL STRUCTURE OF THE HUMAN CENTROMERE (GGA)2 MOTIF.  \\nJRNL        TITL 2 UNPAIRED GUANOSINE RESIDUES STACKED BETWEEN SHEARED G.A      \\nJRNL        TITL 3 PAIRS.                                                       \\nJRNL        REF    J.MOL.BIOL.                   V. 244   259 1994              \\nJRNL        REFN                   ISSN 0022-2836                               \\nJRNL        PMID   7966337                                                      \\nJRNL        DOI    10.1006/JMBI.1994.1727                                       \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION. NOT APPLICABLE.                                          \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : AMBER                                                \\nREMARK   3   AUTHORS     : BIOSYM TECHNOLOGIES, INC.                            \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: R VALUE 0.250 FINAL RMS COORD. SHIFT      \\nREMARK   3  0.65 ANGSTROMS NUMBER OF ATOMS USED IN REFINEMENT. NUMBER OF        \\nREMARK   3  PROTEIN ATOMS 0 NUMBER OF NUCLEIC ACID ATOMS 772 NUMBER OF          \\nREMARK   3  HETEROGEN ATOMS 0 NUMBER OF SOLVENT ATOMS 0. THE DUPLEX CONTAINS    \\nREMARK   3  UNPAIRED STAGGERED GUANOSINE RESIDUES, WHICH CO-STACK BY            \\nREMARK   3  INTERCALATION BETWEEN SHEARED G(DOT)A AND A(DOT)G BASE-PAIRS TO     \\nREMARK   3  FORM AN INTERESTING NEW STRUCTURAL MOTIF, THE GA-BRACKETED G-       \\nREMARK   3  STACK. THE TGGAA REPEAT UNIT CONTAINS SIX \\\"STEPS\\\", FOUR OF WHICH    \\nREMARK   3  ARE NOT WATSON-CRICK BASE-PAIRS.                                    \\nREMARK   4                                                                      \\nREMARK   4 103D COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY BNL.                                \\nREMARK 100 THE DEPOSITION ID IS D_1000170007.                                   \\nREMARK 210                                                                      \\nREMARK 210 EXPERIMENTAL DETAILS                                                 \\nREMARK 210  EXPERIMENT TYPE                : NMR                                \\nREMARK 210  TEMPERATURE           (KELVIN) : NULL                               \\nREMARK 210  PH                             : NULL                               \\nREMARK 210  IONIC STRENGTH                 : NULL                               \\nREMARK 210  PRESSURE                       : NULL                               \\nREMARK 210  SAMPLE CONTENTS                : NULL                               \\nREMARK 210                                                                      \\nREMARK 210  NMR EXPERIMENTS CONDUCTED      : NULL                               \\nREMARK 210  SPECTROMETER FIELD STRENGTH    : NULL                               \\nREMARK 210  SPECTROMETER MODEL             : NULL                               \\nREMARK 210  SPECTROMETER MANUFACTURER      : NULL                               \\nREMARK 210                                                                      \\nREMARK 210  STRUCTURE DETERMINATION.                                            \\nREMARK 210   SOFTWARE USED                 : NULL                               \\nREMARK 210   METHOD USED                   : NULL                               \\nREMARK 210                                                                      \\nREMARK 210 CONFORMERS, NUMBER CALCULATED   : NULL                               \\nREMARK 210 CONFORMERS, NUMBER SUBMITTED    : 1                                  \\nREMARK 210 CONFORMERS, SELECTION CRITERIA  : NULL                               \\nREMARK 210                                                                      \\nREMARK 210 BEST REPRESENTATIVE CONFORMER IN THIS ENSEMBLE : NULL                \\nREMARK 210                                                                      \\nREMARK 210 REMARK: NULL                                                         \\nREMARK 215                                                                      \\nREMARK 215 NMR STUDY                                                            \\nREMARK 215 THE COORDINATES IN THIS ENTRY WERE GENERATED FROM SOLUTION           \\nREMARK 215 NMR DATA.  PROTEIN DATA BANK CONVENTIONS REQUIRE THAT                \\nREMARK 215 CRYST1 AND SCALE RECORDS BE INCLUDED, BUT THE VALUES ON              \\nREMARK 215 THESE RECORDS ARE MEANINGLESS.                                       \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A, B                                  \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \\nREMARK 500                                                                      \\nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \\nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \\nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \\nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \\nREMARK 500     DG A   1   O4' -  C1' -  N9  ANGL. DEV. =   1.9 DEGREES          \\nREMARK 500     DA A   5   O4' -  C1' -  N9  ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500     DA A   6   O4' -  C1' -  N9  ANGL. DEV. =   2.2 DEGREES          \\nREMARK 500     DT A   7   O4' -  C1' -  N1  ANGL. DEV. =   2.5 DEGREES          \\nREMARK 500     DA A  10   O4' -  C1' -  N9  ANGL. DEV. =   2.7 DEGREES          \\nREMARK 500     DA A  11   O4' -  C1' -  N9  ANGL. DEV. =   2.0 DEGREES          \\nREMARK 500     DC A  12   O4' -  C1' -  N1  ANGL. DEV. =   2.5 DEGREES          \\nREMARK 500     DG B  13   O4' -  C1' -  N9  ANGL. DEV. =   1.9 DEGREES          \\nREMARK 500     DT B  14   O4' -  C1' -  N1  ANGL. DEV. =   1.8 DEGREES          \\nREMARK 500     DA B  17   O4' -  C1' -  N9  ANGL. DEV. =   3.1 DEGREES          \\nREMARK 500     DA B  18   O4' -  C1' -  N9  ANGL. DEV. =   2.3 DEGREES          \\nREMARK 500     DT B  19   O4' -  C1' -  N1  ANGL. DEV. =   2.5 DEGREES          \\nREMARK 500     DA B  22   O4' -  C1' -  N9  ANGL. DEV. =   2.7 DEGREES          \\nREMARK 500     DA B  23   O4' -  C1' -  N9  ANGL. DEV. =   1.9 DEGREES          \\nREMARK 500     DC B  24   O4' -  C1' -  N1  ANGL. DEV. =   2.4 DEGREES          \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: PLANAR GROUPS                                              \\nREMARK 500                                                                      \\nREMARK 500 PLANAR GROUPS IN THE FOLLOWING RESIDUES HAVE A TOTAL                 \\nREMARK 500 RMS DISTANCE OF ALL ATOMS FROM THE BEST-FIT PLANE                    \\nREMARK 500 BY MORE THAN AN EXPECTED VALUE OF 6*RMSD, WITH AN                    \\nREMARK 500 RMSD 0.02 ANGSTROMS, OR AT LEAST ONE ATOM HAS                        \\nREMARK 500 AN RMSD GREATER THAN THIS VALUE                                      \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        RMS     TYPE                                    \\nREMARK 500     DT A   2         0.06    SIDE CHAIN                              \\nREMARK 500     DT B  14         0.06    SIDE CHAIN                              \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nDBREF  103D A    1    12  PDB    103D     103D             1     12             \\nDBREF  103D B   13    24  PDB    103D     103D            13     24             \\nSEQRES   1 A   12   DG  DT  DG  DG  DA  DA  DT  DG  DG  DA  DA  DC              \\nSEQRES   1 B   12   DG  DT  DG  DG  DA  DA  DT  DG  DG  DA  DA  DC              \\nCRYST1    1.000    1.000    1.000  90.00  90.00  90.00 P 1           1          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      1.000000  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  1.000000  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  1.000000        0.00000                         \\nATOM      1  O5'  DG A   1      -7.643 -22.640  -1.099  1.00  0.00           O  \\nATOM      2  C5'  DG A   1      -8.313 -23.137   0.040  1.00  0.00           C  \\nATOM      3  C4'  DG A   1      -7.935 -22.349   1.300  1.00  0.00           C  \\nATOM      4  O4'  DG A   1      -6.537 -22.463   1.524  1.00  0.00           O  \\nATOM      5  C3'  DG A   1      -8.273 -20.854   1.190  1.00  0.00           C  \\nATOM      6  O3'  DG A   1      -8.832 -20.418   2.419  1.00  0.00           O  \\nATOM      7  C2'  DG A   1      -6.902 -20.240   0.924  1.00  0.00           C  \\nATOM      8  C1'  DG A   1      -5.995 -21.171   1.722  1.00  0.00           C  \\nATOM      9  N9   DG A   1      -4.593 -21.126   1.247  1.00  0.00           N  \\nATOM     10  C8   DG A   1      -4.107 -21.429  -0.001  1.00  0.00           C  \\nATOM     11  N7   DG A   1      -2.812 -21.335  -0.110  1.00  0.00           N  \\nATOM     12  C5   DG A   1      -2.397 -20.963   1.163  1.00  0.00           C  \\nATOM     13  C6   DG A   1      -1.080 -20.733   1.666  1.00  0.00           C  \\nATOM     14  O6   DG A   1      -0.017 -20.780   1.053  1.00  0.00           O  \\nATOM     15  N1   DG A   1      -1.077 -20.427   3.024  1.00  0.00           N  \\nATOM     16  C2   DG A   1      -2.214 -20.317   3.803  1.00  0.00           C  \\nATOM     17  N2   DG A   1      -2.045 -20.010   5.092  1.00  0.00           N  \\nATOM     18  N3   DG A   1      -3.455 -20.507   3.325  1.00  0.00           N  \\nATOM     19  C4   DG A   1      -3.479 -20.835   2.004  1.00  0.00           C  \\nATOM     20  H5'  DG A   1      -8.042 -24.185   0.178  1.00  0.00           H  \\nATOM     21 H5''  DG A   1      -9.389 -23.072  -0.122  1.00  0.00           H  \\nATOM     22  H4'  DG A   1      -8.473 -22.789   2.142  1.00  0.00           H  \\nATOM     23  H3'  DG A   1      -8.969 -20.658   0.374  1.00  0.00           H  \\nATOM     24  H2'  DG A   1      -6.682 -20.310  -0.141  1.00  0.00           H  \\nATOM     25 H2''  DG A   1      -6.821 -19.205   1.250  1.00  0.00           H  \\nATOM     26  H1'  DG A   1      -6.058 -20.912   2.780  1.00  0.00           H  \\nATOM     27  H8   DG A   1      -4.739 -21.718  -0.828  1.00  0.00           H  \\nATOM     28  H1   DG A   1      -0.169 -20.285   3.449  1.00  0.00           H  \\nATOM     29  H21  DG A   1      -1.114 -19.925   5.477  1.00  0.00           H  \\nATOM     30  H22  DG A   1      -2.853 -19.902   5.687  1.00  0.00           H  \\nATOM     31 HO5'  DG A   1      -7.929 -21.737  -1.254  1.00  0.00           H  \\nATOM     32  P    DT A   2      -9.356 -18.904   2.648  1.00  0.00           P  \\nATOM     33  OP1  DT A   2     -10.579 -18.963   3.478  1.00  0.00           O  \\nATOM     34  OP2  DT A   2      -9.388 -18.212   1.340  1.00  0.00           O  \\nATOM     35  O5'  DT A   2      -8.175 -18.250   3.532  1.00  0.00           O  \\nATOM     36  C5'  DT A   2      -7.959 -18.655   4.871  1.00  0.00           C  \\nATOM     37  C4'  DT A   2      -6.761 -17.920   5.479  1.00  0.00           C  \\nATOM     38  O4'  DT A   2      -5.603 -18.173   4.699  1.00  0.00           O  \\nATOM     39  C3'  DT A   2      -6.955 -16.396   5.550  1.00  0.00           C  \\nATOM     40  O3'  DT A   2      -6.507 -15.961   6.826  1.00  0.00           O  \\nATOM     41  C2'  DT A   2      -6.062 -15.914   4.406  1.00  0.00           C  \\nATOM     42  C1'  DT A   2      -4.938 -16.944   4.495  1.00  0.00           C  \\nATOM     43  N1   DT A   2      -4.057 -17.032   3.297  1.00  0.00           N  \\nATOM     44  C2   DT A   2      -2.677 -17.088   3.499  1.00  0.00           C  \\nATOM     45  O2   DT A   2      -2.146 -16.945   4.598  1.00  0.00           O  \\nATOM     46  N3   DT A   2      -1.900 -17.352   2.381  1.00  0.00           N  \\nATOM     47  C4   DT A   2      -2.368 -17.603   1.102  1.00  0.00           C  \\nATOM     48  O4   DT A   2      -1.578 -17.886   0.204  1.00  0.00           O  \\nATOM     49  C5   DT A   2      -3.808 -17.502   0.964  1.00  0.00           C  \\nATOM     50  C7   DT A   2      -4.451 -17.702  -0.395  1.00  0.00           C  \\nATOM     51  C6   DT A   2      -4.587 -17.231   2.042  1.00  0.00           C  \\nATOM     52  H5'  DT A   2      -7.763 -19.727   4.902  1.00  0.00           H  \\nATOM     53 H5''  DT A   2      -8.845 -18.442   5.472  1.00  0.00           H  \\nATOM     54  H4'  DT A   2      -6.601 -18.314   6.483  1.00  0.00           H  \\nATOM     55  H3'  DT A   2      -7.997 -16.110   5.395  1.00  0.00           H  \\nATOM     56  H2'  DT A   2      -6.630 -15.995   3.481  1.00  0.00           H  \\nATOM     57 H2''  DT A   2      -5.703 -14.894   4.522  1.00  0.00           H  \\nATOM     58  H1'  DT A   2      -4.373 -16.711   5.397  1.00  0.00           H  \\nATOM     59  H3   DT A   2      -0.903 -17.421   2.534  1.00  0.00           H  \\nATOM     60  H71  DT A   2      -3.753 -17.435  -1.188  1.00  0.00           H  \\nATOM     61  H72  DT A   2      -4.736 -18.745  -0.514  1.00  0.00           H  \\nATOM     62  H73  DT A   2      -5.340 -17.077  -0.489  1.00  0.00           H  \\nATOM     63  H6   DT A   2      -5.655 -17.199   1.905  1.00  0.00           H  \\nATOM     64  P    DG A   3      -6.534 -14.414   7.292  1.00  0.00           P  \\nATOM     65  OP1  DG A   3      -6.475 -14.378   8.770  1.00  0.00           O  \\nATOM     66  OP2  DG A   3      -7.637 -13.728   6.584  1.00  0.00           O  \\nATOM     67  O5'  DG A   3      -5.139 -13.848   6.720  1.00  0.00           O  \\nATOM     68  C5'  DG A   3      -3.896 -14.287   7.234  1.00  0.00           C  \\nATOM     69  C4'  DG A   3      -2.748 -13.705   6.405  1.00  0.00           C  \\nATOM     70  O4'  DG A   3      -2.863 -14.119   5.053  1.00  0.00           O  \\nATOM     71  C3'  DG A   3      -2.751 -12.171   6.416  1.00  0.00           C  \\nATOM     72  O3'  DG A   3      -1.406 -11.747   6.547  1.00  0.00           O  \\nATOM     73  C2'  DG A   3      -3.335 -11.844   5.041  1.00  0.00           C  \\nATOM     74  C1'  DG A   3      -2.741 -12.983   4.218  1.00  0.00           C  \\nATOM     75  N9   DG A   3      -3.405 -13.249   2.921  1.00  0.00           N  \\nATOM     76  C8   DG A   3      -4.715 -13.058   2.555  1.00  0.00           C  \\nATOM     77  N7   DG A   3      -4.994 -13.444   1.343  1.00  0.00           N  \\nATOM     78  C5   DG A   3      -3.784 -13.924   0.857  1.00  0.00           C  \\nATOM     79  C6   DG A   3      -3.457 -14.458  -0.426  1.00  0.00           C  \\nATOM     80  O6   DG A   3      -4.194 -14.609  -1.397  1.00  0.00           O  \\nATOM     81  N1   DG A   3      -2.119 -14.829  -0.523  1.00  0.00           N  \\nATOM     82  C2   DG A   3      -1.196 -14.693   0.496  1.00  0.00           C  \\nATOM     83  N2   DG A   3       0.052 -15.094   0.239  1.00  0.00           N  \\nATOM     84  N3   DG A   3      -1.501 -14.190   1.703  1.00  0.00           N  \\nATOM     85  C4   DG A   3      -2.807 -13.820   1.820  1.00  0.00           C  \\nATOM     86  H5'  DG A   3      -3.836 -15.375   7.202  1.00  0.00           H  \\nATOM     87 H5''  DG A   3      -3.792 -13.963   8.271  1.00  0.00           H  \\nATOM     88  H4'  DG A   3      -1.809 -14.080   6.816  1.00  0.00           H  \\nATOM     89  H3'  DG A   3      -3.366 -11.782   7.227  1.00  0.00           H  \\nATOM     90  H2'  DG A   3      -4.421 -11.917   5.096  1.00  0.00           H  \\nATOM     91 H2''  DG A   3      -3.043 -10.865   4.661  1.00  0.00           H  \\nATOM     92  H1'  DG A   3      -1.692 -12.735   4.043  1.00  0.00           H  \\nATOM     93  H8   DG A   3      -5.463 -12.632   3.207  1.00  0.00           H  \\nATOM     94  H1   DG A   3      -1.821 -15.221  -1.404  1.00  0.00           H  \\nATOM     95  H21  DG A   3       0.298 -15.465  -0.666  1.00  0.00           H  \\nATOM     96  H22  DG A   3       0.772 -14.962   0.941  1.00  0.00           H  \\nATOM     97  P    DG A   4      -0.997 -10.224   6.884  1.00  0.00           P  \\nATOM     98  OP1  DG A   4      -0.439 -10.175   8.254  1.00  0.00           O  \\nATOM     99  OP2  DG A   4      -2.112  -9.328   6.505  1.00  0.00           O  \\nATOM    100  O5'  DG A   4       0.196 -10.007   5.828  1.00  0.00           O  \\nATOM    101  C5'  DG A   4       1.508 -10.482   6.064  1.00  0.00           C  \\nATOM    102  C4'  DG A   4       2.254 -10.592   4.729  1.00  0.00           C  \\nATOM    103  O4'  DG A   4       1.640 -11.522   3.855  1.00  0.00           O  \\nATOM    104  C3'  DG A   4       2.288  -9.262   3.978  1.00  0.00           C  \\nATOM    105  O3'  DG A   4       3.326  -8.456   4.505  1.00  0.00           O  \\nATOM    106  C2'  DG A   4       2.518  -9.741   2.552  1.00  0.00           C  \\nATOM    107  C1'  DG A   4       1.787 -11.086   2.512  1.00  0.00           C  \\nATOM    108  N9   DG A   4       0.429 -11.010   1.915  1.00  0.00           N  \\nATOM    109  C8   DG A   4      -0.746 -10.615   2.504  1.00  0.00           C  \\nATOM    110  N7   DG A   4      -1.792 -10.688   1.732  1.00  0.00           N  \\nATOM    111  C5   DG A   4      -1.287 -11.185   0.538  1.00  0.00           C  \\nATOM    112  C6   DG A   4      -1.954 -11.480  -0.691  1.00  0.00           C  \\nATOM    113  O6   DG A   4      -3.141 -11.323  -0.965  1.00  0.00           O  \\nATOM    114  N1   DG A   4      -1.096 -12.013  -1.650  1.00  0.00           N  \\nATOM    115  C2   DG A   4       0.251 -12.248  -1.443  1.00  0.00           C  \\nATOM    116  N2   DG A   4       0.952 -12.759  -2.460  1.00  0.00           N  \\nATOM    117  N3   DG A   4       0.873 -11.977  -0.287  1.00  0.00           N  \\nATOM    118  C4   DG A   4       0.061 -11.433   0.657  1.00  0.00           C  \\nATOM    119  H5'  DG A   4       1.495 -11.457   6.552  1.00  0.00           H  \\nATOM    120 H5''  DG A   4       2.027  -9.777   6.714  1.00  0.00           H  \\nATOM    121  H4'  DG A   4       3.271 -10.944   4.919  1.00  0.00           H  \\nATOM    122  H3'  DG A   4       1.317  -8.768   4.033  1.00  0.00           H  \\nATOM    123  H2'  DG A   4       2.150  -9.039   1.815  1.00  0.00           H  \\nATOM    124 H2''  DG A   4       3.578  -9.904   2.373  1.00  0.00           H  \\nATOM    125  H1'  DG A   4       2.409 -11.766   1.933  1.00  0.00           H  \\nATOM    126  H8   DG A   4      -0.813 -10.291   3.528  1.00  0.00           H  \\nATOM    127  H1   DG A   4      -1.500 -12.225  -2.549  1.00  0.00           H  \\nATOM    128  H21  DG A   4       0.523 -12.928  -3.357  1.00  0.00           H  \\nATOM    129  H22  DG A   4       1.950 -12.906  -2.334  1.00  0.00           H  \\nATOM    130  P    DA A   5       3.562  -6.924   4.058  1.00  0.00           P  \\nATOM    131  OP1  DA A   5       4.379  -6.268   5.103  1.00  0.00           O  \\nATOM    132  OP2  DA A   5       2.259  -6.334   3.677  1.00  0.00           O  \\nATOM    133  O5'  DA A   5       4.464  -7.037   2.730  1.00  0.00           O  \\nATOM    134  C5'  DA A   5       5.721  -7.693   2.730  1.00  0.00           C  \\nATOM    135  C4'  DA A   5       6.616  -7.135   1.615  1.00  0.00           C  \\nATOM    136  O4'  DA A   5       5.843  -6.978   0.432  1.00  0.00           O  \\nATOM    137  C3'  DA A   5       7.201  -5.758   1.969  1.00  0.00           C  \\nATOM    138  O3'  DA A   5       8.471  -5.625   1.344  1.00  0.00           O  \\nATOM    139  C2'  DA A   5       6.152  -4.834   1.366  1.00  0.00           C  \\nATOM    140  C1'  DA A   5       5.814  -5.603   0.093  1.00  0.00           C  \\nATOM    141  N9   DA A   5       4.510  -5.219  -0.496  1.00  0.00           N  \\nATOM    142  C8   DA A   5       3.302  -4.987   0.119  1.00  0.00           C  \\nATOM    143  N7   DA A   5       2.320  -4.747  -0.706  1.00  0.00           N  \\nATOM    144  C5   DA A   5       2.915  -4.819  -1.959  1.00  0.00           C  \\nATOM    145  C6   DA A   5       2.422  -4.678  -3.273  1.00  0.00           C  \\nATOM    146  N6   DA A   5       1.142  -4.414  -3.547  1.00  0.00           N  \\nATOM    147  N1   DA A   5       3.285  -4.794  -4.301  1.00  0.00           N  \\nATOM    148  C2   DA A   5       4.562  -5.063  -4.040  1.00  0.00           C  \\nATOM    149  N3   DA A   5       5.145  -5.238  -2.858  1.00  0.00           N  \\nATOM    150  C4   DA A   5       4.254  -5.095  -1.840  1.00  0.00           C  \\nATOM    151  H5'  DA A   5       5.562  -8.757   2.560  1.00  0.00           H  \\nATOM    152 H5''  DA A   5       6.232  -7.572   3.686  1.00  0.00           H  \\nATOM    153  H4'  DA A   5       7.427  -7.844   1.444  1.00  0.00           H  \\nATOM    154  H3'  DA A   5       7.295  -5.615   3.047  1.00  0.00           H  \\nATOM    155  H2'  DA A   5       5.300  -4.777   2.042  1.00  0.00           H  \\nATOM    156 H2''  DA A   5       6.527  -3.836   1.162  1.00  0.00           H  \\nATOM    157  H1'  DA A   5       6.610  -5.423  -0.628  1.00  0.00           H  \\nATOM    158  H8   DA A   5       3.159  -5.017   1.188  1.00  0.00           H  \\nATOM    159  H61  DA A   5       0.850  -4.267  -4.502  1.00  0.00           H  \\nATOM    160  H62  DA A   5       0.451  -4.437  -2.806  1.00  0.00           H  \\nATOM    161  H2   DA A   5       5.213  -5.162  -4.894  1.00  0.00           H  \\nATOM    162  P    DA A   6       9.363  -4.276   1.441  1.00  0.00           P  \\nATOM    163  OP1  DA A   6      10.782  -4.649   1.246  1.00  0.00           O  \\nATOM    164  OP2  DA A   6       8.960  -3.530   2.654  1.00  0.00           O  \\nATOM    165  O5'  DA A   6       8.887  -3.431   0.150  1.00  0.00           O  \\nATOM    166  C5'  DA A   6       9.220  -3.846  -1.162  1.00  0.00           C  \\nATOM    167  C4'  DA A   6       8.472  -3.000  -2.195  1.00  0.00           C  \\nATOM    168  O4'  DA A   6       7.079  -3.068  -1.929  1.00  0.00           O  \\nATOM    169  C3'  DA A   6       8.883  -1.518  -2.181  1.00  0.00           C  \\nATOM    170  O3'  DA A   6       9.162  -1.103  -3.510  1.00  0.00           O  \\nATOM    171  C2'  DA A   6       7.638  -0.843  -1.609  1.00  0.00           C  \\nATOM    172  C1'  DA A   6       6.534  -1.778  -2.095  1.00  0.00           C  \\nATOM    173  N9   DA A   6       5.266  -1.649  -1.337  1.00  0.00           N  \\nATOM    174  C8   DA A   6       5.062  -1.659   0.023  1.00  0.00           C  \\nATOM    175  N7   DA A   6       3.811  -1.603   0.382  1.00  0.00           N  \\nATOM    176  C5   DA A   6       3.128  -1.551  -0.825  1.00  0.00           C  \\nATOM    177  C6   DA A   6       1.757  -1.487  -1.146  1.00  0.00           C  \\nATOM    178  N6   DA A   6       0.795  -1.473  -0.218  1.00  0.00           N  \\nATOM    179  N1   DA A   6       1.406  -1.437  -2.443  1.00  0.00           N  \\nATOM    180  C2   DA A   6       2.358  -1.447  -3.373  1.00  0.00           C  \\nATOM    181  N3   DA A   6       3.677  -1.511  -3.202  1.00  0.00           N  \\nATOM    182  C4   DA A   6       4.005  -1.564  -1.882  1.00  0.00           C  \\nATOM    183  H5'  DA A   6       8.939  -4.890  -1.301  1.00  0.00           H  \\nATOM    184 H5''  DA A   6      10.295  -3.750  -1.321  1.00  0.00           H  \\nATOM    185  H4'  DA A   6       8.666  -3.422  -3.183  1.00  0.00           H  \\nATOM    186  H3'  DA A   6       9.752  -1.349  -1.542  1.00  0.00           H  \\nATOM    187  H2'  DA A   6       7.729  -0.858  -0.523  1.00  0.00           H  \\nATOM    188 H2''  DA A   6       7.497   0.178  -1.959  1.00  0.00           H  \\nATOM    189  H1'  DA A   6       6.357  -1.604  -3.158  1.00  0.00           H  \\nATOM    190  H8   DA A   6       5.858  -1.702   0.751  1.00  0.00           H  \\nATOM    191  H61  DA A   6      -0.176  -1.429  -0.493  1.00  0.00           H  \\nATOM    192  H62  DA A   6       1.049  -1.512   0.759  1.00  0.00           H  \\nATOM    193  H2   DA A   6       2.014  -1.412  -4.395  1.00  0.00           H  \\nATOM    194  P    DT A   7       9.713   0.376  -3.872  1.00  0.00           P  \\nATOM    195  OP1  DT A   7      10.872   0.227  -4.781  1.00  0.00           O  \\nATOM    196  OP2  DT A   7       9.854   1.156  -2.622  1.00  0.00           O  \\nATOM    197  O5'  DT A   7       8.487   0.997  -4.715  1.00  0.00           O  \\nATOM    198  C5'  DT A   7       8.196   0.539  -6.023  1.00  0.00           C  \\nATOM    199  C4'  DT A   7       6.841   1.074  -6.496  1.00  0.00           C  \\nATOM    200  O4'  DT A   7       5.858   0.807  -5.506  1.00  0.00           O  \\nATOM    201  C3'  DT A   7       6.839   2.587  -6.766  1.00  0.00           C  \\nATOM    202  O3'  DT A   7       6.155   2.808  -7.992  1.00  0.00           O  \\nATOM    203  C2'  DT A   7       6.075   3.121  -5.556  1.00  0.00           C  \\nATOM    204  C1'  DT A   7       5.086   1.977  -5.335  1.00  0.00           C  \\nATOM    205  N1   DT A   7       4.411   1.977  -4.004  1.00  0.00           N  \\nATOM    206  C2   DT A   7       3.021   1.846  -3.966  1.00  0.00           C  \\nATOM    207  O2   DT A   7       2.314   1.799  -4.970  1.00  0.00           O  \\nATOM    208  N3   DT A   7       2.445   1.746  -2.708  1.00  0.00           N  \\nATOM    209  C4   DT A   7       3.120   1.734  -1.498  1.00  0.00           C  \\nATOM    210  O4   DT A   7       2.496   1.594  -0.449  1.00  0.00           O  \\nATOM    211  C5   DT A   7       4.558   1.885  -1.617  1.00  0.00           C  \\nATOM    212  C7   DT A   7       5.422   1.939  -0.371  1.00  0.00           C  \\nATOM    213  C6   DT A   7       5.142   1.992  -2.837  1.00  0.00           C  \\nATOM    214  H5'  DT A   7       8.155  -0.551  -6.034  1.00  0.00           H  \\nATOM    215 H5''  DT A   7       8.978   0.859  -6.713  1.00  0.00           H  \\nATOM    216  H4'  DT A   7       6.575   0.544  -7.412  1.00  0.00           H  \\nATOM    217  H3'  DT A   7       7.851   2.993  -6.814  1.00  0.00           H  \\nATOM    218  H2'  DT A   7       6.781   3.227  -4.736  1.00  0.00           H  \\nATOM    219 H2''  DT A   7       5.578   4.072  -5.733  1.00  0.00           H  \\nATOM    220  H1'  DT A   7       4.370   2.022  -6.154  1.00  0.00           H  \\nATOM    221  H3   DT A   7       1.442   1.618  -2.682  1.00  0.00           H  \\nATOM    222  H71  DT A   7       4.825   2.215   0.499  1.00  0.00           H  \\nATOM    223  H72  DT A   7       5.875   0.967  -0.192  1.00  0.00           H  \\nATOM    224  H73  DT A   7       6.214   2.678  -0.498  1.00  0.00           H  \\nATOM    225  H6   DT A   7       6.217   2.067  -2.881  1.00  0.00           H  \\nATOM    226  P    DG A   8       5.963   4.274  -8.646  1.00  0.00           P  \\nATOM    227  OP1  DG A   8       5.644   4.099 -10.080  1.00  0.00           O  \\nATOM    228  OP2  DG A   8       7.110   5.120  -8.246  1.00  0.00           O  \\nATOM    229  O5'  DG A   8       4.646   4.830  -7.903  1.00  0.00           O  \\nATOM    230  C5'  DG A   8       3.371   4.257  -8.123  1.00  0.00           C  \\nATOM    231  C4'  DG A   8       2.339   4.891  -7.185  1.00  0.00           C  \\nATOM    232  O4'  DG A   8       2.698   4.651  -5.832  1.00  0.00           O  \\nATOM    233  C3'  DG A   8       2.247   6.411  -7.372  1.00  0.00           C  \\nATOM    234  O3'  DG A   8       0.879   6.770  -7.293  1.00  0.00           O  \\nATOM    235  C2'  DG A   8       3.050   6.921  -6.174  1.00  0.00           C  \\nATOM    236  C1'  DG A   8       2.659   5.879  -5.130  1.00  0.00           C  \\nATOM    237  N9   DG A   8       3.547   5.807  -3.945  1.00  0.00           N  \\nATOM    238  C8   DG A   8       4.888   6.082  -3.843  1.00  0.00           C  \\nATOM    239  N7   DG A   8       5.390   5.876  -2.658  1.00  0.00           N  \\nATOM    240  C5   DG A   8       4.305   5.439  -1.907  1.00  0.00           C  \\nATOM    241  C6   DG A   8       4.229   5.086  -0.526  1.00  0.00           C  \\nATOM    242  O6   DG A   8       5.128   5.089   0.311  1.00  0.00           O  \\nATOM    243  N1   DG A   8       2.946   4.704  -0.143  1.00  0.00           N  \\nATOM    244  C2   DG A   8       1.856   4.677  -0.992  1.00  0.00           C  \\nATOM    245  N2   DG A   8       0.685   4.316  -0.461  1.00  0.00           N  \\nATOM    246  N3   DG A   8       1.926   5.007  -2.291  1.00  0.00           N  \\nATOM    247  C4   DG A   8       3.174   5.384  -2.687  1.00  0.00           C  \\nATOM    248  H5'  DG A   8       3.402   3.183  -7.939  1.00  0.00           H  \\nATOM    249 H5''  DG A   8       3.067   4.426  -9.157  1.00  0.00           H  \\nATOM    250  H4'  DG A   8       1.370   4.430  -7.385  1.00  0.00           H  \\nATOM    251  H3'  DG A   8       2.676   6.727  -8.324  1.00  0.00           H  \\nATOM    252  H2'  DG A   8       4.113   6.867  -6.415  1.00  0.00           H  \\nATOM    253 H2''  DG A   8       2.783   7.932  -5.866  1.00  0.00           H  \\nATOM    254  H1'  DG A   8       1.641   6.111  -4.807  1.00  0.00           H  \\nATOM    255  H8   DG A   8       5.491   6.434  -4.667  1.00  0.00           H  \\nATOM    256  H1   DG A   8       2.822   4.440   0.822  1.00  0.00           H  \\nATOM    257  H21  DG A   8       0.614   4.079   0.517  1.00  0.00           H  \\nATOM    258  H22  DG A   8      -0.150   4.335  -1.037  1.00  0.00           H  \\nATOM    259  P    DG A   9       0.330   8.218  -7.744  1.00  0.00           P  \\nATOM    260  OP1  DG A   9      -0.439   8.061  -8.998  1.00  0.00           O  \\nATOM    261  OP2  DG A   9       1.438   9.197  -7.672  1.00  0.00           O  \\nATOM    262  O5'  DG A   9      -0.690   8.535  -6.543  1.00  0.00           O  \\nATOM    263  C5'  DG A   9      -2.012   8.031  -6.519  1.00  0.00           C  \\nATOM    264  C4'  DG A   9      -2.566   8.145  -5.094  1.00  0.00           C  \\nATOM    265  O4'  DG A   9      -1.825   7.353  -4.184  1.00  0.00           O  \\nATOM    266  C3'  DG A   9      -2.506   9.580  -4.566  1.00  0.00           C  \\nATOM    267  O3'  DG A   9      -3.619  10.313  -5.044  1.00  0.00           O  \\nATOM    268  C2'  DG A   9      -2.497   9.337  -3.063  1.00  0.00           C  \\nATOM    269  C1'  DG A   9      -1.771   7.994  -2.919  1.00  0.00           C  \\nATOM    270  N9   DG A   9      -0.338   8.108  -2.538  1.00  0.00           N  \\nATOM    271  C8   DG A   9       0.746   8.359  -3.343  1.00  0.00           C  \\nATOM    272  N7   DG A   9       1.893   8.366  -2.727  1.00  0.00           N  \\nATOM    273  C5   DG A   9       1.561   8.070  -1.412  1.00  0.00           C  \\nATOM    274  C6   DG A   9       2.401   7.935  -0.265  1.00  0.00           C  \\nATOM    275  O6   DG A   9       3.614   8.107  -0.190  1.00  0.00           O  \\nATOM    276  N1   DG A   9       1.693   7.555   0.872  1.00  0.00           N  \\nATOM    277  C2   DG A   9       0.328   7.335   0.900  1.00  0.00           C  \\nATOM    278  N2   DG A   9      -0.210   6.988   2.072  1.00  0.00           N  \\nATOM    279  N3   DG A   9      -0.460   7.469  -0.177  1.00  0.00           N  \\nATOM    280  C4   DG A   9       0.207   7.851  -1.298  1.00  0.00           C  \\nATOM    281  H5'  DG A   9      -2.045   6.989  -6.838  1.00  0.00           H  \\nATOM    282 H5''  DG A   9      -2.628   8.623  -7.197  1.00  0.00           H  \\nATOM    283  H4'  DG A   9      -3.595   7.780  -5.082  1.00  0.00           H  \\nATOM    284  H3'  DG A   9      -1.573  10.062  -4.854  1.00  0.00           H  \\nATOM    285  H2'  DG A   9      -2.015  10.137  -2.518  1.00  0.00           H  \\nATOM    286 H2''  DG A   9      -3.515   9.237  -2.692  1.00  0.00           H  \\nATOM    287  H1'  DG A   9      -2.311   7.429  -2.161  1.00  0.00           H  \\nATOM    288  H8   DG A   9       0.678   8.518  -4.403  1.00  0.00           H  \\nATOM    289  H1   DG A   9       2.224   7.458   1.723  1.00  0.00           H  \\nATOM    290  H21  DG A   9       0.352   6.926   2.907  1.00  0.00           H  \\nATOM    291  H22  DG A   9      -1.215   6.851   2.123  1.00  0.00           H  \\nATOM    292  P    DA A  10      -3.768  11.908  -4.844  1.00  0.00           P  \\nATOM    293  OP1  DA A  10      -4.648  12.420  -5.917  1.00  0.00           O  \\nATOM    294  OP2  DA A  10      -2.422  12.498  -4.662  1.00  0.00           O  \\nATOM    295  O5'  DA A  10      -4.564  12.045  -3.452  1.00  0.00           O  \\nATOM    296  C5'  DA A  10      -5.795  11.380  -3.232  1.00  0.00           C  \\nATOM    297  C4'  DA A  10      -6.574  12.044  -2.090  1.00  0.00           C  \\nATOM    298  O4'  DA A  10      -5.681  12.349  -1.028  1.00  0.00           O  \\nATOM    299  C3'  DA A  10      -7.245  13.358  -2.515  1.00  0.00           C  \\nATOM    300  O3'  DA A  10      -8.433  13.513  -1.751  1.00  0.00           O  \\nATOM    301  C2'  DA A  10      -6.168  14.372  -2.152  1.00  0.00           C  \\nATOM    302  C1'  DA A  10      -5.636  13.757  -0.862  1.00  0.00           C  \\nATOM    303  N9   DA A  10      -4.269  14.207  -0.519  1.00  0.00           N  \\nATOM    304  C8   DA A  10      -3.167  14.362  -1.327  1.00  0.00           C  \\nATOM    305  N7   DA A  10      -2.080  14.697  -0.690  1.00  0.00           N  \\nATOM    306  C5   DA A  10      -2.485  14.770   0.637  1.00  0.00           C  \\nATOM    307  C6   DA A  10      -1.807  15.061   1.839  1.00  0.00           C  \\nATOM    308  N6   DA A  10      -0.504  15.345   1.885  1.00  0.00           N  \\nATOM    309  N1   DA A  10      -2.508  15.070   2.989  1.00  0.00           N  \\nATOM    310  C2   DA A  10      -3.806  14.776   2.951  1.00  0.00           C  \\nATOM    311  N3   DA A  10      -4.552  14.464   1.896  1.00  0.00           N  \\nATOM    312  C4   DA A  10      -3.822  14.485   0.749  1.00  0.00           C  \\nATOM    313  H5'  DA A  10      -5.589  10.343  -2.966  1.00  0.00           H  \\nATOM    314 H5''  DA A  10      -6.412  11.388  -4.132  1.00  0.00           H  \\nATOM    315  H4'  DA A  10      -7.333  11.339  -1.748  1.00  0.00           H  \\nATOM    316  H3'  DA A  10      -7.475  13.385  -3.581  1.00  0.00           H  \\nATOM    317  H2'  DA A  10      -5.414  14.386  -2.937  1.00  0.00           H  \\nATOM    318 H2''  DA A  10      -6.554  15.376  -2.003  1.00  0.00           H  \\nATOM    319  H1'  DA A  10      -6.323  14.024  -0.061  1.00  0.00           H  \\nATOM    320  H8   DA A  10      -3.184  14.211  -2.396  1.00  0.00           H  \\nATOM    321  H61  DA A  10      -0.075  15.609   2.761  1.00  0.00           H  \\nATOM    322  H62  DA A  10       0.069  15.235   1.057  1.00  0.00           H  \\nATOM    323  H2   DA A  10      -4.325  14.780   3.896  1.00  0.00           H  \\nATOM    324  P    DA A  11      -9.394  14.810  -1.869  1.00  0.00           P  \\nATOM    325  OP1  DA A  11     -10.740  14.433  -1.386  1.00  0.00           O  \\nATOM    326  OP2  DA A  11      -9.234  15.394  -3.219  1.00  0.00           O  \\nATOM    327  O5'  DA A  11      -8.742  15.826  -0.800  1.00  0.00           O  \\nATOM    328  C5'  DA A  11      -8.827  15.586   0.592  1.00  0.00           C  \\nATOM    329  C4'  DA A  11      -7.934  16.582   1.339  1.00  0.00           C  \\nATOM    330  O4'  DA A  11      -6.600  16.494   0.866  1.00  0.00           O  \\nATOM    331  C3'  DA A  11      -8.405  18.029   1.139  1.00  0.00           C  \\nATOM    332  O3'  DA A  11      -8.526  18.646   2.413  1.00  0.00           O  \\nATOM    333  C2'  DA A  11      -7.288  18.627   0.284  1.00  0.00           C  \\nATOM    334  C1'  DA A  11      -6.088  17.802   0.739  1.00  0.00           C  \\nATOM    335  N9   DA A  11      -4.945  17.818  -0.203  1.00  0.00           N  \\nATOM    336  C8   DA A  11      -4.941  17.661  -1.568  1.00  0.00           C  \\nATOM    337  N7   DA A  11      -3.754  17.670  -2.108  1.00  0.00           N  \\nATOM    338  C5   DA A  11      -2.903  17.845  -1.024  1.00  0.00           C  \\nATOM    339  C6   DA A  11      -1.500  17.949  -0.911  1.00  0.00           C  \\nATOM    340  N6   DA A  11      -0.681  17.883  -1.966  1.00  0.00           N  \\nATOM    341  N1   DA A  11      -0.968  18.143   0.310  1.00  0.00           N  \\nATOM    342  C2   DA A  11      -1.779  18.218   1.364  1.00  0.00           C  \\nATOM    343  N3   DA A  11      -3.106  18.128   1.391  1.00  0.00           N  \\nATOM    344  C4   DA A  11      -3.618  17.939   0.144  1.00  0.00           C  \\nATOM    345  H5'  DA A  11      -8.503  14.572   0.827  1.00  0.00           H  \\nATOM    346 H5''  DA A  11      -9.859  15.703   0.925  1.00  0.00           H  \\nATOM    347  H4'  DA A  11      -7.947  16.329   2.400  1.00  0.00           H  \\nATOM    348  H3'  DA A  11      -9.358  18.055   0.610  1.00  0.00           H  \\nATOM    349  H2'  DA A  11      -7.517  18.432  -0.763  1.00  0.00           H  \\nATOM    350 H2''  DA A  11      -7.149  19.695   0.441  1.00  0.00           H  \\nATOM    351  H1'  DA A  11      -5.765  18.170   1.712  1.00  0.00           H  \\nATOM    352  H8   DA A  11      -5.838  17.538  -2.157  1.00  0.00           H  \\nATOM    353  H61  DA A  11       0.317  17.992  -1.843  1.00  0.00           H  \\nATOM    354  H62  DA A  11      -1.069  17.748  -2.888  1.00  0.00           H  \\nATOM    355  H2   DA A  11      -1.298  18.375   2.317  1.00  0.00           H  \\nATOM    356  P    DC A  12      -9.152  20.124   2.613  1.00  0.00           P  \\nATOM    357  OP1  DC A  12     -10.116  20.076   3.734  1.00  0.00           O  \\nATOM    358  OP2  DC A  12      -9.578  20.645   1.295  1.00  0.00           O  \\nATOM    359  O5'  DC A  12      -7.866  20.968   3.088  1.00  0.00           O  \\nATOM    360  C5'  DC A  12      -7.409  20.939   4.430  1.00  0.00           C  \\nATOM    361  C4'  DC A  12      -5.979  21.486   4.471  1.00  0.00           C  \\nATOM    362  O4'  DC A  12      -5.178  20.822   3.513  1.00  0.00           O  \\nATOM    363  C3'  DC A  12      -5.913  22.975   4.122  1.00  0.00           C  \\nATOM    364  O3'  DC A  12      -5.844  23.770   5.287  1.00  0.00           O  \\nATOM    365  C2'  DC A  12      -4.638  23.102   3.292  1.00  0.00           C  \\nATOM    366  C1'  DC A  12      -4.097  21.674   3.191  1.00  0.00           C  \\nATOM    367  N1   DC A  12      -3.608  21.408   1.811  1.00  0.00           N  \\nATOM    368  C2   DC A  12      -2.235  21.410   1.557  1.00  0.00           C  \\nATOM    369  O2   DC A  12      -1.416  21.525   2.466  1.00  0.00           O  \\nATOM    370  N3   DC A  12      -1.814  21.268   0.267  1.00  0.00           N  \\nATOM    371  C4   DC A  12      -2.689  21.136  -0.741  1.00  0.00           C  \\nATOM    372  N4   DC A  12      -2.217  20.980  -1.981  1.00  0.00           N  \\nATOM    373  C5   DC A  12      -4.105  21.125  -0.502  1.00  0.00           C  \\nATOM    374  C6   DC A  12      -4.508  21.262   0.783  1.00  0.00           C  \\nATOM    375  H5'  DC A  12      -7.415  19.923   4.828  1.00  0.00           H  \\nATOM    376 H5''  DC A  12      -8.057  21.555   5.054  1.00  0.00           H  \\nATOM    377  H4'  DC A  12      -5.546  21.325   5.460  1.00  0.00           H  \\nATOM    378  H3'  DC A  12      -6.768  23.275   3.514  1.00  0.00           H  \\nATOM    379 HO3'  DC A  12      -6.657  23.652   5.783  1.00  0.00           H  \\nATOM    380  H2'  DC A  12      -4.895  23.497   2.309  1.00  0.00           H  \\nATOM    381 H2''  DC A  12      -3.902  23.753   3.761  1.00  0.00           H  \\nATOM    382  H1'  DC A  12      -3.331  21.535   3.953  1.00  0.00           H  \\nATOM    383  H41  DC A  12      -1.216  20.953  -2.129  1.00  0.00           H  \\nATOM    384  H42  DC A  12      -2.850  20.847  -2.756  1.00  0.00           H  \\nATOM    385  H5   DC A  12      -4.842  21.010  -1.284  1.00  0.00           H  \\nATOM    386  H6   DC A  12      -5.562  21.282   1.013  1.00  0.00           H  \\nTER     387       DC A  12                                                      \\nATOM    388  O5'  DG B  13       8.692  22.125  -1.628  1.00  0.00           O  \\nATOM    389  C5'  DG B  13       8.994  22.802  -0.426  1.00  0.00           C  \\nATOM    390  C4'  DG B  13       8.201  22.220   0.750  1.00  0.00           C  \\nATOM    391  O4'  DG B  13       6.811  22.374   0.501  1.00  0.00           O  \\nATOM    392  C3'  DG B  13       8.479  20.725   0.965  1.00  0.00           C  \\nATOM    393  O3'  DG B  13       8.592  20.475   2.357  1.00  0.00           O  \\nATOM    394  C2'  DG B  13       7.237  20.084   0.356  1.00  0.00           C  \\nATOM    395  C1'  DG B  13       6.171  21.124   0.683  1.00  0.00           C  \\nATOM    396  N9   DG B  13       4.994  21.017  -0.212  1.00  0.00           N  \\nATOM    397  C8   DG B  13       4.951  21.135  -1.579  1.00  0.00           C  \\nATOM    398  N7   DG B  13       3.756  21.031  -2.089  1.00  0.00           N  \\nATOM    399  C5   DG B  13       2.936  20.853  -0.981  1.00  0.00           C  \\nATOM    400  C6   DG B  13       1.518  20.705  -0.906  1.00  0.00           C  \\nATOM    401  O6   DG B  13       0.711  20.664  -1.832  1.00  0.00           O  \\nATOM    402  N1   DG B  13       1.063  20.603   0.405  1.00  0.00           N  \\nATOM    403  C2   DG B  13       1.884  20.606   1.519  1.00  0.00           C  \\nATOM    404  N2   DG B  13       1.294  20.495   2.713  1.00  0.00           N  \\nATOM    405  N3   DG B  13       3.220  20.717   1.448  1.00  0.00           N  \\nATOM    406  C4   DG B  13       3.684  20.847   0.175  1.00  0.00           C  \\nATOM    407  H5'  DG B  13       8.746  23.858  -0.542  1.00  0.00           H  \\nATOM    408 H5''  DG B  13      10.062  22.711  -0.224  1.00  0.00           H  \\nATOM    409  H4'  DG B  13       8.472  22.781   1.647  1.00  0.00           H  \\nATOM    410  H3'  DG B  13       9.390  20.408   0.453  1.00  0.00           H  \\nATOM    411  H2'  DG B  13       7.377  20.000  -0.722  1.00  0.00           H  \\nATOM    412 H2''  DG B  13       7.005  19.107   0.775  1.00  0.00           H  \\nATOM    413  H1'  DG B  13       5.878  21.020   1.728  1.00  0.00           H  \\nATOM    414  H8   DG B  13       5.829  21.295  -2.187  1.00  0.00           H  \\nATOM    415  H1   DG B  13       0.061  20.531   0.528  1.00  0.00           H  \\nATOM    416  H21  DG B  13       0.285  20.475   2.782  1.00  0.00           H  \\nATOM    417  H22  DG B  13       1.862  20.474   3.547  1.00  0.00           H  \\nATOM    418 HO5'  DG B  13       9.205  22.519  -2.337  1.00  0.00           H  \\nATOM    419  P    DT B  14       8.934  19.011   2.956  1.00  0.00           P  \\nATOM    420  OP1  DT B  14       9.823  19.191   4.125  1.00  0.00           O  \\nATOM    421  OP2  DT B  14       9.354  18.131   1.842  1.00  0.00           O  \\nATOM    422  O5'  DT B  14       7.499  18.499   3.486  1.00  0.00           O  \\nATOM    423  C5'  DT B  14       6.884  19.096   4.612  1.00  0.00           C  \\nATOM    424  C4'  DT B  14       5.519  18.461   4.891  1.00  0.00           C  \\nATOM    425  O4'  DT B  14       4.686  18.604   3.750  1.00  0.00           O  \\nATOM    426  C3'  DT B  14       5.603  16.962   5.226  1.00  0.00           C  \\nATOM    427  O3'  DT B  14       4.750  16.719   6.336  1.00  0.00           O  \\nATOM    428  C2'  DT B  14       5.097  16.325   3.932  1.00  0.00           C  \\nATOM    429  C1'  DT B  14       4.057  17.362   3.513  1.00  0.00           C  \\nATOM    430  N1   DT B  14       3.606  17.278   2.096  1.00  0.00           N  \\nATOM    431  C2   DT B  14       2.237  17.366   1.835  1.00  0.00           C  \\nATOM    432  O2   DT B  14       1.379  17.386   2.714  1.00  0.00           O  \\nATOM    433  N3   DT B  14       1.869  17.464   0.502  1.00  0.00           N  \\nATOM    434  C4   DT B  14       2.730  17.523  -0.582  1.00  0.00           C  \\nATOM    435  O4   DT B  14       2.281  17.673  -1.715  1.00  0.00           O  \\nATOM    436  C5   DT B  14       4.133  17.400  -0.235  1.00  0.00           C  \\nATOM    437  C7   DT B  14       5.185  17.396  -1.328  1.00  0.00           C  \\nATOM    438  C6   DT B  14       4.516  17.289   1.062  1.00  0.00           C  \\nATOM    439  H5'  DT B  14       6.743  20.162   4.431  1.00  0.00           H  \\nATOM    440 H5''  DT B  14       7.517  18.970   5.492  1.00  0.00           H  \\nATOM    441  H4'  DT B  14       5.064  18.997   5.726  1.00  0.00           H  \\nATOM    442  H3'  DT B  14       6.624  16.653   5.456  1.00  0.00           H  \\nATOM    443  H2'  DT B  14       5.932  16.270   3.237  1.00  0.00           H  \\nATOM    444 H2''  DT B  14       4.669  15.334   4.062  1.00  0.00           H  \\nATOM    445  H1'  DT B  14       3.224  17.267   4.210  1.00  0.00           H  \\nATOM    446  H3   DT B  14       0.880  17.558   0.315  1.00  0.00           H  \\nATOM    447  H71  DT B  14       4.756  17.043  -2.266  1.00  0.00           H  \\nATOM    448  H72  DT B  14       5.565  18.405  -1.473  1.00  0.00           H  \\nATOM    449  H73  DT B  14       6.011  16.740  -1.054  1.00  0.00           H  \\nATOM    450  H6   DT B  14       5.569  17.235   1.281  1.00  0.00           H  \\nATOM    451  P    DG B  15       4.549  15.254   6.990  1.00  0.00           P  \\nATOM    452  OP1  DG B  15       4.020  15.432   8.361  1.00  0.00           O  \\nATOM    453  OP2  DG B  15       5.783  14.468   6.774  1.00  0.00           O  \\nATOM    454  O5'  DG B  15       3.381  14.619   6.080  1.00  0.00           O  \\nATOM    455  C5'  DG B  15       2.063  15.133   6.103  1.00  0.00           C  \\nATOM    456  C4'  DG B  15       1.210  14.441   5.036  1.00  0.00           C  \\nATOM    457  O4'  DG B  15       1.766  14.658   3.749  1.00  0.00           O  \\nATOM    458  C3'  DG B  15       1.137  12.925   5.254  1.00  0.00           C  \\nATOM    459  O3'  DG B  15      -0.200  12.527   5.011  1.00  0.00           O  \\nATOM    460  C2'  DG B  15       2.104  12.403   4.190  1.00  0.00           C  \\nATOM    461  C1'  DG B  15       1.853  13.416   3.078  1.00  0.00           C  \\nATOM    462  N9   DG B  15       2.902  13.491   2.034  1.00  0.00           N  \\nATOM    463  C8   DG B  15       4.250  13.246   2.131  1.00  0.00           C  \\nATOM    464  N7   DG B  15       4.913  13.452   1.028  1.00  0.00           N  \\nATOM    465  C5   DG B  15       3.939  13.858   0.124  1.00  0.00           C  \\nATOM    466  C6   DG B  15       4.056  14.199  -1.258  1.00  0.00           C  \\nATOM    467  O6   DG B  15       5.066  14.207  -1.957  1.00  0.00           O  \\nATOM    468  N1   DG B  15       2.835  14.554  -1.824  1.00  0.00           N  \\nATOM    469  C2   DG B  15       1.633  14.569  -1.140  1.00  0.00           C  \\nATOM    470  N2   DG B  15       0.548  14.927  -1.832  1.00  0.00           N  \\nATOM    471  N3   DG B  15       1.521  14.247   0.158  1.00  0.00           N  \\nATOM    472  C4   DG B  15       2.706  13.896   0.732  1.00  0.00           C  \\nATOM    473  H5'  DG B  15       2.071  16.205   5.906  1.00  0.00           H  \\nATOM    474 H5''  DG B  15       1.619  14.963   7.085  1.00  0.00           H  \\nATOM    475  H4'  DG B  15       0.208  14.873   5.072  1.00  0.00           H  \\nATOM    476  H3'  DG B  15       1.450  12.654   6.263  1.00  0.00           H  \\nATOM    477  H2'  DG B  15       3.120  12.479   4.576  1.00  0.00           H  \\nATOM    478 H2''  DG B  15       1.898  11.382   3.872  1.00  0.00           H  \\nATOM    479  H1'  DG B  15       0.900  13.151   2.615  1.00  0.00           H  \\nATOM    480  H8   DG B  15       4.735  12.918   3.038  1.00  0.00           H  \\nATOM    481  H1   DG B  15       2.846  14.814  -2.798  1.00  0.00           H  \\nATOM    482  H21  DG B  15       0.616  15.164  -2.810  1.00  0.00           H  \\nATOM    483  H22  DG B  15      -0.361  14.900  -1.383  1.00  0.00           H  \\nATOM    484  P    DG B  16      -0.764  11.068   5.404  1.00  0.00           P  \\nATOM    485  OP1  DG B  16      -1.722  11.216   6.523  1.00  0.00           O  \\nATOM    486  OP2  DG B  16       0.371  10.125   5.520  1.00  0.00           O  \\nATOM    487  O5'  DG B  16      -1.578  10.705   4.067  1.00  0.00           O  \\nATOM    488  C5'  DG B  16      -2.876  11.208   3.813  1.00  0.00           C  \\nATOM    489  C4'  DG B  16      -3.169  11.119   2.311  1.00  0.00           C  \\nATOM    490  O4'  DG B  16      -2.280  11.917   1.551  1.00  0.00           O  \\nATOM    491  C3'  DG B  16      -3.020   9.694   1.781  1.00  0.00           C  \\nATOM    492  O3'  DG B  16      -4.198   8.962   2.067  1.00  0.00           O  \\nATOM    493  C2'  DG B  16      -2.782   9.959   0.301  1.00  0.00           C  \\nATOM    494  C1'  DG B  16      -2.023  11.290   0.303  1.00  0.00           C  \\nATOM    495  N9   DG B  16      -0.551  11.139   0.173  1.00  0.00           N  \\nATOM    496  C8   DG B  16       0.368  10.841   1.147  1.00  0.00           C  \\nATOM    497  N7   DG B  16       1.604  10.808   0.735  1.00  0.00           N  \\nATOM    498  C5   DG B  16       1.510  11.123  -0.614  1.00  0.00           C  \\nATOM    499  C6   DG B  16       2.534  11.240  -1.603  1.00  0.00           C  \\nATOM    500  O6   DG B  16       3.740  11.050  -1.471  1.00  0.00           O  \\nATOM    501  N1   DG B  16       2.035  11.623  -2.845  1.00  0.00           N  \\nATOM    502  C2   DG B  16       0.701  11.877  -3.102  1.00  0.00           C  \\nATOM    503  N2   DG B  16       0.368  12.229  -4.348  1.00  0.00           N  \\nATOM    504  N3   DG B  16      -0.258  11.773  -2.170  1.00  0.00           N  \\nATOM    505  C4   DG B  16       0.202  11.377  -0.955  1.00  0.00           C  \\nATOM    506  H5'  DG B  16      -2.970  12.244   4.140  1.00  0.00           H  \\nATOM    507 H5''  DG B  16      -3.600  10.605   4.362  1.00  0.00           H  \\nATOM    508  H4'  DG B  16      -4.180  11.488   2.125  1.00  0.00           H  \\nATOM    509  H3'  DG B  16      -2.132   9.221   2.201  1.00  0.00           H  \\nATOM    510  H2'  DG B  16      -2.231   9.161  -0.180  1.00  0.00           H  \\nATOM    511 H2''  DG B  16      -3.729  10.087  -0.218  1.00  0.00           H  \\nATOM    512  H1'  DG B  16      -2.411  11.875  -0.530  1.00  0.00           H  \\nATOM    513  H8   DG B  16       0.105  10.668   2.176  1.00  0.00           H  \\nATOM    514  H1   DG B  16       2.705  11.704  -3.595  1.00  0.00           H  \\nATOM    515  H21  DG B  16       1.060  12.268  -5.081  1.00  0.00           H  \\nATOM    516  H22  DG B  16      -0.613  12.385  -4.562  1.00  0.00           H  \\nATOM    517  P    DA B  17      -4.335   7.377   1.798  1.00  0.00           P  \\nATOM    518  OP1  DA B  17      -5.461   6.874   2.615  1.00  0.00           O  \\nATOM    519  OP2  DA B  17      -3.000   6.754   1.940  1.00  0.00           O  \\nATOM    520  O5'  DA B  17      -4.768   7.278   0.251  1.00  0.00           O  \\nATOM    521  C5'  DA B  17      -5.940   7.904  -0.242  1.00  0.00           C  \\nATOM    522  C4'  DA B  17      -6.463   7.161  -1.478  1.00  0.00           C  \\nATOM    523  O4'  DA B  17      -5.368   6.847  -2.328  1.00  0.00           O  \\nATOM    524  C3'  DA B  17      -7.159   5.840  -1.116  1.00  0.00           C  \\nATOM    525  O3'  DA B  17      -8.183   5.587  -2.069  1.00  0.00           O  \\nATOM    526  C2'  DA B  17      -6.000   4.859  -1.228  1.00  0.00           C  \\nATOM    527  C1'  DA B  17      -5.273   5.437  -2.438  1.00  0.00           C  \\nATOM    528  N9   DA B  17      -3.863   4.998  -2.544  1.00  0.00           N  \\nATOM    529  C8   DA B  17      -2.903   4.877  -1.566  1.00  0.00           C  \\nATOM    530  N7   DA B  17      -1.726   4.537  -2.013  1.00  0.00           N  \\nATOM    531  C5   DA B  17      -1.913   4.421  -3.384  1.00  0.00           C  \\nATOM    532  C6   DA B  17      -1.051   4.100  -4.455  1.00  0.00           C  \\nATOM    533  N6   DA B  17       0.243   3.819  -4.289  1.00  0.00           N  \\nATOM    534  N1   DA B  17      -1.560   4.059  -5.701  1.00  0.00           N  \\nATOM    535  C2   DA B  17      -2.848   4.344  -5.879  1.00  0.00           C  \\nATOM    536  N3   DA B  17      -3.755   4.674  -4.965  1.00  0.00           N  \\nATOM    537  C4   DA B  17      -3.217   4.689  -3.716  1.00  0.00           C  \\nATOM    538  H5'  DA B  17      -5.702   8.931  -0.517  1.00  0.00           H  \\nATOM    539 H5''  DA B  17      -6.727   7.926   0.514  1.00  0.00           H  \\nATOM    540  H4'  DA B  17      -7.165   7.816  -1.997  1.00  0.00           H  \\nATOM    541  H3'  DA B  17      -7.579   5.858  -0.109  1.00  0.00           H  \\nATOM    542  H2'  DA B  17      -5.394   4.922  -0.325  1.00  0.00           H  \\nATOM    543 H2''  DA B  17      -6.320   3.833  -1.385  1.00  0.00           H  \\nATOM    544  H1'  DA B  17      -5.820   5.137  -3.330  1.00  0.00           H  \\nATOM    545  H8   DA B  17      -3.088   5.063  -0.519  1.00  0.00           H  \\nATOM    546  H61  DA B  17       0.806   3.543  -5.082  1.00  0.00           H  \\nATOM    547  H62  DA B  17       0.680   3.957  -3.385  1.00  0.00           H  \\nATOM    548  H2   DA B  17      -3.208   4.315  -6.895  1.00  0.00           H  \\nATOM    549  P    DA B  18      -9.096   4.248  -2.047  1.00  0.00           P  \\nATOM    550  OP1  DA B  18     -10.379   4.557  -2.718  1.00  0.00           O  \\nATOM    551  OP2  DA B  18      -9.100   3.703  -0.672  1.00  0.00           O  \\nATOM    552  O5'  DA B  18      -8.271   3.230  -2.990  1.00  0.00           O  \\nATOM    553  C5'  DA B  18      -8.181   3.435  -4.388  1.00  0.00           C  \\nATOM    554  C4'  DA B  18      -7.174   2.461  -5.006  1.00  0.00           C  \\nATOM    555  O4'  DA B  18      -5.925   2.601  -4.346  1.00  0.00           O  \\nATOM    556  C3'  DA B  18      -7.604   0.989  -4.895  1.00  0.00           C  \\nATOM    557  O3'  DA B  18      -7.484   0.375  -6.170  1.00  0.00           O  \\nATOM    558  C2'  DA B  18      -6.601   0.434  -3.886  1.00  0.00           C  \\nATOM    559  C1'  DA B  18      -5.382   1.315  -4.152  1.00  0.00           C  \\nATOM    560  N9   DA B  18      -4.402   1.326  -3.039  1.00  0.00           N  \\nATOM    561  C8   DA B  18      -4.614   1.542  -1.697  1.00  0.00           C  \\nATOM    562  N7   DA B  18      -3.529   1.560  -0.976  1.00  0.00           N  \\nATOM    563  C5   DA B  18      -2.519   1.342  -1.902  1.00  0.00           C  \\nATOM    564  C6   DA B  18      -1.117   1.255  -1.784  1.00  0.00           C  \\nATOM    565  N6   DA B  18      -0.476   1.392  -0.618  1.00  0.00           N  \\nATOM    566  N1   DA B  18      -0.395   1.024  -2.896  1.00  0.00           N  \\nATOM    567  C2   DA B  18      -1.026   0.881  -4.059  1.00  0.00           C  \\nATOM    568  N3   DA B  18      -2.333   0.946  -4.302  1.00  0.00           N  \\nATOM    569  C4   DA B  18      -3.039   1.185  -3.163  1.00  0.00           C  \\nATOM    570  H5'  DA B  18      -7.845   4.453  -4.590  1.00  0.00           H  \\nATOM    571 H5''  DA B  18      -9.158   3.290  -4.850  1.00  0.00           H  \\nATOM    572  H4'  DA B  18      -7.051   2.726  -6.058  1.00  0.00           H  \\nATOM    573  H3'  DA B  18      -8.627   0.896  -4.526  1.00  0.00           H  \\nATOM    574  H2'  DA B  18      -7.010   0.605  -2.891  1.00  0.00           H  \\nATOM    575 H2''  DA B  18      -6.382  -0.623  -4.025  1.00  0.00           H  \\nATOM    576  H1'  DA B  18      -4.899   0.992  -5.075  1.00  0.00           H  \\nATOM    577  H8   DA B  18      -5.589   1.678  -1.256  1.00  0.00           H  \\nATOM    578  H61  DA B  18       0.532   1.327  -0.581  1.00  0.00           H  \\nATOM    579  H62  DA B  18      -1.009   1.569   0.221  1.00  0.00           H  \\nATOM    580  H2   DA B  18      -0.392   0.704  -4.915  1.00  0.00           H  \\nATOM    581  P    DT B  19      -7.936  -1.153  -6.457  1.00  0.00           P  \\nATOM    582  OP1  DT B  19      -8.773  -1.166  -7.678  1.00  0.00           O  \\nATOM    583  OP2  DT B  19      -8.456  -1.744  -5.203  1.00  0.00           O  \\nATOM    584  O5'  DT B  19      -6.529  -1.863  -6.800  1.00  0.00           O  \\nATOM    585  C5'  DT B  19      -5.855  -1.593  -8.016  1.00  0.00           C  \\nATOM    586  C4'  DT B  19      -4.433  -2.160  -7.984  1.00  0.00           C  \\nATOM    587  O4'  DT B  19      -3.781  -1.731  -6.798  1.00  0.00           O  \\nATOM    588  C3'  DT B  19      -4.383  -3.697  -8.021  1.00  0.00           C  \\nATOM    589  O3'  DT B  19      -3.372  -4.076  -8.945  1.00  0.00           O  \\nATOM    590  C2'  DT B  19      -4.022  -4.033  -6.575  1.00  0.00           C  \\nATOM    591  C1'  DT B  19      -3.120  -2.847  -6.238  1.00  0.00           C  \\nATOM    592  N1   DT B  19      -2.873  -2.640  -4.781  1.00  0.00           N  \\nATOM    593  C2   DT B  19      -1.555  -2.477  -4.349  1.00  0.00           C  \\nATOM    594  O2   DT B  19      -0.580  -2.560  -5.093  1.00  0.00           O  \\nATOM    595  N3   DT B  19      -1.378  -2.186  -3.004  1.00  0.00           N  \\nATOM    596  C4   DT B  19      -2.383  -2.014  -2.066  1.00  0.00           C  \\nATOM    597  O4   DT B  19      -2.096  -1.712  -0.910  1.00  0.00           O  \\nATOM    598  C5   DT B  19      -3.723  -2.209  -2.586  1.00  0.00           C  \\nATOM    599  C7   DT B  19      -4.919  -2.099  -1.660  1.00  0.00           C  \\nATOM    600  C6   DT B  19      -3.919  -2.501  -3.897  1.00  0.00           C  \\nATOM    601  H5'  DT B  19      -5.789  -0.516  -8.171  1.00  0.00           H  \\nATOM    602 H5''  DT B  19      -6.406  -2.027  -8.852  1.00  0.00           H  \\nATOM    603  H4'  DT B  19      -3.896  -1.762  -8.847  1.00  0.00           H  \\nATOM    604  H3'  DT B  19      -5.344  -4.128  -8.307  1.00  0.00           H  \\nATOM    605  H2'  DT B  19      -4.942  -4.038  -5.994  1.00  0.00           H  \\nATOM    606 H2''  DT B  19      -3.515  -4.987  -6.458  1.00  0.00           H  \\nATOM    607  H1'  DT B  19      -2.194  -2.994  -6.792  1.00  0.00           H  \\nATOM    608  H3   DT B  19      -0.426  -2.036  -2.699  1.00  0.00           H  \\nATOM    609  H71  DT B  19      -5.373  -1.115  -1.755  1.00  0.00           H  \\nATOM    610  H72  DT B  19      -5.662  -2.855  -1.918  1.00  0.00           H  \\nATOM    611  H73  DT B  19      -4.617  -2.249  -0.623  1.00  0.00           H  \\nATOM    612  H6   DT B  19      -4.933  -2.601  -4.249  1.00  0.00           H  \\nATOM    613  P    DG B  20      -3.026  -5.616  -9.294  1.00  0.00           P  \\nATOM    614  OP1  DG B  20      -2.294  -5.641 -10.580  1.00  0.00           O  \\nATOM    615  OP2  DG B  20      -4.256  -6.422  -9.133  1.00  0.00           O  \\nATOM    616  O5'  DG B  20      -1.998  -6.029  -8.124  1.00  0.00           O  \\nATOM    617  C5'  DG B  20      -0.704  -5.463  -8.038  1.00  0.00           C  \\nATOM    618  C4'  DG B  20      -0.007  -5.932  -6.756  1.00  0.00           C  \\nATOM    619  O4'  DG B  20      -0.749  -5.516  -5.619  1.00  0.00           O  \\nATOM    620  C3'  DG B  20       0.111  -7.460  -6.691  1.00  0.00           C  \\nATOM    621  O3'  DG B  20       1.386  -7.767  -6.152  1.00  0.00           O  \\nATOM    622  C2'  DG B  20      -1.024  -7.820  -5.730  1.00  0.00           C  \\nATOM    623  C1'  DG B  20      -0.947  -6.633  -4.774  1.00  0.00           C  \\nATOM    624  N9   DG B  20      -2.149  -6.417  -3.934  1.00  0.00           N  \\nATOM    625  C8   DG B  20      -3.464  -6.707  -4.203  1.00  0.00           C  \\nATOM    626  N7   DG B  20      -4.295  -6.345  -3.267  1.00  0.00           N  \\nATOM    627  C5   DG B  20      -3.479  -5.781  -2.295  1.00  0.00           C  \\nATOM    628  C6   DG B  20      -3.817  -5.231  -1.021  1.00  0.00           C  \\nATOM    629  O6   DG B  20      -4.927  -5.135  -0.503  1.00  0.00           O  \\nATOM    630  N1   DG B  20      -2.701  -4.768  -0.328  1.00  0.00           N  \\nATOM    631  C2   DG B  20      -1.405  -4.837  -0.802  1.00  0.00           C  \\nATOM    632  N2   DG B  20      -0.442  -4.372  -0.002  1.00  0.00           N  \\nATOM    633  N3   DG B  20      -1.086  -5.351  -2.000  1.00  0.00           N  \\nATOM    634  C4   DG B  20      -2.164  -5.811  -2.696  1.00  0.00           C  \\nATOM    635  H5'  DG B  20      -0.767  -4.375  -8.026  1.00  0.00           H  \\nATOM    636 H5''  DG B  20      -0.111  -5.770  -8.901  1.00  0.00           H  \\nATOM    637  H4'  DG B  20       0.985  -5.478  -6.723  1.00  0.00           H  \\nATOM    638  H3'  DG B  20      -0.015  -7.917  -7.673  1.00  0.00           H  \\nATOM    639  H2'  DG B  20      -1.964  -7.827  -6.285  1.00  0.00           H  \\nATOM    640 H2''  DG B  20      -0.880  -8.771  -5.219  1.00  0.00           H  \\nATOM    641  H1'  DG B  20      -0.078  -6.793  -4.131  1.00  0.00           H  \\nATOM    642  H8   DG B  20      -3.797  -7.185  -5.112  1.00  0.00           H  \\nATOM    643  H1   DG B  20      -2.869  -4.365   0.581  1.00  0.00           H  \\nATOM    644  H21  DG B  20      -0.665  -3.996   0.907  1.00  0.00           H  \\nATOM    645  H22  DG B  20       0.527  -4.453  -0.290  1.00  0.00           H  \\nATOM    646  P    DG B  21       2.024  -9.247  -6.211  1.00  0.00           P  \\nATOM    647  OP1  DG B  21       3.136  -9.244  -7.188  1.00  0.00           O  \\nATOM    648  OP2  DG B  21       0.932 -10.238  -6.335  1.00  0.00           O  \\nATOM    649  O5'  DG B  21       2.634  -9.366  -4.728  1.00  0.00           O  \\nATOM    650  C5'  DG B  21       3.892  -8.820  -4.379  1.00  0.00           C  \\nATOM    651  C4'  DG B  21       3.984  -8.711  -2.853  1.00  0.00           C  \\nATOM    652  O4'  DG B  21       3.006  -7.827  -2.336  1.00  0.00           O  \\nATOM    653  C3'  DG B  21       3.757 -10.058  -2.164  1.00  0.00           C  \\nATOM    654  O3'  DG B  21       4.960 -10.804  -2.168  1.00  0.00           O  \\nATOM    655  C2'  DG B  21       3.289  -9.606  -0.788  1.00  0.00           C  \\nATOM    656  C1'  DG B  21       2.562  -8.286  -1.068  1.00  0.00           C  \\nATOM    657  N9   DG B  21       1.080  -8.399  -1.129  1.00  0.00           N  \\nATOM    658  C8   DG B  21       0.293  -8.799  -2.181  1.00  0.00           C  \\nATOM    659  N7   DG B  21      -0.987  -8.757  -1.951  1.00  0.00           N  \\nATOM    660  C5   DG B  21      -1.072  -8.267  -0.656  1.00  0.00           C  \\nATOM    661  C6   DG B  21      -2.223  -7.998   0.146  1.00  0.00           C  \\nATOM    662  O6   DG B  21      -3.403  -8.196  -0.130  1.00  0.00           O  \\nATOM    663  N1   DG B  21      -1.895  -7.440   1.378  1.00  0.00           N  \\nATOM    664  C2   DG B  21      -0.602  -7.172   1.790  1.00  0.00           C  \\nATOM    665  N2   DG B  21      -0.446  -6.644   3.007  1.00  0.00           N  \\nATOM    666  N3   DG B  21       0.477  -7.430   1.037  1.00  0.00           N  \\nATOM    667  C4   DG B  21       0.183  -7.989  -0.167  1.00  0.00           C  \\nATOM    668  H5'  DG B  21       4.032  -7.833  -4.822  1.00  0.00           H  \\nATOM    669 H5''  DG B  21       4.678  -9.482  -4.745  1.00  0.00           H  \\nATOM    670  H4'  DG B  21       4.963  -8.310  -2.583  1.00  0.00           H  \\nATOM    671  H3'  DG B  21       2.955 -10.613  -2.649  1.00  0.00           H  \\nATOM    672  H2'  DG B  21       2.659 -10.339  -0.303  1.00  0.00           H  \\nATOM    673 H2''  DG B  21       4.145  -9.415  -0.143  1.00  0.00           H  \\nATOM    674  H1'  DG B  21       2.846  -7.600  -0.272  1.00  0.00           H  \\nATOM    675  H8   DG B  21       0.683  -9.104  -3.135  1.00  0.00           H  \\nATOM    676  H1   DG B  21      -2.661  -7.241   2.003  1.00  0.00           H  \\nATOM    677  H21  DG B  21      -1.236  -6.484   3.613  1.00  0.00           H  \\nATOM    678  H22  DG B  21       0.497  -6.466   3.341  1.00  0.00           H  \\nATOM    679  P    DA B  22       5.037 -12.344  -1.689  1.00  0.00           P  \\nATOM    680  OP1  DA B  22       6.199 -12.970  -2.359  1.00  0.00           O  \\nATOM    681  OP2  DA B  22       3.697 -12.958  -1.830  1.00  0.00           O  \\nATOM    682  O5'  DA B  22       5.377 -12.240  -0.119  1.00  0.00           O  \\nATOM    683  C5'  DA B  22       6.485 -11.495   0.353  1.00  0.00           C  \\nATOM    684  C4'  DA B  22       6.880 -11.944   1.766  1.00  0.00           C  \\nATOM    685  O4'  DA B  22       5.707 -12.137   2.543  1.00  0.00           O  \\nATOM    686  C3'  DA B  22       7.656 -13.269   1.773  1.00  0.00           C  \\nATOM    687  O3'  DA B  22       8.548 -13.254   2.879  1.00  0.00           O  \\nATOM    688  C2'  DA B  22       6.523 -14.273   1.937  1.00  0.00           C  \\nATOM    689  C1'  DA B  22       5.614 -13.508   2.894  1.00  0.00           C  \\nATOM    690  N9   DA B  22       4.209 -13.970   2.861  1.00  0.00           N  \\nATOM    691  C8   DA B  22       3.416 -14.290   1.784  1.00  0.00           C  \\nATOM    692  N7   DA B  22       2.184 -14.584   2.096  1.00  0.00           N  \\nATOM    693  C5   DA B  22       2.154 -14.452   3.478  1.00  0.00           C  \\nATOM    694  C6   DA B  22       1.134 -14.604   4.440  1.00  0.00           C  \\nATOM    695  N6   DA B  22      -0.118 -14.938   4.121  1.00  0.00           N  \\nATOM    696  N1   DA B  22       1.441 -14.424   5.739  1.00  0.00           N  \\nATOM    697  C2   DA B  22       2.685 -14.079   6.062  1.00  0.00           C  \\nATOM    698  N3   DA B  22       3.723 -13.880   5.257  1.00  0.00           N  \\nATOM    699  C4   DA B  22       3.389 -14.092   3.956  1.00  0.00           C  \\nATOM    700  H5'  DA B  22       6.211 -10.440   0.380  1.00  0.00           H  \\nATOM    701 H5''  DA B  22       7.346 -11.609  -0.308  1.00  0.00           H  \\nATOM    702  H4'  DA B  22       7.495 -11.158   2.207  1.00  0.00           H  \\nATOM    703  H3'  DA B  22       8.206 -13.437   0.846  1.00  0.00           H  \\nATOM    704  H2'  DA B  22       6.050 -14.437   0.971  1.00  0.00           H  \\nATOM    705 H2''  DA B  22       6.847 -15.224   2.347  1.00  0.00           H  \\nATOM    706  H1'  DA B  22       6.017 -13.627   3.898  1.00  0.00           H  \\nATOM    707  H8   DA B  22       3.765 -14.289   0.763  1.00  0.00           H  \\nATOM    708  H61  DA B  22      -0.799 -15.098   4.849  1.00  0.00           H  \\nATOM    709  H62  DA B  22      -0.403 -14.971   3.149  1.00  0.00           H  \\nATOM    710  H2   DA B  22       2.882 -13.931   7.112  1.00  0.00           H  \\nATOM    711  P    DA B  23       9.498 -14.507   3.262  1.00  0.00           P  \\nATOM    712  OP1  DA B  23      10.631 -13.997   4.067  1.00  0.00           O  \\nATOM    713  OP2  DA B  23       9.762 -15.291   2.035  1.00  0.00           O  \\nATOM    714  O5'  DA B  23       8.550 -15.384   4.227  1.00  0.00           O  \\nATOM    715  C5'  DA B  23       8.201 -14.937   5.523  1.00  0.00           C  \\nATOM    716  C4'  DA B  23       7.125 -15.855   6.111  1.00  0.00           C  \\nATOM    717  O4'  DA B  23       6.004 -15.909   5.243  1.00  0.00           O  \\nATOM    718  C3'  DA B  23       7.640 -17.289   6.298  1.00  0.00           C  \\nATOM    719  O3'  DA B  23       7.347 -17.705   7.624  1.00  0.00           O  \\nATOM    720  C2'  DA B  23       6.857 -18.064   5.238  1.00  0.00           C  \\nATOM    721  C1'  DA B  23       5.569 -17.250   5.162  1.00  0.00           C  \\nATOM    722  N9   DA B  23       4.787 -17.460   3.922  1.00  0.00           N  \\nATOM    723  C8   DA B  23       5.218 -17.504   2.618  1.00  0.00           C  \\nATOM    724  N7   DA B  23       4.265 -17.645   1.740  1.00  0.00           N  \\nATOM    725  C5   DA B  23       3.113 -17.702   2.513  1.00  0.00           C  \\nATOM    726  C6   DA B  23       1.748 -17.856   2.192  1.00  0.00           C  \\nATOM    727  N6   DA B  23       1.307 -17.978   0.936  1.00  0.00           N  \\nATOM    728  N1   DA B  23       0.854 -17.899   3.198  1.00  0.00           N  \\nATOM    729  C2   DA B  23       1.287 -17.784   4.452  1.00  0.00           C  \\nATOM    730  N3   DA B  23       2.536 -17.627   4.883  1.00  0.00           N  \\nATOM    731  C4   DA B  23       3.419 -17.594   3.847  1.00  0.00           C  \\nATOM    732  H5'  DA B  23       7.817 -13.917   5.486  1.00  0.00           H  \\nATOM    733 H5''  DA B  23       9.080 -14.949   6.168  1.00  0.00           H  \\nATOM    734  H4'  DA B  23       6.806 -15.445   7.070  1.00  0.00           H  \\nATOM    735  H3'  DA B  23       8.712 -17.341   6.108  1.00  0.00           H  \\nATOM    736  H2'  DA B  23       7.407 -18.009   4.298  1.00  0.00           H  \\nATOM    737 H2''  DA B  23       6.682 -19.104   5.506  1.00  0.00           H  \\nATOM    738  H1'  DA B  23       4.956 -17.492   6.028  1.00  0.00           H  \\nATOM    739  H8   DA B  23       6.256 -17.425   2.330  1.00  0.00           H  \\nATOM    740  H61  DA B  23       0.323 -18.116   0.752  1.00  0.00           H  \\nATOM    741  H62  DA B  23       1.969 -17.957   0.174  1.00  0.00           H  \\nATOM    742  H2   DA B  23       0.527 -17.826   5.218  1.00  0.00           H  \\nATOM    743  P    DC B  24       7.880 -19.103   8.239  1.00  0.00           P  \\nATOM    744  OP1  DC B  24       8.444 -18.837   9.581  1.00  0.00           O  \\nATOM    745  OP2  DC B  24       8.699 -19.794   7.218  1.00  0.00           O  \\nATOM    746  O5'  DC B  24       6.510 -19.929   8.421  1.00  0.00           O  \\nATOM    747  C5'  DC B  24       5.652 -19.718   9.530  1.00  0.00           C  \\nATOM    748  C4'  DC B  24       4.280 -20.317   9.208  1.00  0.00           C  \\nATOM    749  O4'  DC B  24       3.831 -19.851   7.950  1.00  0.00           O  \\nATOM    750  C3'  DC B  24       4.318 -21.845   9.110  1.00  0.00           C  \\nATOM    751  O3'  DC B  24       3.855 -22.448  10.301  1.00  0.00           O  \\nATOM    752  C2'  DC B  24       3.395 -22.164   7.937  1.00  0.00           C  \\nATOM    753  C1'  DC B  24       2.914 -20.799   7.442  1.00  0.00           C  \\nATOM    754  N1   DC B  24       2.896 -20.769   5.955  1.00  0.00           N  \\nATOM    755  C2   DC B  24       1.679 -20.878   5.280  1.00  0.00           C  \\nATOM    756  O2   DC B  24       0.610 -20.900   5.887  1.00  0.00           O  \\nATOM    757  N3   DC B  24       1.696 -20.948   3.918  1.00  0.00           N  \\nATOM    758  C4   DC B  24       2.848 -20.923   3.233  1.00  0.00           C  \\nATOM    759  N4   DC B  24       2.801 -20.973   1.898  1.00  0.00           N  \\nATOM    760  C5   DC B  24       4.112 -20.809   3.906  1.00  0.00           C  \\nATOM    761  C6   DC B  24       4.079 -20.734   5.257  1.00  0.00           C  \\nATOM    762  H5'  DC B  24       5.537 -18.654   9.743  1.00  0.00           H  \\nATOM    763 H5''  DC B  24       6.068 -20.202  10.415  1.00  0.00           H  \\nATOM    764  H4'  DC B  24       3.557 -20.018   9.969  1.00  0.00           H  \\nATOM    765  H3'  DC B  24       5.324 -22.197   8.879  1.00  0.00           H  \\nATOM    766 HO3'  DC B  24       2.968 -22.129  10.481  1.00  0.00           H  \\nATOM    767  H2'  DC B  24       3.970 -22.680   7.169  1.00  0.00           H  \\nATOM    768 H2''  DC B  24       2.548 -22.782   8.229  1.00  0.00           H  \\nATOM    769  H1'  DC B  24       1.943 -20.587   7.888  1.00  0.00           H  \\nATOM    770  H41  DC B  24       1.902 -21.016   1.437  1.00  0.00           H  \\nATOM    771  H42  DC B  24       3.650 -20.922   1.355  1.00  0.00           H  \\nATOM    772  H5   DC B  24       5.062 -20.776   3.392  1.00  0.00           H  \\nATOM    773  H6   DC B  24       5.004 -20.670   5.810  1.00  0.00           H  \\nTER     774       DC B  24                                                      \\nMASTER      121    0    0    0    0    0    0    6  500    2    0    2          \\nEND                                                                             \\n\");\n",
       "\tviewer_1684696742130174.setStyle({\"model\": -1, \"chain\": \"B\"},{\"cartoon\": {\"color\": \"spectrum\"}});\n",
       "\tviewer_1684696742130174.zoomTo();\n",
       "viewer_1684696742130174.render();\n",
       "});\n",
       "</script>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "sequence = \"DTDADADCDCDC\"\n",
    "get_similars(sequence, algorithm=\"msa\", save_to_db=False, count=5)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "529c74d8",
   "metadata": {},
   "outputs": [],
   "source": [
    "import unittest\n",
    "from unittest.mock import patch\n",
    "\n",
    "class TestGetSimilars(unittest.TestCase):\n",
    "    def setUp(self):\n",
    "        self.sequence = \"MKLTIYPDELVQIVSDKIASNIVAAQCVQGFNADPTVEIKLVEEVNNGDPEILQVSPVAKKVGVIGLNFGRQKASIIICLPDDDQIETIYQA\"\n",
    "        self.only_chain_id = None\n",
    "        self.save_to_db = False\n",
    "        self.algorithm = \"esm\"\n",
    "        self.count = 5\n",
    "\n",
    "    @patch(\"fold_sequence\")\n",
    "    @patch(\"save_generated_pdb\")\n",
    "    @patch(\"view_pdb\")\n",
    "    @patch(\"get_most_similars_by_esm\")\n",
    "    def test_get_similars_with_esm_algorithm(self, mock_similars, mock_view_pdb, mock_save_pdb, mock_fold_sequence):\n",
    "        mock_similars.return_value = [\n",
    "            (0.8, {\n",
    "                \"protein\": {\n",
    "                    \"name\": \"Protein A\",\n",
    "                    \"id\": \"ABC123\"\n",
    "                },\n",
    "                \"chain_id\": \"A\",\n",
    "                \"sequence\": \"MKLTIYPDELVQIVSDKIASNIVAAQCVQGFNADPTVEIKLVEEVNNGDPEILQVSPVAKKVGVIGLNFGRQKASIIICLPDDDQIETIYQA\"\n",
    "            })\n",
    "        ]\n",
    "        mock_fold_sequence.return_value = \"Generated PDB\"\n",
    "        get_similars(self.sequence, self.only_chain_id, self.save_to_db, self.algorithm, self.count)\n",
    "        mock_similars.assert_called_once_with(\n",
    "            self.sequence, self.only_chain_id, top_count=self.count\n",
    "        )\n",
    "        mock_view_pdb.assert_called_once_with(data=\"Generated PDB\", chain_id=\"A\")\n",
    "        mock_save_pdb.assert_not_called()\n",
    "\n",
    "    @patch(\"get_most_similars_by_MSA\")\n",
    "    def test_get_similars_with_msa_algorithm(self, mock_similars):\n",
    "        mock_similars.return_value = [\n",
    "            (0.7, {\n",
    "                \"protein\": {\n",
    "                    \"name\": \"Protein B\",\n",
    "                    \"id\": \"XYZ789\"\n",
    "                },\n",
    "                \"chain_id\": \"B\",\n",
    "                \"sequence\": \"MKLTIYPDELVQIVSDKIASNIVAAQCVQGFNADPTVEIKLVEEVNNGDPEILQVSPVAKKVGVIGLNFGRQKASIIICLPDDDQIETIYQA\"\n",
    "            })\n",
    "        ]\n",
    "        get_similars(self.sequence, self.only_chain_id, self.save_to_db, self.algorithm, self.count)\n",
    "        mock_similars.assert_called_once_with(\n",
    "            self.sequence, self.only_chain_id, top_count=self.count\n",
    "        )\n",
    "\n",
    "if __name__ == '__main__':\n",
    "    unittest.main()"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.16"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
